diff --git a/brave_paks.gni b/brave_paks.gni
index 9e1fe03cf4e..c326497ddc8 100644
--- a/brave_paks.gni
+++ b/brave_paks.gni
@@ -2,6 +2,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
+# test
 
 import("//brave/components/brave_webtorrent/browser/buildflags/buildflags.gni")
 import("//brave/components/tor/buildflags/buildflags.gni")
