diff --git a/browser/ui/webui/brave_rewards_page_ui.cc b/browser/ui/webui/brave_rewards_page_ui.cc
index c8e5ee069eed81d5bb1702a8acdfefff7813be79..2a476803ab2d78b9c4c43d88836ffaf02344822f 100644
--- a/browser/ui/webui/brave_rewards_page_ui.cc
+++ b/browser/ui/webui/brave_rewards_page_ui.cc
@@ -462,8 +462,8 @@ void RewardsDOMHandler::Init() {
     rewards_service_->OnRewardsPageShown();
   }
 
-  // Configure a pref change registrar to update brave://rewards when settings
-  // are changed via brave://settings
+  // Configure a pref change registrar to update ibrowe://rewards when settings
+  // are changed via ibrowe://settings
   InitPrefChangeRegistrar();
 }
 
