diff --git a/browser/brave_rewards/test/rewards_ofac_browsertest.cc b/browser/brave_rewards/test/rewards_ofac_browsertest.cc
index 1feffca62f2ec7b087ea7a3f3f1b37bb3d548959..b1eb06a2d32d79a2d3f1e69adad4f5d4a897d74a 100644
--- a/browser/brave_rewards/test/rewards_ofac_browsertest.cc
+++ b/browser/brave_rewards/test/rewards_ofac_browsertest.cc
@@ -100,7 +100,7 @@ IN_PROC_BROWSER_TEST_F(BraveRewardsOFACTest, AppMenuItemEnabled) {
   }
 }
 
-// Verify that brave://rewards page is reachable even when  in an OFAC
+// Verify that ibrowe://rewards page is reachable even when  in an OFAC
 // sanctioned region.
 IN_PROC_BROWSER_TEST_F(BraveRewardsOFACTest, RewardsPagesAccess) {
   const GURL url("chrome://rewards");
