diff --git a/browser/ui/views/sidebar/sidebar_control_view.cc b/browser/ui/views/sidebar/sidebar_control_view.cc
index 046927edfe70796eb6fe41dd2458f7956e6eb1ce..ddc26865c9219a5a30ef14774698e9a5458516d4 100644
--- a/browser/ui/views/sidebar/sidebar_control_view.cc
+++ b/browser/ui/views/sidebar/sidebar_control_view.cc
@@ -205,7 +205,7 @@ void SidebarControlView::OnButtonPressed(views::View* view) {
   if (view == sidebar_settings_view_) {
     ShowSingletonTabOverwritingNTP(
         browser_,
-        GURL("brave://settings?search=" +
+        GURL("ibrowe://settings?search=" +
              l10n_util::GetStringUTF8(
                  IDS_SETTINGS_APPEARNCE_SETTINGS_SIDEBAR_PART_TITLE)));
   }
