diff --git a/browser/ui/toolbar/brave_location_bar_model_delegate_unittest.cc b/browser/ui/toolbar/brave_location_bar_model_delegate_unittest.cc
index 49ad205541b21dd7d6644d3a58ce74e888bb90f2..f213f44f635292c552d7107ca8aa68f0995c0bd8 100644
--- a/browser/ui/toolbar/brave_location_bar_model_delegate_unittest.cc
+++ b/browser/ui/toolbar/brave_location_bar_model_delegate_unittest.cc
@@ -16,26 +16,26 @@ TEST_F(BraveLocationBarModelDelegateTest, ResolvesEthereumRemoteClient) {
   GURL url(kEthereumRemoteClientBaseUrl);
   std::u16string formatted_url = base::UTF8ToUTF16(url.spec());
   BraveLocationBarModelDelegate::FormattedStringFromURL(url, &formatted_url);
-  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "brave://wallet");
+  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "ibrowe://wallet");
 }
 TEST_F(BraveLocationBarModelDelegateTest,
     ResolvesEthereumRemoteClientPhishingRoute) {
   GURL url(kEthereumRemoteClientPhishingUrl);
   std::u16string formatted_url = base::UTF8ToUTF16(url.spec());
   BraveLocationBarModelDelegate::FormattedStringFromURL(url, &formatted_url);
-  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "brave://wallet");
+  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "ibrowe://wallet");
 }
 TEST_F(BraveLocationBarModelDelegateTest,
     ResolvesEthereumRemoteClientENSRoute) {
   GURL url(kEthereumRemoteClientEnsRedirectUrl);
   std::u16string formatted_url = base::UTF8ToUTF16(url.spec());
   BraveLocationBarModelDelegate::FormattedStringFromURL(url, &formatted_url);
-  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "brave://wallet");
+  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "ibrowe://wallet");
 }
 
 TEST_F(BraveLocationBarModelDelegateTest, ResolvesChromeSchemeToBrave) {
   GURL url("chrome://sync/");
   std::u16string formatted_url = base::UTF8ToUTF16(url.spec());
   BraveLocationBarModelDelegate::FormattedStringFromURL(url, &formatted_url);
-  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "brave://sync/");
+  ASSERT_STREQ(base::UTF16ToASCII(formatted_url).c_str(), "ibrowe://sync/");
 }
