diff --git a/chromium_src/components/search_engines/template_url_prepopulate_data.cc b/chromium_src/components/search_engines/template_url_prepopulate_data.cc
index d413835642fc1e9f042efc822334d91651caeea8..d58ec70cb07e590b5ada2977e7b9d8435ae02b42 100644
--- a/chromium_src/components/search_engines/template_url_prepopulate_data.cc
+++ b/chromium_src/components/search_engines/template_url_prepopulate_data.cc
@@ -193,7 +193,7 @@ constexpr auto kDefaultEnginesByCountryIdMap =
 // - `SetDefaultSearchVersion` in `//brave/browser/profiles/profile_util.cc`
 // - `//brave/browser/profiles/brave_profile_manager.cc` where it is called
 //
-// If that person resets the profile using brave://settings/reset, we need to
+// If that person resets the profile using ibrowe://settings/reset, we need to
 // set the default search engine back to what it was when the profile was
 // originally created. This way, a person doesn't get a new unexpected default
 // when they reset the profile; it goes back to the original value.
