diff --git a/chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc b/chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
index 74764458eab8327029bed2c2995d06d9c92b0f36..7423538c50dec83fc3af0e1cc2f0540b419d5def 100644
--- a/chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
+++ b/chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc
@@ -27,7 +27,7 @@
 void TabHoverCardBubbleView_ChromiumImpl::BraveUpdateCardContent(
     const Tab* tab) {
   TabHoverCardBubbleView_ChromiumImpl::UpdateCardContent(tab);
-  // Replace chrome:// with brave://. Since this is purely in the UI we can
+  // Replace chrome:// with ibrowe://. Since this is purely in the UI we can
   // just do a sub-string replacement instead of parsing into GURL.
   auto domain = std::u16string(domain_label_->GetText());
   if (brave_utils::ReplaceChromeToBraveScheme(domain)) {
