diff --git a/common/brave_content_client_unittest.cc b/common/brave_content_client_unittest.cc
index 758fbaa129eb7de9a814f6f3efde5f7a636dabbf..16ac69db6fd409b187c0366bfebf446e661c5cb9 100644
--- a/common/brave_content_client_unittest.cc
+++ b/common/brave_content_client_unittest.cc
@@ -17,7 +17,7 @@ TEST(BraveContentClientTest, AdditionalSchemesTest) {
   content::SetContentClient(&content_client);
   content::ReRegisterContentSchemesForTests();
 
-  const GURL sync_url("brave://sync");
+  const GURL sync_url("ibrowe://sync");
   EXPECT_TRUE(sync_url.is_valid());
   EXPECT_TRUE(sync_url.has_host());
   EXPECT_EQ("sync", sync_url.host());
