diff --git a/components/ntp_background_images/browser/ntp_sponsored_images_data.cc b/components/ntp_background_images/browser/ntp_sponsored_images_data.cc
index be9fa1a7f06539333b273057eaf8621e01dd1690..e030185164a35f9bb29c729bd7403fc44648543d 100644
--- a/components/ntp_background_images/browser/ntp_sponsored_images_data.cc
+++ b/components/ntp_background_images/browser/ntp_sponsored_images_data.cc
@@ -25,7 +25,7 @@
     "imageUrl": "logo.png",
     "alt": "Visit Brave Software",
     "companyName": "Brave Software",
-    "destinationUrl": "https://www.brave.com/"
+    "destinationUrl": "https://www.ibrowe.com/"
   },
   "wallpapers": [
     {
