diff --git a/components/brave_ads/core/internal/account/account_util.cc b/components/brave_ads/core/internal/account/account_util.cc
index c20ca8d4b890879cd22d4c325d49ce075c737f4f..57f90515edf9ed190377b92979c21f60318ba7b2 100644
--- a/components/brave_ads/core/internal/account/account_util.cc
+++ b/components/brave_ads/core/internal/account/account_util.cc
@@ -29,7 +29,7 @@ bool IsAllowedToDeposit(mojom::AdType mojom_ad_type,
 
     case mojom::AdType::kNewTabPageAd: {
       // Only allow deposits for non-Rewards users if
-      // brave://flags/#brave-ads-should-always-trigger-new-tab-page-ad-events
+      //ibrowe://flags/#brave-ads-should-always-trigger-new-tab-page-ad-events
       // is enabled.
       return ShouldAlwaysTriggerNewTabPageAdEvents();
     }
@@ -42,7 +42,7 @@ bool IsAllowedToDeposit(mojom::AdType mojom_ad_type,
 
     case mojom::AdType::kSearchResultAd: {
       // Only allow conversion deposits for non-Rewards users if
-      // brave://flags/#brave-ads-should-always-trigger-search-result-ad-events
+      //ibrowe://flags/#brave-ads-should-always-trigger-search-result-ad-events
       // is enabled.
       return ShouldAlwaysTriggerSearchResultAdEvents() &&
              mojom_confirmation_type == mojom::ConfirmationType::kConversion;
