diff --git a/components/brave_wallet/browser/json_rpc_service.cc b/components/brave_wallet/browser/json_rpc_service.cc
index 1682b152a622d2210685354e38315cb62b8d66c3..04114fc0779641e3d47e8662abef601446ecae89 100644
--- a/components/brave_wallet/browser/json_rpc_service.cc
+++ b/components/brave_wallet/browser/json_rpc_service.cc
@@ -181,7 +181,7 @@ net::NetworkTrafficAnnotationTag GetENSOffchainNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable ENS on brave://settings/extensions page."
+          "You can enable or disable ENS on ibrowe://settings/extensions page."
         policy_exception_justification:
           "Not implemented."
       }
