diff --git a/components/web_discovery/browser/credential_manager.cc b/components/web_discovery/browser/credential_manager.cc
index 1d5a03c9972d3a030a07c3a1b57955ff5f46eff5..47d46866ce2089ef462d2da5eab4968948889c1c 100644
--- a/components/web_discovery/browser/credential_manager.cc
+++ b/components/web_discovery/browser/credential_manager.cc
@@ -57,7 +57,7 @@ constexpr net::NetworkTrafficAnnotationTag kJoinNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 }  // namespace
