diff --git a/components/commands/common/features.cc b/components/commands/common/features.cc
index 4f1a61ac8f34f147d79242cc7e7d1057524a9dd6..80c1bad55eec85f13bf12b90d6d880fa3554d6b1 100644
--- a/components/commands/common/features.cc
+++ b/components/commands/common/features.cc
@@ -8,7 +8,7 @@
 
 namespace commands::features {
 
-// This feature enables a page a brave://commands which lists all available
+// This feature enables a page a ibrowe://commands which lists all available
 // commands in Brave, and the shortcut for executing them. In future, this will
 // allow configuring the shortcuts for various commands.
 BASE_FEATURE(kBraveCommands, "BraveCommands", base::FEATURE_ENABLED_BY_DEFAULT);
