diff --git a/components/brave_ads/core/internal/ad_units/inline_content_ad/inline_content_ad_test.cc b/components/brave_ads/core/internal/ad_units/inline_content_ad/inline_content_ad_test.cc
index aaf1b95095c2f8361d8dae4640599fed994858fb..b37abd3733ac1fead9ea742a9408218d70fe883f 100644
--- a/components/brave_ads/core/internal/ad_units/inline_content_ad/inline_content_ad_test.cc
+++ b/components/brave_ads/core/internal/ad_units/inline_content_ad/inline_content_ad_test.cc
@@ -36,7 +36,7 @@ class BraveAdsInlineContentAdIntegrationTest : public test::TestBase {
     test::TestBase::SetUp(/*is_integration_test=*/true);
 
     SimulateOpeningNewTab(/*tab_id=*/1,
-                          /*redirect_chain=*/{GURL("brave://newtab")},
+                          /*redirect_chain=*/{GURL("ibrowe://newtab")},
                           net::HTTP_OK);
   }
 
