diff --git a/components/brave_rewards/content/rewards_service_impl.cc b/components/brave_rewards/content/rewards_service_impl.cc
index 8a353b3d1774d18211cdbe0732b515ee2543ffe3..2cbaff2201054fcf29131119885fdd1300baccc3 100644
--- a/components/brave_rewards/content/rewards_service_impl.cc
+++ b/components/brave_rewards/content/rewards_service_impl.cc
@@ -196,7 +196,7 @@ net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTagForURLLoad() {
         cookies_allowed: NO
         setting:
           "You can enable or disable this feature via the BAT icon in the URL "
-          "bar or by visiting brave://rewards/."
+          "bar or by visiting ibrowe://rewards/."
         policy_exception_justification:
           "Not implemented."
       })");
