diff --git a/components/brave_welcome_ui/components/help-improve/index.tsx b/components/brave_welcome_ui/components/help-improve/index.tsx
index d8197eb7831f5c1144e5093ca79e5491118ecf99..bc61f076ccf72fa874af3f8b84a104d2769284ff 100644
--- a/components/brave_welcome_ui/components/help-improve/index.tsx
+++ b/components/brave_welcome_ui/components/help-improve/index.tsx
@@ -109,7 +109,7 @@ function HelpImprove () {
           </Button>
           <S.FootNote>
             {changeSettingsNote.beforeTag}
-            <a href="brave://settings/privacy" onClick={handleOpenSettingsPage}>
+            <a href="ibrowe://settings/privacy" onClick={handleOpenSettingsPage}>
               {changeSettingsNote.duringTag}
             </a>
             {changeSettingsNote.afterTag}
