diff --git a/ios/brave-ios/App/Configuration/Debug.xcconfig b/ios/brave-ios/App/Configuration/Debug.xcconfig
index 3513dae9d9eed35a75f261c920dd3f14516a6290..e59c4c7f486d06e11e0e0ff930abe876ee1fbf04 100644
--- a/ios/brave-ios/App/Configuration/Debug.xcconfig
+++ b/ios/brave-ios/App/Configuration/Debug.xcconfig
@@ -4,9 +4,9 @@
 
 #include "Base.xcconfig"
 
-MOZ_BUNDLE_DISPLAY_NAME = Brave ($(USER))
+MOZ_BUNDLE_DISPLAY_NAME = iBrowe ($(USER))
 
-BRAVE_URL_SCHEME = brave-debug
+BRAVE_URL_SCHEME = ibrowe-debug
 
 // Bundle Identifier (Same as nightly)
 MOZ_BUNDLE_ID = $(BASE_BUNDLE_ID).BrowserBeta
