diff --git a/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift b/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
index 783b98b122df025d8915793388fbd1765b486ef7..4730ffc1c27977fbb667da721acea2acb96c963a 100644
--- a/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
+++ b/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
@@ -11,9 +11,9 @@ extension Strings {
     public static let introducingBravePlaylist = NSLocalizedString(
       "introducingBravePlaylist",
       bundle: .module,
-      value: "Introducing Brave Playlist",
+      value: "Introducing iBrowe Playlist",
       comment:
-        "A title shown on the first 2 steps of onboarding for Brave Playlist. Brave Playlist is a product name"
+        "A title shown on the first 2 steps of onboarding for iBrowe Playlist. iBrowe Playlist is a product name"
     )
 
     // -------------------
@@ -26,21 +26,21 @@ extension Strings {
       value:
         "Create playlists with content you love. Add media from any page.\nPlay anytime. Even offline!",
       comment:
-        "A body of text explaining the feature that will be displayed under a title reading 'Introducing Brave Playlist'"
+        "A body of text explaining the feature that will be displayed under a title reading 'Introducing iBrowe Playlist'"
     )
     public static let advanceInitialOnboardingButtonTitle = NSLocalizedString(
       "advanceInitialOnboardingButtonTitle",
       bundle: .module,
       value: "Discover How",
       comment:
-        "A button title shown below language introducing the Brave Playlist feature. i.e. 'Discover how to use use this feature'"
+        "A button title shown below language introducing the iBrowe Playlist feature. i.e. 'Discover how to use use this feature'"
     )
     public static let dismissOnboardingButtonTitle = NSLocalizedString(
       "dismissOnboardingButtonTitle",
       bundle: .module,
       value: "Dismiss",
       comment:
-        "A button title that allows the user to dismiss the popup explaining the Brave Playlist feature"
+        "A button title that allows the user to dismiss the popup explaining the iBrowe Playlist feature"
     )
 
     // -------------------
@@ -50,7 +50,7 @@ extension Strings {
     public static let playlistInfoFeaturePointTitleOne = NSLocalizedString(
       "playlistInfoFeaturePointTitleOne",
       bundle: .module,
-      value: "Add videos to Brave Playlist. Playback anytime. Even offline",
+      value: "Add videos to iBrowe Playlist. Playback anytime. Even offline",
       comment: "A list of features in Playlist"
     )
     public static let playlistInfoFeaturePointSubtitleOne = NSLocalizedString(
@@ -95,7 +95,7 @@ extension Strings {
       bundle: .module,
       value: "Maybe Later",
       comment:
-        "A button title that allows the user to dismiss the popup explaining the Brave Playlist feature"
+        "A button title that allows the user to dismiss the popup explaining the iBrowe Playlist feature"
     )
 
     // -------------------
@@ -146,14 +146,14 @@ extension Strings {
       "confirmationActionButtonTitle",
       bundle: .module,
       value: "Take me to my Playlist",
-      comment: "A button title that when tapped opens Brave Playlist"
+      comment: "A button title that when tapped opens iBrowe Playlist"
     )
     public static let dismissStep4OnboardingButtonTitle = NSLocalizedString(
       "dismissStep4OnboardingButtonTitle",
       bundle: .module,
       value: "Close",
       comment:
-        "A button title that allows the user to dismiss the popup explaining the Brave Playlist feature"
+        "A button title that allows the user to dismiss the popup explaining the iBrowe Playlist feature"
     )
   }
 }
@@ -182,7 +182,7 @@ extension Strings {
       bundle: .module,
       value: "Disable This Feature",
       comment:
-        "A button that allows the user to disable the brave translate feature. When pressed, the page will not be automatically translated to the user's language."
+        "A button that allows the user to disable the iBrowe translate feature. When pressed, the page will not be automatically translated to the user's language."
     )
   }
 }
