diff --git a/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements b/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
index 6fd26ea80ec06ea489a1706bd0000256beacb42a..2643610aad7180fed4b301e5025387c3e6d64318 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
+++ b/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
@@ -28,7 +28,7 @@
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID).unique</string>
 	</array>
-	<key>com.apple.developer.carplay-audio</key>
-	<true/>
+<!--	<key>com.apple.developer.carplay-audio</key>-->
+<!--	<true/>-->
 </dict>
 </plist>
