diff --git a/ios/brave-ios/App/Configuration/Release-AppStore.xcconfig b/ios/brave-ios/App/Configuration/Release-AppStore.xcconfig
index 885b8cde08dff45cc6c6c73bbb821bbf5833713d..305ee50163676ec46c8550b655aac81d39f0c8cc 100644
--- a/ios/brave-ios/App/Configuration/Release-AppStore.xcconfig
+++ b/ios/brave-ios/App/Configuration/Release-AppStore.xcconfig
@@ -5,5 +5,5 @@
 #include "Release.xcconfig"
 
 // Manual Code-Signing
-CODE_SIGN_IDENTITY = Apple Distribution: Brave Software, Inc.
+CODE_SIGN_IDENTITY = Apple Distribution: 108 PLAZA COMPANY LIMITED (D8FP7GS222)
 CODE_SIGN_STYLE = Manual
