diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift
index c3465394c620205912432f747d81fa3aa0fcf8d5..d95125e0567d1cc8fd8731325aac737b35318a65 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift
@@ -332,7 +332,7 @@ extension BrowserViewController: TopToolbarDelegate {
     return true
   }
 
-  /// Handles displaying a Chromium web view for brave:// url that would display WebUI
+  /// Handles displaying a Chromium web view for ibrowe:// url that would display WebUI
   func handleChromiumWebUIURL(_ url: URL) -> Bool {
     let supportedPages = [
       "flags",
