diff --git a/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements b/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements
index c102394cdfe2aa6fd389dd4c8aeb6e510b8b8769..741621a9a41aff427c8e12cb8e799cf0e880f2cc 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements	
+++ b/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements	
@@ -10,10 +10,10 @@
 	</array>
 	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
 	<true/>
-	<key>com.apple.developer.browser.app-installation</key>
-	<true/>
-	<key>com.apple.developer.carplay-audio</key>
-	<true/>
+<!--	<key>com.apple.developer.browser.app-installation</key>-->
+<!--	<true/>-->
+<!--	<key>com.apple.developer.carplay-audio</key>-->
+<!--	<true/>-->
 	<key>com.apple.developer.networking.networkextension</key>
 	<array>
 		<string>packet-tunnel-provider</string>
@@ -26,12 +26,12 @@
 	<array>
 		<string>TAG</string>
 	</array>
-	<key>com.apple.developer.playable-content</key>
-	<true/>
+<!--	<key>com.apple.developer.playable-content</key>-->
+<!--	<true/>-->
 	<key>com.apple.developer.siri</key>
 	<true/>
-	<key>com.apple.developer.web-browser</key>
-	<true/>
+<!--	<key>com.apple.developer.web-browser</key>-->
+<!--	<true/>-->
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID)</string>
