diff --git a/test/filters/browser_tests.filter b/test/filters/browser_tests.filter
index 4bd6097d06ea11f2d6cf33c1d72a56a917995748..a6eed526be45fee79d858da1acee1bd4220502d7 100644
--- a/test/filters/browser_tests.filter
+++ b/test/filters/browser_tests.filter
@@ -1995,7 +1995,7 @@
 -BrowserCommandsTest.AddingToReadingListOpensToast
 -BrowserCommandsTest.CopyingUrlOpensToast
 
-# This test fails because we show brave://welcome page on first run
+# This test fails because we show ibrowe://welcome page on first run
 -StartupBrowserCreatorTriggeredResetFirstRunTest.TestTriggeredResetDoesNotShowWithFirstRunURLs
 
 # These tests fail because we change the default for TOP_LEVEL_STORAGE_ACCESS
