diff --git a/android/java/org/chromium/chrome/browser/feedback/BraveHelpAndFeedbackLauncherImpl.java b/android/java/org/chromium/chrome/browser/feedback/BraveHelpAndFeedbackLauncherImpl.java
index 14ef0aea99664e4ca18172b9bf5afc0c5ffeca16..a9d02243b91b1c14d55e675ce068a251cc59c82b 100644
--- a/android/java/org/chromium/chrome/browser/feedback/BraveHelpAndFeedbackLauncherImpl.java
+++ b/android/java/org/chromium/chrome/browser/feedback/BraveHelpAndFeedbackLauncherImpl.java
@@ -19,9 +19,9 @@ import org.chromium.chrome.browser.profiles.Profile;
 import java.util.Map;
 
 public class BraveHelpAndFeedbackLauncherImpl implements HelpAndFeedbackLauncher {
-    private static final String FALLBACK_SUPPORT_URL = "https://community.brave.com/";
+    private static final String FALLBACK_SUPPORT_URL = "https://community.ibrowe.com/";
     private static final String SAFE_BROWSING_URL =
-            "https://support.brave.com/hc/articles/15222663599629-Safe-Browsing-in-Brave";
+            "https://support.ibrowe.com/hc/articles/15222663599629-Safe-Browsing-in-Brave";
 
     public BraveHelpAndFeedbackLauncherImpl(Profile profile) {}
 
