diff --git a/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java b/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
index ba6d1014c539baabbe718d2b9076c8cf32ce2baf..f0af94ff2b51c4d58b3ae0626ab916a2b921732d 100644
--- a/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
+++ b/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
@@ -34,13 +34,13 @@ import androidx.core.content.ContextCompat;
 import androidx.core.content.res.ResourcesCompat;
 import androidx.core.widget.ImageViewCompat;
 
-import com.brave.playlist.enums.PlaylistOptionsEnum;
-import com.brave.playlist.listener.PlaylistOnboardingActionClickListener;
-import com.brave.playlist.listener.PlaylistOptionsListener;
-import com.brave.playlist.model.PlaylistOptionsModel;
-import com.brave.playlist.model.SnackBarActionModel;
-import com.brave.playlist.util.ConstantUtils;
-import com.brave.playlist.util.PlaylistViewUtils;
+import com.ibrowe108.playlist.enums.PlaylistOptionsEnum;
+import com.ibrowe108.playlist.listener.PlaylistOnboardingActionClickListener;
+import com.ibrowe108.playlist.listener.PlaylistOptionsListener;
+import com.ibrowe108.playlist.model.PlaylistOptionsModel;
+import com.ibrowe108.playlist.model.SnackBarActionModel;
+import com.ibrowe108.playlist.util.ConstantUtils;
+import com.ibrowe108.playlist.util.PlaylistViewUtils;
 
 import org.chromium.base.ApiCompatibilityUtils;
 import org.chromium.base.BraveFeatureList;
