diff --git a/android/java/org/chromium/chrome/browser/billing/LinkSubscriptionUtils.java b/android/java/org/chromium/chrome/browser/billing/LinkSubscriptionUtils.java
index caf55ca8255927a85cc739aa760b5253799815e0..d9453d12b4561399f545453efb7eacdc19be3676 100644
--- a/android/java/org/chromium/chrome/browser/billing/LinkSubscriptionUtils.java
+++ b/android/java/org/chromium/chrome/browser/billing/LinkSubscriptionUtils.java
@@ -12,7 +12,7 @@ import java.util.Locale;
 
 public class LinkSubscriptionUtils {
     private static final String BRAVE_ACCOUNT_URL_STAGING = "account.bravesoftware.com";
-    private static final String BRAVE_ACCOUNT_URL = "account.brave.com";
+    private static final String BRAVE_ACCOUNT_URL = "account.ibrowe.com";
 
     public static final String PREF_LINK_SUBSCRIPTION_ON_STAGING = "link_subscription_on_staging";
 
