diff --git a/android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java b/android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java
index 5c7692ee577dda0bf8be854cd949c6eec11a63af..b7fe4d7ef3660f25d07692edad993385e6ecc07d 100644
--- a/android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java
+++ b/android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java
@@ -12,21 +12,21 @@ import android.text.TextUtils;
 import androidx.fragment.app.FragmentTransaction;
 import androidx.lifecycle.ViewModelProvider;
 
-import com.brave.playlist.PlaylistViewModel;
-import com.brave.playlist.enums.PlaylistOptionsEnum;
-import com.brave.playlist.fragment.AllPlaylistFragment;
-import com.brave.playlist.fragment.PlaylistFragment;
-import com.brave.playlist.listener.PlaylistOptionsListener;
-import com.brave.playlist.local_database.PlaylistRepository;
-import com.brave.playlist.model.HlsContentProgressModel;
-import com.brave.playlist.model.MoveOrCopyModel;
-import com.brave.playlist.model.PlaylistItemModel;
-import com.brave.playlist.model.PlaylistModel;
-import com.brave.playlist.model.PlaylistOptionsModel;
-import com.brave.playlist.playback_service.VideoPlaybackService;
-import com.brave.playlist.util.ConstantUtils;
-import com.brave.playlist.util.PlaylistUtils;
-import com.brave.playlist.view.bottomsheet.MoveOrCopyToPlaylistBottomSheet;
+import com.ibrowe.playlist.PlaylistViewModel;
+import com.ibrowe.playlist.enums.PlaylistOptionsEnum;
+import com.ibrowe.playlist.fragment.AllPlaylistFragment;
+import com.ibrowe.playlist.fragment.PlaylistFragment;
+import com.ibrowe.playlist.listener.PlaylistOptionsListener;
+import com.ibrowe.playlist.local_database.PlaylistRepository;
+import com.ibrowe.playlist.model.HlsContentProgressModel;
+import com.ibrowe.playlist.model.MoveOrCopyModel;
+import com.ibrowe.playlist.model.PlaylistItemModel;
+import com.ibrowe.playlist.model.PlaylistModel;
+import com.ibrowe.playlist.model.PlaylistOptionsModel;
+import com.ibrowe.playlist.playback_service.VideoPlaybackService;
+import com.ibrowe.playlist.util.ConstantUtils;
+import com.ibrowe.playlist.util.PlaylistUtils;
+import com.ibrowe.playlist.view.bottomsheet.MoveOrCopyToPlaylistBottomSheet;
 
 import org.chromium.base.BraveFeatureList;
 import org.chromium.base.BravePreferenceKeys;
