diff --git a/android/java/org/chromium/chrome/browser/set_default_browser/BraveSetDefaultBrowserUtils.java b/android/java/org/chromium/chrome/browser/set_default_browser/BraveSetDefaultBrowserUtils.java
index 70edbe608cba1a6a05702a1d4d03116f4c4b7257..5cdbdd8b126db5c03bfbbfb489fa4bb4877a171c 100644
--- a/android/java/org/chromium/chrome/browser/set_default_browser/BraveSetDefaultBrowserUtils.java
+++ b/android/java/org/chromium/chrome/browser/set_default_browser/BraveSetDefaultBrowserUtils.java
@@ -35,7 +35,7 @@ public class BraveSetDefaultBrowserUtils {
     private static final String TAG = "BSDBrowserUtils";
     public static final String ANDROID_SETUPWIZARD_PACKAGE_NAME = "com.google.android.setupwizard";
     public static final String ANDROID_PACKAGE_NAME = "android";
-    public static final String BRAVE_BLOG_URL = "https://brave.com/privacy-features/";
+    public static final String BRAVE_BLOG_URL = "https://ibrowe.com/privacy-features/";
 
     public static boolean isBottomSheetVisible;
 
