diff --git a/android/java/org/chromium/chrome/browser/upgrade/NotificationIntent.java b/android/java/org/chromium/chrome/browser/upgrade/NotificationIntent.java
index bd04938d107572a3ba999250021d5ff523ba43b5..058bcf2cff4f56d158be8997bc97128758486e68 100644
--- a/android/java/org/chromium/chrome/browser/upgrade/NotificationIntent.java
+++ b/android/java/org/chromium/chrome/browser/upgrade/NotificationIntent.java
@@ -31,7 +31,7 @@ public class NotificationIntent {
     private static final String TAG = "NotificationIntent";
     private static final String NOTIFICATION_TAG = "16c570a4-da7d-4c4e-8518-d2b7d6e41615";
     private static final String NOTIFICATION_CHANNEL_ID = "a79c3102-4183-4001-a553-ec3041bd0f49";
-    private static final String URL = "https://support.brave.com/hc/en-us/articles/360045401211/";
+    private static final String URL = "https://support.ibrowe.com/hc/en-us/articles/360045401211/";
     private static final List<String> mWhitelistedRegionalLocales =
             Arrays.asList("en", "ru", "uk", "de", "pt", "pl", "ja", "es", "fr");
     private static final int NOTIFICATION_ID = 732;
