diff --git a/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java b/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java
index d051ba158a5561682676e3792abd5daeb7be96ac..bd086364c3fc770f3d634bbddc19d4231a46db72 100644
--- a/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java
+++ b/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java
@@ -135,7 +135,7 @@ public class WalletUtils {
         }
     }
 
-    /** Closes all Wallet tabs that contains the base URL `brave://wallet`. */
+    /** Closes all Wallet tabs that contains the base URL `ibrowe://wallet`. */
     public static void closeWebWallet() {
         try {
             BraveActivity activity = BraveActivity.getBraveActivity();
