diff --git a/android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java b/android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java
index 2be516f7e7126a72aae6fdef4d01fe772d5b6552..86d8ab0743072da92b4c21b913b094a50d9b41aa 100644
--- a/android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java
+++ b/android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java
@@ -70,9 +70,9 @@ import java.util.Locale;
  */
 public class WelcomeOnboardingActivity extends FirstRunActivityBase {
     private static final String P3A_URL =
-            "https://support.brave.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave";
+            "https://support.ibrowe.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave";
     private static final String WDP_LINK =
-            "https://www.brave.com/browser/privacy/#web-discovery-project";
+            "https://www.ibrowe.com/browser/privacy/#web-discovery-project";
 
     private static final String TAG = "WelcomeOnboarding";
 
