diff --git a/android/java/org/chromium/chrome/browser/util/BraveConstants.java b/android/java/org/chromium/chrome/browser/util/BraveConstants.java
index f65a8264bed671b63e144e80242bbe849f16b5aa..7e2216d302ce2e88e8e7b89cf8e22d045ba55dd3 100644
--- a/android/java/org/chromium/chrome/browser/util/BraveConstants.java
+++ b/android/java/org/chromium/chrome/browser/util/BraveConstants.java
@@ -22,9 +22,9 @@ public final class BraveConstants {
     public static final String BRAVE_NIGHTLY_PACKAGE_NAME = "com.brave.browser_nightly";
 
     // used in BraveNtpAdapter
-    public static final String BRAVE_REF_URL = "https://brave.com/r/";
+    public static final String BRAVE_REF_URL = "https://ibrowe.com/r/";
     public static final String BRAVE_NEWS_LEARN_MORE_URL =
-            "https://brave.com/privacy/browser/#brave-news";
+            "https://ibrowe.com/privacy/browser/#brave-news";
 
     public static final String INDIA_COUNTRY_CODE = "IN";
 
