diff --git a/android/java/org/chromium/chrome/browser/playlist/hls_content/HlsServiceImpl.java b/android/java/org/chromium/chrome/browser/playlist/hls_content/HlsServiceImpl.java
index 9022dbb66d98703f286f37ab54929a1cf95175b3..efe6f3329f2140d4c4f6c0dff90d821c6457feb8 100644
--- a/android/java/org/chromium/chrome/browser/playlist/hls_content/HlsServiceImpl.java
+++ b/android/java/org/chromium/chrome/browser/playlist/hls_content/HlsServiceImpl.java
@@ -15,13 +15,13 @@ import android.os.IBinder;
 import androidx.annotation.Nullable;
 import androidx.media3.exoplayer.hls.playlist.HlsMediaPlaylist.Segment;
 
-import com.brave.playlist.local_database.PlaylistRepository;
-import com.brave.playlist.model.HlsContentProgressModel;
-import com.brave.playlist.model.HlsContentQueueModel;
-import com.brave.playlist.model.PlaylistItemModel;
-import com.brave.playlist.playback_service.VideoPlaybackService;
-import com.brave.playlist.util.MediaUtils;
-import com.brave.playlist.util.PlaylistUtils;
+import com.ibrowe108.playlist.local_database.PlaylistRepository;
+import com.ibrowe108.playlist.model.HlsContentProgressModel;
+import com.ibrowe108.playlist.model.HlsContentQueueModel;
+import com.ibrowe108.playlist.model.PlaylistItemModel;
+import com.ibrowe108.playlist.playback_service.VideoPlaybackService;
+import com.ibrowe108.playlist.util.MediaUtils;
+import com.ibrowe108.playlist.util.PlaylistUtils;
 
 import org.chromium.base.BravePreferenceKeys;
 import org.chromium.base.ContextUtils;
