diff --git a/android/java/org/chromium/chrome/browser/app/BraveActivity.java b/android/java/org/chromium/chrome/browser/app/BraveActivity.java
index d0f6ffc0f7b4bc05e4233d795cd27817209c0334..8c8244e9203f0d68b4ff74f89a9bdb42412641d5 100644
--- a/android/java/org/chromium/chrome/browser/app/BraveActivity.java
+++ b/android/java/org/chromium/chrome/browser/app/BraveActivity.java
@@ -43,9 +43,9 @@ import androidx.lifecycle.MutableLiveData;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 
-import com.brave.playlist.util.ConstantUtils;
-import com.brave.playlist.util.PlaylistPreferenceUtils;
-import com.brave.playlist.util.PlaylistUtils;
+import com.ibrowe108.playlist.util.ConstantUtils;
+import com.ibrowe108.playlist.util.PlaylistPreferenceUtils;
+import com.ibrowe108.playlist.util.PlaylistUtils;
 import com.google.android.material.dialog.MaterialAlertDialogBuilder;
 import com.google.android.play.core.appupdate.AppUpdateInfo;
 import com.google.android.play.core.appupdate.AppUpdateManager;
@@ -250,28 +250,28 @@ public abstract class BraveActivity extends ChromeActivity
                 KeyboardVisibilityHelper.KeyboardVisibilityListener,
                 OnSharedPreferenceChangeListener {
     public static final String BRAVE_WALLET_HOST = "wallet";
-    public static final String BRAVE_WALLET_ORIGIN = "brave://wallet/";
-    public static final String BRAVE_WALLET_URL = "brave://wallet/crypto/portfolio/assets";
-    public static final String BRAVE_BUY_URL = "brave://wallet/crypto/fund-wallet";
-    public static final String BRAVE_SEND_URL = "brave://wallet/send";
-    public static final String BRAVE_SWAP_URL = "brave://wallet/swap";
-    public static final String BRAVE_DEPOSIT_URL = "brave://wallet/crypto/deposit-funds";
-    public static final String BRAVE_REWARDS_SETTINGS_URL = "brave://rewards/";
+    public static final String BRAVE_WALLET_ORIGIN = "ibrowe://wallet/";
+    public static final String BRAVE_WALLET_URL = "ibrowe://wallet/crypto/portfolio/assets";
+    public static final String BRAVE_BUY_URL = "ibrowe://wallet/crypto/fund-wallet";
+    public static final String BRAVE_SEND_URL = "ibrowe://wallet/send";
+    public static final String BRAVE_SWAP_URL = "ibrowe://wallet/swap";
+    public static final String BRAVE_DEPOSIT_URL = "ibrowe://wallet/crypto/deposit-funds";
+    public static final String BRAVE_REWARDS_SETTINGS_URL = "ibrowe://rewards/";
     public static final String BRAVE_REWARDS_SETTINGS_WALLET_VERIFICATION_URL =
-            "brave://rewards/#verify";
-    public static final String BRAVE_REWARDS_WALLET_RECONNECT_URL = "brave://rewards/reconnect";
-    public static final String BRAVE_REWARDS_SETTINGS_MONTHLY_URL = "brave://rewards/#monthly";
-    public static final String REWARDS_AC_SETTINGS_URL = "brave://rewards/contribute";
-    public static final String BRAVE_REWARDS_RESET_PAGE = "brave://rewards/#reset";
+            "ibrowe://rewards/#verify";
+    public static final String BRAVE_REWARDS_WALLET_RECONNECT_URL = "ibrowe://rewards/reconnect";
+    public static final String BRAVE_REWARDS_SETTINGS_MONTHLY_URL = "ibrowe://rewards/#monthly";
+    public static final String REWARDS_AC_SETTINGS_URL = "ibrowe://rewards/contribute";
+    public static final String BRAVE_REWARDS_RESET_PAGE = "ibrowe://rewards/#reset";
     public static final String BRAVE_AI_CHAT_URL = "chrome-untrusted://chat/tab";
     public static final String REWARDS_LEARN_MORE_URL =
-            "https://brave.com/faq-rewards/#unclaimed-funds";
+            "https://ibrowe.com/faq-rewards/#unclaimed-funds";
     public static final String BRAVE_TERMS_PAGE =
             "https://basicattentiontoken.org/user-terms-of-service/";
-    public static final String BRAVE_PRIVACY_POLICY = "https://brave.com/privacy/browser/#rewards";
+    public static final String BRAVE_PRIVACY_POLICY = "https://ibrowe.com/privacy/browser/#rewards";
     public static final String OPEN_URL = "open_url";
     public static final String BRAVE_WEBCOMPAT_INFO_WIKI_URL =
-            "https://github.com/brave/brave-browser/wiki/Web-compatibility-reports";
+            "https://github.com/yuthstyle88/ibrowe-browser/wiki/Web-compatibility-reports";
 
     private static final int DAYS_4 = 4;
     private static final int DAYS_7 = 7;
@@ -292,7 +292,7 @@ public abstract class BraveActivity extends ChromeActivity
             Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE;
 
     /** Settings for sending local notification reminders. */
-    public static final String CHANNEL_ID = "com.brave.browser";
+    public static final String CHANNEL_ID = "com.ibrowe108.browser";
 
     // Explicitly declare this variable to avoid build errors.
     // It will be removed in asm and parent variable will be used instead.
