diff --git a/android/java/org/chromium/chrome/browser/tabbed_mode/BraveTabbedAppMenuPropertiesDelegate.java b/android/java/org/chromium/chrome/browser/tabbed_mode/BraveTabbedAppMenuPropertiesDelegate.java
index 5df52a29108edc104547bd1d990cf1a3427f725c..8ab73e80d0d6cd691fbabc28a2a41ebf6f861fdb 100644
--- a/android/java/org/chromium/chrome/browser/tabbed_mode/BraveTabbedAppMenuPropertiesDelegate.java
+++ b/android/java/org/chromium/chrome/browser/tabbed_mode/BraveTabbedAppMenuPropertiesDelegate.java
@@ -306,19 +306,19 @@ public class BraveTabbedAppMenuPropertiesDelegate extends TabbedAppMenuPropertie
                             R.id.brave_wallet_id,
                             R.id.all_bookmarks_menu_id));
         }
-        if (BraveLeoPrefUtils.isLeoEnabled()) {
-            Tab tab = mActivityTabProvider.get();
-            if (tab != null && !tab.isIncognito()) {
-                addMenuItemAfter(
-                        modelList,
-                        buildBraveLeoItem(),
-                        Arrays.asList(
-                                R.id.add_to_playlist_id,
-                                R.id.brave_playlist_id,
-                                R.id.brave_wallet_id,
-                                R.id.all_bookmarks_menu_id));
-            }
-        }
+//         if (BraveLeoPrefUtils.isLeoEnabled()) {
+//             Tab tab = mActivityTabProvider.get();
+//             if (tab != null && !tab.isIncognito()) {
+//                 addMenuItemAfter(
+//                         modelList,
+//                         buildBraveLeoItem(),
+//                         Arrays.asList(
+//                                 R.id.add_to_playlist_id,
+//                                 R.id.brave_playlist_id,
+//                                 R.id.brave_wallet_id,
+//                                 R.id.all_bookmarks_menu_id));
+//             }
+//         }
         if (ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_SPEEDREADER)
                 && UserPrefs.get(assumeNonNull(mTabModelSelector.getCurrentModel().getProfile()))
                         .getBoolean(BravePref.SPEEDREADER_PREF_ENABLED)) {
@@ -332,13 +332,13 @@ public class BraveTabbedAppMenuPropertiesDelegate extends TabbedAppMenuPropertie
             modelList.add(buildSetDefaultBrowserItem());
         }
         if (!mJunitIsTesting) {
-            if (BraveVpnUtils.isVpnFeatureSupported(mContext)) {
-                modelList.add(buildBraveVpnItem());
-                if (BraveVpnPrefUtils.isSubscriptionPurchase()
-                        && !TextUtils.isEmpty(BraveVpnPrefUtils.getRegionIsoCode())) {
-                    modelList.add(buildBraveVpnLocationIconItem());
-                }
-            }
+//             if (BraveVpnUtils.isVpnFeatureSupported(mContext)) {
+//                 modelList.add(buildBraveVpnItem());
+//                 if (BraveVpnPrefUtils.isSubscriptionPurchase()
+//                         && !TextUtils.isEmpty(BraveVpnPrefUtils.getRegionIsoCode())) {
+//                     modelList.add(buildBraveVpnLocationIconItem());
+//                 }
+//             }
             BraveRewardsNativeWorker braveRewardsNativeWorker =
                     BraveRewardsNativeWorker.getInstance();
             if (braveRewardsNativeWorker != null && braveRewardsNativeWorker.isSupported()) {
