diff --git a/android/java/org/chromium/chrome/browser/onboarding/v2/HighlightDialogFragment.java b/android/java/org/chromium/chrome/browser/onboarding/v2/HighlightDialogFragment.java
index fe7b8f50f83e8218fe5ac3086622b51d51a57de0..fcdd338965d65a4973405d0a15f0b5d821dc20f7 100644
--- a/android/java/org/chromium/chrome/browser/onboarding/v2/HighlightDialogFragment.java
+++ b/android/java/org/chromium/chrome/browser/onboarding/v2/HighlightDialogFragment.java
@@ -35,7 +35,7 @@ import java.util.Locale;
 
 public class HighlightDialogFragment extends DialogFragment {
     final public static String TAG_FRAGMENT = "HIGHLIGHT_FRAG";
-    private final static String NTP_TUTORIAL_PAGE = "https://brave.com/ja/android-ntp-tutorial";
+    private final static String NTP_TUTORIAL_PAGE = "https://ibrowe.com/ja/android-ntp-tutorial";
 
     public interface HighlightDialogListener {
         void onNextPage();
