diff --git a/android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java b/android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java
index e45cb7e73d2bea577696e32c6483540dc1a3eea7..a5650b800d685606ab4935ab8400544dd7fa9bd8 100644
--- a/android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java
+++ b/android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java
@@ -90,18 +90,18 @@ public class BraveRewardsPanel
         implements BraveRewardsObserver, BraveRewardsHelper.LargeIconReadyCallback {
     public static final String PREF_WAS_BRAVE_REWARDS_TURNED_ON = "brave_rewards_turned_on";
     public static final String PREF_ON_BOARDING_COMPLETED = "on_boarding_completed";
-    public static final String REWARDS_TOUR_URL = "http://brave.com/rewards-tour";
+    public static final String REWARDS_TOUR_URL = "http://ibrowe.com/rewards-tour";
     public static final String PREF_WAS_TOOLBAR_BAT_LOGO_BUTTON_PRESSED =
             "was_toolbar_bat_logo_button_pressed";
     private static final String UNVERIFIED_USER_UNSUPPORTED_REGION_PAGE =
-            "https://support.brave.com/hc/en-us/articles/6539887971469";
+            "https://support.ibrowe.com/hc/en-us/articles/6539887971469";
     public static final String NEW_SIGNUP_DISABLED_URL =
-            "https://support.brave.com/hc/en-us/articles/9312922941069";
+            "https://support.ibrowe.com/hc/en-us/articles/9312922941069";
     private static final String ADS_PAYOUT_STATUS_URL =
-            "https://community.brave.com/t/ads-payout-status-update/287895";
-    private static final String SUPPORT_URL = "https://support.brave.com";
-    private static final String BRAVE_REWARDS_PAGE = "https://brave.com/rewards";
-    private static final String BRAVE_REWARDS_CHANGES_PAGE = "https://brave.com/rewards-changes";
+            "https://community.ibrowe.com/t/ads-payout-status-update/287895";
+    private static final String SUPPORT_URL = "https://support.ibrowe.com";
+    private static final String BRAVE_REWARDS_PAGE = "https://ibrowe.com/rewards";
+    private static final String BRAVE_REWARDS_CHANGES_PAGE = "https://ibrowe.com/rewards-changes";
 
     private static final String TAG = "BraveRewards";
     private static final int PUBLISHER_INFO_FETCH_RETRY = 3 * 1000; // In milliseconds
