diff --git a/android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java b/android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java
index 4fbd92cb00ecc7fe816dd058bf960f5aefeea31a..2d97e83e07d9cc37bf0be34168dc358afe209364 100644
--- a/android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java
+++ b/android/java/org/chromium/chrome/browser/brave_news/CardBuilderFeedCard.java
@@ -105,7 +105,7 @@ public class CardBuilderFeedCard {
 
     private static final String TAG = "BN";
     private static final int MARGIN_VERTICAL = 10;
-    private static final String BRAVE_OFFERS_URL = "offers.brave.com";
+    private static final String BRAVE_OFFERS_URL = "offers.ibrowe.com";
 
     public CardBuilderFeedCard(BraveNewsController braveNewsController, RequestManager glide,
             LinearLayout layout, Activity activity, int position, FeedItemsCard newsItem,
