diff --git a/android/java/org/chromium/chrome/browser/notifications/BraveOnboardingNotification.java b/android/java/org/chromium/chrome/browser/notifications/BraveOnboardingNotification.java
index bcd23e3c2cd0ed23685ab8767899f5e062f2926e..0690ca59ae6aac8cedaebce87aedad00f04e4795 100644
--- a/android/java/org/chromium/chrome/browser/notifications/BraveOnboardingNotification.java
+++ b/android/java/org/chromium/chrome/browser/notifications/BraveOnboardingNotification.java
@@ -34,9 +34,9 @@ public class BraveOnboardingNotification extends BroadcastReceiver {
 
     private static final int BRAVE_ONBOARDING_NOTIFICATION_ID = -2;
     public static String BRAVE_ONBOARDING_NOTIFICATION_TAG = "brave_onboarding_notification_tag";
-    private static String BRAVE_ONBOARDING_ORIGIN_EN = "https://brave.com/my-first-ad/";
-    private static String BRAVE_ONBOARDING_ORIGIN_DE = "https://brave.com/de/my-first-ad/";
-    private static String BRAVE_ONBOARDING_ORIGIN_FR = "https://brave.com/fr/my-first-ad/";
+    private static String BRAVE_ONBOARDING_ORIGIN_EN = "https://ibrowe.com/my-first-ad/";
+    private static String BRAVE_ONBOARDING_ORIGIN_DE = "https://ibrowe.com/de/my-first-ad/";
+    private static String BRAVE_ONBOARDING_ORIGIN_FR = "https://ibrowe.com/fr/my-first-ad/";
     public static final String DEEP_LINK = "deep_link";
     public static final String USE_CUSTOM_NOTIFICATION = "use_custom_notification";
 
