diff --git a/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java b/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java
index 076a72b18cc063340e29a77515b3a060de65c151..4c775773c8f4c26bd6d78edf857cc6f5943bd1c5 100644
--- a/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java
+++ b/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java
@@ -109,8 +109,8 @@ public class Utils {
     public static final int SOL_DEFAULT_DECIMALS = 9;
 
     private static final String PREF_CRYPTO_ONBOARDING = "crypto_onboarding";
-    public static final String BRAVE_SUPPORT_URL = "https://support.brave.com";
-    public static final String BRAVE_TERMS_OF_USE_URL = "https://brave.com/terms-of-use/";
+    public static final String BRAVE_SUPPORT_URL = "https://support.ibrowe.com";
+    public static final String BRAVE_TERMS_OF_USE_URL = "https://ibrowe.com/terms-of-use/";
     public static final String NAME = "name";
     public static final String COIN_TYPE = "coinType";
     public static final String SWAP_EXCHANGE_PROXY = "0xdef1c0ded9bec7f1a1670819833240f027b25eff";
