diff --git a/android/java/org/chromium/chrome/browser/vpn/settings/BraveVpnPreferences.java b/android/java/org/chromium/chrome/browser/vpn/settings/BraveVpnPreferences.java
index ae0fe6e5972bfd1923d2a02033a7aff46171ebc9..91e1030918154da9034905abd0f43188649e43d9 100644
--- a/android/java/org/chromium/chrome/browser/vpn/settings/BraveVpnPreferences.java
+++ b/android/java/org/chromium/chrome/browser/vpn/settings/BraveVpnPreferences.java
@@ -78,7 +78,7 @@ public class BraveVpnPreferences extends BravePreferenceFragment implements Brav
     private static final int INVALIDATE_CREDENTIAL_TIMER_COUNT = 5000;
 
     private static final String VPN_SUPPORT_PAGE =
-            "https://support.brave.com/hc/en-us/articles/4410838268429";
+            "https://support.ibrowe.com/hc/en-us/articles/4410838268429";
 
     private static final String DATE_FORMAT = "dd/MM/yyyy";
 
