diff --git a/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java b/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java
index 8aff723844d704d882a8388e5434f8f25d7d5102..53bcec0a5d17db125b640656b86fd8af571f856e 100644
--- a/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java
+++ b/android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java
@@ -28,14 +28,14 @@ public final class WalletConstants {
 
     // Brave
     public static final String URL_SIGN_TRANSACTION_REQUEST =
-            "https://support.brave.com/hc/en-us/articles/4409513799693";
+            "https://support.ibrowe.com/hc/en-us/articles/4409513799693";
 
     // NFT Auto Discovery
     public static final String NFT_DISCOVERY_LEARN_MORE_LINK =
             "https://github.com/brave/brave-browser/wiki/NFT-Discovery";
 
     public static final String WALLET_HELP_CENTER =
-            "https://support.brave.com/hc/en-us/categories/360001059151-Brave-Wallet";
+            "https://support.ibrowe.com/hc/en-us/categories/360001059151-Brave-Wallet";
 
     // Regex
     public static final String REGX_ANY_ETH_ADDRESS = ".*(0x[a-fA-F0-9]{40}).*";
