diff --git a/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java b/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
index 536b0f25303887e4d8ea44c08c55255dd2837c55..61bf18c39e5132c1939de1741a1a16f220182e0a 100644
--- a/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
+++ b/android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java
@@ -32,13 +32,13 @@ import androidx.core.content.ContextCompat;
 import androidx.core.content.res.ResourcesCompat;
 import androidx.core.widget.ImageViewCompat;
 
-import com.brave.playlist.enums.PlaylistOptionsEnum;
-import com.brave.playlist.listener.PlaylistOnboardingActionClickListener;
-import com.brave.playlist.listener.PlaylistOptionsListener;
-import com.brave.playlist.model.PlaylistOptionsModel;
-import com.brave.playlist.model.SnackBarActionModel;
-import com.brave.playlist.util.ConstantUtils;
-import com.brave.playlist.util.PlaylistViewUtils;
+import com.ibrowe.playlist.enums.PlaylistOptionsEnum;
+import com.ibrowe.playlist.listener.PlaylistOnboardingActionClickListener;
+import com.ibrowe.playlist.listener.PlaylistOptionsListener;
+import com.ibrowe.playlist.model.PlaylistOptionsModel;
+import com.ibrowe.playlist.model.SnackBarActionModel;
+import com.ibrowe.playlist.util.ConstantUtils;
+import com.ibrowe.playlist.util.PlaylistViewUtils;
 
 import org.chromium.base.ApiCompatibilityUtils;
 import org.chromium.base.BraveFeatureList;
