diff --git a/app/theme/brave/BRANDING b/app/theme/brave/BRANDING
index 6cd0a54cd16cdf9c8b64a0b743b6000300f63a3c..fa21b74a0408f88719d0d8b21c8f33b56372cc04 100644
--- a/app/theme/brave/BRANDING
+++ b/app/theme/brave/BRANDING
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=Brave Software, Inc.
-COMPANY_SHORTNAME=Brave Software
-PRODUCT_FULLNAME=Brave Browser
-PRODUCT_SHORTNAME=Brave
-PRODUCT_INSTALLER_FULLNAME=Brave Installer
-PRODUCT_INSTALLER_SHORTNAME=Brave Installer
-COPYRIGHT=Copyright 2016 The Brave Authors. All rights reserved.
-MAC_BUNDLE_ID=com.brave.Browser
+COMPANY_FULLNAME=iBrowe Software, Inc.
+COMPANY_SHORTNAME=iBrowe Software
+PRODUCT_FULLNAME=iBrowe Browser
+PRODUCT_SHORTNAME=iBrowe
+PRODUCT_INSTALLER_FULLNAME=iBrowe Installer
+PRODUCT_INSTALLER_SHORTNAME=iBrowe Installer
+COPYRIGHT=Copyright 2016 The iBrowe Authors. All rights reserved.
+MAC_BUNDLE_ID=com.ibrowe.Browser
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=KL8N8XSYF4
+MAC_TEAM_ID=D8FP7GS222
