diff --git a/app/theme/brave/BRANDING.nightly b/app/theme/brave/BRANDING.nightly
index 48194457ccc198f290baa278f7a0c18b87f60d33..fcc104cf857fe7b7c265e5e959a5dfcea6d49902 100644
--- a/app/theme/brave/BRANDING.nightly
+++ b/app/theme/brave/BRANDING.nightly
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=Brave Software, Inc.
-COMPANY_SHORTNAME=Brave Software
-PRODUCT_FULLNAME=Brave Browser Nightly
-PRODUCT_SHORTNAME=Brave Nightly
-PRODUCT_INSTALLER_FULLNAME=Brave Installer
-PRODUCT_INSTALLER_SHORTNAME=Brave Installer
-COPYRIGHT=Copyright 2016 The Brave Authors. All rights reserved.
-MAC_BUNDLE_ID=com.brave.Browser.nightly
+COMPANY_FULLNAME=iBrowe Software, Inc.
+COMPANY_SHORTNAME=iBrowe Software
+PRODUCT_FULLNAME=iBrowe Browser Nightly
+PRODUCT_SHORTNAME=iBrowe Nightly
+PRODUCT_INSTALLER_FULLNAME=iBrowe Installer
+PRODUCT_INSTALLER_SHORTNAME=iBrowe Installer
+COPYRIGHT=Copyright 2016 The iBrowe Authors. All rights reserved.
+MAC_BUNDLE_ID=com.ibrowe.browser.nightly
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=KL8N8XSYF4
+MAC_TEAM_ID=VJ4PQMKSUA
