diff --git a/app/theme/brave/BRANDING.dev b/app/theme/brave/BRANDING.dev
index cc68f036e27053be9841bf8cb253e29c0829f51e..b952d026745214fcf6f8b22f0b34de3d09c5f3c4 100644
--- a/app/theme/brave/BRANDING.dev
+++ b/app/theme/brave/BRANDING.dev
@@ -1,10 +1,10 @@
-COMPANY_FULLNAME=Brave Software, Inc.
-COMPANY_SHORTNAME=Brave Software
-PRODUCT_FULLNAME=Brave Browser Dev
-PRODUCT_SHORTNAME=Brave Dev
-PRODUCT_INSTALLER_FULLNAME=Brave Installer
-PRODUCT_INSTALLER_SHORTNAME=Brave Installer
-COPYRIGHT=Copyright 2016 The Brave Authors. All rights reserved.
-MAC_BUNDLE_ID=com.brave.Browser.dev
+COMPANY_FULLNAME=iBrowe Software, Inc.
+COMPANY_SHORTNAME=iBrowe Software
+PRODUCT_FULLNAME=iBrowe Browser Dev
+PRODUCT_SHORTNAME=iBrowe Dev
+PRODUCT_INSTALLER_FULLNAME=iBrowe Installer
+PRODUCT_INSTALLER_SHORTNAME=iBrowe Installer
+COPYRIGHT=Copyright 2016 The iBrowe Authors. All rights reserved.
+MAC_BUNDLE_ID=com.ibrowe.Browser.dev
 MAC_CREATOR_CODE=Cr24
-MAC_TEAM_ID=KL8N8XSYF4
+MAC_TEAM_ID=VJ4PQMKSUA
