diff --git a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
index 6416841cc80525f1621995aa3846566f107ec34b..7c42d8837d85f6cb351363c764a28b85dca104e9 100644
--- a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
+++ b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
@@ -1222,7 +1222,7 @@ void BraveAddLocalizedStrings(content::WebUIDataSource* html_source,
   BraveAddEmailAliasesStrings(html_source);
   BraveAddBraveAccountStrings(html_source);
 
-  // Load time data for brave://settings/extensions
+  // Load time data for ibrowe://settings/extensions
   html_source->AddBoolean(
       "signInAllowedOnNextStartupInitialValue",
       profile->GetPrefs()->GetBoolean(prefs::kSigninAllowedOnNextStartup));
