diff --git a/browser/ui/views/frame/brave_browser_view.cc b/browser/ui/views/frame/brave_browser_view.cc
index e892480c42a0441a74b1abf1a9f088de7aed2a3c..dc3d8c52c8c0f2e5119266a66ce5efb501c57726 100644
--- a/browser/ui/views/frame/brave_browser_view.cc
+++ b/browser/ui/views/frame/brave_browser_view.cc
@@ -267,12 +267,12 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
   // Show the correct value in settings on initial start
   UpdateSearchTabsButtonState();
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  pref_change_registrar_.Add(
-      brave_vpn::prefs::kBraveVPNShowButton,
-      base::BindRepeating(&BraveBrowserView::OnPreferenceChanged,
-                          base::Unretained(this)));
-#endif
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+//   pref_change_registrar_.Add(
+//       brave_vpn::prefs::kBraveVPNShowButton,
+//       base::BindRepeating(&BraveBrowserView::OnPreferenceChanged,
+//                           base::Unretained(this)));
+// #endif
 
   // Only normal window (tabbed) should have sidebar.
   const bool can_have_sidebar = sidebar::CanUseSidebar(browser_.get());
@@ -340,12 +340,12 @@ void BraveBrowserView::OnPreferenceChanged(const std::string& pref_name) {
     return;
   }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  if (pref_name == brave_vpn::prefs::kBraveVPNShowButton) {
-    vpn_panel_controller_.ResetBubbleManager();
-    return;
-  }
-#endif
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+//   if (pref_name == brave_vpn::prefs::kBraveVPNShowButton) {
+//     vpn_panel_controller_.ResetBubbleManager();
+//     return;
+//   }
+// #endif
 }
 
 void BraveBrowserView::UpdateSideBarHorizontalAlignment() {
@@ -390,22 +390,22 @@ void BraveBrowserView::ToggleSidebar() {
 }
 
 void BraveBrowserView::ShowBraveVPNBubble(bool show_select) {
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  vpn_panel_controller_.ShowBraveVPNPanel(show_select);
-#endif
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+//   vpn_panel_controller_.ShowBraveVPNPanel(show_select);
+// #endif
 }
 
 views::View* BraveBrowserView::GetAnchorViewForBraveVPNPanel() {
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  auto* vpn_button =
-      static_cast<BraveToolbarView*>(toolbar())->brave_vpn_button();
-  if (vpn_button->GetVisible()) {
-    return vpn_button;
-  }
-  return toolbar()->app_menu_button();
-#else
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+//   auto* vpn_button =
+//       static_cast<BraveToolbarView*>(toolbar())->brave_vpn_button();
+//   if (vpn_button->GetVisible()) {
+//     return vpn_button;
+//   }
+//   return toolbar()->app_menu_button();
+// #else
   return nullptr;
-#endif
+// #endif
 }
 
 gfx::Rect BraveBrowserView::GetShieldsBubbleRect() {
