diff --git a/browser/profiles/brave_profile_manager.cc b/browser/profiles/brave_profile_manager.cc
index 8d5b8aedb0e1b37914ce17bde85e2e1f082bc473..878b63efc6d2157a1f70b7b8bf7b141d69f9b3ce 100644
--- a/browser/profiles/brave_profile_manager.cc
+++ b/browser/profiles/brave_profile_manager.cc
@@ -123,7 +123,7 @@ void BraveProfileManager::InitProfileUserPrefs(Profile* profile) {
     pref_service->SetBoolean(kEnableMediaRouterOnRestart, enabled);
   } else {
     // For Desktop, kEnableMediaRouterOnRestart is used to track the current
-    // state of the media router switch in brave://settings/extensions. The
+    // state of the media router switch in ibrowe://settings/extensions. The
     // value of kEnableMediaRouter is only updated to match
     // kEnableMediaRouterOnRestart on restart
     auto enabled = pref_service->GetBoolean(kEnableMediaRouterOnRestart);
