diff --git a/browser/ui/views/playlist/playlist_action_dialogs.cc b/browser/ui/views/playlist/playlist_action_dialogs.cc
index 6cbd2eb77e150d94900abd4ddba466ffde5390bd..0e7bea7ed8c7b151c2c92973e8c752a146334e64 100644
--- a/browser/ui/views/playlist/playlist_action_dialogs.cc
+++ b/browser/ui/views/playlist/playlist_action_dialogs.cc
@@ -232,7 +232,7 @@ void ShowPlaylistSettings(content::WebContents* contents) {
   auto* browser_view = FindBrowserViewFromWebContents(contents);
   CHECK(browser_view);
   ShowSingletonTab(browser_view->browser(),
-                   GURL("brave://settings/braveContent#playlist-section"));
+                   GURL("ibrowe://settings/braveContent#playlist-section"));
 }
 
 void ShowPlaylistAddBubble(content::WebContents* contents) {
