diff --git a/browser/farbling/brave_dark_mode_fingerprint_protection_browsertest.cc b/browser/farbling/brave_dark_mode_fingerprint_protection_browsertest.cc
index e47ff82963d509a52d5421865fa338af8c8334d8..030cd9f32b0edd9e6b53fad496c8619217a884d4 100644
--- a/browser/farbling/brave_dark_mode_fingerprint_protection_browsertest.cc
+++ b/browser/farbling/brave_dark_mode_fingerprint_protection_browsertest.cc
@@ -229,7 +229,7 @@ IN_PROC_BROWSER_TEST_F(BraveDarkModeFingerprintProtectionTest,
 
   BlockFingerprinting();
   ASSERT_TRUE(
-      ui_test_utils::NavigateToURL(browser(), GURL("brave://settings")));
+      ui_test_utils::NavigateToURL(browser(), GURL("ibrowe://settings")));
   ASSERT_TRUE(IsReportingDarkMode());
 }
 
