diff --git a/browser/ui/whats_new/whats_new_util.cc b/browser/ui/whats_new/whats_new_util.cc
index 168c0358c14f734c3352116aed1644b345348229..10f27c137a3d3013604caae3aca0040a0d4cd2a6 100644
--- a/browser/ui/whats_new/whats_new_util.cc
+++ b/browser/ui/whats_new/whats_new_util.cc
@@ -189,7 +189,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
 }
 
 void StartBraveWhatsNew(Browser* browser) {
-  constexpr char kBraveWhatsNewURL[] = "https://brave.com/whats-new/";
+  constexpr char kBraveWhatsNewURL[] = "https://ibrowe.com/whats-new/";
   // Load whats-new url in the first foreground tab.
   chrome::AddTabAt(browser, GURL(kBraveWhatsNewURL), 0, true);
   browser->tab_strip_model()->ActivateTabAt(
