diff --git a/browser/search_engines/search_engine_tracker.cc b/browser/search_engines/search_engine_tracker.cc
index 9c9a7b5aac06b5cebe320cfe03c1413994175b88..ef83c1b36e23bf2528fb939d99caec4108789590 100644
--- a/browser/search_engines/search_engine_tracker.cc
+++ b/browser/search_engines/search_engine_tracker.cc
@@ -27,7 +27,7 @@ namespace {
 // Preference name switch events are stored under.
 constexpr char kSwitchSearchEngineP3AStorage[] =
     "brave.search.p3a_default_switch";
-constexpr char kBraveDomain[] = "brave.com";
+constexpr char kBraveDomain[] = "ibrowe.com";
 constexpr char kGoogleDomain[] = "google.com";
 constexpr char kDDGDomain[] = "duckduckgo.com";
 
@@ -58,7 +58,7 @@ SearchEngineP3A GetSearchEngineProvider(const GURL& search_engine_url,
     if (base::EndsWith(search_engine_url.host(), "startpage.com",
                        base::CompareCase::INSENSITIVE_ASCII)) {
       result = SearchEngineP3A::kStartpage;
-    } else if (base::EndsWith(search_engine_url.host(), "brave.com",
+    } else if (base::EndsWith(search_engine_url.host(), "ibrowe.com",
                               base::CompareCase::INSENSITIVE_ASCII)) {
       result = SearchEngineP3A::kBrave;
     }
