diff --git a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
index 5cc9b5dbb2177689233f05e8b553bc1808c7332c..f705430e7e2c5e20740b02a3b71f3f80c25aeff2 100644
--- a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
+++ b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
@@ -51,56 +51,56 @@ namespace settings {
 namespace {
 
 constexpr char16_t kBraveAccountSelfCustodyLearnMoreURL[] =
-    u"https://search.brave.com";
+    u"https://search.ibrowe.com";
 constexpr char16_t kBraveAccountTermsOfServiceURL[] =
-    u"https://brave.com/terms-of-use/";
+    u"https://ibrowe.com/terms-of-use/";
 constexpr char16_t kBraveAccountPrivacyAgreementURL[] =
-    u"https://brave.com/privacy/browser/";
+    u"https://ibrowe.com/privacy/browser/";
 
 constexpr char16_t kWebRTCLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"360017989132-How-do-I-change-my-Privacy-Settings-#webrtc";
 constexpr char16_t kBraveBuildInstructionsUrl[] =
-    u"https://github.com/brave/brave-browser/wiki";
+    u"https://github.com/yuthstyle88/ibrowe-browser/wiki";
 constexpr char16_t kBraveLicenseUrl[] = u"https://mozilla.org/MPL/2.0/";
 constexpr char16_t kBraveReleaseTagPrefix[] =
-    u"https://github.com/brave/brave-browser/releases/tag/v";
+    u"https://github.com/yuthstyle88/ibrowe-browser/releases/tag/v";
 constexpr char16_t kGoogleLoginLearnMoreURL[] =
-    u"https://github.com/brave/brave-browser/wiki/"
+    u"https://github.comyuthstyle88/ibrowe-browser/wiki/"
     u"Allow-Google-login---Third-Parties-and-Extensions";
 constexpr char16_t kUnstoppableDomainsLearnMoreURL[] =
-    u"https://github.com/brave/brave-browser/wiki/"
+    u"https://github.com/yuthstyle88/ibrowe-browser/wiki/"
     u"Resolve-Methods-for-Unstoppable-Domains";
 constexpr char16_t kEnsOffchainLookupLearnMoreURL[] =
-    u"https://github.com/brave/brave-browser/wiki/ENS-offchain-lookup";
+    u"https://github.com/yuthstyle88/ibrowe-browser/wiki/ENS-offchain-lookup";
 constexpr char16_t kBraveSyncGuideUrl[] =
-    u"https://support.brave.com/hc/en-us/articles/360047642371-Sync-FAQ";
+    u"https://support.ibrowe.com/hc/en-us/articles/360047642371-Sync-FAQ";
 constexpr char16_t kDeAmpLearnMoreUrl[] =
-    u"https://support.brave.com/hc/en-us/articles/8611298579981";
+    u"https://support.ibrowe.com/hc/en-us/articles/8611298579981";
 constexpr char16_t kDebounceLearnMoreUrl[] =
-    u"https://brave.com/privacy-updates/11-debouncing/";
+    u"https://ibrowe.com/privacy-updates/11-debouncing/";
 constexpr char16_t kEnableNftDiscoveryLearnMoreUrl[] =
-    u"https://github.com/brave/brave-browser/wiki/"
+    u"https://github.com//yuthstyle88/ibrowe-browser/wiki/"
     u"NFT-Discovery";
 constexpr char16_t kBlockAllCookiesLearnMoreUrl[] =
-    u"https://github.com/brave/brave-browser/wiki/"
+    u"https://github.com/yuthstyle88/ibrowe-browser/wiki/"
     u"Block-all-cookies-global-Shields-setting";
 constexpr char16_t kLeoCustomModelsLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"34070140231821-How-do-I-use-the-Bring-Your-Own-Model-BYOM-with-Brave-Leo";
 
 constexpr char16_t kTabOrganizationLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"35200007195917-How-to-use-Tab-Focus-Mode";
 
 constexpr char16_t kLeoPrivacyPolicyURL[] =
-    u"https://brave.com/privacy/browser/#brave-leo";
+    u"https://ibrowe.com/privacy/browser/#ibrowe-leo";
 
 constexpr char16_t kSurveyPanelistLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/36550092449165";
+    u"https://support.ibrowe.com/hc/en-us/articles/36550092449165";
 
 constexpr char16_t kExtensionsV2LearnMoreURL[] =
-    u"https://brave.com/blog/brave-shields-manifest-v3/";
+    u"https://ibrowe.com/blog/ibrowe-shields-manifest-v3/";
 
 void BraveAddCommonStrings(content::WebUIDataSource* html_source,
                            Profile* profile) {
