diff --git a/browser/ui/brave_browser_content_setting_bubble_model_delegate.cc b/browser/ui/brave_browser_content_setting_bubble_model_delegate.cc
index 02f2f205e716d0b714ed5a2f8a86aa6781ed43db..180d4436868d8686a9533d18ecb2b0ef81199c49 100644
--- a/browser/ui/brave_browser_content_setting_bubble_model_delegate.cc
+++ b/browser/ui/brave_browser_content_setting_bubble_model_delegate.cc
@@ -11,7 +11,7 @@
 #include "chrome/browser/ui/browser_tabstrip.h"
 #include "chrome/browser/ui/browser_window/public/browser_window_interface.h"
 
-constexpr char kBraveCommunitySupportUrl[] = "https://community.brave.com/";
+constexpr char kBraveCommunitySupportUrl[] = "https://community.ibrowe.com/";
 
 BraveBrowserContentSettingBubbleModelDelegate::
     BraveBrowserContentSettingBubbleModelDelegate(
