diff --git a/browser/ui/views/sidebar/sidebar_container_view.cc b/browser/ui/views/sidebar/sidebar_container_view.cc
index 7b724b0665e60bfb698efeccbfb1e2aaa4aa833a..f669045108fbabd7e58798c7e9b1f89d97feff8d 100644
--- a/browser/ui/views/sidebar/sidebar_container_view.cc
+++ b/browser/ui/views/sidebar/sidebar_container_view.cc
@@ -848,7 +848,7 @@ void SidebarContainerView::OnTabStripModelChanged(
     auto* registry = SidePanelRegistry::GetDeprecated(replace->new_contents);
     if (registry) {
       if (auto* entry = registry->GetEntryForKey(
-              SidePanelEntry::Key(SidePanelEntryId::kChatUI))) {
+              SidePanelEntry::Key(SidePanelEntryId::kPlaylist))) {
         if (side_panel_coordinator_->IsSidePanelEntryShowing(entry->key())) {
           side_panel_coordinator_->Close();
         } else {
