diff --git a/browser/brave_content_browser_client.cc b/browser/brave_content_browser_client.cc
index 3d71a28d027fa63044926d9de78819fe8aafc142..ebb117f508f870d023cc9c0f0ba7155c004dbbc9 100644
--- a/browser/brave_content_browser_client.cc
+++ b/browser/brave_content_browser_client.cc
@@ -282,12 +282,12 @@ bool HandleURLReverseOverrideRewrite(GURL* url,
   }
 
 // For wallet pages, return true to update the displayed URL to react-routed
-// URL rather than showing brave://wallet for everything. This is needed
-// because of a side effect from rewriting brave:// to chrome:// in
-// HandleURLRewrite handler which makes brave://wallet the virtual URL here
+// URL rather than showing ibrowe://wallet for everything. This is needed
+// because of a side effect from rewriting ibrowe:// to chrome:// in
+// HandleURLRewrite handler which makes ibrowe://wallet the virtual URL here
 // unless we return true to trigger an update of virtual URL here to the routed
-// URL. For example, we will display brave://wallet/send instead of
-// brave://wallet with this. This is Android only because currently both
+// URL. For example, we will display ibrowe://wallet/send instead of
+//ibrowe://wallet with this. This is Android only because currently both
 // virtual and real URLs are chrome:// on desktop, so it doesn't have this
 // issue.
 #if BUILDFLAG(IS_ANDROID)
@@ -313,7 +313,7 @@ bool HandleURLRewrite(GURL* url, content::BrowserContext* browser_context) {
   }
 
 // For wallet pages, return true so we can handle it in the reverse handler.
-// Also update the real URL from brave:// to chrome://.
+// Also update the real URL from ibrowe:// to chrome://.
 #if BUILDFLAG(IS_ANDROID)
   if ((url->SchemeIs(content::kBraveUIScheme) ||
        url->SchemeIs(content::kChromeUIScheme)) &&
@@ -1154,7 +1154,7 @@ bool BraveContentBrowserClient::HandleURLOverrideRewrite(
     return false;
   }
 
-  // brave://sync => brave://settings/braveSync
+  // ibrowe://sync => ibrowe://settings/braveSync
   if (url->host() == chrome::kBraveUISyncHost) {
     GURL::Replacements replacements;
     replacements.SetSchemeStr(content::kChromeUIScheme);
@@ -1165,7 +1165,7 @@ bool BraveContentBrowserClient::HandleURLOverrideRewrite(
   }
 
 #if !BUILDFLAG(IS_ANDROID)
-  // brave://adblock => brave://settings/shields/filters
+  // ibrowe://adblock => ibrowe://settings/shields/filters
   if (url->host() == kAdblockHost) {
     GURL::Replacements replacements;
     replacements.SetSchemeStr(content::kChromeUIScheme);
