diff --git a/browser/ui/views/brave_news/brave_news_bubble_view.cc b/browser/ui/views/brave_news/brave_news_bubble_view.cc
index b9bca2e01416f47033eecd4c441d49bcf17aab1b..8f052070102db1621aa7e8dafb0f7eefa1495480 100644
--- a/browser/ui/views/brave_news/brave_news_bubble_view.cc
+++ b/browser/ui/views/brave_news/brave_news_bubble_view.cc
@@ -149,7 +149,7 @@ BraveNewsBubbleView::~BraveNewsBubbleView() = default;
 void BraveNewsBubbleView::OpenManageFeeds() {
   auto* browser = chrome::FindBrowserWithTab(contents_);
   browser->OpenURL(
-      {GURL("brave://newtab/?openSettings=BraveNews"), content::Referrer(),
+      {GURL("ibrowe://newtab/?openSettings=BraveNews"), content::Referrer(),
        WindowOpenDisposition::NEW_FOREGROUND_TAB, ui::PAGE_TRANSITION_LINK,
        false},
       /*navigation_handle_callback=*/{});
