diff --git a/browser/sessions/brave_session_restore_browsertest.cc b/browser/sessions/brave_session_restore_browsertest.cc
index e843b0684075fd2ac199f97bb8c7be307162ca43..f2efe6b39d178f5c9763721804edd159a1b1a873 100644
--- a/browser/sessions/brave_session_restore_browsertest.cc
+++ b/browser/sessions/brave_session_restore_browsertest.cc
@@ -31,7 +31,7 @@ IN_PROC_BROWSER_TEST_F(BraveSessionRestoreBrowserTest,
   SessionService* const session_service =
       SessionServiceFactory::GetForProfile(browser()->profile());
   ui_test_utils::NavigateToURLBlockUntilNavigationsComplete(
-      browser(), GURL("brave://newtab/"), 1);
+      browser(), GURL("ibrowe://newtab/"), 1);
   ASSERT_EQ(true, EvalJs(web_contents,
                          R"(
         var textarea = document.createElement('textarea')
@@ -74,7 +74,7 @@ IN_PROC_BROWSER_TEST_F(BraveSessionRestoreBrowserTest,
   SessionService* const session_service =
       SessionServiceFactory::GetForProfile(browser()->profile());
   ui_test_utils::NavigateToURLBlockUntilNavigationsComplete(
-      browser(), GURL("brave://rewards/"), 1);
+      browser(), GURL("ibrowe://rewards/"), 1);
   ASSERT_EQ(true, EvalJs(web_contents,
                          R"(
         var textarea = document.createElement('textarea')
