diff --git a/browser/brave_wallet/notifications/wallet_notification_service.cc b/browser/brave_wallet/notifications/wallet_notification_service.cc
index 7b29aeae681ec75f31a5e31c8458db6090f2e5f8..bf4d0abaa77f63c0f39657a8b4bc1602e583379e 100644
--- a/browser/brave_wallet/notifications/wallet_notification_service.cc
+++ b/browser/brave_wallet/notifications/wallet_notification_service.cc
@@ -63,7 +63,7 @@ void PushNotification(content::BrowserContext* context,
                       const std::u16string& body) {
   auto notification = CreateMessageCenterNotification(
       title, body, uuid,
-      GURL("brave://wallet/crypto/accounts/" + from + "#" + uuid));
+      GURL("ibrowe://wallet/crypto/accounts/" + from + "#" + uuid));
   auto* profile = Profile::FromBrowserContext(context);
   NotificationDisplayServiceFactory::GetForProfile(profile)->Display(
       NotificationHandler::Type::SEND_TAB_TO_SELF, *notification, nullptr);
