diff --git a/browser/ui/webui/welcome_page/welcome_dom_handler.cc b/browser/ui/webui/welcome_page/welcome_dom_handler.cc
index 8ec9aeafa06efc1bb7a8ea72c18367539eefce13..0cc9e057e6a723461ea2fa8c240a24be21d3964e 100644
--- a/browser/ui/webui/welcome_page/welcome_dom_handler.cc
+++ b/browser/ui/webui/welcome_page/welcome_dom_handler.cc
@@ -80,7 +80,7 @@ WelcomeDOMHandler::WelcomeDOMHandler(Profile* profile)
       brave_education_server_checker_(*profile->GetPrefs(),
                                       profile->GetURLLoaderFactory()) {
   base::MakeRefCounted<shell_integration::DefaultSchemeClientWorker>(
-      GURL("https://browser-education.brave.com"))
+      GURL("https://browser-education.ibrowe.com"))
       ->StartCheckIsDefaultAndGetDefaultClientName(
           base::BindOnce(&WelcomeDOMHandler::OnGetDefaultBrowser,
                          weak_ptr_factory_.GetWeakPtr()));
