diff --git a/browser/brave_wallet/notifications/wallet_notification_service.cc b/browser/brave_wallet/notifications/wallet_notification_service.cc
index 7135802e2b2d285e4ad2991c28a1b9af88b17a1c..d95db0d4db528d7943df4e5bbcbddb523f9bf68c 100644
--- a/browser/brave_wallet/notifications/wallet_notification_service.cc
+++ b/browser/brave_wallet/notifications/wallet_notification_service.cc
@@ -62,7 +62,7 @@ void PushNotification(content::BrowserContext* context,
                       const std::u16string& body) {
   auto notification = CreateMessageCenterNotification(
       title, body, uuid,
-      GURL("brave://wallet/crypto/accounts/" + from + "#" + uuid));
+      GURL("ibrowe://wallet/crypto/accounts/" + from + "#" + uuid));
   auto* profile = Profile::FromBrowserContext(context);
   NotificationDisplayServiceFactory::GetForProfile(profile)->Display(
       NotificationHandler::Type::SEND_TAB_TO_SELF, *notification, nullptr);
