diff --git a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
index 5cc9b5dbb2177689233f05e8b553bc1808c7332c..931769e87aae1f55060843849ebfc7c5f1360851 100644
--- a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
+++ b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
@@ -53,12 +53,12 @@ namespace {
 constexpr char16_t kBraveAccountSelfCustodyLearnMoreURL[] =
     u"https://search.brave.com";
 constexpr char16_t kBraveAccountTermsOfServiceURL[] =
-    u"https://brave.com/terms-of-use/";
+    u"https://ibrowe.com/terms-of-use/";
 constexpr char16_t kBraveAccountPrivacyAgreementURL[] =
-    u"https://brave.com/privacy/browser/";
+    u"https://ibrowe.com/ibrowe-privacy";
 
 constexpr char16_t kWebRTCLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"360017989132-How-do-I-change-my-Privacy-Settings-#webrtc";
 constexpr char16_t kBraveBuildInstructionsUrl[] =
     u"https://github.com/brave/brave-browser/wiki";
