diff --git a/browser/ui/browser_commands.cc b/browser/ui/browser_commands.cc
index 19d590d60c957d0e845e7f5657842cd36a7afec7..0e216734ad19c253d49a580db5f6a367bc683f8f 100644
--- a/browser/ui/browser_commands.cc
+++ b/browser/ui/browser_commands.cc
@@ -319,11 +319,11 @@ void OpenBraveVPNUrls(Browser* browser, int command_id) {
 }
 
 void ToggleAIChat(Browser* browser) {
-#if defined(TOOLKIT_VIEWS)
-  SidePanelUI* side_panel_ui = browser->GetFeatures().side_panel_ui();
-  side_panel_ui->Toggle(SidePanelEntry::Key(SidePanelEntryId::kChatUI),
-                        SidePanelOpenTrigger::kToolbarButton);
-#endif
+// #if defined(TOOLKIT_VIEWS)
+//   SidePanelUI* side_panel_ui = browser->GetFeatures().side_panel_ui();
+//   side_panel_ui->Toggle(SidePanelEntry::Key(SidePanelEntryId::kChatUI),
+//                         SidePanelOpenTrigger::kToolbarButton);
+// #endif
 }
 
 void ShowWalletBubble(Browser* browser) {
