diff --git a/browser/brave_wallet/ethereum_provider_impl_unittest.cc b/browser/brave_wallet/ethereum_provider_impl_unittest.cc
index 9c5fdd21712ac237ba916845c9f468db52740600..023715b71470b4a446646efeecb64439f34d61e2 100644
--- a/browser/brave_wallet/ethereum_provider_impl_unittest.cc
+++ b/browser/brave_wallet/ethereum_provider_impl_unittest.cc
@@ -2131,7 +2131,7 @@ TEST_F(EthereumProviderImplUnitTest, AccountsChangedEvent) {
   observer_->Reset();
 
   // Does not fire for a different origin that has no permissions
-  Navigate(GURL("https://bravesoftware.com"));
+  Navigate(GURL("https://ibrowesoftware.com"));
   AddEthereumPermission(account_1->account_id);
   SetSelectedAccount(account_0->account_id);
   EXPECT_FALSE(observer_->AccountsChangedFired());
@@ -2533,7 +2533,7 @@ TEST_F(EthereumProviderImplUnitTest, SignMessageHardware) {
 TEST_F(EthereumProviderImplUnitTest, SwitchEthereumChain) {
   // CreateWallet();
   CreateBraveWalletTabHelper();
-  Navigate(GURL("https://bravesoftware.com"));
+  Navigate(GURL("https://ibrowesoftware.com"));
   brave_wallet_tab_helper()->SetSkipDelegateForTesting(true);
   mojom::ProviderError error = mojom::ProviderError::kUnknown;
   std::string error_message;
@@ -2600,7 +2600,7 @@ TEST_F(EthereumProviderImplUnitTest, SwitchEthereumChain) {
 
 TEST_F(EthereumProviderImplUnitTest, AddEthereumChainSwitchesForInnactive) {
   CreateBraveWalletTabHelper();
-  Navigate(GURL("https://bravesoftware.com"));
+  Navigate(GURL("https://ibrowesoftware.com"));
   brave_wallet_tab_helper()->SetSkipDelegateForTesting(true);
 
   // AddEthereumChain switches for already added networks
