diff --git a/browser/ui/views/tabs/brave_tab_context_menu_contents_browsertest.cc b/browser/ui/views/tabs/brave_tab_context_menu_contents_browsertest.cc
index 00fca218cdcc0aacff3ba9c9d538896d06c400dd..d0ba9c9aa5a667bbb8f6580c44c5b8af15ce82f8 100644
--- a/browser/ui/views/tabs/brave_tab_context_menu_contents_browsertest.cc
+++ b/browser/ui/views/tabs/brave_tab_context_menu_contents_browsertest.cc
@@ -88,7 +88,7 @@ IN_PROC_BROWSER_TEST_F(BraveTabContextMenuContentsTest, Basics) {
 
   // When a tab is closed, restore tab menu item is enabled.
   ASSERT_TRUE(
-      ui_test_utils::NavigateToURL(browser(), GURL("brave://version/")));
+      ui_test_utils::NavigateToURL(browser(), GURL("ibrowe://version/")));
   chrome::CloseTab(browser());
   EXPECT_TRUE(menu->IsCommandIdEnabled(BraveTabMenuModel::CommandRestoreTab));
   EXPECT_FALSE(
