diff --git a/browser/ui/webui/brave_settings_ui.cc b/browser/ui/webui/brave_settings_ui.cc
index 30dde7a4c2575056c3b67520fc150191d425d7e9..5e05a2bef45ae366cb971e6197bf44cc9dd2b0d6 100644
--- a/browser/ui/webui/brave_settings_ui.cc
+++ b/browser/ui/webui/brave_settings_ui.cc
@@ -205,10 +205,10 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
                           base::FeatureList::IsEnabled(
                               extensions_mv2::features::kExtensionsManifestV2));
 
-  html_source->AddBoolean("isLeoAssistantAllowed",
-                          ai_chat::IsAIChatEnabled(profile->GetPrefs()));
-  html_source->AddBoolean("isLeoAssistantHistoryAllowed",
-                          ai_chat::features::IsAIChatHistoryEnabled());
+  // html_source->AddBoolean("isLeoAssistantAllowed",
+  //                         ai_chat::IsAIChatEnabled(profile->GetPrefs()));
+  // html_source->AddBoolean("isLeoAssistantHistoryAllowed",
+  //                         ai_chat::features::IsAIChatHistoryEnabled());
 
   html_source->AddBoolean("isSurveyPanelistAllowed",
                           base::FeatureList::IsEnabled(
