diff --git a/browser/brave_profile_prefs.cc b/browser/brave_profile_prefs.cc
index 00b4265518e17741c01ce0601cfe09906d7b2b19..f4b98f4b27eeaf3f5cf25e84f6cdcffc2b70bf0d 100644
--- a/browser/brave_profile_prefs.cc
+++ b/browser/brave_profile_prefs.cc
@@ -296,7 +296,7 @@ void RegisterProfilePrefsForMigration(
 #endif
 
   // Added Feb 2023
-  registry->RegisterBooleanPref(brave_rewards::prefs::kShowButton, true);
+  registry->RegisterBooleanPref(brave_rewards::prefs::kShowButton, false);
 
   brave_rewards::RegisterProfilePrefsForMigration(registry);
 
@@ -415,13 +415,13 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(kNewTabPageShowClock, false);
   registry->RegisterStringPref(kNewTabPageClockFormat, "");
   registry->RegisterBooleanPref(kNewTabPageShowStats, true);
-  registry->RegisterBooleanPref(kNewTabPageShowRewards, true);
-  registry->RegisterBooleanPref(kNewTabPageShowBraveTalk, true);
+  registry->RegisterBooleanPref(kNewTabPageShowRewards, false);
+  registry->RegisterBooleanPref(kNewTabPageShowBraveTalk, false);
   registry->RegisterBooleanPref(kBraveTalkDisabledByPolicy, false);
   registry->RegisterBooleanPref(kNewTabPageHideAllWidgets, false);
 
 #if BUILDFLAG(ENABLE_BRAVE_VPN)
-  registry->RegisterBooleanPref(kNewTabPageShowBraveVPN, true);
+  registry->RegisterBooleanPref(kNewTabPageShowBraveVPN, false);
 #endif
 
 // Private New Tab Page
