diff --git a/browser/ui/webui/brave_wallet/market/market_ui.cc b/browser/ui/webui/brave_wallet/market/market_ui.cc
index a616d9d13793b4b9c84eb39360e8a9432a79aa59..c1bf17501c295d32fc5876293feaf7a72c88dbc3 100644
--- a/browser/ui/webui/brave_wallet/market/market_ui.cc
+++ b/browser/ui/webui/brave_wallet/market/market_ui.cc
@@ -57,7 +57,7 @@ UntrustedMarketUI::UntrustedMarketUI(content::WebUI* web_ui)
       std::string("font-src 'self' data: chrome-untrusted://resources;"));
   untrusted_source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::ImgSrc,
-      std::string("img-src 'self' https://assets.cgproxy.brave.com "
+      std::string("img-src 'self' https://assets.cgproxy.ibrowe.com "
                   "chrome-untrusted://resources chrome-untrusted://image;"));
 
   untrusted_source->AddResourcePath("load_time_data_deprecated.js",
