diff --git a/browser/ui/views/toolbar/brave_toolbar_view.cc b/browser/ui/views/toolbar/brave_toolbar_view.cc
index 9c898fac32696d14620c97a8d554ad63ea21d938..4547349dffa3fd034b0467df7ab60de9826b6505 100644
--- a/browser/ui/views/toolbar/brave_toolbar_view.cc
+++ b/browser/ui/views/toolbar/brave_toolbar_view.cc
@@ -265,7 +265,7 @@ void BraveToolbarView::Init() {
     UpdateAIChatButtonVisibility();
   }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
+/* #if BUILDFLAG(ENABLE_BRAVE_VPN)
   if (brave_vpn::BraveVpnServiceFactory::GetForProfile(profile)) {
     brave_vpn_ = container_view->AddChildViewAt(
         std::make_unique<BraveVPNButton>(browser()),
@@ -280,7 +280,7 @@ void BraveToolbarView::Init() {
                             base::Unretained(this)));
     brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
   }
-#endif
+#endif */
 
   // Make sure that avatar button should be located right before the app menu.
   if (auto* avatar = GetAvatarToolbarButton()) {
@@ -292,7 +292,7 @@ void BraveToolbarView::Init() {
   UpdateHorizontalPadding();
 }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
+/*#if BUILDFLAG(ENABLE_BRAVE_VPN)
 bool BraveToolbarView::IsBraveVPNButtonVisible() const {
   return show_brave_vpn_button_.GetValue() &&
          !hide_brave_vpn_button_by_policy_.GetValue();
@@ -301,7 +301,7 @@ void BraveToolbarView::OnVPNButtonVisibilityChanged() {
   DCHECK(brave_vpn_);
   brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
 }
-#endif
+#endif */
 
 void BraveToolbarView::OnEditBookmarksEnabledChanged() {
   DCHECK_EQ(DisplayMode::NORMAL, display_mode_);
