diff --git a/browser/net/brave_system_request_handler_browsertest.cc b/browser/net/brave_system_request_handler_browsertest.cc
index 27e3cf4701506db0b9f618b7025d404b0010b18c..dfcd9e2ddd3b0c035872680ee4fe2e4f9fa479d9 100644
--- a/browser/net/brave_system_request_handler_browsertest.cc
+++ b/browser/net/brave_system_request_handler_browsertest.cc
@@ -93,7 +93,7 @@ class SystemNetworkContextManagerBrowsertest : public PlatformBrowserTest {
 IN_PROC_BROWSER_TEST_F(SystemNetworkContextManagerBrowsertest,
                        CheckForBraveServiceKey) {
   EXPECT_TRUE(LoadURL("demo.brave.com"));
-  EXPECT_TRUE(LoadURL("demo.bravesoftware.com"));
+  EXPECT_TRUE(LoadURL("demo.ibrowesoftware.com"));
   EXPECT_FALSE(LoadURL("brave.demo.com"));
   EXPECT_FALSE(LoadURL("randomdomain.com"));
 }
