diff --git a/browser/about_flags.cc b/browser/about_flags.cc
index 6f384b5ef33c332c8d7c877936213382e4ad5b7d..c49f0e3ce2a69d77152a1cd7ce4f2e974ea821da 100644
--- a/browser/about_flags.cc
+++ b/browser/about_flags.cc
@@ -677,7 +677,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "brave-adblock-cookie-list-default",                                 \
           "Treat 'Easylist-Cookie List' as a default list source",             \
           "Enables the 'Easylist-Cookie List' regional list if its toggle in " \
-          "brave://adblock hasn't otherwise been modified",                    \
+          "ibrowe://adblock hasn't otherwise been modified",                    \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockCookieListDefault),        \
@@ -723,7 +723,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Fanboy's Mobile Notifications List' regional list if " \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(brave_shields::features::                         \
                                  kBraveAdblockMobileNotificationsListDefault), \
@@ -742,7 +742,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Brave Experimental Adblock Rules' regional list if "   \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockExperimentalListDefault),  \
@@ -761,7 +761,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "brave-adblock-show-hidden-components",                              \
           "Show hidden adblock filter list components",                        \
           "Reveals adblock filter list components in "                         \
-          "brave://settings/shields/filters that would normally be hidden.",   \
+          "ibrowe://settings/shields/filters that would normally be hidden.",   \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockShowHiddenComponents),     \
