diff --git a/browser/ui/sidebar/sidebar_controller.cc b/browser/ui/sidebar/sidebar_controller.cc
index 1478620edbe73bcff1ba6f5734e0e3e5cfd59391..7e763ee847dd805f7c5070a6a3d6db01c29254a1 100644
--- a/browser/ui/sidebar/sidebar_controller.cc
+++ b/browser/ui/sidebar/sidebar_controller.cc
@@ -107,11 +107,11 @@ void SidebarController::ActivateItemAt(std::optional<size_t> index,
   if (item.open_in_panel) {
     sidebar_model_->SetActiveIndex(index);
 
-    if (sidebar::features::kOpenOneShotLeoPanel.Get() &&
-        item.built_in_item_type == SidebarItem::BuiltInItemType::kChatUI) {
-      // Prevent one-time Leo panel open.
-      profile_->GetPrefs()->SetBoolean(kLeoPanelOneShotOpen, true);
-    }
+    // if (sidebar::features::kOpenOneShotLeoPanel.Get() &&
+    //     item.built_in_item_type == SidebarItem::BuiltInItemType::kChatUI) {
+    //   // Prevent one-time Leo panel open.
+    //   profile_->GetPrefs()->SetBoolean(kLeoPanelOneShotOpen, true);
+    // }
     return;
   }
 
