diff --git a/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc b/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc
index 505df5a78a74a3941a0c2f05b7e42c8e9fa37551..36ea6f2707af646f0c2e16fd46b15c5a2a563320 100644
--- a/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc
+++ b/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc
@@ -38,15 +38,15 @@
 namespace {
 constexpr uint32_t kDesiredFaviconSizePixels = 32;
 constexpr char kURLRefreshPremiumSession[] =
-    "https://account.brave.com/?intent=recover&product=leo";
+    "https://account.ibrowe.com/?intent=recover&product=leo";
 constexpr char kURLLearnMoreAboutStorage[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "32663367857549-How-do-I-use-Chat-History-in-Brave-Leo";
 
 #if !BUILDFLAG(IS_ANDROID)
 constexpr char kURLGoPremium[] =
-    "https://account.brave.com/account/?intent=checkout&product=leo";
-constexpr char kURLManagePremium[] = "https://account.brave.com/";
+    "https://account.ibrowe.com/account/?intent=checkout&product=leo";
+constexpr char kURLManagePremium[] = "https://account.ibrowe.com/";
 #endif
 }  // namespace
 
@@ -98,7 +98,7 @@ void AIChatUIPageHandler::OpenAIChatSettings() {
       (active_chat_tab_helper_) ? active_chat_tab_helper_->web_contents()
                                 : owner_web_contents_.get();
 #if !BUILDFLAG(IS_ANDROID)
-  const GURL url("brave://settings/leo-ai");
+  const GURL url("ibrowe://settings/leo-ai");
   if (auto* browser = chrome::FindBrowserWithTab(contents_to_navigate)) {
     ShowSingletonTab(browser, url);
   } else {
