diff --git a/browser/about_flags.cc b/browser/about_flags.cc
index c78f9fe8cad6cf5bd236ff3a86eaeded3f951470..7f59719e9d3b5ed111d4823e866b5f08827e5d99 100644
--- a/browser/about_flags.cc
+++ b/browser/about_flags.cc
@@ -584,7 +584,7 @@ const flags_ui::FeatureEntry::FeatureVariation kZCashFeatureVariations[] = {
           "brave-adblock-cookie-list-default",                                 \
           "Treat 'Easylist-Cookie List' as a default list source",             \
           "Enables the 'Easylist-Cookie List' regional list if its toggle in " \
-          "brave://adblock hasn't otherwise been modified",                    \
+          "ibrowe://adblock hasn't otherwise been modified",                    \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockCookieListDefault),        \
@@ -630,7 +630,7 @@ const flags_ui::FeatureEntry::FeatureVariation kZCashFeatureVariations[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Fanboy's Mobile Notifications List' regional list if " \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(brave_shields::features::                         \
                                  kBraveAdblockMobileNotificationsListDefault), \
@@ -649,7 +649,7 @@ const flags_ui::FeatureEntry::FeatureVariation kZCashFeatureVariations[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Brave Experimental Adblock Rules' regional list if "   \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockExperimentalListDefault),  \
