diff --git a/browser/ui/toolbar/brave_app_menu_model.cc b/browser/ui/toolbar/brave_app_menu_model.cc
index 67a336f13005bbf1f90508c03b2a31658f3ca373..68395c17bb0639702d5f1600c6cfc1fde6f92b9c 100644
--- a/browser/ui/toolbar/brave_app_menu_model.cc
+++ b/browser/ui/toolbar/brave_app_menu_model.cc
@@ -158,7 +158,7 @@ void BraveAppMenuModel::BuildBraveProductsSection() {
     need_separator = true;
   }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
+/* #if BUILDFLAG(ENABLE_BRAVE_VPN)
   if (IsCommandIdEnabled(IDC_BRAVE_VPN_MENU)) {
     sub_menus().push_back(std::make_unique<BraveVPNMenuModel>(
         browser(), browser()->profile()->GetPrefs()));
@@ -171,7 +171,7 @@ void BraveAppMenuModel::BuildBraveProductsSection() {
                              IDC_SHOW_BRAVE_VPN_PANEL, IDS_BRAVE_VPN_MENU);
     need_separator = true;
   }
-#endif
+#endif */
 
 #if defined(TOOLKIT_VIEWS)
   if (sidebar::CanUseSidebar(browser())) {
