diff --git a/browser/ui/brave_browser_browsertest.cc b/browser/ui/brave_browser_browsertest.cc
index b53e15514a926346a543557f905bdfe962ef13cb..ac2499daf8f215befc51650d9e7b94f4179e9979 100644
--- a/browser/ui/brave_browser_browsertest.cc
+++ b/browser/ui/brave_browser_browsertest.cc
@@ -41,7 +41,7 @@ Browser* OpenNewBrowser(Profile* profile) {
 }  // namespace
 
 IN_PROC_BROWSER_TEST_F(BraveBrowserBrowserTest, NTPFaviconTest) {
-  ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL("brave://newtab/")));
+  ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL("ibrowe://newtab/")));
 
   auto* tab_model = browser()->tab_strip_model();
   EXPECT_FALSE(
