diff --git a/browser/about_flags.cc b/browser/about_flags.cc
index 8a4df860cd2a11fb604471db87d1fca0ae4d9dca..c321cbcf4d5c6818ce3c6ca600bd14eee0237bda 100644
--- a/browser/about_flags.cc
+++ b/browser/about_flags.cc
@@ -715,7 +715,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "brave-adblock-cookie-list-default",                                 \
           "Treat 'Easylist-Cookie List' as a default list source",             \
           "Enables the 'Easylist-Cookie List' regional list if its toggle in " \
-          "brave://adblock hasn't otherwise been modified",                    \
+          "ibrowe://adblock hasn't otherwise been modified",                    \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockCookieListDefault),        \
@@ -751,7 +751,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Fanboy's Mobile Notifications List' regional list if " \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(brave_shields::features::                         \
                                  kBraveAdblockMobileNotificationsListDefault), \
@@ -770,7 +770,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "source",                                                            \
                                                                                \
           "Enables the 'Brave Experimental Adblock Rules' regional list if "   \
-          "its toggle in brave://adblock hasn't otherwise been modified",      \
+          "its toggle in ibrowe://adblock hasn't otherwise been modified",      \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockExperimentalListDefault),  \
@@ -789,7 +789,7 @@ constexpr flags_ui::FeatureEntry::Choice kVerticalTabCollapseDelayChoices[] = {
           "brave-adblock-show-hidden-components",                              \
           "Show hidden adblock filter list components",                        \
           "Reveals adblock filter list components in "                         \
-          "brave://settings/shields/filters that would normally be hidden.",   \
+          "ibrowe://settings/shields/filters that would normally be hidden.",   \
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(                                                  \
               brave_shields::features::kBraveAdblockShowHiddenComponents),     \
