diff --git a/browser/resources/settings/brave_overrides/about_page.ts b/browser/resources/settings/brave_overrides/about_page.ts
index 19fd5f28fc727640bd4b6a96c26c03aa8373d5d0..0e109c48ce28ae1cd1616c7f1101d0324c7f21b6 100644
--- a/browser/resources/settings/brave_overrides/about_page.ts
+++ b/browser/resources/settings/brave_overrides/about_page.ts
@@ -45,7 +45,7 @@ RegisterPolymerTemplateModifications({
       wrapper.setAttribute('id', 'release-notes')
       wrapper.setAttribute('target', '_blank')
       wrapper.setAttribute('rel', 'noopener noreferrer')
-      wrapper.setAttribute('href', 'https://brave.com/latest/')
+      wrapper.setAttribute('href', 'https://ibrowe.com/latest/')
 
       const parent = version.parentNode
       parent?.replaceChild(wrapper, version)
@@ -58,12 +58,12 @@ RegisterPolymerTemplateModifications({
     if (updateStatusMessageLink) {
       // <if expr="is_win">
       updateStatusMessageLink.href =
-        'https://support.brave.com/hc/en-us/articles/360042816611-Why-isn-t-Brave-updating-automatically-on-Windows-'
+        'https://support.ibrowe.com/hc/en-us/articles/360042816611-Why-isn-t-Brave-updating-automatically-on-Windows-'
       // </if>
 
       // <if expr="not is_win">
         updateStatusMessageLink.href =
-          'https://community.brave.com?p=update_error'
+          'https://community.ibrowe.com?p=update_error'
       // </if>
     }
   }
