diff --git a/browser/ui/views/playlist/playlist_action_dialogs.cc b/browser/ui/views/playlist/playlist_action_dialogs.cc
index c5a72a8aeae212f833cca84c014df1049ba043eb..341b02af3a39ba4b363d4d5f6b0bc6e7bd0caa45 100644
--- a/browser/ui/views/playlist/playlist_action_dialogs.cc
+++ b/browser/ui/views/playlist/playlist_action_dialogs.cc
@@ -225,7 +225,7 @@ void ShowPlaylistSettings(content::WebContents* contents) {
   auto* browser_view = FindBrowserViewFromWebContents(contents);
   CHECK(browser_view);
   ShowSingletonTab(browser_view->browser(),
-                   GURL("brave://settings/braveContent#playlist-section"));
+                   GURL("ibrowe://settings/braveContent#playlist-section"));
 }
 
 void ShowPlaylistAddBubble(content::WebContents* contents) {
