diff --git a/browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc b/browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc
index 19b0d14b597a6e4273db105567131ef893d9a2b6..5592be7084586721b12f652271ccf01e496c10a5 100644
--- a/browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc
+++ b/browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc
@@ -80,16 +80,16 @@ base::Value::Dict GetPreferencesDictionary(PrefService* prefs) {
   pref_data.Set("showStats", prefs->GetBoolean(kNewTabPageShowStats));
   pref_data.Set("showToday",
                 prefs->GetBoolean(brave_news::prefs::kNewTabPageShowToday));
-  pref_data.Set("showRewards", prefs->GetBoolean(kNewTabPageShowRewards));
+//  pref_data.Set("showRewards", prefs->GetBoolean(kNewTabPageShowRewards));
   pref_data.Set("isBrandedWallpaperNotificationDismissed",
                 prefs->GetBoolean(kBrandedWallpaperNotificationDismissed));
   pref_data.Set("isBraveNewsOptedIn",
                 prefs->GetBoolean(brave_news::prefs::kBraveNewsOptedIn));
   pref_data.Set("hideAllWidgets", prefs->GetBoolean(kNewTabPageHideAllWidgets));
-  pref_data.Set("showBraveTalk", prefs->GetBoolean(kNewTabPageShowBraveTalk));
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  pref_data.Set("showBraveVPN", prefs->GetBoolean(kNewTabPageShowBraveVPN));
-#endif
+//  pref_data.Set("showBraveTalk", prefs->GetBoolean(kNewTabPageShowBraveTalk));
+//#if BUILDFLAG(ENABLE_BRAVE_VPN)
+//  pref_data.Set("showBraveVPN", prefs->GetBoolean(kNewTabPageShowBraveVPN));
+//#endif
   pref_data.Set(
       "showSearchBox",
       prefs->GetBoolean(brave_search_conversion::prefs::kShowNTPSearchBox));
@@ -299,16 +299,16 @@ void BraveNewTabMessageHandler::OnJavascriptAllowed() {
       kBrandedWallpaperNotificationDismissed,
       base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
                           base::Unretained(this)));
-  pref_change_registrar_.Add(
-      kNewTabPageShowBraveTalk,
-      base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
-                          base::Unretained(this)));
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  pref_change_registrar_.Add(
-      kNewTabPageShowBraveVPN,
-      base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
-                          base::Unretained(this)));
-#endif
+//  pref_change_registrar_.Add(
+//      kNewTabPageShowBraveTalk,
+//      base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
+//                          base::Unretained(this)));
+//#if BUILDFLAG(ENABLE_BRAVE_VPN)
+//  pref_change_registrar_.Add(
+//      kNewTabPageShowBraveVPN,
+//      base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
+//                          base::Unretained(this)));
+//#endif
   pref_change_registrar_.Add(
       kNewTabPageHideAllWidgets,
       base::BindRepeating(&BraveNewTabMessageHandler::OnPreferencesChanged,
@@ -407,18 +407,18 @@ void BraveNewTabMessageHandler::HandleSaveNewTabPagePref(
     settings_key = brave_news::prefs::kNewTabPageShowToday;
   } else if (settings_key_input == "isBraveNewsOptedIn") {
     settings_key = brave_news::prefs::kBraveNewsOptedIn;
-  } else if (settings_key_input == "showRewards") {
-    settings_key = kNewTabPageShowRewards;
+//  } else if (settings_key_input == "showRewards") {
+//    settings_key = kNewTabPageShowRewards;
   } else if (settings_key_input == "isBrandedWallpaperNotificationDismissed") {
     settings_key = kBrandedWallpaperNotificationDismissed;
   } else if (settings_key_input == "hideAllWidgets") {
     settings_key = kNewTabPageHideAllWidgets;
-  } else if (settings_key_input == "showBraveTalk") {
-    settings_key = kNewTabPageShowBraveTalk;
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  } else if (settings_key_input == "showBraveVPN") {
-    settings_key = kNewTabPageShowBraveVPN;
-#endif
+//  } else if (settings_key_input == "showBraveTalk") {
+//    settings_key = kNewTabPageShowBraveTalk;
+//#if BUILDFLAG(ENABLE_BRAVE_VPN)
+//  } else if (settings_key_input == "showBraveVPN") {
+//    settings_key = kNewTabPageShowBraveVPN;
+//#endif
   } else if (settings_key_input == "showSearchBox") {
     settings_key = brave_search_conversion::prefs::kShowNTPSearchBox;
   } else if (settings_key_input == "promptEnableSearchSuggestions") {
