diff --git a/browser/ui/views/toolbar/brave_toolbar_view.cc b/browser/ui/views/toolbar/brave_toolbar_view.cc
index 4d6c31ef0c5ae0c3aea731c3999c2d2753a7d1cb..0aa3cded5aac68593b6079075fc51146abddae82 100644
--- a/browser/ui/views/toolbar/brave_toolbar_view.cc
+++ b/browser/ui/views/toolbar/brave_toolbar_view.cc
@@ -264,38 +264,38 @@ void BraveToolbarView::Init() {
 
   // Don't check policy status since we're going to
   // setup a watcher for policy pref.
-  if (ai_chat::IsAllowedForContext(browser_->profile(), false)) {
-    ai_chat_button_ = container_view->AddChildViewAt(
-        std::make_unique<AIChatButton>(browser()),
-        *container_view->GetIndexOf(GetAppMenuButton()) - 1);
-    show_ai_chat_button_.Init(
-        ai_chat::prefs::kBraveAIChatShowToolbarButton,
-        browser_->profile()->GetPrefs(),
-        base::BindRepeating(&BraveToolbarView::UpdateAIChatButtonVisibility,
-                            base::Unretained(this)));
-    hide_ai_chat_button_by_policy_.Init(
-        ai_chat::prefs::kEnabledByPolicy, profile->GetPrefs(),
-        base::BindRepeating(&BraveToolbarView::UpdateAIChatButtonVisibility,
-                            base::Unretained(this)));
-    UpdateAIChatButtonVisibility();
-  }
-
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-  if (brave_vpn::BraveVpnServiceFactory::GetForProfile(profile)) {
-    brave_vpn_ = container_view->AddChildViewAt(
-        std::make_unique<BraveVPNButton>(browser()),
-        *container_view->GetIndexOf(GetAppMenuButton()) - 1);
-    show_brave_vpn_button_.Init(
-        brave_vpn::prefs::kBraveVPNShowButton, profile->GetPrefs(),
-        base::BindRepeating(&BraveToolbarView::OnVPNButtonVisibilityChanged,
-                            base::Unretained(this)));
-    hide_brave_vpn_button_by_policy_.Init(
-        brave_vpn::prefs::kManagedBraveVPNDisabled, profile->GetPrefs(),
-        base::BindRepeating(&BraveToolbarView::OnVPNButtonVisibilityChanged,
-                            base::Unretained(this)));
-    brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
-  }
-#endif
+  // if (ai_chat::IsAllowedForContext(browser_->profile(), false)) {
+  //   ai_chat_button_ = container_view->AddChildViewAt(
+  //       std::make_unique<AIChatButton>(browser()),
+  //       *container_view->GetIndexOf(GetAppMenuButton()) - 1);
+  //   show_ai_chat_button_.Init(
+  //       ai_chat::prefs::kBraveAIChatShowToolbarButton,
+  //       browser_->profile()->GetPrefs(),
+  //       base::BindRepeating(&BraveToolbarView::UpdateAIChatButtonVisibility,
+  //                           base::Unretained(this)));
+  //   hide_ai_chat_button_by_policy_.Init(
+  //       ai_chat::prefs::kEnabledByPolicy, profile->GetPrefs(),
+  //       base::BindRepeating(&BraveToolbarView::UpdateAIChatButtonVisibility,
+  //                           base::Unretained(this)));
+  //   UpdateAIChatButtonVisibility();
+  // }
+
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+//   if (brave_vpn::BraveVpnServiceFactory::GetForProfile(profile)) {
+//     brave_vpn_ = container_view->AddChildViewAt(
+//         std::make_unique<BraveVPNButton>(browser()),
+//         *container_view->GetIndexOf(GetAppMenuButton()) - 1);
+//     show_brave_vpn_button_.Init(
+//         brave_vpn::prefs::kBraveVPNShowButton, profile->GetPrefs(),
+//         base::BindRepeating(&BraveToolbarView::OnVPNButtonVisibilityChanged,
+//                             base::Unretained(this)));
+//     hide_brave_vpn_button_by_policy_.Init(
+//         brave_vpn::prefs::kManagedBraveVPNDisabled, profile->GetPrefs(),
+//         base::BindRepeating(&BraveToolbarView::OnVPNButtonVisibilityChanged,
+//                             base::Unretained(this)));
+//     brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
+//   }
+// #endif
 
   // Make sure that avatar button should be located right before the app menu.
   if (auto* avatar = GetAvatarToolbarButton()) {
@@ -307,16 +307,16 @@ void BraveToolbarView::Init() {
   UpdateHorizontalPadding();
 }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN)
-bool BraveToolbarView::IsBraveVPNButtonVisible() const {
-  return show_brave_vpn_button_.GetValue() &&
-         !hide_brave_vpn_button_by_policy_.GetValue();
-}
-void BraveToolbarView::OnVPNButtonVisibilityChanged() {
-  DCHECK(brave_vpn_);
-  brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
-}
-#endif
+// #if BUILDFLAG(ENABLE_BRAVE_VPN)
+// bool BraveToolbarView::IsBraveVPNButtonVisible() const {
+//   return show_brave_vpn_button_.GetValue() &&
+//          !hide_brave_vpn_button_by_policy_.GetValue();
+// }
+// void BraveToolbarView::OnVPNButtonVisibilityChanged() {
+//   DCHECK(brave_vpn_);
+//   brave_vpn_->SetVisible(IsBraveVPNButtonVisible());
+// }
+// #endif
 
 void BraveToolbarView::OnEditBookmarksEnabledChanged() {
   DCHECK_EQ(DisplayMode::NORMAL, display_mode_);
@@ -509,11 +509,11 @@ void BraveToolbarView::ResetBookmarkButtonBounds() {
   }
 }
 
-void BraveToolbarView::UpdateAIChatButtonVisibility() {
-  bool should_show = ai_chat::IsAllowedForContext(browser()->profile()) &&
-                     show_ai_chat_button_.GetValue();
-  ai_chat_button_->SetVisible(should_show);
-}
+// void BraveToolbarView::UpdateAIChatButtonVisibility() {
+//   bool should_show = ai_chat::IsAllowedForContext(browser()->profile()) &&
+//                      show_ai_chat_button_.GetValue();
+//   ai_chat_button_->SetVisible(should_show);
+// }
 
 void BraveToolbarView::UpdateWalletButtonVisibility() {
   Profile* profile = browser()->profile();
