diff --git a/browser/net/brave_network_audit_rewards_browsertest.cc b/browser/net/brave_network_audit_rewards_browsertest.cc
index ba6790aba6671743df5bdc278f3430c7bf75cfb9..285108e2e4ec4cc6e9880503e3f1b6afa0fdeba8 100644
--- a/browser/net/brave_network_audit_rewards_browsertest.cc
+++ b/browser/net/brave_network_audit_rewards_browsertest.cc
@@ -100,18 +100,18 @@ class BraveRewardsNetworkAuditTest : public InProcessBrowserTest {
             "https://grant.rewards.brave.com/v1/promotions",
 
             // Brave Rewards staging & dev.
-            "https://api.rewards.bravesoftware.com/v1/cards",
-            "https://api.rewards.bravesoftware.com/v1/parameters",
-            "https://rewards-stg.bravesoftware.com/publishers/prefix-list",
-            "https://grant.rewards.bravesoftware.com/v1/promotions",
+            "https://api.rewards.ibrowesoftware.com/v1/cards",
+            "https://api.rewards.ibrowesoftware.com/v1/parameters",
+            "https://rewards-stg.ibrowesoftware.com/publishers/prefix-list",
+            "https://grant.rewards.ibrowesoftware.com/v1/promotions",
 
             // Brave Ads production.
             "https://geo.ads.brave.com/v1/getstate",
             "https://static.ads.brave.com/v9/catalog",
 
             // Brave Ads staging.
-            "https://geo.ads.bravesoftware.com/v1/getstate",
-            "https://static.ads.bravesoftware.com/v9/catalog",
+            "https://geo.ads.ibrowesoftware.com/v1/getstate",
+            "https://static.ads.ibrowesoftware.com/v9/catalog",
         });
   }
 
