diff --git a/browser/ui/webui/settings/brave_extensions_manifest_v2_browsertest.cc b/browser/ui/webui/settings/brave_extensions_manifest_v2_browsertest.cc
index ca17f719079cfddbaff80e11d3d45a5651a328a0..53d8914854b2e111cf5d09c7ed3c76239d67cf07 100644
--- a/browser/ui/webui/settings/brave_extensions_manifest_v2_browsertest.cc
+++ b/browser/ui/webui/settings/brave_extensions_manifest_v2_browsertest.cc
@@ -119,7 +119,7 @@ class BraveExtensionsManifestV2BrowserTest : public InProcessBrowserTest {
 
 IN_PROC_BROWSER_TEST_F(BraveExtensionsManifestV2BrowserTest, InstallFail) {
   ASSERT_TRUE(ui_test_utils::NavigateToURL(
-      browser(), GURL("brave://settings/extensions/v2")));
+      browser(), GURL("ibrowe://settings/extensions/v2")));
   auto* web_contents = browser()->tab_strip_model()->GetActiveWebContents();
   content::WebContentsConsoleObserver console_observer(web_contents);
   console_observer.SetPattern("Could not fetch data from the Chrome Web Store");
@@ -135,7 +135,7 @@ IN_PROC_BROWSER_TEST_F(BraveExtensionsManifestV2BrowserTest, InstallFail) {
 IN_PROC_BROWSER_TEST_F(BraveExtensionsManifestV2BrowserTest,
                        InstallDisableEnableUninstall) {
   ASSERT_TRUE(ui_test_utils::NavigateToURL(
-      browser(), GURL("brave://settings/extensions/v2")));
+      browser(), GURL("ibrowe://settings/extensions/v2")));
   auto* web_contents = browser()->tab_strip_model()->GetActiveWebContents();
   EXPECT_FALSE(IsExtensionToggled(web_contents));
   EXPECT_TRUE(IsExtensionToggleEnabled(web_contents));
