diff --git a/browser/brave_ads/ads_service_delegate.cc b/browser/brave_ads/ads_service_delegate.cc
index 23d1ec6d58600daf3048fdd58c4ca8c69ea1faa9..fc11dffdde9a964d92ae6da1f55197dfcf9011eb 100644
--- a/browser/brave_ads/ads_service_delegate.cc
+++ b/browser/brave_ads/ads_service_delegate.cc
@@ -52,7 +52,7 @@ constexpr char kSkuOrderCreatedAtKey[] = "created_at";
 constexpr char kSkuOrderExpiresAtKey[] = "expires_at";
 constexpr char kSkuOrderLastPaidAtKey[] = "last_paid_at";
 constexpr char kSkuOrderStatusKey[] = "status";
-constexpr char kNotificationAdUrlPrefix[] = "https://www.brave.com/ads/?";
+constexpr char kNotificationAdUrlPrefix[] = "https://www.ibrowe.com/ads/?";
 
 std::string StripSkuEnvironmentPrefix(const std::string& environment) {
   const size_t pos = environment.find(':');
