diff --git a/browser/brave_wallet/notifications/wallet_notification_service.cc b/browser/brave_wallet/notifications/wallet_notification_service.cc
index 1c96ee4522cea0a7065cc651e026123092ba92c7..091da39f21dce6b2efa611899129215dfcd00257 100644
--- a/browser/brave_wallet/notifications/wallet_notification_service.cc
+++ b/browser/brave_wallet/notifications/wallet_notification_service.cc
@@ -62,7 +62,7 @@ void PushNotification(content::BrowserContext* context,
                       const std::u16string& body) {
   auto notification = CreateMessageCenterNotification(
       title, body, uuid,
-      GURL("brave://wallet/crypto/accounts/" + from + "#" + uuid));
+      GURL("ibrowe://wallet/crypto/accounts/" + from + "#" + uuid));
   auto* profile = Profile::FromBrowserContext(context);
   NotificationDisplayServiceFactory::GetForProfile(profile)->Display(
       NotificationHandler::Type::SEND_TAB_TO_SELF, *notification, nullptr);
