diff --git a/browser/brave_wallet/solana_provider_renderer_browsertest.cc b/browser/brave_wallet/solana_provider_renderer_browsertest.cc
index fadc5fbe17a612e999d42726b0060e3e8720e96c..de0bca06eb6abffd3d00c2318e474fa3cad25772 100644
--- a/browser/brave_wallet/solana_provider_renderer_browsertest.cc
+++ b/browser/brave_wallet/solana_provider_renderer_browsertest.cc
@@ -561,7 +561,7 @@ class SolanaProviderRendererTest : public InProcessBrowserTest {
     // This is intentional to trigger
     // TestBraveContentBrowserClient::RegisterBrowserInterfaceBindersForFrame
     ASSERT_TRUE(
-        ui_test_utils::NavigateToURL(browser(), GURL("brave://settings")));
+        ui_test_utils::NavigateToURL(browser(), GURL("ibrowe://settings")));
 
     GURL url = embedded_test_server()->GetURL("/empty.html");
     ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), url));
