diff --git a/browser/ui/brave_pages.cc b/browser/ui/brave_pages.cc
index c8d2e523ae9d6b8b6417067f89fbb9a82cd42e2a..d4253f5313dc1b6c3e29091d0cee902927791955 100644
--- a/browser/ui/brave_pages.cc
+++ b/browser/ui/brave_pages.cc
@@ -35,7 +35,7 @@ void ShowSync(Browser* browser) {
 
 void ShowBraveNewsConfigure(Browser* browser) {
   ShowSingletonTabOverwritingNTP(
-      browser, GURL("brave://newtab/?openSettings=BraveNews"));
+      browser, GURL("ibrowe://newtab/?openSettings=BraveNews"));
 }
 
 void ShowShortcutsPage(Browser* browser) {
