diff --git a/browser/resources/settings/brave_overrides/page_visibility.ts b/browser/resources/settings/brave_overrides/page_visibility.ts
index 09a34c7d73f62f4bb6946296fe7ae8a5e5e4300c..cf56834f93d0dd34e12a912e4e28297615d61391 100644
--- a/browser/resources/settings/brave_overrides/page_visibility.ts
+++ b/browser/resources/settings/brave_overrides/page_visibility.ts
@@ -76,7 +76,7 @@ function getPageVisibility () {
     // custom properties
     braveSync: !loadTimeData.getBoolean('isSyncDisabled'),
     braveWallet: loadTimeData.getBoolean('isBraveWalletAllowed'),
-    leoAssistant: loadTimeData.getBoolean('isLeoAssistantAllowed'),
+    leoAssistant: false,
     surveyPanelist: loadTimeData.getBoolean('isSurveyPanelistAllowed'),
     content: alwaysTrueProxy,
     playlist: loadTimeData.getBoolean('isPlaylistAllowed'),
