diff --git a/browser/ui/views/brave_vpn/brave_vpn_dns_settings_notificiation_dialog_view.cc b/browser/ui/views/brave_vpn/brave_vpn_dns_settings_notificiation_dialog_view.cc
index 743ef563bd6d7f2f43b21ac12b1c4b254c481f4c..a23c69e3098327f6cc9ce70b7c349d3001a17e0c 100644
--- a/browser/ui/views/brave_vpn/brave_vpn_dns_settings_notificiation_dialog_view.cc
+++ b/browser/ui/views/brave_vpn/brave_vpn_dns_settings_notificiation_dialog_view.cc
@@ -27,7 +27,7 @@ namespace brave_vpn {
 namespace {
 
 constexpr char kBraveVPNLearnMoreURL[] =
-    "https://support.brave.com/hc/en-us/articles/10864482160141";
+    "https://support.ibrowe.com/hc/en-us/articles/10864482160141";
 
 constexpr int kChildSpacing = 16;
 constexpr int kPadding = 24;
