diff --git a/browser/ui/views/playlist/playlist_action_dialogs.cc b/browser/ui/views/playlist/playlist_action_dialogs.cc
index c02e0524e9cdb2ba8a553168bd6b37a9f1b72046..a6d63d1ef087ce94488e3ed9807896a2f8e803c8 100644
--- a/browser/ui/views/playlist/playlist_action_dialogs.cc
+++ b/browser/ui/views/playlist/playlist_action_dialogs.cc
@@ -231,7 +231,7 @@ void ShowPlaylistSettings(content::WebContents* contents) {
   auto* browser_view = FindBrowserViewFromWebContents(contents);
   CHECK(browser_view);
   ShowSingletonTab(browser_view->browser(),
-                   GURL("brave://settings/braveContent#playlist-section"));
+                   GURL("ibrowe://settings/braveContent#playlist-section"));
 }
 
 void ShowPlaylistAddBubble(content::WebContents* contents) {
