diff --git a/browser/ui/views/brave_news/brave_news_bubble_view.cc b/browser/ui/views/brave_news/brave_news_bubble_view.cc
index 89f757f6edfc6da91d6df16bb2d428a9ae5ebc68..ae74151525094c4d0caea3176566547b1776241b 100644
--- a/browser/ui/views/brave_news/brave_news_bubble_view.cc
+++ b/browser/ui/views/brave_news/brave_news_bubble_view.cc
@@ -148,7 +148,7 @@ BraveNewsBubbleView::~BraveNewsBubbleView() = default;
 void BraveNewsBubbleView::OpenManageFeeds() {
   auto* browser = chrome::FindBrowserWithTab(contents_);
   browser->OpenURL(
-      {GURL("brave://newtab/?openSettings=BraveNews"), content::Referrer(),
+      {GURL("ibrowe://newtab/?openSettings=BraveNews"), content::Referrer(),
        WindowOpenDisposition::NEW_FOREGROUND_TAB, ui::PAGE_TRANSITION_LINK,
        false},
       /*navigation_handle_callback=*/{});
