diff --git a/browser/ui/views/playlist/playlist_action_dialogs.cc b/browser/ui/views/playlist/playlist_action_dialogs.cc
index 805a988232498be081f95b6611373b2600f885d8..9024ae9cae7fc6c5d9acabce3e7a3e188e19d03b 100644
--- a/browser/ui/views/playlist/playlist_action_dialogs.cc
+++ b/browser/ui/views/playlist/playlist_action_dialogs.cc
@@ -232,7 +232,7 @@ void ShowPlaylistSettings(content::WebContents* contents) {
   auto* browser_view = FindBrowserViewFromWebContents(contents);
   CHECK(browser_view);
   ShowSingletonTab(browser_view->browser(),
-                   GURL("brave://settings/braveContent#playlist-section"));
+                   GURL("ibrowe://settings/braveContent#playlist-section"));
 }
 
 void ShowPlaylistAddBubble(content::WebContents* contents) {
