diff --git a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
index 2ecda32f3b717b3bcea7885f60e1e06840d7c903..61777ab4d312003ed2411286df99c454f0c3b024 100644
--- a/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
+++ b/browser/ui/webui/settings/brave_settings_localized_strings_provider.cc
@@ -49,14 +49,14 @@ namespace settings {
 namespace {
 
 constexpr char16_t kBraveAccountSelfCustodyLearnMoreURL[] =
-    u"https://search.brave.com";
+    u"https://search.ibrowe.com";
 constexpr char16_t kBraveAccountTermsOfServiceURL[] =
-    u"https://brave.com/terms-of-use/";
+    u"https://ibrowe.com/terms-of-use/";
 constexpr char16_t kBraveAccountPrivacyAgreementURL[] =
-    u"https://brave.com/privacy/browser/";
+    u"https://ibrowe.com/privacy/browser/";
 
 constexpr char16_t kWebRTCLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"360017989132-How-do-I-change-my-Privacy-Settings-#webrtc";
 constexpr char16_t kBraveBuildInstructionsUrl[] =
     u"https://github.com/brave/brave-browser/wiki";
@@ -72,11 +72,11 @@ constexpr char16_t kUnstoppableDomainsLearnMoreURL[] =
 constexpr char16_t kEnsOffchainLookupLearnMoreURL[] =
     u"https://github.com/brave/brave-browser/wiki/ENS-offchain-lookup";
 constexpr char16_t kBraveSyncGuideUrl[] =
-    u"https://support.brave.com/hc/en-us/articles/360047642371-Sync-FAQ";
+    u"https://support.ibrowe.com/hc/en-us/articles/360047642371-Sync-FAQ";
 constexpr char16_t kDeAmpLearnMoreUrl[] =
-    u"https://support.brave.com/hc/en-us/articles/8611298579981";
+    u"https://support.ibrowe.com/hc/en-us/articles/8611298579981";
 constexpr char16_t kDebounceLearnMoreUrl[] =
-    u"https://brave.com/privacy-updates/11-debouncing/";
+    u"https://ibrowe.com/privacy-updates/11-debouncing/";
 constexpr char16_t kEnableNftDiscoveryLearnMoreUrl[] =
     u"https://github.com/brave/brave-browser/wiki/"
     u"NFT-Discovery";
@@ -84,15 +84,15 @@ constexpr char16_t kBlockAllCookiesLearnMoreUrl[] =
     u"https://github.com/brave/brave-browser/wiki/"
     u"Block-all-cookies-global-Shields-setting";
 constexpr char16_t kLeoCustomModelsLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"34070140231821-How-do-I-use-the-Bring-Your-Own-Model-BYOM-with-Brave-Leo";
 
 constexpr char16_t kTabOrganizationLearnMoreURL[] =
-    u"https://support.brave.com/hc/en-us/articles/"
+    u"https://support.ibrowe.com/hc/en-us/articles/"
     u"35200007195917-How-to-use-Tab-Focus-Mode";
 
 constexpr char16_t kLeoPrivacyPolicyURL[] =
-    u"https://brave.com/privacy/browser/#brave-leo";
+    u"https://ibrowe.com/privacy/browser/#brave-leo";
 
 void BraveAddCommonStrings(content::WebUIDataSource* html_source,
                            Profile* profile) {
