diff --git a/browser/ui/views/omnibox/brave_omnibox_view_views_browsertest.cc b/browser/ui/views/omnibox/brave_omnibox_view_views_browsertest.cc
index 88f99dd59e59b7a7fd9e7039965473f223b34f54..989991f40603ae25cdf879538e93e6a38d237cc9 100644
--- a/browser/ui/views/omnibox/brave_omnibox_view_views_browsertest.cc
+++ b/browser/ui/views/omnibox/brave_omnibox_view_views_browsertest.cc
@@ -168,7 +168,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, PasteAndSearchTest) {
 // Load brave url and check copied url also has brave scheme.
 IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest,
                        CopyInternalURLToClipboardTest) {
-  const std::string test_url("brave://version/");
+  const std::string test_url("ibrowe://version/");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
 
   omnibox_view()->SelectAll(true);
@@ -191,7 +191,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyCleanURLToClipboardTest) {
     { "include": [ "*://*/*"], "params": ["utm_content"] }
   ])");
   const std::string test_url(
-      "https://dev-pages.bravesoftware.com/clean-urls/"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/"
       "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&"
       "d&utm_content=removethis&e=&f=g&=end");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
@@ -203,7 +203,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyCleanURLToClipboardTest) {
   clipboard->ReadAsciiText(ui::ClipboardBuffer::kCopyPaste,
                            /* data_dst = */ nullptr, &text_from_clipboard);
   EXPECT_EQ(text_from_clipboard,
-            "https://dev-pages.bravesoftware.com/clean-urls/"
+            "https://dev-pages.ibrowesoftware.com/clean-urls/"
             "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&d&"
             "e=&f=g&=end");
 }
@@ -213,7 +213,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyURLToClipboardTest) {
     { "include": [ "*://*/*"], "params": ["utm_content"] }
   ])");
   const std::string test_url(
-      "https://dev-pages.bravesoftware.com/clean-urls/"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/"
       "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&"
       "d&utm_content=removethis&e=&f=g&=end");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
@@ -225,7 +225,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyURLToClipboardTest) {
   clipboard->ReadAsciiText(ui::ClipboardBuffer::kCopyPaste,
                            /* data_dst = */ nullptr, &text_from_clipboard);
   EXPECT_EQ(text_from_clipboard,
-            "https://dev-pages.bravesoftware.com/clean-urls/"
+            "https://dev-pages.ibrowesoftware.com/clean-urls/"
             "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&d&"
             "utm_content=removethis&e=&f=g&=end");
 }
@@ -236,7 +236,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsEnabledFeatureTest,
     { "include": [ "*://*/*"], "params": ["utm_content"] }
   ])");
   const std::string test_url(
-      "https://dev-pages.bravesoftware.com/clean-urls/"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/"
       "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&"
       "d&utm_content=removethis&e=&f=g&=end");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
@@ -251,13 +251,13 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsEnabledFeatureTest,
   clipboard->ReadAsciiText(ui::ClipboardBuffer::kCopyPaste,
                            /* data_dst = */ nullptr, &text_from_clipboard);
   EXPECT_EQ(text_from_clipboard,
-            "https://dev-pages.bravesoftware.com/clean-urls/"
+            "https://dev-pages.ibrowesoftware.com/clean-urls/"
             "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&d&"
             "e=&f=g&=end");
 }
 
 IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, DoNotSanitizeInternalURLS) {
-  const std::string test_url("brave://settings/?utm_ad=1");
+  const std::string test_url("ibrowe://settings/?utm_ad=1");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
   SetSanitizerRules(R"([
     { "include": [ "*://*/*"], "params": ["utm_ad"] }
@@ -270,7 +270,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, DoNotSanitizeInternalURLS) {
   std::string text_from_clipboard;
   clipboard->ReadAsciiText(ui::ClipboardBuffer::kCopyPaste,
                            /* data_dst = */ nullptr, &text_from_clipboard);
-  EXPECT_EQ(text_from_clipboard, "brave://settings/?utm_ad=1");
+  EXPECT_EQ(text_from_clipboard, "ibrowe://settings/?utm_ad=1");
 }
 
 IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsDisabledFeatureTest,
@@ -279,7 +279,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsDisabledFeatureTest,
     { "include": [ "*://*/*"], "params": ["utm_content"] }
   ])");
   const std::string test_url(
-      "https://dev-pages.bravesoftware.com/clean-urls/"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/"
       "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&"
       "d&utm_content=removethis&e=&f=g&=end");
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), GURL(test_url)));
@@ -294,7 +294,7 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsDisabledFeatureTest,
   clipboard->ReadAsciiText(ui::ClipboardBuffer::kCopyPaste,
                            /* data_dst = */ nullptr, &text_from_clipboard);
   EXPECT_EQ(text_from_clipboard,
-            "https://dev-pages.bravesoftware.com/clean-urls/"
+            "https://dev-pages.ibrowesoftware.com/clean-urls/"
             "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&d&"
             "utm_content=removethis&e=&f=g&=end");
 }
