diff --git a/browser/ui/views/sidebar/sidebar_items_contents_view.cc b/browser/ui/views/sidebar/sidebar_items_contents_view.cc
index 573713f6e97ce047a2c8f70f0cadad7ce0beeb77..e12ef9554a41f82d7797657ba6c1ece08a6b0409 100644
--- a/browser/ui/views/sidebar/sidebar_items_contents_view.cc
+++ b/browser/ui/views/sidebar/sidebar_items_contents_view.cc
@@ -153,14 +153,14 @@ void SidebarItemsContentsView::UpdateAllBuiltInItemsViewState() {
 
     // If browser window has tab that loads brave talk, brave talk panel icon
     // will use colored one for normal state also.
-    if (item.built_in_item_type ==
-        sidebar::SidebarItem::BuiltInItemType::kBraveTalk) {
-      UpdateItemViewStateAt(
-          item_index,
-          browser_->sidebar_controller()->DoesBrowserHaveOpenedTabForItem(
-              item));
-      continue;
-    }
+//    if (item.built_in_item_type ==
+//        sidebar::SidebarItem::BuiltInItemType::kBraveTalk) {
+//      UpdateItemViewStateAt(
+//          item_index,
+//          browser_->sidebar_controller()->DoesBrowserHaveOpenedTabForItem(
+//              item));
+//      continue;
+//    }
 
     UpdateItemViewStateAt(item_index, item_index == active_index);
   }
@@ -556,8 +556,8 @@ ui::ImageModel SidebarItemsContentsView::GetImageForBuiltInItems(
   switch (type) {
     case sidebar::SidebarItem::BuiltInItemType::kWallet:
       return get_image_model(kLeoProductBraveWalletIcon, state);
-    case sidebar::SidebarItem::BuiltInItemType::kBraveTalk:
-      return get_image_model(kLeoProductBraveTalkIcon, state);
+//    case sidebar::SidebarItem::BuiltInItemType::kBraveTalk:
+//      return get_image_model(kLeoProductBraveTalkIcon, state);
     case sidebar::SidebarItem::BuiltInItemType::kBookmarks:
       return get_image_model(kLeoProductBookmarksIcon, state);
     case sidebar::SidebarItem::BuiltInItemType::kReadingList:
