diff --git a/browser/brave_wallet/brave_wallet_policy_browsertest.cc b/browser/brave_wallet/brave_wallet_policy_browsertest.cc
index ea20d26e04c383aac133f5a12a5af66f1706bace..5bbe6039980c971839c3b01c0658804fb2464497 100644
--- a/browser/brave_wallet/brave_wallet_policy_browsertest.cc
+++ b/browser/brave_wallet/brave_wallet_policy_browsertest.cc
@@ -145,7 +145,7 @@ IN_PROC_BROWSER_TEST_P(BraveWalletPolicyTest, AppMenuItemDisabled) {
   }
 }
 
-// Verify that brave://wallet page isn't reachable when Brave Wallet is
+// Verify that ibrowe://wallet page isn't reachable when Brave Wallet is
 // disabled by policy.
 IN_PROC_BROWSER_TEST_P(BraveWalletPolicyTest, WalletPageAccess) {
   const GURL url("chrome://wallet");
