diff --git a/browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc b/browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc
index b177505f3b9f32ed4753baeb84b680c6ebef4adb..b109929ee81048f80821f748537b0fc46ae678a9 100644
--- a/browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc
+++ b/browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc
@@ -139,7 +139,7 @@ void BravePrivateNewTabPageHandler::GoToBraveSupport() {
     web_contents = web_contents_;
 
   web_contents->OpenURL(
-      content::OpenURLParams(GURL("https://support.brave.com/"),
+      content::OpenURLParams(GURL("https://support.ibrowe.com/"),
                              content::Referrer(),
                              WindowOpenDisposition::NEW_FOREGROUND_TAB,
                              ui::PageTransition::PAGE_TRANSITION_LINK, false),
