diff --git a/browser/ui/views/side_panel/brave_side_panel_utils.cc b/browser/ui/views/side_panel/brave_side_panel_utils.cc
index 94f98c927c9bffdd82561d01f1393a6e0a4e173b..383d5548e4db6690f5ccb54cef8bcddf71d6bf33 100644
--- a/browser/ui/views/side_panel/brave_side_panel_utils.cc
+++ b/browser/ui/views/side_panel/brave_side_panel_utils.cc
@@ -60,19 +60,19 @@ class AIChatSidePanelWebView : public SidePanelWebUIViewT<AIChatUI> {
   bool should_focus_ = true;
 };
 
-std::unique_ptr<views::View> CreateAIChatSidePanelWebView(
-    base::WeakPtr<Profile> profile,
-    SidePanelEntryScope& scope) {
-  CHECK(profile);
-
-  auto web_view = std::make_unique<AIChatSidePanelWebView>(
-      scope, std::make_unique<WebUIContentsWrapperT<AIChatUI>>(
-                 ai_chat::TabAssociatedConversationUrl(), profile.get(),
-                 IDS_SIDEBAR_CHAT_SUMMARIZER_ITEM_TITLE,
-                 /*esc_closes_ui=*/false));
-  web_view->ShowUI();
-  return web_view;
-}
+// std::unique_ptr<views::View> CreateAIChatSidePanelWebView(
+//     base::WeakPtr<Profile> profile,
+//     SidePanelEntryScope& scope) {
+//   CHECK(profile);
+//
+//   auto web_view = std::make_unique<AIChatSidePanelWebView>(
+//       scope, std::make_unique<WebUIContentsWrapperT<AIChatUI>>(
+//                  ai_chat::TabAssociatedConversationUrl(), profile.get(),
+//                  IDS_SIDEBAR_CHAT_SUMMARIZER_ITEM_TITLE,
+//                  /*esc_closes_ui=*/false));
+//   web_view->ShowUI();
+//   return web_view;
+// }
 
 }  // namespace
 
@@ -83,16 +83,16 @@ namespace brave {
 // registering it.
 void RegisterContextualSidePanel(SidePanelRegistry* registry,
                                  content::WebContents* web_contents) {
-  content::BrowserContext* context = web_contents->GetBrowserContext();
-  if (ai_chat::IsAIChatEnabled(user_prefs::UserPrefs::Get(context)) &&
-      Profile::FromBrowserContext(context)->IsRegularProfile()) {
-    // If |registry| already has it, it's no-op.
-    registry->Register(std::make_unique<SidePanelEntry>(
-        SidePanelEntry::Key(SidePanelEntry::Id::kChatUI),
-        base::BindRepeating(&CreateAIChatSidePanelWebView,
-                            Profile::FromBrowserContext(context)->GetWeakPtr()),
-        /*default_content_width_callback=*/base::NullCallback()));
-  }
+  // content::BrowserContext* context = web_contents->GetBrowserContext();
+  // if (ai_chat::IsAIChatEnabled(user_prefs::UserPrefs::Get(context)) &&
+  //     Profile::FromBrowserContext(context)->IsRegularProfile()) {
+  //   // If |registry| already has it, it's no-op.
+  //   registry->Register(std::make_unique<SidePanelEntry>(
+  //       SidePanelEntry::Key(SidePanelEntry::Id::kChatUI),
+  //       base::BindRepeating(&CreateAIChatSidePanelWebView,
+  //                           Profile::FromBrowserContext(context)->GetWeakPtr()),
+  //       /*default_content_width_callback=*/base::NullCallback()));
+  // }
 }
 
 }  // namespace brave
