diff --git a/build/config.gni b/build/config.gni
index bc57498324ac6214bf49f20023548ea03e22444c..d7dc0d0b4b0e878a5b949cff7b769d4655617c29 100644
--- a/build/config.gni
+++ b/build/config.gni
@@ -108,19 +108,19 @@ if (is_win) {
   if (target_cpu == "x86") {
     _arch = "32"
   }
-  brave_installer_exe = "brave_installer.exe"
+  brave_installer_exe = "ibrowe_installer.exe"
   brave_stub_installer_exe =
-      "BraveBrowser$_channel" + "Setup$_arch$brave_underline_full_version.exe"
+  "iBroweBrowser$_channel" + "Setup$_arch$brave_underline_full_version.exe"
   brave_untagged_stub_installer_exe =
-      "BraveBrowserUntagged$_channel" +
-      "Setup$_arch$brave_underline_full_version.exe"
+  "iBroweBrowserUntagged$_channel" +
+  "Setup$_arch$brave_underline_full_version.exe"
   brave_standalone_installer_exe =
-      "BraveBrowserStandalone$_channel" +
-      "Setup$_arch$brave_underline_full_version.exe"
-  brave_silent_installer_exe = "BraveBrowserStandaloneSilent$_channel" +
-                               "Setup$_arch$brave_underline_full_version.exe"
+  "iBroweBrowserStandalone$_channel" +
+  "Setup$_arch$brave_underline_full_version.exe"
+  brave_silent_installer_exe = "iBroweBrowserStandaloneSilent$_channel" +
+  "Setup$_arch$brave_underline_full_version.exe"
   brave_untagged_installer_exe = "BraveBrowserStandaloneUntagged$_channel" +
-                                 "Setup$_arch$brave_underline_full_version.exe"
+  "Setup$_arch$brave_underline_full_version.exe"
 } else if (is_mac) {
   brave_exe = "$chrome_product_full_name.app"
   brave_dmg = "$chrome_product_full_name.dmg"
@@ -143,11 +143,11 @@ if (is_win) {
     brave_product_dir_name_suffix = "-Development"
   }
   brave_product_dir_name =
-      "BraveSoftware/Brave-Browser$brave_product_dir_name_suffix"
+  "iBroweSoftware/iBrowe-Browser$brave_product_dir_name_suffix"
 
   if (base_sparkle_update_url == "") {
     base_sparkle_update_url =
-        "https://updates.bravesoftware.com/sparkle/Brave-Browser"
+    "https://updates.ibrowesoftware.com/sparkle/iBrowe-Browser"
   }
 }
 
@@ -164,7 +164,7 @@ is_release_channel = brave_channel == ""
 
 brave_android_output = ""
 if (is_android) {
-  brave_android_output += "$root_out_dir/apks/Brave"
+  brave_android_output += "$root_out_dir/apks/iBrowe"
   if (target_android_base == "modern") {
     assert(target_cpu != "arm64" && target_cpu != "x64")
     brave_android_output += "Modern"
