diff --git a/build/commands/lib/config.js b/build/commands/lib/config.js
index 7109a5573c2b5acf7da7de8b32b5872a084150c4..f09d225ad08095910e9a2979bd42e27eb64bb0fe 100644
--- a/build/commands/lib/config.js
+++ b/build/commands/lib/config.js
@@ -592,17 +592,17 @@ Config.prototype.buildArgs = function () {
     args.android_channel = this.channel
     if (!this.isReleaseBuild()) {
       args.android_channel = 'default'
-      args.chrome_public_manifest_package = 'com.brave.browser_default'
+      args.chrome_public_manifest_package = 'com.ibrowe.browser_default'
     } else if (this.channel === '') {
       args.android_channel = 'stable'
-      args.chrome_public_manifest_package = 'com.brave.browser'
+      args.chrome_public_manifest_package = 'com.ibrowe.browser'
     } else if (this.channel === 'beta') {
-      args.chrome_public_manifest_package = 'com.brave.browser_beta'
+      args.chrome_public_manifest_package = 'com.ibrowe.browser_beta'
     } else if (this.channel === 'dev') {
-      args.chrome_public_manifest_package = 'com.brave.browser_dev'
+      args.chrome_public_manifest_package = 'com.ibrowe.browser_dev'
     } else if (this.channel === 'nightly') {
       args.android_channel = 'canary'
-      args.chrome_public_manifest_package = 'com.brave.browser_nightly'
+      args.chrome_public_manifest_package = 'com.ibrowe.browser_nightly'
     }
     // exclude_unwind_tables is inherited form upstream and is false for any
     // Android build
