diff --git a/build/commands/lib/config.js b/build/commands/lib/config.js
index 4cf855d5f7984d79663d6ce8571d02f85f4c919e..536b1c3897c4711c06542f938b23063209e0c525 100644
--- a/build/commands/lib/config.js
+++ b/build/commands/lib/config.js
@@ -20,6 +20,7 @@ if (process.platform === 'win32') {
 }
 const rootDir = path.resolve(dirName, '..', '..', '..', '..', '..')
 const braveCoreDir = path.join(rootDir, 'src', 'brave')
+const ibroweCoreDir = path.join(rootDir, 'src', 'ibrowe')
 
 var packageConfig = function (key, sourceDir = braveCoreDir) {
   let packages = { config: {} }
@@ -96,7 +97,7 @@ const parseExtraInputs = (inputs, accumulator, callback) => {
 }
 
 const getBraveVersion = (ignorePatchVersionNumber) => {
-  const braveVersion = packageConfig(['version'])
+  const braveVersion = packageConfig(['version'], ibroweCoreDir)
   if (!ignorePatchVersionNumber) {
     return braveVersion
   }
