diff --git a/build/commands/lib/config.js b/build/commands/lib/config.js
index d2c495ea406960fb7f467eca3c1ce7aaf7fa0c5d..0a91e94d8c0a9cb17ffdc57c3f076b3f560c8b5e 100644
--- a/build/commands/lib/config.js
+++ b/build/commands/lib/config.js
@@ -20,6 +20,7 @@ if (process.platform === 'win32') {
 }
 const rootDir = path.resolve(dirName, '..', '..', '..', '..', '..')
 const braveCoreDir = path.join(rootDir, 'src', 'brave')
+const ibroweCoreDir = path.join(rootDir, 'src', 'ibrowe')
 
 var packageConfig = function (key, sourceDir = braveCoreDir) {
   let packages = { config: {} }
@@ -96,7 +97,7 @@ const parseExtraInputs = (inputs, accumulator, callback) => {
 }
 
 const getBraveVersion = (ignorePatchVersionNumber) => {
-  const braveVersion = packageConfig(['version'])
+  const braveVersion = packageConfig(['version'], ibroweCoreDir)
   if (!ignorePatchVersionNumber) {
     return braveVersion
   }
