diff --git a/chromium_src/chrome/browser/ui/safety_hub/password_status_check_service.cc b/chromium_src/chrome/browser/ui/safety_hub/password_status_check_service.cc
index cb9fc75ac8f6ec7c944ecc4c108c68445e885f0c..a5a00403df2f7d1f31cd7479b79a413e1f8f93ad 100644
--- a/chromium_src/chrome/browser/ui/safety_hub/password_status_check_service.cc
+++ b/chromium_src/chrome/browser/ui/safety_hub/password_status_check_service.cc
@@ -9,7 +9,7 @@
 #include "src/chrome/browser/ui/safety_hub/password_status_check_service.cc"
 #undef GetPasswordCardData
 
-// We hide the password card in brave://settings/safetyCheck, so we don't want
+// We hide the password card in ibrowe://settings/safetyCheck, so we don't want
 // to return recommendations that involve that card
 base::Value::Dict PasswordStatusCheckService::GetPasswordCardData(
     bool signed_in) {
