diff --git a/chromium_src/components/os_crypt/sync/keychain_password_mac.mm b/chromium_src/components/os_crypt/sync/keychain_password_mac.mm
index b845ac4bda36720d843374f884110235e612c6ff..7e748032b1a70fef1a45c9e8a4bdf10e6ff141ab 100644
--- a/chromium_src/components/os_crypt/sync/keychain_password_mac.mm
+++ b/chromium_src/components/os_crypt/sync/keychain_password_mac.mm
@@ -53,7 +53,7 @@ KeychainPassword::KeychainNameType& GetBraveServiceName() {
     return *kOperaServiceName;
   } else {
     static KeychainNameContainerType kBraveDefaultServiceName(
-        "Brave Safe Storage");
+        "iBrowe Safe Storage");
     return *kBraveDefaultServiceName;
   }
 }
