diff --git a/chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc b/chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
index 216915e26b39d6253076068958c5db9ab64aedc1..0a42dd6a089d62d707f31c770e777c3abc2772de 100644
--- a/chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
+++ b/chromium_src/chrome/browser/ui/views/permissions/permission_prompt_bubble_base_view.cc
@@ -67,7 +67,7 @@ constexpr char kGeolocationPermissionLearnMoreURL[] =
     "allow-apps-to-detect-the-location-of-your-mac-mh35873/mac";
 #else
     // Not used now. Set proper link when detailed bubble is enabled on linux.
-    "https://www.brave.com/";
+    "https://www.ibrowe.com/";
 #endif
 
 #if BUILDFLAG(ENABLE_WIDEVINE)
