diff --git a/chromium_src/components/search_engines/template_url_starter_pack_data.cc b/chromium_src/components/search_engines/template_url_starter_pack_data.cc
index edf7ef59175255179b71a5f2ab08323205fcfd75..2395b929cccab53869a1e1e6b2edee5cc5310479 100644
--- a/chromium_src/components/search_engines/template_url_starter_pack_data.cc
+++ b/chromium_src/components/search_engines/template_url_starter_pack_data.cc
@@ -17,7 +17,7 @@
 namespace {
 
 constexpr char kChromeSchema[] = "chrome://";
-constexpr char kBraveSchema[] = "brave://";
+constexpr char kBraveSchema[] = "ibrowe://";
 }  // namespace
 
 namespace TemplateURLStarterPackData {
