diff --git a/chromium_src/components/policy/tools/template_writers/writer_configuration.py b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
index fdcbbfbb1a92471a74a10f09fd07adf954b4471c..13edac5e1f3887e1fed1b11ec4c40b64afe2f932 100644
--- a/chromium_src/components/policy/tools/template_writers/writer_configuration.py
+++ b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
@@ -38,7 +38,7 @@ _BRAVE_VALUES = {
     },
     'admx_prefix': 'brave',
     'linux_policy_path': '/etc/brave/policies/',
-    'bundle_id': 'com.brave.ios.core',
+    'bundle_id': 'com.ibrowe.ios.core',
 }
 
 def _merge_dicts(src, dst):
