diff --git a/chromium_src/chrome/browser/devtools/url_constants.cc b/chromium_src/chrome/browser/devtools/url_constants.cc
index fb2a8264b2b4f62b00eb4ad49b91a57c5d8f1901..db4c676224e20c3a9ae4fb77ef39afd307e02c90 100644
--- a/chromium_src/chrome/browser/devtools/url_constants.cc
+++ b/chromium_src/chrome/browser/devtools/url_constants.cc
@@ -5,6 +5,6 @@
 
 #include "chrome/browser/devtools/url_constants.h"
 
-constexpr char kRemoteFrontendDomain[] = "devtools.brave.com";
-constexpr char kRemoteFrontendBase[] = "https://devtools.brave.com/";
+constexpr char kRemoteFrontendDomain[] = "devtools.ibrowe.com";
+constexpr char kRemoteFrontendBase[] = "https://devtools.ibrowe.com/";
 constexpr char kRemoteFrontendPath[] = "serve_file";
