diff --git a/chromium_src/chrome/common/url_constants.h b/chromium_src/chrome/common/url_constants.h
index b52b92443aa90e061404707dc8360cd27f62a20c..d227326646a75ce40a2bf87b50a7d8c57bc99489 100644
--- a/chromium_src/chrome/common/url_constants.h
+++ b/chromium_src/chrome/common/url_constants.h
@@ -18,24 +18,23 @@ namespace chrome {
 
 // "Learn more" URL linked in the dialog to cast using a code.
 inline constexpr char kAccessCodeCastLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "Learn more" URL for accessibility image labels, linked from the permissions
 // dialog shown when a user enables the feature.
 inline constexpr char kAccessibilityLabelsLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "Learn more" URL for Ad Privacy.
-inline constexpr char kAdPrivacyLearnMoreURL[] = "https://support.brave.com/";
+inline constexpr char kAdPrivacyLearnMoreURL[] = "https://support.ibrowe.com/";
 
 // "Learn more" URL for when profile settings are automatically reset.
 inline constexpr char kAutomaticSettingsResetLearnMoreURL[] =
-    "https://support.brave.com/hc/en-us/articles/"
-    "360017903152-How-do-I-reset-Brave-settings-to-default-";
+    "https://community.ibrowe.com/post/74";
 
 // "Learn more" URL for Advanced Protection download warnings.
 inline constexpr char kAdvancedProtectionDownloadLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "Chrome Settings" URL for the appearance page.
 inline constexpr char kBrowserSettingsSearchEngineURL[] =
@@ -49,16 +48,16 @@ inline constexpr char16_t kBatterySaverModeLearnMoreUrl[] =
 
 // The URL for providing help when the Bluetooth adapter is off.
 inline constexpr char kBluetoothAdapterOffHelpURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "Learn more" URL shown in the dialog to enable cloud services for Cast.
 inline constexpr char kCastCloudServicesHelpURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the help center article to show when no Cast destination has been
 // found.
 inline constexpr char kCastNoDestinationFoundURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the WebHID API help center article.
 inline constexpr char kChooserHidOverviewUrl[] =
@@ -78,18 +77,18 @@ inline constexpr char kChromeBetaForumURL[] =
 
 // The URL for the help center article to fix Chrome update problems.
 inline constexpr char16_t kChromeFixUpdateProblems[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // General help links for Chrome, opened using various actions.
 inline constexpr char kChromeHelpViaKeyboardURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
-inline constexpr char kChromeHelpViaMenuURL[] = "https://support.brave.com/";
+inline constexpr char kChromeHelpViaMenuURL[] = "https://support.ibrowe.com/";
 
-inline constexpr char kChromeHelpViaWebUIURL[] = "https://support.brave.com/";
+inline constexpr char kChromeHelpViaWebUIURL[] = "https://support.ibrowe.com/";
 
 inline constexpr char kRelatedWebsiteSetsLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The isolated-app: scheme is used for Isolated Web Apps. A public explainer
 // can be found here: https://github.com/reillyeon/isolated-web-apps
@@ -101,7 +100,7 @@ inline constexpr char16_t kIsolatedAppSchemeUtf16[] = u"isolated-app";
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 
 // The URL of safe section in Chrome page.
-inline constexpr char16_t kChromeSafePageURL[] = u"https://support.brave.com/";
+inline constexpr char16_t kChromeSafePageURL[] = u"https://support.ibrowe.com/";
 
 // Pages under chrome-search.
 inline constexpr char kChromeSearchLocalNtpHost[] = "local-ntp";
@@ -207,14 +206,14 @@ inline constexpr char kExtensionInvalidRequestURL[] =
 
 // Link for creating family group with Google Families.
 inline constexpr char16_t kFamilyGroupCreateURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // Link for viewing family group with Google Families.
-inline constexpr char16_t kFamilyGroupViewURL[] = u"https://support.brave.com/";
+inline constexpr char16_t kFamilyGroupViewURL[] = u"https://support.ibrowe.com/";
 
 // "Learn more" URL for Incognito tracking protections.
 inline constexpr char kIncognitoTrackingProtectionsLearnMoreUrl[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // Url to a blogpost about Flash deprecation.
 inline constexpr char kFlashDeprecationLearnMoreURL[] =
@@ -222,41 +221,41 @@ inline constexpr char kFlashDeprecationLearnMoreURL[] =
 
 // URL of the 'Activity controls' section of the privacy settings page.
 inline constexpr char kGoogleAccountActivityControlsURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the 'Activity controls' section of the privacy settings page, with
 // privacy guide parameters and a link for users to manage data.
 inline constexpr char kGoogleAccountActivityControlsURLInPrivacyGuide[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the 'Linked services' section of the privacy settings page.
 inline constexpr char kGoogleAccountLinkedServicesURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the Google Account.
-inline constexpr char kGoogleAccountURL[] = "https://support.brave.com/";
+inline constexpr char kGoogleAccountURL[] = "https://support.ibrowe.com/";
 
 // URL of the Google Account chooser.
-inline constexpr char kGoogleAccountChooserURL[] = "https://support.brave.com/";
+inline constexpr char kGoogleAccountChooserURL[] = "https://support.ibrowe.com/";
 
 // URL of the Google Account page showing the known user devices.
 inline constexpr char kGoogleAccountDeviceActivityURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the Google Account home address page.
 inline constexpr char kGoogleAccountHomeAddressURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the Google Account work address page.
 inline constexpr char kGoogleAccountWorkAddressURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the two factor authentication setup required intersitial.
 inline constexpr char kGoogleTwoFactorIntersitialURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // URL of the Google Password Manager.
-inline constexpr char kGooglePasswordManagerURL[] = "https://support.brave.com";
+inline constexpr char kGooglePasswordManagerURL[] = "https://support.ibrowe.com";
 
 // The URL for the "Learn more" link for the High Efficiency Mode.
 inline constexpr char16_t kMemorySaverModeLearnMoreUrl[] =
@@ -265,11 +264,11 @@ inline constexpr char16_t kMemorySaverModeLearnMoreUrl[] =
 // The URL in the help text for the High Efficiency Mode tab discarding
 // exceptions add dialog.
 inline constexpr char16_t kMemorySaverModeTabDiscardingHelpUrl[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL to the help center article of Incognito mode.
 inline constexpr char16_t kIncognitoHelpCenterURL[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // The URL for the Help Center page about IP Protection.
 inline constexpr char kIpProtectionHelpCenterURL[] =
@@ -282,22 +281,22 @@ inline constexpr char kLearnMoreReportingURL[] =
     "360017905872-How-do-I-enable-or-disable-automatic-crash-reporting-";
 
 // The URL for the Help Center page about managing third-party cookies.
-inline constexpr char kManage3pcHelpCenterURL[] = "https://support.brave.com/";
+inline constexpr char kManage3pcHelpCenterURL[] = "https://support.ibrowe.com/";
 
 // The URL for the tab group sync help center page.
-inline constexpr char kTabGroupsLearnMoreURL[] = "https://support.brave.com/";
+inline constexpr char kTabGroupsLearnMoreURL[] = "https://support.ibrowe.com/";
 
 // The URL for the Learn More page about policies and enterprise enrollment.
 inline constexpr char16_t kManagedUiLearnMoreUrl[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for insecure download blocking.
 inline constexpr char kInsecureDownloadBlockingLearnMoreUrl[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "myactivity.google.com" URL for the history checkbox in ClearBrowsingData.
 inline constexpr char16_t kMyActivityUrlInClearBrowsingData[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // Help URL for the Omnibox setting.
 inline constexpr char16_t kOmniboxLearnMoreURL[] =
@@ -311,11 +310,11 @@ inline constexpr char kPageInfoHelpCenterURL[] =
 
 // Help URL for the bulk password check.
 inline constexpr char kPasswordCheckLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // Help URL for password generation.
 inline constexpr char kPasswordGenerationLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 inline constexpr char16_t kPasswordManagerLearnMoreURL[] =
     u"https://support.brave.com/hc/en-us/articles/"
@@ -323,46 +322,46 @@ inline constexpr char16_t kPasswordManagerLearnMoreURL[] =
 
 // Help URL for passwords import.
 inline constexpr char kPasswordManagerImportLearnMoreURL[] =
-    "https://support.brave.com";
+    "https://support.ibrowe.com";
 
 // Help URL for password sharing.
 inline constexpr char kPasswordSharingLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // Help URL for troubleshooting password sharing.
 inline constexpr char kPasswordSharingTroubleshootURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // Help URL for the Payment methods page of the Google Pay site.
-inline constexpr char16_t kPaymentMethodsURL[] = u"https://support.brave.com";
+inline constexpr char16_t kPaymentMethodsURL[] = u"https://support.ibrowe.com";
 
 // Help URL for the newer GPay Web site instead of the legacy Payments Center.
 inline constexpr char16_t kPaymentMethodsURLForGPayWeb[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // The URL for the "Fill out forms automatically" support page.
 inline constexpr char kAddressesAndPaymentMethodsLearnMoreURL[] =
-    "https://support.brave.com";
+    "https://support.ibrowe.com";
 
 // The URL for the "Pay over time" support page.
 inline constexpr char16_t kPayOverTimeLearnMoreUrl[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // Help URL for Autofill Prediction Improvements.
 inline constexpr char16_t kAutofillAiLearnMoreURL[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // "Learn more" URL for the autofill show card benefits setting.
 inline constexpr char16_t kCardBenefitsLearnMoreURL[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // "Learn more" URL for the performance intervention notification setting.
 inline constexpr char16_t kPerformanceInterventionLearnMoreUrl[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // "Learn more" URL for the preloading section in Performance settings.
 inline constexpr char16_t kPreloadingLearnMoreUrl[] =
-    u"https://support.brave.com";
+    u"https://support.ibrowe.com";
 
 // "Learn more" URL for the Privacy section under Options.
 inline constexpr char kPrivacyLearnMoreURL[] =
@@ -371,8 +370,7 @@ inline constexpr char kPrivacyLearnMoreURL[] =
 
 // The URL for the Learn More link of the non-CWS bubble.
 inline constexpr char kRemoveNonCWSExtensionURL[] =
-    "https://support.brave.com/hc/en-us/articles/"
-    "360017914832-Why-am-I-seeing-the-message-extensions-disabled-by-Brave-";
+    "https://community.ibrowe.com/post/6";
 
 // "Learn more" URL for resetting profile preferences.
 inline constexpr char kResetProfileSettingsLearnMoreURL[] =
@@ -386,90 +384,90 @@ inline constexpr char kSafeBrowsingHelpCenterURL[] =
 
 // Updated "Info icon" URL for Safebrowsing
 inline constexpr char kSafeBrowsingHelpCenterUpdatedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // "Learn more" URL for Enhanced Protection
 inline constexpr char16_t kSafeBrowsingInChromeHelpCenterURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL of Safe Browsing p-tour.
 inline constexpr char16_t kSafeBrowsingUseInChromeURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // "Learn more" URL for Safety Check page.
 inline constexpr char16_t kSafetyHubHelpCenterURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // "Learn more" URL for safety tip bubble.
 inline constexpr char kSafetyTipHelpCenterURL[] =
-    "https://support.brave.com/hc/en-us/articles/17550072876045-Lookalike-URLs";
+    "https://community.ibrowe.com/post/76";
 
 // Google search history URL that leads users of the CBD dialog to their search
 // history in their Google account.
 inline constexpr char16_t kSearchHistoryUrlInClearBrowsingData[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL for the "See more security tips" with advices how to create a strong
 // password.
-inline constexpr char kSeeMoreSecurityTipsURL[] = "https://support.brave.com/";
+inline constexpr char kSeeMoreSecurityTipsURL[] = "https://support.ibrowe.com/";
 
 // Help URL for the settings page's search feature.
 inline constexpr char16_t kSettingsSearchHelpURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL for the Learn More page about Sync and Google services.
 inline constexpr char kSyncAndGoogleServicesLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page on sync encryption.
 inline constexpr char16_t kSyncEncryptionHelpURL[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL for the "Learn more" link when there is a sync error.
-inline constexpr char kSyncErrorsHelpURL[] = "https://support.brave.com/";
+inline constexpr char kSyncErrorsHelpURL[] = "https://support.ibrowe.com/";
 
-inline constexpr char kSyncGoogleDashboardURL[] = "https://support.brave.com/";
+inline constexpr char kSyncGoogleDashboardURL[] = "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for sync setup on the personal stuff page.
-inline constexpr char16_t kSyncLearnMoreURL[] = u"https://support.brave.com/";
+inline constexpr char16_t kSyncLearnMoreURL[] = u"https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for signing in to chrome with expanded
 // section on "Sign in and turn on sync" in the Computer/Desktop tab.
 inline constexpr char kSigninOnDesktopLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for adding a new profile to Chrome.
 inline constexpr char kAddNewProfileOnDesktopLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for AI settings for managed users.
 inline constexpr char16_t kAiSettingsLearnMorePageManagedUrl[] =
-    u"https://support.brave.com/";
+    u"https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for Help me Write.
-inline constexpr char kComposeLearnMorePageURL[] = "https://support.brave.com/";
+inline constexpr char kComposeLearnMorePageURL[] = "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for Help me Write for managed users.
 inline constexpr char kComposeLearnMorePageManagedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" links for pages related to History search.
 // TODO(crbug.com/328300718): Update help article URL.
 inline constexpr char kHistorySearchLearnMorePageURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more about shortcuts" page for Glic.
 inline constexpr char kGlicKeyboardShortcutLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more about shortcuts" page for Glic for managed users.
 inline constexpr char kGlicKeyboardShortcutLearnMoreManagedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" links for pages related to History search for
 // managed users.
 inline constexpr char kHistorySearchLearnMorePageManagedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the Settings page to enable history search.
 inline constexpr char16_t kHistorySearchSettingURL[] =
@@ -482,25 +480,24 @@ inline constexpr char16_t kHistorySearchV2SettingURL[] =
 
 // The URL for the "Learn more" page for Wallpaper Search.
 inline constexpr char kWallpaperSearchLearnMorePageURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for Wallpaper Search for managed users.
 inline constexpr char kWallpaperSearchLearnMorePageManagedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" page for Tab Organization.
 inline constexpr char kTabOrganizationLearnMorePageURL[] =
-    "https://support.brave.com/hc/en-us/articles/"
-    "35200007195917-How-to-use-Tab-Focus-Mode";
+    "https://community.ibrowe.com/post/114";
 
 // The URL for the "Learn more" page for Tab Organization for managed users.
 inline constexpr char kTabOrganizationLearnMorePageManagedURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "Learn more" link in the enterprise disclaimer for managed
 // profile in the Signin Intercept bubble.
 inline constexpr char kSigninInterceptManagedDisclaimerLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 #if !BUILDFLAG(IS_ANDROID)
 // The URL for the trusted vault sync passphrase opt in.
@@ -565,18 +562,17 @@ inline constexpr char kEnhancedPlaybackNotificationLearnMoreURL[] =
 #if BUILDFLAG(IS_MAC)
 // "Learn more" URL for the enterprise sign-in confirmation dialog.
 inline constexpr char kChromeEnterpriseSignInLearnMoreURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the "learn more" link on the macOS version obsolescence infobar.
 inline constexpr char kMacOsObsoleteURL[] =
-    "https://support.brave.com/hc/en-us/articles/"
-    "18347246446733-Changes-to-macOS-desktop-browser-requirements";
+    "https://community.ibrowe.com/post/374";
 #endif
 
 #if BUILDFLAG(IS_WIN)
 // The URL for the Windows XP/Vista deprecation help center article.
 inline constexpr char kWindowsXPVistaDeprecationURL[] =
-    "https://support.brave.com/";
+    "https://support.ibrowe.com/";
 
 // The URL for the Windows 7/8.1 deprecation help center article.
 inline constexpr char kWindows78DeprecationURL[] =
@@ -584,7 +580,7 @@ inline constexpr char kWindows78DeprecationURL[] =
 #endif  // BUILDFLAG(IS_WIN)
 
 // "Learn more" URL for the one click signin infobar.
-inline constexpr char kChromeSyncLearnMoreURL[] = "https://support.brave.com/";
+inline constexpr char kChromeSyncLearnMoreURL[] = "https://support.ibrowe.com/";
 
 #if BUILDFLAG(ENABLE_PLUGINS)
 // The URL for the "Learn more" page for the outdated plugin infobar.
