diff --git a/chromium_src/components/policy/tools/template_writers/writer_configuration.py b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
index 5951be07f4a7ce58dc198a7b32772d1b9f3b1e2a..3c592a80e7c6b3cbe4d970eb1febb5117e352335 100644
--- a/chromium_src/components/policy/tools/template_writers/writer_configuration.py
+++ b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
@@ -14,7 +14,7 @@ _BRAVE_VALUES = {
   'build': 'brave',
   'app_name': 'Brave',
   'doc_url':
-    'https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy',
+    'https://support.ibrowe.com/hc/en-us/articles/360039248271-Group-Policy',
   'frame_name': 'Brave Frame',
   'webview_name': 'Brave WebView',
   'win_config': {
