diff --git a/chromium_src/components/policy/tools/template_writers/writer_configuration.py b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
index 5951be07f4a7ce58dc198a7b32772d1b9f3b1e2a..585c6dd07606e5268a50dc6fb0cd49e5077ae44b 100644
--- a/chromium_src/components/policy/tools/template_writers/writer_configuration.py
+++ b/chromium_src/components/policy/tools/template_writers/writer_configuration.py
@@ -11,38 +11,37 @@ def GetConfigurationForBuild(original_function, defines):
   return _merge_dicts(_BRAVE_VALUES, base)
 
 _BRAVE_VALUES = {
-  'build': 'brave',
-  'app_name': 'Brave',
-  'doc_url':
-    'https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy',
-  'frame_name': 'Brave Frame',
-  'webview_name': 'Brave WebView',
+  'build': 'ibrowe',
+  'app_name': 'iBrowe',
+  'doc_url': 'https://support.ibrowe.com/docs/policy',
+  'frame_name': 'iBrowe Frame',
+  'webview_name': 'iBrowe WebView',
   'win_config': {
     'win': {
-      'reg_mandatory_key_name': 'Software\\Policies\\BraveSoftware\\Brave',
+      'reg_mandatory_key_name': 'Software\\Policies\\iBroweSoftware\\iBrowe',
       'reg_recommended_key_name':
-        'Software\\Policies\\BraveSoftware\\Brave\\Recommended',
-      'mandatory_category_path': ['Brave:Cat_Brave', 'brave'],
-      'recommended_category_path': ['Brave:Cat_Brave', 'brave_recommended'],
+        'Software\\Policies\\iBroweSoftware\\iBrowe\\Recommended',
+      'mandatory_category_path': ['iBrowe:Cat_iBrowe', 'ibrowe'],
+      'recommended_category_path': ['iBrowe:Cat_iBrowe', 'ibrowe_recommended'],
       'category_path_strings': {
-        'brave': 'Brave',
-        'brave_recommended':
-        'Brave - {doc_recommended}'
+        'ibrowe': 'iBrowe',
+        'ibrowe_recommended':
+        'iBrowe - {doc_recommended}'
       },
-      'namespace': 'BraveSoftware.Policies.Brave',
+      'namespace': 'iBroweSoftware.Policies.iBrowe',
     },
   },
   # The string 'Brave' is defined in brave.adml for ADMX, but ADM doesn't
   # support external references, so we define this map here.
   'adm_category_path_strings': {
-    'Brave:Cat_Brave': 'Brave'
+    'iBrowe:Cat_iBrowe': 'iBrowe'
   },
-  'admx_prefix': 'brave',
+  'admx_prefix': 'ibrowe',
   'admx_using_namespaces': {
-    'Brave': 'BraveSoftware.Policies'  # prefix: namespace
+    'iBrowe': 'iBroweSoftware.Policies'  # prefix: namespace
   },
-  'linux_policy_path': '/etc/brave/policies/',
-  'bundle_id': 'com.brave.ios.core',
+  'linux_policy_path': '/etc/ibrowe/policies/',
+  'bundle_id': 'com.ibrowe.ios.core',
 }
 
 def _merge_dicts(src, dst):
