diff --git a/chromium_src/chrome/browser/net/profile_network_context_service.cc b/chromium_src/chrome/browser/net/profile_network_context_service.cc
index 3e5e80e4322c67ed1bf578a215cb276d3d0b95b5..7e6fdf4973dd644aab135b648f3eca5efb794145 100644
--- a/chromium_src/chrome/browser/net/profile_network_context_service.cc
+++ b/chromium_src/chrome/browser/net/profile_network_context_service.cc
@@ -6,7 +6,7 @@ static const char* kBraveCTExcludedHosts[] = {
     // Critical endpoints that shouldn't require SCTs so they always work
     "updates.bravesoftware.com",
     "updates-cdn.bravesoftware.com",
-    "usage-ping.brave.com",
+    "usage-ping.ibrowe.com",
     // Test host for manual testing
     "sct-exempted.bravesoftware.com",
 };
