diff --git a/chromium_src/chrome/updater/branding.gni b/chromium_src/chrome/updater/branding.gni
index db0ddb391250ad503d8689adfe5ef62031e84d1b..b5a476cc9ac5362e7b58cd331c38746c707ec912 100644
--- a/chromium_src/chrome/updater/branding.gni
+++ b/chromium_src/chrome/updater/branding.gni
@@ -14,25 +14,25 @@
 import("//build/util/branding.gni")
 
 brave_updater_branding = {
-  browser_name = "Brave"
+  browser_name = "iBrowe"
   browser_product_name = chrome_product_full_name
-  crash_product_name = "BraveUpdater"
-  keystone_app_name = "BraveSoftwareUpdate"
-  keystone_bundle_identifier = "com.brave.Keystone"
-  mac_browser_bundle_identifier = "com.brave.Browser"
-  mac_updater_bundle_identifier = "com.brave.BraveUpdater"
-  privileged_helper_bundle_name = "BraveUpdaterPrivilegedHelper"
-  privileged_helper_name = "com.brave.Browser.UpdaterPrivilegedHelper"
-  updater_company_full_name = "Brave Software, Inc"
-  updater_company_short_name = "BraveSoftware"
+  crash_product_name = "iBroweUpdater"
+  keystone_app_name = "iBroweSoftwareUpdate"
+  keystone_bundle_identifier = "com.ibrowe.Keystone"
+  mac_browser_bundle_identifier = "com.ibrowe.Browser"
+  mac_updater_bundle_identifier = "com.ibrowe.iBroweUpdater"
+  privileged_helper_bundle_name = "iBroweUpdaterPrivilegedHelper"
+  privileged_helper_name = "com.ibrowe.Browser.UpdaterPrivilegedHelper"
+  updater_company_full_name = "iBrowe Software, Inc"
+  updater_company_short_name = "iBroweSoftware"
   updater_company_short_name_uppercase = "BRAVESOFTWARE"
-  updater_copyright = "Copyright 2024 The Brave Authors. All rights reserved."
-  updater_product_full_name = "BraveUpdater"
+  updater_copyright = "Copyright 2024 The iBrowe Authors. All rights reserved."
+  updater_product_full_name = "iBroweUpdater"
   mac_team_identifier = "KL8N8XSYF4"
-  grdfile_name = "brave_strings"
+  grdfile_name = "ibrowe_strings"
 
   # The following values come from our Omaha 3 implementation:
-  legacy_service_name_prefix = "brave"
+  legacy_service_name_prefix = "ibrowe"
   UpdaterLegacyLibGUID = "6FF4F468-0B5A-4878-ACEB-86263DAAD877"
   GoogleUpdate3WebUserClassGUID = "2F78AECB-0A7F-4474-89D5-C325293DE960"
   GoogleUpdate3WebSystemClassGUID = "00B16F95-319A-4F01-AC81-CE69B8F4E387"
@@ -84,6 +84,6 @@ brave_updater_branding = {
   declare_args() {
     # Omaha 4 update server:
     update_check_url =
-        "https://updates.bravesoftware.com/non-stable/service/update2/json"
+        "https://updates.ibrowesoftware.com/non-stable/service/update2/json"
   }
 }
