diff --git a/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc b/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
index 0a98589aecbf408a3a228719f588ee165de33f12..91529f51420ec6d801eeea41e49d233a9acab223 100644
--- a/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
+++ b/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
@@ -92,9 +92,9 @@ void ChromeAutocompleteProviderClient::OpenLeo(const std::u16string& query) {
     conversation_handler->MaybeUnlinkAssociatedContent();
 
     // Activate the panel.
-    auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
-    sidebar_controller->ActivatePanelItem(
-        sidebar::SidebarItem::BuiltInItemType::kChatUI);
+    // auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
+    // sidebar_controller->ActivatePanelItem(
+    //     sidebar::SidebarItem::BuiltInItemType::kChatUI);
   }
 
   if (!conversation_handler) {
