diff --git a/chromium_src/chrome/app/resources/manpage.1.in b/chromium_src/chrome/app/resources/manpage.1.in
index f8d6fa1bf3fd5e93e4b1e9b41356f16928ea98e9..c35700c15083cc7ac3057a350fb63dffd9d86399 100644
--- a/chromium_src/chrome/app/resources/manpage.1.in
+++ b/chromium_src/chrome/app/resources/manpage.1.in
@@ -12,7 +12,7 @@
 .SH DESCRIPTION
 See the Brave Help Center for help on using the browser.
 .IP
-<https://support.brave.com/>
+<https://support.ibrowe.com/>
 .PP
 This manpage only describes invocation, environment, and arguments.
 
