diff --git a/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index 645cef8c1a1bc2a8d56941d4381ca77abf9cc77c..8f469a1fe82ca9ba8a72a26338c72d24546b548c 100644
--- a/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -11,7 +11,7 @@
 namespace {
 
 constexpr char kBraveStubSessionTag[] = "brave_stub_more_session_tag";
-constexpr char kBraveSyncedTabsUrl[] = "brave://history/syncedTabs";
+constexpr char kBraveSyncedTabsUrl[] = "ibrowe://history/syncedTabs";
 
 }  //  namespace
 
