diff --git a/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info b/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info
index 2f94be147d8cf84d05c9b45eb15e21341e244518..0affb112d8988414e3db0005d478040f9c589cbe 100644
--- a/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info
+++ b/chromium_src/chrome/installer/linux/common/brave-browser/chromium-browser.info
@@ -33,4 +33,4 @@ PRODUCTURL="https://brave.com/"
 VERSION=$(cat $OUTPUTDIR/version | tr -d 'v,')
 VERSIONFULL=$VERSION
 
-RDN="com.brave.Browser"
+RDN="com.ibrowe.Browser"
