diff --git a/chromium_src/chrome/browser/media/router/media_router_feature.cc b/chromium_src/chrome/browser/media/router/media_router_feature.cc
index f1c631e16a93f669fd46fc9ee798c97e818dee81..117ae9b31d417d1170a5d6fe9d0c1852c7016929 100644
--- a/chromium_src/chrome/browser/media/router/media_router_feature.cc
+++ b/chromium_src/chrome/browser/media/router/media_router_feature.cc
@@ -29,7 +29,7 @@ bool MediaRouterEnabled(content::BrowserContext* context) {
   CHECK(pref->GetValue()->is_bool());
   // Chromium has a pref for Media Router but it is only controlled via
   // enterprise policy. In Brave, the pref can be controlled both via
-  // brave://settings/extensions and enterprise policy, with the latter taking
+  //ibrowe://settings/extensions and enterprise policy, with the latter taking
   // precedence.
   if (pref->IsManaged()) {
     return MediaRouterEnabled_ChromiumImpl(context);
