diff --git a/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/android_browser_backend_settings.py b/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
index 1265530343cae1dc1fa28f1e3ab681d58e914537..6067d9e48a05e0f68163efa305d2340327cd219b 100644
--- a/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
+++ b/chromium_src/third_party/catapult/telemetry/telemetry/internal/backends/android_browser_backend_settings.py
@@ -5,19 +5,19 @@
 """A inline part of android_browser_backend_settings.py"""
 
 ANDROID_BRAVE = GenericChromeBackendSettings(browser_type='android-brave',
-                                             package='com.brave.browser')
+                                             package='com.ibrowe108.browser')
 
 ANDROID_BRAVE_DEFAULT = GenericChromeBackendSettings(
-    browser_type='android-brave-default', package='com.brave.browser_default')
+    browser_type='android-brave-default', package='com.ibrowe108.browser_default')
 
 ANDROID_BRAVE_BETA = GenericChromeBackendSettings(
-    browser_type='android-brave-beta', package='com.brave.browser_beta')
+    browser_type='android-brave-beta', package='com.ibrowe108.browser_beta')
 
 ANDROID_BRAVE_DEV = GenericChromeBackendSettings(
-    browser_type='android-brave-dev', package='com.brave.browser_dev')
+    browser_type='android-brave-dev', package='com.ibrowe108.browser_dev')
 
 ANDROID_BRAVE_NIGHTLY = GenericChromeBackendSettings(
-    browser_type='android-brave-nightly', package='com.brave.browser_nightly')
+    browser_type='android-brave-nightly', package='com.ibrowe108.browser_nightly')
 
 BRAVE_ANDROID_BACKEND_SETTINGS = (ANDROID_BRAVE_DEFAULT, ANDROID_BRAVE,
                                   ANDROID_BRAVE_BETA, ANDROID_BRAVE_DEV,
