diff --git a/chromium_src/components/qr_code_generator/dino_image.h b/chromium_src/components/qr_code_generator/dino_image.h
index f2ab254f28509850a7132ee189ad90e886663047..ee1c70ec13604d9dddad9db48eb762aa248b8a6d 100644
--- a/chromium_src/components/qr_code_generator/dino_image.h
+++ b/chromium_src/components/qr_code_generator/dino_image.h
@@ -6,63 +6,62 @@
 #ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_QR_CODE_GENERATOR_DINO_IMAGE_H_
 #define BRAVE_CHROMIUM_SRC_COMPONENTS_QR_CODE_GENERATOR_DINO_IMAGE_H_
 
-// Contains constants clients use to to render a lion on top of a QR image.
+// Contains constants clients use to render a lion on top of a QR image.
 namespace dino_image {
 
-// Width of the lion pixel data.
+// Width of the dino pixel data.
 static constexpr int kDinoWidth = 22;
-// Height of the lion pixel data.
+// Height of the dino pixel data.
 static constexpr int kDinoHeight = 24;
-// Height of the lion pixel data, top segment.
+// Height of the dino pixel data, top segment.
 static constexpr int kDinoHeadHeight = 1;
-// Height of the lion image data, body segment.
+// Height of the dino image data, body segment.
 static constexpr int kDinoBodyHeight = kDinoHeight - kDinoHeadHeight;
-// Width of the lion image data.
-static constexpr int kDinoWidthBytes = (kDinoWidth + 7) / 8;
+// Width of the dino image data.
+static constexpr int kDinoWidthBytes = 3;
 
-// Pixel data for the lion's head (original lion)
-static const unsigned char kDinoHeadRight[kDinoWidthBytes * kDinoHeadHeight] = {
+// Pixel data for the lion's head (left side)
+static const unsigned char kDinoHeadLeft[kDinoWidthBytes * kDinoHeadHeight] = {
     // clang-format off
-  0b00000000, 0b11111100, 0b00000000,
+    0x10, 0x38, 0x10,
     // clang-format on
 };
 
-// Pixel data for the lion's head (original lion, maybe add sunglasses later?)
-static const unsigned char kDinoHeadLeft[kDinoWidthBytes * kDinoHeadHeight] = {
+// Pixel data for the lion's head (right side, mirrored from left)
+static const unsigned char kDinoHeadRight[kDinoWidthBytes * kDinoHeadHeight] = {
     // clang-format off
-  0b00000000, 0b11111100, 0b00000000,
+    0x08, 0x1C, 0x08,
     // clang-format on
 };
 
-// Pixel data for the lion's body.
+// Pixel data for the dino's body.
 static const unsigned char kDinoBody[kDinoWidthBytes * kDinoBodyHeight] = {
     // clang-format off
-  0b00000011, 0b11111111, 0b00000000,
-  0b00011111, 0b11111111, 0b11100000,
-  0b00111111, 0b11111111, 0b11110000,
-  0b01111000, 0b00000000, 0b01111000,
-  0b00110000, 0b00000000, 0b00110000,
-  0b01110011, 0b11001111, 0b00111000,
-  0b01100000, 0b11001100, 0b00011000,
-  0b01100000, 0b10000100, 0b00011000,
-  0b01110000, 0b00000000, 0b00111000,
-  0b00111000, 0b11001100, 0b01110000,
-  0b00111100, 0b01111000, 0b11110000,
-  0b00011100, 0b00110000, 0b11100000,
-  0b00011000, 0b00110000, 0b01100000,
-  0b00011100, 0b01111000, 0b11100000,
-  0b00011111, 0b10000111, 0b11100000,
-  0b00001111, 0b00000011, 0b11000000,
-  0b00001111, 0b10000111, 0b11000000,
-  0b00001111, 0b11001111, 0b11000000,
-  0b00000111, 0b11111111, 0b10000000,
-  0b00000011, 0b11111111, 0b00000000,
-  0b00000001, 0b11111110, 0b00000000,
-  0b00000000, 0b01111000, 0b00000000,
-  0b00000000, 0b00110000, 0b00000000,
+  0x06, 0x01, 0x40,
+  0x0B, 0x01, 0x40,
+  0x13, 0xBA, 0x40,
+  0x12, 0x7F, 0x80,
+  0x16, 0x85, 0x00,
+  0x02, 0xCB, 0xC0,
+  0x0B, 0x41, 0x80,
+  0x1D, 0xD6, 0xC0,
+  0x07, 0xC7, 0x60,
+  0x39, 0x8C, 0x50,
+  0x74, 0x80, 0xC0,
+  0x30, 0x30, 0x50,
+  0x1C, 0x30, 0xD0,
+  0x1C, 0x20, 0x90,
+  0x24, 0xE1, 0x90,
+  0x2C, 0x0E, 0x60,
+  0x14, 0x48, 0xA0,
+  0x08, 0x50, 0x80,
+  0x0C, 0x00, 0x00,
+  0x04, 0x00, 0x00,
+  0x02, 0x02, 0x00,
+  0x00, 0x3C, 0x00,
 };
 
 }  // namespace dino_image
-
 #endif  // BRAVE_CHROMIUM_SRC_COMPONENTS_QR_CODE_GENERATOR_DINO_IMAGE_H_
