diff --git a/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc b/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
index 9831da879aecf121b94bf836b9b7e2ce4c191950..6fc945c859799a77c618eed8011912b536e0aa96 100644
--- a/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
+++ b/chromium_src/chrome/browser/ui/tabs/recent_tabs_sub_menu_model.cc
@@ -10,7 +10,7 @@
 namespace {
 
 constexpr char kBraveStubSessionTag[] = "brave_stub_more_session_tag";
-constexpr char kBraveSyncedTabsUrl[] = "brave://history/syncedTabs";
+constexpr char kBraveSyncedTabsUrl[] = "ibrowe://history/syncedTabs";
 
 }  //  namespace
 
