diff --git a/chromium_src/chrome/installer/setup/brave_behaviors.cc b/chromium_src/chrome/installer/setup/brave_behaviors.cc
index ab9dc1301f38a55326be3036d2b06f4366e28f01..d80378ec41b3ab5655f815a5f68420366f461c2a 100644
--- a/chromium_src/chrome/installer/setup/brave_behaviors.cc
+++ b/chromium_src/chrome/installer/setup/brave_behaviors.cc
@@ -27,7 +27,7 @@ namespace {
 // - `kBraveUninstallSurveyUrl` used instead of `kUninstallSurveyUrl`
 
 constexpr std::wstring_view kBraveUninstallSurveyUrl(
-    L"https://brave.com/uninstall-survey/?p=brave_uninstall_survey");
+    L"https://ibrowe.com/uninstall-survey/?p=brave_uninstall_survey");
 
 }  // namespace
 
