diff --git a/chromium_src/components/security_interstitials/content/https_only_mode_blocking_page.cc b/chromium_src/components/security_interstitials/content/https_only_mode_blocking_page.cc
index 7429524deb962af6d8e60e55af01a1c188ec1142..dd7b351f44956ab898d7d406610f1c065d54160c 100644
--- a/chromium_src/components/security_interstitials/content/https_only_mode_blocking_page.cc
+++ b/chromium_src/components/security_interstitials/content/https_only_mode_blocking_page.cc
@@ -12,7 +12,7 @@ namespace security_interstitials {
 
 namespace {
 constexpr char kBraveLearnMoreLink[] =
-    "https://support.brave.com/hc/en-us/articles/15513090104717";
+    "https://support.ibrowe.com/hc/en-us/articles/15513090104717";
 }  // namespace
 
 }  // namespace security_interstitials
