diff --git a/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index 3280169f7c014ff8a32c39d7d69bbd0b085307c8..aefe06de491b788fd76f4b7abce17341a1de1ca3 100644
--- a/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -333,8 +333,8 @@ void OnRewriteSuggestionCompleted(
 
     auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
     CHECK(sidebar_controller);
-    sidebar_controller->ActivatePanelItem(
-        sidebar::SidebarItem::BuiltInItemType::kChatUI);
+    // sidebar_controller->ActivatePanelItem(
+    //     sidebar::SidebarItem::BuiltInItemType::kChatUI);
 
     conversation->AddSubmitSelectedTextError(selected_text, action_type,
                                              result.error());
@@ -607,8 +607,8 @@ void BraveRenderViewContextMenu::ExecuteAIChatCommand(int command) {
     // Active the panel.
     auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
     CHECK(sidebar_controller);
-    sidebar_controller->ActivatePanelItem(
-        sidebar::SidebarItem::BuiltInItemType::kChatUI);
+    // sidebar_controller->ActivatePanelItem(
+    //     sidebar::SidebarItem::BuiltInItemType::kChatUI);
     conversation->SubmitSelectedText(selected_text, action_type);
   }
 
