diff --git a/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc b/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
index ea33397d69e7205820459b4270eb220faada682e..10c9696b3b92e62b090217876be4d7e1e0cdb5e9 100644
--- a/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
+++ b/chromium_src/chrome/browser/autocomplete/chrome_autocomplete_provider_client.cc
@@ -92,9 +92,9 @@ void ChromeAutocompleteProviderClient::OpenLeo(const std::u16string& query) {
     conversation_handler->MaybeUnlinkAssociatedContent();
 
     // Activate the panel.
-    auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
-    sidebar_controller->ActivatePanelItem(
-        sidebar::SidebarItem::BuiltInItemType::kChatUI);
+    // auto* sidebar_controller = browser->GetFeatures().sidebar_controller();
+    // sidebar_controller->ActivatePanelItem(
+    //     sidebar::SidebarItem::BuiltInItemType::kChatUI);
   }
 
   if (!conversation_handler) {
