diff --git a/chromium_src/chrome/browser/ui/singleton_tabs.cc b/chromium_src/chrome/browser/ui/singleton_tabs.cc
index 34dddb8fe858741db3d4d421ae16fab9966ef325..1bef282c5caabcbbeb5fcb32e3f63dbe057b4e81 100644
--- a/chromium_src/chrome/browser/ui/singleton_tabs.cc
+++ b/chromium_src/chrome/browser/ui/singleton_tabs.cc
@@ -14,14 +14,14 @@
 // instead.
 void ShowSingletonTab(Browser* browser, const GURL& url) {
   GURL new_url = url.DomainIs("google.com") ?
-    GURL("https://community.brave.com/") : url;
+    GURL("https://community.ibrowe.com/") : url;
 
   ShowSingletonTab_ChromiumImpl(browser, new_url);
 }
 
 void ShowSingletonTab(Profile* profile, const GURL& url) {
   GURL new_url =
-      url.DomainIs("google.com") ? GURL("https://community.brave.com/") : url;
+      url.DomainIs("google.com") ? GURL("https://community.ibrowe.com/") : url;
 
   ShowSingletonTab_ChromiumImpl(profile, new_url);
 }
