diff --git a/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc b/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
index 9da2ea01c4d414e41e459bb448babe46575bbbda..f9d1a6297261797cd7b13ad186617aee22306f95 100644
--- a/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -332,7 +332,7 @@ void OnRewriteSuggestionCompleted(
     }
     conversation->MaybeUnlinkAssociatedContent();
 
-    ai_chat::OpenAIChatForTab(web_contents.get());
+    // ai_chat::OpenAIChatForTab(web_contents.get());
 
     conversation->AddSubmitSelectedTextError(selected_text, action_type,
                                              result.error());
@@ -603,7 +603,7 @@ void BraveRenderViewContextMenu::ExecuteAIChatCommand(int command) {
     conversation->MaybeUnlinkAssociatedContent();
 
     // Active the panel.
-    ai_chat::OpenAIChatForTab(embedder_web_contents_);
+    // ai_chat::OpenAIChatForTab(embedder_web_contents_);
     conversation->SubmitSelectedText(selected_text, action_type);
   }
 
