diff --git a/components/skus/browser/skus_utils_unittest.cc b/components/skus/browser/skus_utils_unittest.cc
index 8cb0f1cd695f792aa053fbbd037bf9a62aef94ea..686ecf5341a132cfb1b3ac2eed6bde1b7cef78ec 100644
--- a/components/skus/browser/skus_utils_unittest.cc
+++ b/components/skus/browser/skus_utils_unittest.cc
@@ -24,11 +24,11 @@ TEST(SkusUtilsUnittest, GetEnvironmentForDomain) {
   EXPECT_EQ(GetEnvironmentForDomain("account.brave.com"), kEnvProduction);
   EXPECT_EQ(GetEnvironmentForDomain("vpn.brave.com"), kEnvProduction);
 
-  EXPECT_EQ(GetEnvironmentForDomain("vpn.bravesoftware.com"), kEnvStaging);
-  EXPECT_EQ(GetEnvironmentForDomain("account.bravesoftware.com"), kEnvStaging);
+  EXPECT_EQ(GetEnvironmentForDomain("vpn.ibrowesoftware.com"), kEnvStaging);
+  EXPECT_EQ(GetEnvironmentForDomain("account.ibrowesoftware.com"), kEnvStaging);
 
-  EXPECT_EQ(GetEnvironmentForDomain("vpn.brave.software"), kEnvDevelopment);
-  EXPECT_EQ(GetEnvironmentForDomain("account.brave.software"), kEnvDevelopment);
+  EXPECT_EQ(GetEnvironmentForDomain("vpn.ibrowe.software"), kEnvDevelopment);
+  EXPECT_EQ(GetEnvironmentForDomain("account.ibrowe.software"), kEnvDevelopment);
 }
 
 TEST(SkusUtilsUnittest, GetDomain) {
