diff --git a/components/sidebar/browser/sidebar_service_unittest.cc b/components/sidebar/browser/sidebar_service_unittest.cc
index f87b28cc25dd2229f5ea413fce50353823a7fb7a..ac9fd291e151c548cc6ac42f2f10077254cab67b 100644
--- a/components/sidebar/browser/sidebar_service_unittest.cc
+++ b/components/sidebar/browser/sidebar_service_unittest.cc
@@ -799,8 +799,8 @@ TEST_F(SidebarServiceTest, BuiltInItemUpdateTestWithBuiltInItemTypeKey) {
     dict.Set(sidebar::kSidebarItemTitleKey, "Brave together");
     dict.Set(sidebar::kSidebarItemTypeKey,
              static_cast<int>(SidebarItem::Type::kTypeBuiltIn));
-    dict.Set(sidebar::kSidebarItemBuiltInItemTypeKey,
-             static_cast<int>(SidebarItem::BuiltInItemType::kBraveTalk));
+    // dict.Set(sidebar::kSidebarItemBuiltInItemTypeKey,
+    //          static_cast<int>(SidebarItem::BuiltInItemType::kBraveTalk));
     dict.Set(sidebar::kSidebarItemOpenInPanelKey, true);
     base::Value::List list;
     list.Append(std::move(dict));
@@ -1012,7 +1012,8 @@ TEST_F(SidebarServiceOrderingTest, BuiltInItemsDefaultOrder) {
   EXPECT_EQ(0UL, service_->GetHiddenDefaultSidebarItems().size());
 
   EXPECT_TRUE(
-      ValidateBuiltInTypesOrdering({SidebarItem::BuiltInItemType::kBraveTalk,
+      ValidateBuiltInTypesOrdering({
+                                    // SidebarItem::BuiltInItemType::kBraveTalk,
                                     SidebarItem::BuiltInItemType::kWallet,
                                     SidebarItem::BuiltInItemType::kChatUI,
                                     SidebarItem::BuiltInItemType::kBookmarks,
@@ -1033,7 +1034,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsAllBuiltInVisible) {
       SidebarItem::BuiltInItemType::kWallet,
       SidebarItem::BuiltInItemType::kReadingList,
       SidebarItem::BuiltInItemType::kBookmarks,
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
   };
 
   auto expected_count = sidebar_items->size();
@@ -1053,7 +1054,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsWalletBuiltInHidden) {
   CHECK(sidebar_items);
 
   std::vector items = {
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
       SidebarItem::BuiltInItemType::kBookmarks,
       SidebarItem::BuiltInItemType::kReadingList,
       SidebarItem::BuiltInItemType::kChatUI,
@@ -1080,7 +1081,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsAIChatBuiltInNotListed) {
   CHECK(sidebar_items);
 
   std::vector items = {
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
       SidebarItem::BuiltInItemType::kBookmarks,
       SidebarItem::BuiltInItemType::kChatUI,
       SidebarItem::BuiltInItemType::kReadingList,
