diff --git a/components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc b/components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc
index 109009c7ae2b89c7deb5e501da10586cd1ce9a78..27fcade2779e003aecdc79cae8880f8ad3d98ca4 100644
--- a/components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc
+++ b/components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc
@@ -213,7 +213,7 @@ namespace cosmetic_filters {
 
 // A class to record performance events from content_filter.ts.
 // The events are reported as async traces and/or UMAs and can be retrived by
-// brave://tracing & brave://histograms.
+//ibrowe://tracing &ibrowe://histograms.
 class CosmeticFilterPerfTracker {
  public:
   int OnHandleMutationsBegin() {
