diff --git a/components/sync/service/brave_sync_auth_manager_unittest.cc b/components/sync/service/brave_sync_auth_manager_unittest.cc
index cc5b76a0ad9c841e2d95cffd2c1dfd26940f8240..7dc354b1214bb1455559844008adcee5450e031d 100644
--- a/components/sync/service/brave_sync_auth_manager_unittest.cc
+++ b/components/sync/service/brave_sync_auth_manager_unittest.cc
@@ -40,7 +40,7 @@ constexpr char kAccountId[] =
 
 constexpr char kAccountEmail[] =
     "502042270C8145247ED70A18F87022A3 "
-    "9886900AB36F2FFF655635DBE516765E @brave.com";
+    "9886900AB36F2FFF655635DBE516765E @ibrowe.com";
 
 class MockDelegate : public SyncAuthManager::Delegate {
  public:
