diff --git a/components/brave_rewards/resources/shared/components/onboarding/rewards_opt_in.tsx b/components/brave_rewards/resources/shared/components/onboarding/rewards_opt_in.tsx
index 8780aed223e81ae728e27957265043c22dde198b..695a924353c00056fd606abe180e154e69e40205 100644
--- a/components/brave_rewards/resources/shared/components/onboarding/rewards_opt_in.tsx
+++ b/components/brave_rewards/resources/shared/components/onboarding/rewards_opt_in.tsx
@@ -60,7 +60,7 @@ export function RewardsOptIn (props: Props) {
             $1: (content) =>
               <NewTabLink
                 key='link'
-                href='https://support.brave.com/hc/en-us/articles/9312922941069'
+                href='https://support.ibrowe.com/hc/en-us/articles/9312922941069'
               >
                 {content}
               </NewTabLink>
