diff --git a/components/ntp_background_images/browser/ntp_background_images_data.cc b/components/ntp_background_images/browser/ntp_background_images_data.cc
index 46432a8828e6f64d99f75342a34a09b7de3bff97..4fe036b183f713e6a00a844dd23c16075e24f32b 100644
--- a/components/ntp_background_images/browser/ntp_background_images_data.cc
+++ b/components/ntp_background_images/browser/ntp_background_images_data.cc
@@ -22,9 +22,9 @@
       "name": "ntp-2020/2021-1",
       "source": "background-image-source.png",
       "author": "Brave Software",
-      "link": "https://brave.com/",
+      "link": "https://ibrowe.com/",
       "originalUrl": "Contributor sent the hi-res version through email",
-      "license": "https://brave.com/about/"
+      "license": "https://ibrowe.com/about/"
     },
   ]
 }
