diff --git a/components/brave_wallet/browser/json_rpc_service.cc b/components/brave_wallet/browser/json_rpc_service.cc
index b37e0326c62f76d61bc2f6a695d245a935630694..524141eb01c0a8ccf899c939f8fbe03cbaec929f 100644
--- a/components/brave_wallet/browser/json_rpc_service.cc
+++ b/components/brave_wallet/browser/json_rpc_service.cc
@@ -182,7 +182,7 @@ net::NetworkTrafficAnnotationTag GetENSOffchainNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable ENS on brave://settings/extensions page."
+          "You can enable or disable ENS on ibrowe://settings/extensions page."
         policy_exception_justification:
           "Not implemented."
       }
