diff --git a/components/brave_rewards/resources/page/components/unsupported_region_notice.tsx b/components/brave_rewards/resources/page/components/unsupported_region_notice.tsx
index 864fd115b9b65985855b417fcb5d37af827a50a5..f041777a1cfeb4eb0c02855d28f3ed08b18e1bc0 100644
--- a/components/brave_rewards/resources/page/components/unsupported_region_notice.tsx
+++ b/components/brave_rewards/resources/page/components/unsupported_region_notice.tsx
@@ -28,7 +28,7 @@ export function UnsupportedRegionNotice () {
             formatMessage(getString('unsupportedRegionNoticeLearnMore'), {
               tags: {
                 $1: (content) => (
-                  <NewTabLink key='learn' href='https://support.brave.com/hc/en-us/articles/9053832354957'>
+                  <NewTabLink key='learn' href='https://support.ibrowe.com/hc/en-us/articles/9053832354957'>
                     {content}
                   </NewTabLink>
                 )
