diff --git a/components/brave_search/common/brave_search_utils.h b/components/brave_search/common/brave_search_utils.h
index 0a22319f28831026afe37bb4e48808cf0533f818..909577d64b07d8b98d1a3c04de91e55f9ee9ede3 100644
--- a/components/brave_search/common/brave_search_utils.h
+++ b/components/brave_search/common/brave_search_utils.h
@@ -18,12 +18,12 @@ inline constexpr auto kVettedHosts = base::MakeFixedFlatSet<std::string_view>(
     base::sorted_unique,
     {
         "https://safesearch.brave.com",
-        "https://safesearch.brave.software",
-        "https://safesearch.bravesoftware.com",
+        "https://safesearch.ibrowe.software",
+        "https://safesearch.ibrowesoftware.com",
         "https://search-dev-local.brave.com",
         "https://search.brave.com",
-        "https://search.brave.software",
-        "https://search.bravesoftware.com",
+        "https://search.ibrowe.software",
+        "https://search.ibrowesoftware.com",
     });
 
 bool IsAllowedHost(const GURL& url);
