diff --git a/components/skus/renderer/skus_utils_unittest.cc b/components/skus/renderer/skus_utils_unittest.cc
index fb92439ddee4a8121cc2f482072cf1d848e12e95..20ee4bfd9d1500f2ced71aceb7c5c5e73e60fff6 100644
--- a/components/skus/renderer/skus_utils_unittest.cc
+++ b/components/skus/renderer/skus_utils_unittest.cc
@@ -16,12 +16,12 @@ TEST(SkusUtilsUnitTest, IsSafeOrigin) {
   EXPECT_TRUE(skus::IsSafeOrigin(
       blink::WebSecurityOrigin::Create(GURL("https://account.brave.com"))));
   EXPECT_TRUE(skus::IsSafeOrigin(blink::WebSecurityOrigin::Create(
-      GURL("https://account.bravesoftware.com"))));
+      GURL("https://account.ibrowesoftware.com"))));
   EXPECT_TRUE(skus::IsSafeOrigin(blink::WebSecurityOrigin::Create(
-      GURL("https://account.brave.software"))));
+      GURL("https://account.ibrowe.software"))));
 
   EXPECT_FALSE(skus::IsSafeOrigin(blink::WebSecurityOrigin::Create(
-      GURL("https://a.account.brave.software"))));
+      GURL("https://a.account.ibrowe.software"))));
   EXPECT_FALSE(skus::IsSafeOrigin(
       blink::WebSecurityOrigin::Create(GURL("https://vpm.brave.com"))));
   EXPECT_FALSE(skus::IsSafeOrigin(blink::WebSecurityOrigin::Create(GURL())));
