diff --git a/components/brave_wallet/browser/json_rpc_service.cc b/components/brave_wallet/browser/json_rpc_service.cc
index c86676cabb752636ae19aceffe66bc19c4c255c1..ac1d2350f272d878aa553b49988b73932d2701b7 100644
--- a/components/brave_wallet/browser/json_rpc_service.cc
+++ b/components/brave_wallet/browser/json_rpc_service.cc
@@ -201,7 +201,7 @@ net::NetworkTrafficAnnotationTag GetENSOffchainNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable ENS on brave://settings/extensions page."
+          "You can enable or disable ENS on ibrowe://settings/extensions page."
         policy_exception_justification:
           "Not implemented."
       }
