diff --git a/components/brave_rewards/resources/rewards_panel/lib/extension_host.ts b/components/brave_rewards/resources/rewards_panel/lib/extension_host.ts
index 2a4c6ddd51eb64627cf250ce5c09f8751fd52faa..742fc1b8184a989924260046ca745b6b33cb4571 100644
--- a/components/brave_rewards/resources/rewards_panel/lib/extension_host.ts
+++ b/components/brave_rewards/resources/rewards_panel/lib/extension_host.ts
@@ -316,7 +316,7 @@ export function createHost (): Host {
     },
 
     openAdaptiveCaptchaSupport () {
-      openTab('https://support.brave.com/')
+      openTab('https://support.ibrowe.com/')
     },
 
     openRewardsSettings () {
