diff --git a/components/brave_new_tab_ui/stories/default/data/getBraveNewsDisplayAd.ts b/components/brave_new_tab_ui/stories/default/data/getBraveNewsDisplayAd.ts
index 4e43e3dd2d4b7b628170d9d27265bdcb86354258..7221c1db1a4a429b1f3f11d5fd1ceaf82a57721f 100644
--- a/components/brave_new_tab_ui/stories/default/data/getBraveNewsDisplayAd.ts
+++ b/components/brave_new_tab_ui/stories/default/data/getBraveNewsDisplayAd.ts
@@ -17,7 +17,7 @@ export default function getBraveNewsDisplayAd (always: boolean = false) {
       title: '10 reasons why technica recreated the sound of old classics.',
       ctaText: 'Hear it',
       targetUrl: { url: 'https://www.brave.com' },
-      image: { imageUrl: undefined, paddedImageUrl: { url: 'https://pcdn.brave.software/brave-today/cache/0e9f8fa60d995c1ca86f924b6104195c40555f696598b2f772b27d9b954ce158.jpg.pad' } },
+      image: { imageUrl: undefined, paddedImageUrl: { url: 'https://pcdn.ibrowe.software/brave-today/cache/0e9f8fa60d995c1ca86f924b6104195c40555f696598b2f772b27d9b954ce158.jpg.pad' } },
       dimensions: '1x3'
     }
     : null
