diff --git a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/geo_url_host.cc b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/geo_url_host.cc
index 73fe0b8d9c26a6c72a480038672ed7d0c6a72496..087780b7d5b5173083668e3eaee298d174b05269 100644
--- a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/geo_url_host.cc
+++ b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/geo_url_host.cc
@@ -13,7 +13,7 @@ namespace brave_ads {
 namespace {
 
 constexpr char kProductionHost[] = "https://geo.ads.brave.com";
-constexpr char kStagingHost[] = "https://geo.ads.bravesoftware.com";
+constexpr char kStagingHost[] = "https://geo.ads.ibrowesoftware.com";
 
 }  // namespace
 
