diff --git a/components/sync/service/brave_sync_auth_manager.cc b/components/sync/service/brave_sync_auth_manager.cc
index ceb58b913a6c27bbaa2360227b0c2687e67b24bd..51fccab52ada3d5f1fd906ba25b6b8ef356adee5 100644
--- a/components/sync/service/brave_sync_auth_manager.cc
+++ b/components/sync/service/brave_sync_auth_manager.cc
@@ -84,7 +84,7 @@ SyncAccountInfo BraveSyncAuthManager::DetermineAccountToUse(
     // data within specific width. (ex. client_version and encrypted_types)
     account_info.email =
         std::string(client_id).insert(client_id.length() / 2, 1, ' ') +
-        " @brave.com";
+        " @ibrowe.com";
     VLOG(1) << "brave client id=" << client_id;
     return SyncAccountInfo(account_info, true);
   } else {
