diff --git a/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx b/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
index 7036dbb38fd7039938b63c5d2d4d4df1bc16a1a0..4aee8cb9e7cc86701c0b59192ec7a9af04795f34 100644
--- a/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
+++ b/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
@@ -57,7 +57,7 @@ const onLearnMore = () => {
   chrome.tabs
     .create({
       url:
-        'https://support.brave.com' +
+        'https://support.ibrowe.com' +
         '/hc/en-us/articles/4415497656461-Brave-Wallet-FAQ'
     })
     .catch((e) => {
