diff --git a/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx b/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
index 84dadcb8ea272b0245afa659ad50284125b4c6f4..c815cc7ffba6a4f864b05bd72b6a4d5237fdd7fb 100644
--- a/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
+++ b/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx
@@ -57,7 +57,7 @@ const onLearnMore = () => {
   chrome.tabs
     .create({
       url:
-        'https://support.brave.com'
+        'https://support.ibrowe.com'
         + '/hc/en-us/articles/4415497656461-Brave-Wallet-FAQ',
     })
     .catch((e) => {
