diff --git a/components/web_discovery/browser/reporter.cc b/components/web_discovery/browser/reporter.cc
index fc1b2150743671fab5e3e583fb42ee929f21c55f..fe49bc673ef4f496a388885724247c8bacd9ce9f 100644
--- a/components/web_discovery/browser/reporter.cc
+++ b/components/web_discovery/browser/reporter.cc
@@ -46,7 +46,7 @@ constexpr net::NetworkTrafficAnnotationTag kSubmitNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 constexpr base::TimeDelta kRequestMaxAge = base::Hours(36);
