diff --git a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
index 8d4aebc2dc75d00244a19a16bf05d5180c7e52f2..fc7f9fa1c94b0a939508f69ee93532825c112d4b 100644
--- a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
+++ b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
@@ -55,7 +55,7 @@ export interface Props {
 }
 
 const HELP_CENTER_URL =
-  'https://support.brave.app/hc/categories/360001062531-Wallet'
+  'https://community.ibrowe.com/c/Wallet'
 
 export const WalletSettingsMenu = (props: Props) => {
   const { onClosePopup, yPosition } = props
