diff --git a/components/constants/url_constants.h b/components/constants/url_constants.h
index 25eb35ec318b58f3084eaab8ab62dd2b21a1d801..97a4b3e64ed2c471d4d6f021e3a7dc457662c7ac 100644
--- a/components/constants/url_constants.h
+++ b/components/constants/url_constants.h
@@ -15,25 +15,25 @@ inline constexpr char kRewardsUpholdSupport[] =
 inline constexpr char kP3ALearnMoreURL[] = "https://brave.com/P3A";
 inline constexpr char kP3ASettingsLink[] = "chrome://settings/privacy";
 inline constexpr char kImportDataHelpURL[] =
-    "https://support.brave.com/hc/en-us/articles/360019782291#safari";
+    "https://support.ibrowe.com/hc/en-us/articles/360019782291#safari";
 inline constexpr char kCryptoWalletsLearnMoreURL[] =
-    "https://support.brave.com/hc/en-us/articles/360034535452";
+    "https://support.ibrowe.com/hc/en-us/articles/360034535452";
 inline constexpr char kPermissionPromptLearnMoreUrl[] =
     "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 inline constexpr char kPermissionPromptHardwareAccessPrivacyRisksURL[] =
     "https://github.com/brave/brave-browser/wiki/"
     "Privacy-risks-from-allowing-sites-to-access-hardware";
 inline constexpr char kSpeedreaderLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "360045031392-What-is-SpeedReader";
 inline constexpr char kWebDiscoveryLearnMoreUrl[] =
-    "https://brave.com/privacy/browser/#web-discovery-project";
+    "https://ibrowe.com/privacy/browser/#web-discovery-project";
 inline constexpr char kBraveSearchHost[] = "search.brave.com";
 inline constexpr char kWidevineLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "360023851591-How-do-I-view-DRM-protected-content-";
 inline constexpr char kDevChannelDeprecationLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "17924707453581-How-do-I-migrate-my-Brave-Dev-data-to-another-channel-"
     "Nightly-Beta-Release-";
 
