diff --git a/components/brave_wallet/browser/brave_wallet_constants.cc b/components/brave_wallet/browser/brave_wallet_constants.cc
index de46375488f9fee20a6ad32424f17285aa174482..61d56c977bb2f6a84c5a177abfacef2aff7d178b 100644
--- a/components/brave_wallet/browser/brave_wallet_constants.cc
+++ b/components/brave_wallet/browser/brave_wallet_constants.cc
@@ -72,7 +72,7 @@ const std::string GetAssetRatioBaseURL() {
       base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII(
           switches::kAssetRatioDevUrl);
   if (ratios_url.empty()) {
-    ratios_url = "https://ratios.wallet.brave.com";
+    ratios_url = "https://ratios.wallet.ibrowe.com";
   }
 
   return ratios_url;
