diff --git a/components/brave_wallet/browser/brave_wallet_utils_unittest.cc b/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
index 7f6b5c19e3746458b043eb219e85b5309d06ef53..62afaad141ab6b9a5f28ef93a442854850a97aee 100644
--- a/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
+++ b/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
@@ -268,7 +268,7 @@ TEST(BraveWalletUtilsTest, IsEndpointUsingBraveWalletProxy) {
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
       GURL("https://ethereum-mainnet.wallet.brave.com")));
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
-      GURL("https://ethereum-mainnet.wallet.bravesoftware.com")));
+      GURL("https://ethereum-mainnet.wallet.ibrowesoftware.com")));
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
       GURL("https://ethereum-mainnet.wallet.s.brave.io")));
 
