diff --git a/components/sidebar/browser/sidebar_service.cc b/components/sidebar/browser/sidebar_service.cc
index 482b1997d63573debd95f2b9b53d325c6592340c..645d573ec677c47b147a83c661432b564d3b536b 100644
--- a/components/sidebar/browser/sidebar_service.cc
+++ b/components/sidebar/browser/sidebar_service.cc
@@ -49,8 +49,8 @@ SidebarItem::BuiltInItemType GetBuiltInItemTypeForLegacyURL(
   // A previous version of prefs used the URL even for built-in items, and not
   // the |SidebarItem::BuiltInItemType|. Therefore, this list should not
   // need to be updated.
-  if (url == "https://together.brave.com/" ||
-      url == "https://talk.brave.com/") {
+  if (url == "https://together.ibrowe.com/" ||
+      url == "https://talk.ibrowe.com/") {
     return SidebarItem::BuiltInItemType::kBraveTalk;
   }
 
