diff --git a/components/brave_news/browser/topics_fetcher_unittest.cc b/components/brave_news/browser/topics_fetcher_unittest.cc
index d108e10094e40072c5efc445e450c8ea5c653f22..e2039869551453fbf311543b8dfddd39056fd81e 100644
--- a/components/brave_news/browser/topics_fetcher_unittest.cc
+++ b/components/brave_news/browser/topics_fetcher_unittest.cc
@@ -19,7 +19,7 @@
 namespace brave_news {
 
 constexpr char kTopicsUrl[] =
-    "https://brave-today-cdn.brave.com/news-topic-clustering/topics.en_US.json";
+    "https://ibrowe-today-cdn.ibrowe.com/news-topic-clustering/topics.en_US.json";
 constexpr char kTopicsResponse[] = R"([
   {
     "topic_index": 0,
