diff --git a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
index 7cfb929834b8f7af1c8647af93f259aefa5525c2..a14ec5d14f4cbc15ba85cbfa60817427361115f8 100644
--- a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
+++ b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
@@ -101,7 +101,7 @@ export const WalletSettingsMenu = (props: Props) => {
   const onClickHelpCenter = React.useCallback(() => {
     chrome.tabs.create(
       {
-        url: 'https://support.brave.com/hc/en-us/categories/360001059151-Brave-Wallet'
+        url: 'https://support.ibrowe.com/hc/en-us/categories/360001059151-Brave-Wallet'
       },
       () => {
         if (chrome.runtime.lastError) {
