diff --git a/components/web_discovery/browser/server_config_loader.cc b/components/web_discovery/browser/server_config_loader.cc
index 279e8ebf12945ed7856582387f1cb0c80448667c..d5c2451553f1c6b7a1c4cbfb99fc949be3d56f62 100644
--- a/components/web_discovery/browser/server_config_loader.cc
+++ b/components/web_discovery/browser/server_config_loader.cc
@@ -57,7 +57,7 @@ constexpr net::NetworkTrafficAnnotationTag kNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 constexpr char kGroupPubKeysFieldName[] = "groupPubKeys";
