diff --git a/components/brave_welcome_ui/components/help-wdp/index.tsx b/components/brave_welcome_ui/components/help-wdp/index.tsx
index a600a55a3ccfb818d4de35e2f3cfe0160068843f..7d7516fe4cb18966fbf6b2c0d725bd1374d61e7d 100644
--- a/components/brave_welcome_ui/components/help-wdp/index.tsx
+++ b/components/brave_welcome_ui/components/help-wdp/index.tsx
@@ -46,7 +46,7 @@ function HelpWDP () {
         {getLocale('braveWelcomeHelpWDPDescription')}
         <span> </span>
         <a
-          href='https://support.brave.com/hc/articles/4409406835469-What-is-the-Web-Discovery-Project'
+          href='https://support.ibrowe.com/hc/articles/4409406835469-What-is-the-Web-Discovery-Project'
           target='_blank'
         >
           {getLocale('braveWelcomeHelpWDPLearnMore')}
