diff --git a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
index 4d42dccabca9013e4f58d635b683c774e64de102..cc3b990f32bfede4d1f4269b493dc40118897799 100644
--- a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
+++ b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
@@ -105,7 +105,7 @@ export const WalletSettingsMenu = (props: Props) => {
         throw new Error('Coin not supported')
     }
 
-    chrome.tabs.create({ url: `brave://settings/content/${route}` }, () => {
+    chrome.tabs.create({ url: `ibrowe://settings/content/${route}` }, () => {
       if (chrome.runtime.lastError) {
         console.error('tabs.create failed: ' + chrome.runtime.lastError.message)
       }
