diff --git a/components/webui/about_ui.cc b/components/webui/about_ui.cc
index 1a2652c33483038714850ada76cae944632fee0a..b55525d9ffea3995568d2b41130d20bf24958b4a 100644
--- a/components/webui/about_ui.cc
+++ b/components/webui/about_ui.cc
@@ -18,15 +18,15 @@ namespace brave {
 std::string ReplaceAboutUIChromeURLs(std::string chrome_urls) {
   // Replace Chrome -> Brave.
   constexpr std::string_view kChromeHeader = "Chrome URLs";
-  constexpr std::string_view kBraveHeader = "Brave URLs";
+  constexpr std::string_view kBraveHeader = "iBrowe URLs";
   constexpr std::string_view kChromePagesHeader = "List of Chrome URLs";
-  constexpr std::string_view kBravePagesHeader = "List of Brave URLs";
+  constexpr std::string_view kBravePagesHeader = "List of iBrowe URLs";
   constexpr std::string_view kChromeInternalPagesHeader =
       "List of chrome://internals pages";
   constexpr std::string_view kBraveInternalPagesHeader =
-      "List of brave://internals pages";
+      "List of ibrowe://internals pages";
   constexpr std::string_view kChromeURLList = ">chrome://";
-  constexpr std::string_view kBraveURLList = ">brave://";
+  constexpr std::string_view kBraveURLList = ">ibrowe://";
 
   RE2::GlobalReplace(&chrome_urls, kChromeHeader, kBraveHeader);
   RE2::GlobalReplace(&chrome_urls, kChromePagesHeader, kBravePagesHeader);
@@ -38,7 +38,7 @@ std::string ReplaceAboutUIChromeURLs(std::string chrome_urls) {
   auto html_lines = base::SplitStringPiece(
       chrome_urls, "\n", base::KEEP_WHITESPACE, base::SPLIT_WANT_ALL);
   constexpr auto kURLsToRemove = base::MakeFixedFlatSet<std::string_view>({
-      "brave://memories",
+      "ibrowe://memories",
   });
   // URLs in html should be sorted so it's okay to iterate over sorted
   // kURLsToRemove.
