diff --git a/components/ai_chat/core/browser/android/ai_chat_iap_subscription_android.cc b/components/ai_chat/core/browser/android/ai_chat_iap_subscription_android.cc
index 31c2b2226769c4a52bffd964f3c23c323e56337a..87af7349708af28282b80922031beae37b9f42f1 100644
--- a/components/ai_chat/core/browser/android/ai_chat_iap_subscription_android.cc
+++ b/components/ai_chat/core/browser/android/ai_chat_iap_subscription_android.cc
@@ -15,8 +15,8 @@
 
 namespace {
 
-inline constexpr char kDefaultPackage[] = "com.brave.browser";
-inline constexpr char kProductId[] = "brave.leo.monthly";
+inline constexpr char kDefaultPackage[] = "com.ibrowe.browser";
+inline constexpr char kProductId[] = "ibrowe.leo.monthly";
 
 }  // namespace
 
