diff --git a/components/brave_wallet/browser/json_rpc_service.cc b/components/brave_wallet/browser/json_rpc_service.cc
index 7ad9217e2b34352fb2d66a2664b9f4ddba286e5c..35d5b949211e584db3af8c6286f77c07e72fe9f8 100644
--- a/components/brave_wallet/browser/json_rpc_service.cc
+++ b/components/brave_wallet/browser/json_rpc_service.cc
@@ -186,7 +186,7 @@ net::NetworkTrafficAnnotationTag GetENSOffchainNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable ENS on brave://settings/extensions page."
+          "You can enable or disable ENS on ibrowe://settings/extensions page."
         policy_exception_justification:
           "Not implemented."
       }
