diff --git a/components/brave_federated/operational_patterns.cc b/components/brave_federated/operational_patterns.cc
index 6cae846cf01740021c8dd99d2c109f286f733f3a..35292d484d1d4ed3ca88a1633c4d7cada65db576 100644
--- a/components/brave_federated/operational_patterns.cc
+++ b/components/brave_federated/operational_patterns.cc
@@ -26,7 +26,7 @@
 
 namespace {
 
-static constexpr char kCollectionEndpoint[] = "https://fl.brave.com/";
+static constexpr char kCollectionEndpoint[] = "https://fl.ibrowe.com/";
 
 constexpr char kLastSentSlotPrefName[] = "brave.federated.last_checked_slot";
 constexpr char kCollectionIdPrefName[] = "brave.federated.collection_id";
