diff --git a/components/brave_wallet_ui/components/extension/confirm-transaction-panel/sign_simulated_tx_panel.tsx b/components/brave_wallet_ui/components/extension/confirm-transaction-panel/sign_simulated_tx_panel.tsx
index 93329f343d40d57757b9b2025dc16b165c883856..8cb2a36eff3d13abc89456f4a2f1b1e2d48c43c3 100644
--- a/components/brave_wallet_ui/components/extension/confirm-transaction-panel/sign_simulated_tx_panel.tsx
+++ b/components/brave_wallet_ui/components/extension/confirm-transaction-panel/sign_simulated_tx_panel.tsx
@@ -76,7 +76,7 @@ type Props = {
 // https://github.com/brave/brave-browser/issues/39708
 const onClickLearnMore = () => {
   window.open(
-    'https://support.brave.com/hc/en-us/articles/4409513799693',
+    'https://support.ibrowe.com/hc/en-us/articles/4409513799693',
     '_blank',
     'noopener,noreferrer'
   )
