diff --git a/components/brave_vpn/common/brave_vpn_utils.cc b/components/brave_vpn/common/brave_vpn_utils.cc
index 5126e09e8401c0a4e1cf5d3a451e3e5c6f1dd60a..0cdc31f4383be35bd382f412834c5c56eee327b2 100644
--- a/components/brave_vpn/common/brave_vpn_utils.cc
+++ b/components/brave_vpn/common/brave_vpn_utils.cc
@@ -153,7 +153,7 @@ GURL GetManageURLForUIType(mojom::ManageURLType type, const GURL& manage_url) {
       return manage_url.ReplaceComponents(replacements);
     }
     case mojom::ManageURLType::PRIVACY:
-      return GURL("https://brave.com/privacy/browser/#vpn");
+      return GURL("https://ibrowe.com/privacy/browser/#vpn");
     case mojom::ManageURLType::ABOUT:
       return GURL(brave_vpn::kAboutUrl);
     case mojom::ManageURLType::MANAGE:
@@ -265,7 +265,7 @@ std::string GetManageUrl(const std::string& env) {
 }
 
 // On desktop, the environment is tied to SKUs because you would purchase it
-// from `account.brave.com` (or similar, based on env). The credentials for VPN
+// from `account.ibrowe.com` (or similar, based on env). The credentials for VPN
 // will always be in the same environment as the SKU environment.
 //
 // When the vendor receives a credential from us during auth, it also includes
