diff --git a/components/debounce/content/browser/debounce_navigation_throttle.cc b/components/debounce/content/browser/debounce_navigation_throttle.cc
index 5700ca9900cb9f781aebd2a8f185a8022abedd8b..e882b19424df31a29e21051bf07e339c3edb6674 100644
--- a/components/debounce/content/browser/debounce_navigation_throttle.cc
+++ b/components/debounce/content/browser/debounce_navigation_throttle.cc
@@ -87,7 +87,7 @@ std::unique_ptr<DebounceNavigationThrottle>
 DebounceNavigationThrottle::MaybeCreateThrottleFor(
     NavigationHandle* navigation_handle,
     DebounceService* debounce_service) {
-  // If debouncing is disabled in brave://flags, debounce service will
+  // If debouncing is disabled in ibrowe://flags, debounce service will
   // never be created (will be null) so we won't create the throttle
   // either. Caller must nullcheck this.
   if (!debounce_service) {
