diff --git a/components/variations/command_line_utils.cc b/components/variations/command_line_utils.cc
index 8994aec5399d7f454b1db98e83947344348bce20..68e9c62fbd9eb6c71b92823522002783507325e1 100644
--- a/components/variations/command_line_utils.cc
+++ b/components/variations/command_line_utils.cc
@@ -21,7 +21,7 @@ namespace {
 // seed and uploads it to a URL with the following template, where $1 is the
 // pull request number.
 constexpr char kVariationsPrTestSeedUrlTemplate[] =
-    "https://griffin.brave.com/pull/$1/seed";
+    "https://griffin.ibrowe.com/pull/$1/seed";
 
 }  // namespace
 
