diff --git a/components/brave_ads/browser/ads_service_impl.cc b/components/brave_ads/browser/ads_service_impl.cc
index b007e7cd337b022b9317bba05a7cda6b1f12c7c5..4240c6aff80512f82c599d7ab522ef03123da116 100644
--- a/components/brave_ads/browser/ads_service_impl.cc
+++ b/components/brave_ads/browser/ads_service_impl.cc
@@ -158,7 +158,7 @@ net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable this feature by visiting brave://rewards."
+          "You can enable or disable this feature by visiting ibrowe://rewards."
         policy_exception_justification:
           "Not implemented."
       }
