diff --git a/components/webui/about_ui.cc b/components/webui/about_ui.cc
index 1a2652c33483038714850ada76cae944632fee0a..1b87c4547577c140e8f73203ecaefb63188ed652 100644
--- a/components/webui/about_ui.cc
+++ b/components/webui/about_ui.cc
@@ -24,9 +24,9 @@ std::string ReplaceAboutUIChromeURLs(std::string chrome_urls) {
   constexpr std::string_view kChromeInternalPagesHeader =
       "List of chrome://internals pages";
   constexpr std::string_view kBraveInternalPagesHeader =
-      "List of brave://internals pages";
+      "List of ibrowe://internals pages";
   constexpr std::string_view kChromeURLList = ">chrome://";
-  constexpr std::string_view kBraveURLList = ">brave://";
+  constexpr std::string_view kBraveURLList = ">ibrowe://";
 
   RE2::GlobalReplace(&chrome_urls, kChromeHeader, kBraveHeader);
   RE2::GlobalReplace(&chrome_urls, kChromePagesHeader, kBravePagesHeader);
@@ -38,7 +38,7 @@ std::string ReplaceAboutUIChromeURLs(std::string chrome_urls) {
   auto html_lines = base::SplitStringPiece(
       chrome_urls, "\n", base::KEEP_WHITESPACE, base::SPLIT_WANT_ALL);
   constexpr auto kURLsToRemove = base::MakeFixedFlatSet<std::string_view>({
-      "brave://memories",
+      "ibrowe://memories",
   });
   // URLs in html should be sorted so it's okay to iterate over sorted
   // kURLsToRemove.
