diff --git a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
index 1d9308a31484886aae1e8d238a9b14a89b1c9a03..f0518b1e8e48fbb35271a18bd72f12a9c4501f69 100644
--- a/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
+++ b/components/brave_wallet_ui/components/desktop/wallet-menus/wallet_settings_menu.tsx
@@ -101,7 +101,7 @@ export const WalletSettingsMenu = (props: Props) => {
   const onClickHelpCenter = React.useCallback(() => {
     chrome.tabs.create(
       {
-        url: 'https://support.brave.com/hc/categories/360001062531-Wallet',
+        url: 'https://ibrowe.com/ibrowe-wallet',
       },
       () => {
         if (chrome.runtime.lastError) {
