diff --git a/components/brave_wallet/browser/brave_wallet_utils.cc b/components/brave_wallet/browser/brave_wallet_utils.cc
index 09d017e9432681de7d2553f1813d64f7fe78e9df..9769ea25ec6e7e3fc947b8ee65f109e4ece55486 100644
--- a/components/brave_wallet/browser/brave_wallet_utils.cc
+++ b/components/brave_wallet/browser/brave_wallet_utils.cc
@@ -202,7 +202,7 @@ bool EncodeStringArrayInternal(base::span<const StringType> input,
 
 bool IsEndpointUsingBraveWalletProxy(const GURL& url) {
   return url.DomainIs("wallet.brave.com") ||
-         url.DomainIs("wallet.bravesoftware.com") ||
+         url.DomainIs("wallet.ibrowesoftware.com") ||
          url.DomainIs("wallet.s.brave.io");
 }
 
