diff --git a/components/sync/service/brave_sync_auth_manager.cc b/components/sync/service/brave_sync_auth_manager.cc
index d03d8ca746ad15406205fcb48d96ab01ff212bd4..c3f11ea8b97d5270db0a3cdb2895bdf1470bf71e 100644
--- a/components/sync/service/brave_sync_auth_manager.cc
+++ b/components/sync/service/brave_sync_auth_manager.cc
@@ -84,7 +84,7 @@ SyncAccountInfo BraveSyncAuthManager::DetermineAccountToUse() const {
     // data within specific width. (ex. client_version and encrypted_types)
     account_info.email =
         std::string(client_id).insert(client_id.length() / 2, 1, ' ') +
-        " @brave.com";
+        " @ibrowe.com";
     VLOG(1) << "brave client id=" << client_id;
     return SyncAccountInfo(account_info, true);
   } else {
