diff --git a/components/brave_new_tab_ui/containers/newTab/index.tsx b/components/brave_new_tab_ui/containers/newTab/index.tsx
index 75e3acb04035dc85513d310340d9bbb23bb02eef..5d7a1421584b4785e1ceedaecd4513a00c4c7a57 100644
--- a/components/brave_new_tab_ui/containers/newTab/index.tsx
+++ b/components/brave_new_tab_ui/containers/newTab/index.tsx
@@ -444,24 +444,13 @@ class NewTabPage extends React.Component<Props, State> {
     const {
       widgetStackOrder,
       braveRewardsSupported,
-      braveTalkSupported,
       showRewards,
-      showBraveTalk,
-      showBraveVPN,
     } = this.props.newTabData
 
     const lookup: { [p: string]: { display: boolean, render: any } } = {
       'rewards': {
         display: braveRewardsSupported && showRewards,
         render: this.renderRewardsWidget.bind(this)
-      },
-      'braveVPN': {
-        display: this.braveVPNSupported && showBraveVPN,
-        render: this.renderBraveVPNWidget
-      },
-      'braveTalk': {
-        display: braveTalkSupported && showBraveTalk,
-        render: this.renderBraveTalkWidget.bind(this)
       }
     }
 
