diff --git a/components/constants/network_constants.h b/components/constants/network_constants.h
index 352f119fb641854d379f7c46da1e6533d4dd020b..b238c7c4eedcb11a58190b634d0e3fe29fa43c0d 100644
--- a/components/constants/network_constants.h
+++ b/components/constants/network_constants.h
@@ -6,18 +6,18 @@
 #ifndef BRAVE_COMPONENTS_CONSTANTS_NETWORK_CONSTANTS_H_
 #define BRAVE_COMPONENTS_CONSTANTS_NETWORK_CONSTANTS_H_
 
-inline constexpr char kExtensionUpdaterDomain[] = "extensionupdater.brave.com";
+inline constexpr char kExtensionUpdaterDomain[] = "extensionupdater.ibrowe.com";
 
 inline constexpr char kBraveUsageStandardPath[] = "/1/usage/brave-core";
 
 inline constexpr char kBraveReferralsInitPath[] = "/promo/initialize/nonua";
 inline constexpr char kBraveReferralsActivityPath[] = "/promo/activity";
 
-inline constexpr char kBraveSafeBrowsing2Proxy[] = "safebrowsing2.brave.com";
-inline constexpr char kBraveSafeBrowsingSslProxy[] = "sb-ssl.brave.com";
-inline constexpr char kBraveRedirectorProxy[] = "redirector.brave.com";
-inline constexpr char kBraveClients4Proxy[] = "clients4.brave.com";
-inline constexpr char kBraveStaticProxy[] = "static1.brave.com";
+inline constexpr char kBraveSafeBrowsing2Proxy[] = "safebrowsing2.ibrowe.com";
+inline constexpr char kBraveSafeBrowsingSslProxy[] = "sb-ssl.ibrowe.com";
+inline constexpr char kBraveRedirectorProxy[] = "redirector.ibrowe.com";
+inline constexpr char kBraveClients4Proxy[] = "clients4.ibrowe.com";
+inline constexpr char kBraveStaticProxy[] = "static1.ibrowe.com";
 
 inline constexpr char kAutofillPrefix[] = "https://www.gstatic.com/autofill/*";
 inline constexpr char kClients4Prefix[] = "*://clients4.google.com/";
