diff --git a/components/brave_rewards/resources/page/components/provider_redirect_modal.tsx b/components/brave_rewards/resources/page/components/provider_redirect_modal.tsx
index 96df1123ae3fc8fc4ee7a9dbca4c6512874a818f..eb9668748d30eda231848e087d2b7c887185a06b 100644
--- a/components/brave_rewards/resources/page/components/provider_redirect_modal.tsx
+++ b/components/brave_rewards/resources/page/components/provider_redirect_modal.tsx
@@ -44,7 +44,7 @@ export function ProviderRedirectModal () {
           id={'redirect-modal-device-limit-reached'}
           errorText={[getString('redirectModalDeviceLimitReachedText').replace('$1', providerName)]}
           titleText={getString('redirectModalDeviceLimitReachedTitle')}
-          learnMore={'https://support.brave.com/hc/en-us/articles/360056508071'}
+          learnMore={'https://support.ibrowe.com/hc/en-us/articles/360056508071'}
           buttonText={getString('redirectModalClose')}
           walletType={walletType}
           onClick={actions.hideRedirectModal}
@@ -59,7 +59,7 @@ export function ProviderRedirectModal () {
             getString('redirectModalFlaggedWalletText2'),
             getString('redirectModalFlaggedWalletText3'),
             getString('redirectModalFlaggedWalletText4')]}
-          errorTextLink={'https://support.brave.com/hc/en-us/articles/4494596374925'}
+          errorTextLink={'https://support.ibrowe.com/hc/en-us/articles/4494596374925'}
           titleText={getString('redirectModalFlaggedWalletTitle')}
           buttonText={getString('redirectModalClose')}
           walletType={walletType}
@@ -83,7 +83,7 @@ export function ProviderRedirectModal () {
           id={'redirect-modal-mismatched-countries'}
           errorText={[getString('redirectModalMismatchedCountriesText').replace('$1', providerName)]}
           titleText={getString('redirectModalMismatchedCountriesTitle')}
-          learnMore={'https://support.brave.com/hc/en-us/articles/9809690466061'}
+          learnMore={'https://support.ibrowe.com/hc/en-us/articles/9809690466061'}
           buttonText={getString('redirectModalClose')}
           walletType={walletType}
           onClick={actions.hideRedirectModal}
@@ -95,7 +95,7 @@ export function ProviderRedirectModal () {
           id={'redirect-modal-mismatched-provider-accounts'}
           errorText={[getString('redirectModalMismatchedProviderAccountsText').replace('$1', providerName)]}
           titleText={getString('redirectModalMismatchedProviderAccountsTitle')}
-          learnMore={'https://support.brave.com/hc/en-us/articles/360034841711-What-is-a-verified-wallet-'}
+          learnMore={'https://support.ibrowe.com/hc/en-us/articles/360034841711-What-is-a-verified-wallet-'}
           buttonText={getString('redirectModalClose')}
           walletType={walletType}
           onClick={actions.hideRedirectModal}
@@ -123,7 +123,7 @@ export function ProviderRedirectModal () {
             getString('redirectModalRegionNotSupportedText1').replaceAll('$1', providerName),
             getString('redirectModalRegionNotSupportedText2')]}
           titleText={getString('redirectModalRegionNotSupportedTitle')}
-          errorTextLink={'https://support.brave.com/hc/en-us/articles/6539887971469'}
+          errorTextLink={'https://support.ibrowe.com/hc/en-us/articles/6539887971469'}
           buttonText={getString('redirectModalClose')}
           walletType={walletType}
           onClick={actions.hideRedirectModal}
