diff --git a/components/brave_welcome_ui/components/help-improve/index.tsx b/components/brave_welcome_ui/components/help-improve/index.tsx
index 9ee335f51e30f37afb3e25138fa1572b51652aee..40c049c117f4d7f0745f50abe9cb79ae5aaa90c6 100644
--- a/components/brave_welcome_ui/components/help-improve/index.tsx
+++ b/components/brave_welcome_ui/components/help-improve/index.tsx
@@ -19,7 +19,7 @@ interface InputCheckboxProps {
 }
 
 const changeSettingsNote = formatLocale('braveWelcomeChangeSettingsNote', {
-  $1: content => <a href="brave://settings/privacy" onClick={() => {
+  $1: content => <a href="ibrowe://settings/privacy" onClick={() => {
     WelcomeBrowserProxyImpl.getInstance().openSettingsPage()
   }}>
     {content}
