diff --git a/components/brave_new_tab_ui/components/default/braveNews/cards/cardOptIn.tsx b/components/brave_new_tab_ui/components/default/braveNews/cards/cardOptIn.tsx
index 43f664c8ab4a8645f72021023e18bf0b9f0de411..97b358d2650e7193f653dcddf01a1ec43e6cc043 100644
--- a/components/brave_new_tab_ui/components/default/braveNews/cards/cardOptIn.tsx
+++ b/components/brave_new_tab_ui/components/default/braveNews/cards/cardOptIn.tsx
@@ -29,7 +29,7 @@ export default function IntroCard (props: Props) {
         </Card.Paragraph>
         <Card.Paragraph>
           {descriptionTwoTextParts.beforeTag}
-          <a href={'https://brave.com/privacy/browser/'}>
+          <a href={'https://ibrowe.com/privacy/browser/'}>
             {descriptionTwoTextParts.duringTag}
           </a>
           {descriptionTwoTextParts.afterTag}
