diff --git a/components/sidebar/browser/sidebar_item.h b/components/sidebar/browser/sidebar_item.h
index e9e1ff431c9fc15121a3cebf5bb76cf10e56eb27..3a399886a4f91c407a756b0540fcb753c426ae58 100644
--- a/components/sidebar/browser/sidebar_item.h
+++ b/components/sidebar/browser/sidebar_item.h
@@ -20,7 +20,7 @@ struct SidebarItem {
   // items.
   enum class BuiltInItemType {
     kNone = 0,
-    kBraveTalk,
+    // kBraveTalk,
     kWallet,
     kBookmarks,
     kReadingList,
