diff --git a/components/brave_referrals/browser/brave_referrals_service.cc b/components/brave_referrals/browser/brave_referrals_service.cc
index b0b4a721f6a08666cdacc7a12bc503197a610d40..2b15bf5fd28b0d73c3c508545c332487849afbd5 100644
--- a/components/brave_referrals/browser/brave_referrals_service.cc
+++ b/components/brave_referrals/browser/brave_referrals_service.cc
@@ -71,7 +71,7 @@ void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
 
 #if BUILDFLAG(IS_MAC)
 std::string ReadPromoCodeFromXattr() {
-  static constexpr char kRefCodeAttr[] = "com.brave.refcode";
+  static constexpr char kRefCodeAttr[] = "com.ibrowe.refcode";
   base::FilePath bundle_path = base::apple::OuterBundlePath();
   std::vector<char> value;
   int result_errno =
