diff --git a/components/brave_referrals/browser/brave_referrals_service.cc b/components/brave_referrals/browser/brave_referrals_service.cc
index 389440d22d116291702d368bc9e8dac0f8ef3fdf..56ff5cb3ca356c046fd4595bc44526f16947f03d 100644
--- a/components/brave_referrals/browser/brave_referrals_service.cc
+++ b/components/brave_referrals/browser/brave_referrals_service.cc
@@ -70,7 +70,7 @@ void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
 
 #if BUILDFLAG(IS_MAC)
 std::string ReadPromoCodeFromXattr() {
-  static constexpr char kRefCodeAttr[] = "com.brave.refcode";
+  static constexpr char kRefCodeAttr[] = "com.ibrowe.refcode";
   base::FilePath bundle_path = base::apple::OuterBundlePath();
   std::vector<char> value;
   int result_errno =
