diff --git a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
index 0beed7b14a414e5648122982383ed12c94ba56a5..078163a26806bc1fe3a3167c03df195c952767a1 100644
--- a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
+++ b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
@@ -31,7 +31,7 @@ const TermsOfUseText: React.FC<{}> = () => {
   const text = formatLocale('braveWalletTermsOfServiceCheckboxText', {
     $1: (content) => (
       <TermsLink
-        href='https://brave.com/terms-of-use/'
+        href='https://ibrowe.com/terms-of-use/'
         target='_blank'
         rel='noopener noreferrer'
         onClick={
