diff --git a/components/ai_chat/core/browser/constants.h b/components/ai_chat/core/browser/constants.h
index 694b53d503fdee71fbf39456bc05a1fe6ee877a4..197650b90a47524daf2c9ca2855ddc903d7aaa67 100644
--- a/components/ai_chat/core/browser/constants.h
+++ b/components/ai_chat/core/browser/constants.h
@@ -31,7 +31,7 @@ inline constexpr auto kPrintPreviewRetrievalHosts =
 
 inline constexpr uint8_t kMaxPreviewPages = 20;
 inline constexpr char kLeoModelSupportUrl[] =
-    "https://support.brave.com/hc/en-us/articles/26727364100493-"
+    "https://support.ibrowe.com/hc/en-us/articles/26727364100493-"
     "What-are-the-differences-between-Leo-s-AI-Models";
 
 // Upon registering a custom model, users have the ability to explicitly
