diff --git a/components/brave_wallet/browser/brave_wallet_utils_unittest.cc b/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
index 9d3c0b72e0817078bafa921e1f2b3e6e7367a4ae..b50bd5709c3705adb0a59a875f42fb2e73c727e3 100644
--- a/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
+++ b/components/brave_wallet/browser/brave_wallet_utils_unittest.cc
@@ -265,7 +265,7 @@ TEST(BraveWalletUtilsTest, IsEndpointUsingBraveWalletProxy) {
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
       GURL("https://ethereum-mainnet.wallet.brave.com")));
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
-      GURL("https://ethereum-mainnet.wallet.bravesoftware.com")));
+      GURL("https://ethereum-mainnet.wallet.ibrowesoftware.com")));
   EXPECT_TRUE(IsEndpointUsingBraveWalletProxy(
       GURL("https://ethereum-mainnet.wallet.s.brave.io")));
 
