diff --git a/components/brave_ads/core/public/serving/targeting/condition_matcher/condition_matcher_util.h b/components/brave_ads/core/public/serving/targeting/condition_matcher/condition_matcher_util.h
index 2e93752f3e019165f987359a351e1c106ac179d5..a1f60ab881fc429deba3dbb36c65dfe10a656558 100644
--- a/components/brave_ads/core/public/serving/targeting/condition_matcher/condition_matcher_util.h
+++ b/components/brave_ads/core/public/serving/targeting/condition_matcher/condition_matcher_util.h
@@ -114,7 +114,7 @@ class PrefProviderInterface;
 // For example, the following condition matcher will serve an ad if the
 // `brave.new_tab_page.show_brave_news` pref path does not exist, the browser
 // was installed 3 or more days ago, the user has fewer than 7 bookmarks, the
-// chosen search provider is Startpage, and both the brave://settings/languages
+// chosen search provider is Startpage, and both the ibrowe://settings/languages
 // preferred website language and the operating system language are Deutsch:
 //
 //  "conditionMatchers": [
@@ -189,7 +189,7 @@ class PrefProviderInterface;
 //       ]
 //
 // NOTE: To identify condition matchers, first create a copy of your `Local
-// State` and `Default/Preferences` files. Next, change a brave://setting or
+// State` and `Default/Preferences` files. Next, change a ibrowe://setting or
 // enable a feature, quit the browser and then compare the original and modified
 // versions to determine which key/value pairs are required. Invalid or
 // malformed condition matchers will be logged to the console, they are not
