diff --git a/components/web_discovery/browser/double_fetcher.cc b/components/web_discovery/browser/double_fetcher.cc
index cabc1d183ae8eb2eaf2944affdf228acc41c2287..5b4358cc6eba6c10e620070914b53ae8192cc318 100644
--- a/components/web_discovery/browser/double_fetcher.cc
+++ b/components/web_discovery/browser/double_fetcher.cc
@@ -53,7 +53,7 @@ constexpr net::NetworkTrafficAnnotationTag kFetchNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 }  // namespace
