diff --git a/components/web_discovery/browser/credential_manager.cc b/components/web_discovery/browser/credential_manager.cc
index d3e5d7c4d08154942b2acbab1def368ed360be71..02c5f0d501e4a483f3cb152c8bf049c56db6133c 100644
--- a/components/web_discovery/browser/credential_manager.cc
+++ b/components/web_discovery/browser/credential_manager.cc
@@ -59,7 +59,7 @@ constexpr net::NetworkTrafficAnnotationTag kJoinNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 }  // namespace
