diff --git a/components/brave_new_tab_ui/containers/newTab/settings.tsx b/components/brave_new_tab_ui/containers/newTab/settings.tsx
index 9465ddce9b766e632786386c8a5adf0b71463926..254109a4a4cab43551bb061a3452fd9ecd02efcb 100644
--- a/components/brave_new_tab_ui/containers/newTab/settings.tsx
+++ b/components/brave_new_tab_ui/containers/newTab/settings.tsx
@@ -40,8 +40,8 @@ const Sidebar = styled(Navigation)`
   margin-inline-start: -24px;
 `
 
-const SidebarItem = styled(NavigationItem)`
-  text-transform: capitalize;
+const SidebarItem = styled(NavigationItem)<{ disableCapitalize?: boolean }>`
+  text-transform: ${props => props.disableCapitalize ? 'none' : 'capitalize'};
 `
 
 export interface Props {
@@ -147,9 +147,17 @@ export default function Settings(props: Props) {
     <SettingsContent id='settingsBody'>
       <Sidebar id="sidebar">
         <NavigationMenu>
-          {allowedTabTypes.map(tabType => <SidebarItem key={tabType} icon={tabIcons[tabType]} isCurrent={activeTab === tabType} onClick={() => changeTab(tabType)}>
-            {getLocale(tabTranslationKeys[tabType])}
-          </SidebarItem>)}
+          {allowedTabTypes.map(tabType => (
+              <SidebarItem
+                  key={tabType}
+                  icon={tabIcons[tabType]}
+                  isCurrent={activeTab === tabType}
+                  onClick={() => changeTab(tabType)}
+                  disableCapitalize={tabType === TabType.BraveNews || tabType === TabType.BraveStats}
+              >
+                {getLocale(tabTranslationKeys[tabType])}
+              </SidebarItem>
+          ))}
         </NavigationMenu>
       </Sidebar>
       <SettingsFeatureBody id='content'>
