diff --git a/components/brave_rewards/core/engine/test/data/publisher_info_db_v27.sql b/components/brave_rewards/core/engine/test/data/publisher_info_db_v27.sql
index 1fd16b5ccc08c158d18dad784f935d4c8503e651..1bc1b5b8d42fa1fc9348d761c0f806974f583d1f 100644
--- a/components/brave_rewards/core/engine/test/data/publisher_info_db_v27.sql
+++ b/components/brave_rewards/core/engine/test/data/publisher_info_db_v27.sql
@@ -199,7 +199,7 @@ INSERT INTO "publisher_info" VALUES ('laurenwags.github.io',0,'laurenwags.github
 INSERT INTO "server_publisher_info" VALUES ('laurenwags.github.io',2,0,'096f1756-9406-4d9b-94c8-5bb566c2ea5f');
 INSERT INTO "server_publisher_info" VALUES ('wikipedia.org',0,0,'');
 INSERT INTO "server_publisher_info" VALUES ('site1.com',1,0,'');
-INSERT INTO "server_publisher_banner" VALUES ('laurenwags.github.io','Staging Banner Test','Lorem ipsum dolor sit amet, sale homero neglegentur ei vix, quo no tacimates vituperatoribus. Per elit luptatum temporibus ad, cibo minimum quaerendum no nec, atqui corpora complectitur te sed. Per ne vulputate neglegentur, id nec alia affert aperiri. Ea melius deserunt pro. Officiis sadipscing at nam, adhuc populo atomorum est.','chrome://rewards-image/https://rewards-stg.bravesoftware.com/xrEJASVGN9nQ5zJUnmoCxjEE','chrome://rewards-image/https://rewards-stg.bravesoftware.com/8eT9LXcpK3D795YHxvDdhrmg');
+INSERT INTO "server_publisher_banner" VALUES ('laurenwags.github.io','Staging Banner Test','Lorem ipsum dolor sit amet, sale homero neglegentur ei vix, quo no tacimates vituperatoribus. Per elit luptatum temporibus ad, cibo minimum quaerendum no nec, atqui corpora complectitur te sed. Per ne vulputate neglegentur, id nec alia affert aperiri. Ea melius deserunt pro. Officiis sadipscing at nam, adhuc populo atomorum est.','chrome://rewards-image/https://rewards-stg.ibrowesoftware.com/xrEJASVGN9nQ5zJUnmoCxjEE','chrome://rewards-image/https://rewards-stg.ibrowesoftware.com/8eT9LXcpK3D795YHxvDdhrmg');
 INSERT INTO "server_publisher_banner" VALUES ('site1.com','Site 1 Banner','Banner test','','');
 INSERT INTO "server_publisher_links" VALUES ('laurenwags.github.io','twitch','https://www.twitch.tv/laurenwags');
 INSERT INTO "server_publisher_links" VALUES ('laurenwags.github.io','twitter','https://twitter.com/bravelaurenwags');
