diff --git a/components/brave_vpn/common/brave_vpn_utils.cc b/components/brave_vpn/common/brave_vpn_utils.cc
index 5126e09e8401c0a4e1cf5d3a451e3e5c6f1dd60a..f91c53c8ba7d67d9451c5a888774bee0c8c6b5af 100644
--- a/components/brave_vpn/common/brave_vpn_utils.cc
+++ b/components/brave_vpn/common/brave_vpn_utils.cc
@@ -48,9 +48,9 @@ void RegisterVPNLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterTimePref(prefs::kBraveVPNLastCredentialExpiry, {});
   registry->RegisterBooleanPref(prefs::kBraveVPNLocalStateMigrated, false);
   registry->RegisterTimePref(prefs::kBraveVPNSessionExpiredDate, {});
-#if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD)
-  registry->RegisterBooleanPref(prefs::kBraveVPNWireguardEnabled, false);
-#endif
+// #if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD)
+//   registry->RegisterBooleanPref(prefs::kBraveVPNWireguardEnabled, false);
+// #endif
 #if BUILDFLAG(IS_MAC)
   registry->RegisterBooleanPref(prefs::kBraveVPNOnDemandEnabled, false);
 #endif
@@ -112,16 +112,17 @@ bool IsBraveVPNWireguardEnabled(PrefService* local_state) {
     return false;
   }
 
-#if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD)
-  auto enabled = local_state->GetBoolean(prefs::kBraveVPNWireguardEnabled);
-#if BUILDFLAG(IS_MAC)
-  enabled = enabled && base::FeatureList::IsEnabled(
-                           brave_vpn::features::kBraveVPNEnableWireguardForOSX);
-#endif  // BUILDFLAG(IS_MAC)
-  return enabled;
-#else
+// #if BUILDFLAG(ENABLE_BRAVE_VPN_WIREGUARD)
+//   auto enabled = local_state->GetBoolean(prefs::kBraveVPNWireguardEnabled);
+// #if BUILDFLAG(IS_MAC)
+//   enabled = enabled && base::FeatureList::IsEnabled(
+//                            brave_vpn::features::kBraveVPNEnableWireguardForOSX);
+// #endif  // BUILDFLAG(IS_MAC)
+//   return enabled;
+// #else
+//   return false;
+// #endif
   return false;
-#endif
 }
 
 #if BUILDFLAG(IS_WIN)
@@ -153,7 +154,7 @@ GURL GetManageURLForUIType(mojom::ManageURLType type, const GURL& manage_url) {
       return manage_url.ReplaceComponents(replacements);
     }
     case mojom::ManageURLType::PRIVACY:
-      return GURL("https://brave.com/privacy/browser/#vpn");
+      return GURL("https://ibrowe.com/privacy/browser/#vpn");
     case mojom::ManageURLType::ABOUT:
       return GURL(brave_vpn::kAboutUrl);
     case mojom::ManageURLType::MANAGE:
@@ -212,7 +213,7 @@ bool IsBraveVPNDisabledByPolicy(PrefService* prefs) {
   return prefs->FindPreference(prefs::kManagedBraveVPNDisabled) &&
   // Need to investigate more about this.
   // IsManagedPreference() gives false on macOS when it's configured by
-  // "defaults write com.brave.Browser.beta BraveVPNDisabled -bool true".
+  // "defaults write com.ibrowe.Browser.beta BraveVPNDisabled -bool true".
   // As kManagedBraveVPNDisabled is false by default and only can be set
   // by policy, I think skipping this condition checking will be fine.
 #if !BUILDFLAG(IS_MAC)
@@ -265,7 +266,7 @@ std::string GetManageUrl(const std::string& env) {
 }
 
 // On desktop, the environment is tied to SKUs because you would purchase it
-// from `account.brave.com` (or similar, based on env). The credentials for VPN
+// from `account.ibrowe.com` (or similar, based on env). The credentials for VPN
 // will always be in the same environment as the SKU environment.
 //
 // When the vendor receives a credential from us during auth, it also includes
