diff --git a/components/brave_welcome_ui/components/help-improve/index.tsx b/components/brave_welcome_ui/components/help-improve/index.tsx
index d8197eb7831f5c1144e5093ca79e5491118ecf99..bf69826f87b9b34c1232815236baa830613a8eef 100644
--- a/components/brave_welcome_ui/components/help-improve/index.tsx
+++ b/components/brave_welcome_ui/components/help-improve/index.tsx
@@ -80,7 +80,7 @@ function HelpImprove () {
             isChecked={isMetricsReportingEnabled}
           >
             {diagnosticReportsLabel.beforeTag}
-            <a href="https://support.brave.com/hc/en-us/articles/360017905872-How-do-I-enable-or-disable-automatic-crash-reporting" target="_blank">
+            <a href="https://support.ibrowe.com/hc/en-us/articles/360017905872-How-do-I-enable-or-disable-automatic-crash-reporting" target="_blank">
               {diagnosticReportsLabel.duringTag}
             </a>
             {diagnosticReportsLabel.afterTag}
@@ -91,7 +91,7 @@ function HelpImprove () {
             isChecked={isP3AEnabled}
           >
             {braveProductUsageDataLabel.beforeTag}
-            <a href="https://support.brave.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-" target="_blank">
+            <a href="https://support.ibrowe.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-" target="_blank">
               {braveProductUsageDataLabel.duringTag}
             </a>
             {braveProductUsageDataLabel.afterTag}
@@ -115,7 +115,7 @@ function HelpImprove () {
             {changeSettingsNote.afterTag}
             <span>
               {readPrivacyPolicy.beforeTag}
-              <a href="https://brave.com/privacy/browser" target="_blank">
+              <a href="https://ibrowe.com/privacy/browser" target="_blank">
                 {readPrivacyPolicy.duringTag}
               </a>
               {readPrivacyPolicy.afterTag}
