diff --git a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
index d207e2c4bd0c7135cf9c159fa5eab7705351b81b..3363163898d4ab4f02df064befe99bd98b0171c8 100644
--- a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
+++ b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
@@ -144,7 +144,7 @@ std::string formatSkusStateValue(const base::Time start_time,
 #if defined(OFFICIAL_BUILD)
   const std::string skus_domain = "leo.brave.com";
 #else
-  const std::string skus_domain = "leo.bravesoftware.com";
+  const std::string skus_domain = "leo.ibrowesoftware.com";
 #endif
 
   replacements.push_back(skus_domain);
