diff --git a/components/brave_referrals/browser/brave_referrals_service.cc b/components/brave_referrals/browser/brave_referrals_service.cc
index 968da66ad00021c45c66d5142090fa07fe41a982..5640f3849082b2575076149a7472d5b35af3ba66 100644
--- a/components/brave_referrals/browser/brave_referrals_service.cc
+++ b/components/brave_referrals/browser/brave_referrals_service.cc
@@ -75,7 +75,7 @@ void DeletePromoCodeFile(const base::FilePath& promo_code_file) {
 
 #if BUILDFLAG(IS_MAC)
 std::string ReadPromoCodeFromXattr() {
-  static constexpr char kRefCodeAttr[] = "com.brave.refcode";
+  static constexpr char kRefCodeAttr[] = "com.ibrowe.refcode";
   base::FilePath bundle_path = base::apple::OuterBundlePath();
   std::vector<char> value;
   int result_errno =
