diff --git a/components/brave_vpn/common/brave_vpn_constants.h b/components/brave_vpn/common/brave_vpn_constants.h
index bba4dc98bd0cc75d83b4380be0ee30dc661ae6de..6d261a698cd0235a96484e14daeea8c004d20e09 100644
--- a/components/brave_vpn/common/brave_vpn_constants.h
+++ b/components/brave_vpn/common/brave_vpn_constants.h
@@ -14,7 +14,7 @@ inline constexpr char kManageUrlProd[] = "https://account.brave.com/account/";
 inline constexpr char kManageUrlStaging[] =
     "https://account.bravesoftware.com/account/";
 inline constexpr char kManageUrlDev[] =
-    "https://account.brave.software/account/";
+    "https://account.ibrowe.software/account/";
 
 // TODO(simonhong): Update when vpn feedback url is ready.
 inline constexpr char kFeedbackUrl[] = "https://support.brave.app/";
