diff --git a/components/brave_extension/extension/brave_extension/background/webDiscoveryProject.ts b/components/brave_extension/extension/brave_extension/background/webDiscoveryProject.ts
index 178c72b817a044484fd9d73defa2eef70649e434..aaf901506a6a8bb7ddc78f6cf7f3106d283c29bd 100644
--- a/components/brave_extension/extension/brave_extension/background/webDiscoveryProject.ts
+++ b/components/brave_extension/extension/brave_extension/background/webDiscoveryProject.ts
@@ -32,7 +32,7 @@ function stopWDP() {
 
 function onCommitted (details: chrome.webNavigation.WebNavigationTransitionCallbackDetails) {
   // Only inject if page is acceptable protocol (to skip internal pages like
-  // devtools://, brave://, etc.). We also make sure to only inject if we
+  // devtools://,ibrowe://, etc.). We also make sure to only inject if we
   // receive the onCommitted for the main_frame (frameId === 0).
   if (
     details.frameId === 0 &&
