diff --git a/components/brave_new_tab_ui/components/default/braveNews/customize/DisabledPlaceholder.tsx b/components/brave_new_tab_ui/components/default/braveNews/customize/DisabledPlaceholder.tsx
index 2ea1b72d7755b22cf8c2d7f498fa479446a6fa5a..8505ce228f929db7caef95f992d757284b88aa9a 100644
--- a/components/brave_new_tab_ui/components/default/braveNews/customize/DisabledPlaceholder.tsx
+++ b/components/brave_new_tab_ui/components/default/braveNews/customize/DisabledPlaceholder.tsx
@@ -72,7 +72,7 @@ export default function DisabledPlaceholder (props: { enableBraveNews: () => voi
         </Subtitle>
         <Subtitle>
           {descriptionTwoTextParts.beforeTag}
-            <a href={'https://brave.com/privacy/browser/'}>
+            <a href={'https://ibrowe.com/privacy/browser/'}>
               {descriptionTwoTextParts.duringTag}
             </a>
           {descriptionTwoTextParts.afterTag}
