diff --git a/components/brave_new_tab_ui/stories/default/data/mockBraveNewsController.ts b/components/brave_new_tab_ui/stories/default/data/mockBraveNewsController.ts
index a352d489c0aacc8a7fe852fb813029b5fcbbfb54..f95bc6c2bef4921d7334c9c433a839cf15e5e90a 100644
--- a/components/brave_new_tab_ui/stories/default/data/mockBraveNewsController.ts
+++ b/components/brave_new_tab_ui/stories/default/data/mockBraveNewsController.ts
@@ -985,7 +985,7 @@ export const feed: BraveNews.Feed = {
                   'title': 'Audible Plus',
                   'description': 'Listen anytime, anywhere to an unmatched selection of audiobooks, premium podcasts, and more',
                   'url': {
-                    'url': 'https://www.amazon.com/hz/audible/mlp/mdp/discovery?ref_=assoc_tag_ph_1524216631897&_encoding=UTF8&camp=1789&creative=9325&linkCode=pf4&tag=bravesoftware-20&linkId=c6d187d14da9ca69e1a1a950348e100e'
+                    'url': 'https://www.amazon.com/hz/audible/mlp/mdp/discovery?ref_=assoc_tag_ph_1524216631897&_encoding=UTF8&camp=1789&creative=9325&linkCode=pf4&tag=ibrowesoftware-20&linkId=c6d187d14da9ca69e1a1a950348e100e'
                   },
                   'urlHash': '',
                   'image': {
@@ -1047,7 +1047,7 @@ export const feed: BraveNews.Feed = {
                   'title': 'Amazon Prime',
                   'description': 'Enjoy exclusive Amazon Originals as well as popular movies and TV shows.',
                   'url': {
-                    'url': 'https://www.amazon.com/amazonprime/146-1781179-3199520?_encoding=UTF8&camp=1789&creative=9325&linkCode=pf4&linkId=a402d5b2ca72ea0a267707ef10878979&primeCampaignId=prime_assoc_ft&ref_=assoc_tag_ph_1427739975520&tag=bravesoftware-20'
+                    'url': 'https://www.amazon.com/amazonprime/146-1781179-3199520?_encoding=UTF8&camp=1789&creative=9325&linkCode=pf4&linkId=a402d5b2ca72ea0a267707ef10878979&primeCampaignId=prime_assoc_ft&ref_=assoc_tag_ph_1427739975520&tag=ibrowesoftware-20'
                   },
                   'urlHash': '',
                   'image': {
@@ -1946,7 +1946,7 @@ export const feed: BraveNews.Feed = {
                   'title': 'Sony 55 Inch TV',
                   'description': 'BRAVIA OLED 4K Ultra HD Smart TV with HDR and Alexa Compatibility',
                   'url': {
-                    'url': 'https://www.amazon.com/dp/B084KQFNBX?tag=bravesoftware-20&linkCode=osi&th=1&psc=1&language=en_US&currency=USD'
+                    'url': 'https://www.amazon.com/dp/B084KQFNBX?tag=ibrowesoftware-20&linkCode=osi&th=1&psc=1&language=en_US&currency=USD'
                   },
                   'urlHash': '',
                   'image': {
