diff --git a/components/web_discovery/browser/credential_manager.cc b/components/web_discovery/browser/credential_manager.cc
index cff9aa074d2479976ca964e598c44ea5f7f09f3c..4fb244de635d8523a0bb0ebb39f61382dd5e130c 100644
--- a/components/web_discovery/browser/credential_manager.cc
+++ b/components/web_discovery/browser/credential_manager.cc
@@ -59,7 +59,7 @@ constexpr net::NetworkTrafficAnnotationTag kJoinNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 }  // namespace
