diff --git a/components/ai_chat/resources/page/components/privacy_message/index.tsx b/components/ai_chat/resources/page/components/privacy_message/index.tsx
index 50da308e029cae20163d28450e7001d7eeaf54d3..4cda6ebd0aaa9e91c7acb88afc07a66826368f5a 100644
--- a/components/ai_chat/resources/page/components/privacy_message/index.tsx
+++ b/components/ai_chat/resources/page/components/privacy_message/index.tsx
@@ -12,8 +12,8 @@ import { Url } from 'gen/url/mojom/url.mojom.m.js'
 import { useAIChat } from '../../state/ai_chat_context'
 import styles from './style.module.scss'
 
-const WIKI_URL = "https://support.brave.com/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models"
-const PRIVACY_URL = "https://brave.com/privacy/browser/#brave-leo"
+const WIKI_URL = "https://support.ibrowe.com/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models"
+const PRIVACY_URL = "https://ibrowe.com/privacy/browser/#brave-leo"
 
 function PrivacyMessage () {
   const context = useAIChat()
