diff --git a/components/brave_ads/browser/ads_service_impl.cc b/components/brave_ads/browser/ads_service_impl.cc
index 5c6d26f2c00b6b05326c30e4726897299250e343..27eb25f6cc6bfb7f52e531273e5eef98a9a59db6 100644
--- a/components/brave_ads/browser/ads_service_impl.cc
+++ b/components/brave_ads/browser/ads_service_impl.cc
@@ -157,7 +157,7 @@ net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable this feature by visiting brave://rewards."
+          "You can enable or disable this feature by visiting ibrowe://rewards."
         policy_exception_justification:
           "Not implemented."
       }
@@ -342,8 +342,8 @@ void AdsServiceImpl::GetDeviceIdAndMaybeStartBatAdsServiceCallback(
 
 bool AdsServiceImpl::CanStartBatAdsService() const {
   if (UserHasJoinedBraveRewards()) {
-    // Always start the service to update brave://ads-internals,
-    // brave://rewards, and brave://rewards-internals if the user has joined
+    // Always start the service to update ibrowe://ads-internals,
+    // ibrowe://rewards, and ibrowe://rewards-internals if the user has joined
     // Brave Rewards, even if all ad units are opted out.
     return true;
   }
