diff --git a/components/skus/browser/skus_service_unittest.cc b/components/skus/browser/skus_service_unittest.cc
index 626a9c61dd2fabc3d72e78194ab4c290b6764e26..0f62e6612f9472ec2de24c0cd6779cf4d4c71c69 100644
--- a/components/skus/browser/skus_service_unittest.cc
+++ b/components/skus/browser/skus_service_unittest.cc
@@ -311,9 +311,9 @@ TEST_F(SkusServiceTestUnitTest, CredentialSummaryFailed) {
 
 TEST_F(SkusServiceTestUnitTest, CredentialSummaryWrongEnv) {
   base::Value::Dict state;
-  auto testing_payload = GenerateTestingCreds("vpn.brave.software");
+  auto testing_payload = GenerateTestingCreds("vpn.ibrowe.software");
   state.Set("skus:staging", testing_payload);
   prefs()->SetDict(skus::prefs::kSkusState, std::move(state));
-  auto credentials = GetCredentialsSummary("vpn.brave.software");
+  auto credentials = GetCredentialsSummary("vpn.ibrowe.software");
   EXPECT_EQ(credentials, "{}");
 }
