diff --git a/components/brave_wallet_ui/components/extension/sign-panel/sign-transaction-panel.tsx b/components/brave_wallet_ui/components/extension/sign-panel/sign-transaction-panel.tsx
index d1011627c9f725a09387e8cd6ad7982526322bad..a4e9dcaaf1bcac3093ad3fb31cd4071dae11a3f6 100644
--- a/components/brave_wallet_ui/components/extension/sign-panel/sign-transaction-panel.tsx
+++ b/components/brave_wallet_ui/components/extension/sign-panel/sign-transaction-panel.tsx
@@ -81,7 +81,7 @@ interface Props {
 // https://github.com/brave/brave-browser/issues/39708
 const onClickLearnMore = () => {
   window.open(
-    'https://support.brave.com/hc/en-us/articles/4409513799693',
+    'https://support.ibrowe.com/hc/en-us/articles/4409513799693',
     '_blank',
     'noreferrer'
   )
