diff --git a/components/brave_mobile_subscription/renderer/android/subscription_render_frame_observer_browsertest.cc b/components/brave_mobile_subscription/renderer/android/subscription_render_frame_observer_browsertest.cc
index 400b8266c275996c10bf9439750c04fe1d08a4b7..daadd3f0eabf193629f3a82fb5eb8baf4a3a9a33 100644
--- a/components/brave_mobile_subscription/renderer/android/subscription_render_frame_observer_browsertest.cc
+++ b/components/brave_mobile_subscription/renderer/android/subscription_render_frame_observer_browsertest.cc
@@ -79,57 +79,57 @@ TEST_F(SubscriptionRenderFrameObserverBrowserTest, IsAllowed) {
 
   EXPECT_FALSE(observer.IsAllowed());
 
-  // https://account.bravesoftware.com
+  // https://account.ibrowesoftware.com
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.bravesoftware.com/?intent=connect-receipt&product=vpn");
+      "https://account.ibrowesoftware.com/?intent=connect-receipt&product=vpn");
 
   EXPECT_TRUE(observer.IsAllowed());
 
-  // https://account.brave.software
+  // https://account.ibrowe.software
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=connect-receipt&product=vpn");
+      "https://account.ibrowe.software/?intent=connect-receipt&product=vpn");
 
   EXPECT_TRUE(observer.IsAllowed());
 
   // no receipt
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=&product=vpn");
+      "https://account.ibrowe.software/?intent=&product=vpn");
 
   EXPECT_FALSE(observer.IsAllowed());
 
   // wrong receipt
   LoadHTMLWithUrlOverride(R"(<html><body></body></html>)",
-                          "https://account.brave.software/?product=vpn");
+                          "https://account.ibrowe.software/?product=vpn");
 
   EXPECT_FALSE(observer.IsAllowed());
 
   // wrong receipt
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=wrong&product=vpn");
+      "https://account.ibrowe.software/?intent=wrong&product=vpn");
 
   EXPECT_FALSE(observer.IsAllowed());
 
   // no product
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=connect-receipt&product=");
+      "https://account.ibrowe.software/?intent=connect-receipt&product=");
 
   EXPECT_FALSE(observer.IsAllowed());
 
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=connect-receipt");
+      "https://account.ibrowe.software/?intent=connect-receipt");
 
   EXPECT_FALSE(observer.IsAllowed());
 
   // wrong product
   LoadHTMLWithUrlOverride(
       R"(<html><body></body></html>)",
-      "https://account.brave.software/?intent=connect-receipt&product=wrong");
+      "https://account.ibrowe.software/?intent=connect-receipt&product=wrong");
 
   EXPECT_FALSE(observer.IsAllowed());
 }
