diff --git a/components/skus/renderer/skus_utils.cc b/components/skus/renderer/skus_utils.cc
index 18f6462f8856e2c6e087d5eaddc0b6610d587216..41b131057222eed675a0320509c8bc3827eb83e7 100644
--- a/components/skus/renderer/skus_utils.cc
+++ b/components/skus/renderer/skus_utils.cc
@@ -17,7 +17,7 @@ namespace skus {
 bool IsSafeOrigin(const blink::WebSecurityOrigin& origin) {
   // NOTE: please open a security review when appending to this list.
   static base::NoDestructor<std::vector<blink::WebSecurityOrigin>> safe_origins{
-      {{blink::WebSecurityOrigin::Create(GURL("https://account.brave.com"))},
+      {{blink::WebSecurityOrigin::Create(GURL("https://account.ibrowe.com"))},
        {blink::WebSecurityOrigin::Create(
            GURL("https://account.bravesoftware.com"))},
        {blink::WebSecurityOrigin::Create(
