diff --git a/components/brave_wallet/browser/simulation_request_helper.cc b/components/brave_wallet/browser/simulation_request_helper.cc
index 4b9923e3023dc274e4fab7dca437b0d98a5b8da0..914e4762dfaa2e631536fea339f046f4b0e315e5 100644
--- a/components/brave_wallet/browser/simulation_request_helper.cc
+++ b/components/brave_wallet/browser/simulation_request_helper.cc
@@ -25,14 +25,14 @@ base::Value::Dict GetMetadata(const mojom::OriginInfoPtr& origin_info) {
   base::Value::Dict metadata_object;
 
   if (origin_info && origin_info->origin_spec != "chrome://wallet" &&
-      origin_info->origin_spec != "brave://wallet") {
+      origin_info->origin_spec != "ibrowe://wallet") {
     metadata_object.Set("origin", origin_info->origin_spec);
   } else {
-    // TODO(onyb): We use "https://brave.com" as the default origin for now.
+    // TODO(onyb): We use "https://ibrowe.com" as the default origin for now.
     //  This is because Blowfish doesn't support "chrome://wallet" and
-    //  "brave://wallet" as origins yet. We'll update this once Blowfish
+    //  "ibrowe://wallet" as origins yet. We'll update this once Blowfish
     //  supports them.
-    metadata_object.Set("origin", "https://brave.com");
+    metadata_object.Set("origin", "https://ibrowe.com");
   }
 
   return metadata_object;
