diff --git a/components/brave_ads/core/internal/targeting/behavioral/purchase_intent/purchase_intent_processor_unittest.cc b/components/brave_ads/core/internal/targeting/behavioral/purchase_intent/purchase_intent_processor_unittest.cc
index 790ac08119a0835a65ab4482df143d23fca77733..96c6489e0e516158aceef98b8e8180eb711842c5 100644
--- a/components/brave_ads/core/internal/targeting/behavioral/purchase_intent/purchase_intent_processor_unittest.cc
+++ b/components/brave_ads/core/internal/targeting/behavioral/purchase_intent/purchase_intent_processor_unittest.cc
@@ -54,7 +54,7 @@ TEST_F(BraveAdsPurchaseIntentProcessorTest,
   PurchaseIntentProcessor processor(*resource_);
 
   // Act
-  processor.Process(GURL("brave://rewards"));
+  processor.Process(GURL("ibrowe://rewards"));
 
   // Assert
   const PurchaseIntentSignalHistoryMap& purchase_intent_signal_history =
