diff --git a/components/brave_wallet/browser/simulation_request_helper.cc b/components/brave_wallet/browser/simulation_request_helper.cc
index 3dac89b4f6e63a507f0c54aea2c6db4403e5c8cc..4c1d3ecdae07faa86488d46e4c3ab6ffcb5cea28 100644
--- a/components/brave_wallet/browser/simulation_request_helper.cc
+++ b/components/brave_wallet/browser/simulation_request_helper.cc
@@ -25,12 +25,12 @@ base::Value::Dict GetMetadata(const mojom::OriginInfoPtr& origin_info) {
   base::Value::Dict metadata_object;
 
   if (origin_info && origin_info->origin_spec != "chrome://wallet" &&
-      origin_info->origin_spec != "brave://wallet") {
+      origin_info->origin_spec != "ibrowe://wallet") {
     metadata_object.Set("origin", origin_info->origin_spec);
   } else {
     // TODO(onyb): We use "https://brave.com" as the default origin for now.
     //  This is because Blowfish doesn't support "chrome://wallet" and
-    //  "brave://wallet" as origins yet. We'll update this once Blowfish
+    //  "ibrowe://wallet" as origins yet. We'll update this once Blowfish
     //  supports them.
     metadata_object.Set("origin", "https://brave.com");
   }
