diff --git a/components/url_sanitizer/browser/url_sanitizer_service_unittest.cc b/components/url_sanitizer/browser/url_sanitizer_service_unittest.cc
index ba6a278fe249bee1e8629380ff1192bfcbdc1aca..7496a08ea443e9860aa74f6608a9e513676576ac 100644
--- a/components/url_sanitizer/browser/url_sanitizer_service_unittest.cc
+++ b/components/url_sanitizer/browser/url_sanitizer_service_unittest.cc
@@ -39,10 +39,10 @@ constexpr char kTestPatterns[] = R"(
   },
   {
     "include": [
-      "https://dev-pages.bravesoftware.com/clean-urls/*"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/*"
     ],
     "exclude": [
-      "https://dev-pages.bravesoftware.com/clean-urls/exempted/*"
+      "https://dev-pages.ibrowesoftware.com/clean-urls/exempted/*"
     ],
     "params": [
       "brave_testing1",
@@ -97,19 +97,19 @@ TEST_F(URLSanitizerServiceUnitTest, ClearURLS) {
 
   EXPECT_EQ(
       SanitizeURL(
-          GURL("https://dev-pages.bravesoftware.com/clean-urls/"
+          GURL("https://dev-pages.ibrowesoftware.com/clean-urls/"
                "exempted/"
                "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&"
                "d&utm_content=removethis&e=&f=g&=end")),
-      GURL("https://dev-pages.bravesoftware.com/clean-urls/"
+      GURL("https://dev-pages.ibrowesoftware.com/clean-urls/"
            "exempted/?brave_testing1=foo&brave_testing2=bar&brave_"
            "testing3=keep&&;b&d&e=&f=g&=end"));
   EXPECT_EQ(
       SanitizeURL(GURL(
-          "https://dev-pages.bravesoftware.com/clean-urls/"
+          "https://dev-pages.ibrowesoftware.com/clean-urls/"
           "?brave_testing1=foo&brave_testing2=bar&brave_testing3=keep&&;b&d&"
           "utm_content=removethis&e=&f=g&=end")),
-      GURL("https://dev-pages.bravesoftware.com/clean-urls/"
+      GURL("https://dev-pages.ibrowesoftware.com/clean-urls/"
            "?brave_testing3=keep&&;b&d&e=&f=g&=end"));
 
   WaitInitialization(R"([
