diff --git a/components/skus/browser/skus_utils.cc b/components/skus/browser/skus_utils.cc
index 570f2bf74104b8c59c8e14f454428636d298f220..42dc6da0c3b48381c68dfddd27914a733eddcea4 100644
--- a/components/skus/browser/skus_utils.cc
+++ b/components/skus/browser/skus_utils.cc
@@ -37,11 +37,11 @@ std::string GetDomain(const std::string& prefix,
   DCHECK(prefix == kProductTalk || prefix == kProductVPN);
 
   if (environment == kEnvProduction) {
-    return prefix + ".brave.com";
+    return prefix + ".ibrowe.com";
   } else if (environment == kEnvStaging) {
-    return prefix + ".bravesoftware.com";
+    return prefix + ".ibrowesoft.com";
   } else if (environment == kEnvDevelopment) {
-    return prefix + ".brave.software";
+    return prefix + ".ibrowe.software";
   }
 
   NOTREACHED() << "Unsupported environment: " << environment;
@@ -50,11 +50,11 @@ std::string GetDomain(const std::string& prefix,
 std::string GetEnvironmentForDomain(const std::string& domain) {
   auto base_domain = net::registry_controlled_domains::GetDomainAndRegistry(
       domain, net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES);
-  if (base_domain == "brave.com")
+  if (base_domain == "ibrowe.com")
     return kEnvProduction;
-  if (base_domain == "bravesoftware.com")
+  if (base_domain == "ibrowesoft.com")
     return kEnvStaging;
-  if (base_domain == "brave.software")
+  if (base_domain == "ibrowe.software")
     return kEnvDevelopment;
   NOTIMPLEMENTED();
   return "";
