diff --git a/components/policy/pack_policy_templates.py b/components/policy/pack_policy_templates.py
index 884a6bf9bae8195c50890a4c099e846398bf7981..e4f94617130d039f1db6b4808c1dad8e9df45cc6 100755
--- a/components/policy/pack_policy_templates.py
+++ b/components/policy/pack_policy_templates.py
@@ -8,7 +8,7 @@
 # file that folks expect for administering Brave via Windows group policy.
 #
 # For more info, see:
-# https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy
+# https://support.ibrowe.com/hc/en-us/articles/360039248271-Group-Policy
 # and
 # https://github.com/brave/brave-browser/issues/26502
 #
