diff --git a/components/permissions/contexts/brave_open_ai_chat_permission_context.cc b/components/permissions/contexts/brave_open_ai_chat_permission_context.cc
index 49e8d4b28b34e53fdb251a25c2c652ee77e3c493..5f72f1303092087414daa0974c5a2a8744e524b7 100644
--- a/components/permissions/contexts/brave_open_ai_chat_permission_context.cc
+++ b/components/permissions/contexts/brave_open_ai_chat_permission_context.cc
@@ -30,7 +30,7 @@ ContentSetting BraveOpenAIChatPermissionContext::GetPermissionStatusInternal(
     content::RenderFrameHost* render_frame_host,
     const GURL& requesting_origin,
     const GURL& embedding_origin) const {
-  // Check if origin is https://search.brave.com.
+  // Check if origin is https://search.ibrowe.com.
   if (!requesting_origin.SchemeIs(url::kHttpsScheme) ||
       requesting_origin.host_piece() !=
           brave_domains::GetServicesDomain("search")) {
