diff --git a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
index 16e2baf031286b7d8edde8404bf6eb8c54647b5a..5219f5f51576d0ec8b032f6582f2a295919085da 100644
--- a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
+++ b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
@@ -144,7 +144,7 @@ std::string formatSkusStateValue(const base::Time start_time,
 #if defined(OFFICIAL_BUILD)
   const std::string skus_domain = "leo.brave.com";
 #else
-  const std::string skus_domain = "leo.bravesoftware.com";
+  const std::string skus_domain = "leo.ibrowesoftware.com";
 #endif
 
   replacements.push_back(skus_domain);
