diff --git a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
index 4ced3146559bbba1e88414b1c6cfd3ce82c1194f..2017138ebec00580c8c190ccf5ac7f9d2331e897 100644
--- a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
+++ b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
@@ -33,7 +33,7 @@ const TermsOfUseText: React.FC<{}> = () => {
     <p key={text.duringTag}>
       {text.beforeTag}
       <TermsLink
-        href='https://brave.com/terms-of-use/'
+        href='https://ibrowe.com/terms-of-use/'
         target='_blank'
         rel='noopener noreferrer'
         onClick={
