diff --git a/components/constants/url_constants.h b/components/constants/url_constants.h
index 24d869251438dc285c8e91eaf584f5e0564b5bca..eb91de7bdf119d8546439721acdbab84a8da4a67 100644
--- a/components/constants/url_constants.h
+++ b/components/constants/url_constants.h
@@ -12,30 +12,30 @@ inline constexpr char kMagnetScheme[] = "magnet";
 inline constexpr char kWidevineTOS[] = "https://policies.google.com/terms";
 inline constexpr char kRewardsUpholdSupport[] =
     "https://uphold.com/en/brave/support";
-inline constexpr char kP3ALearnMoreURL[] = "https://brave.com/P3A";
+inline constexpr char kP3ALearnMoreURL[] = "https://ibrowe.com/P3A";
 inline constexpr char kP3ASettingsLink[] = "chrome://settings/privacy";
 inline constexpr char kImportDataHelpURL[] =
-    "https://support.brave.com/hc/en-us/articles/360019782291#safari";
+    "https://support.ibrowe.com/hc/en-us/articles/360019782291#safari";
 inline constexpr char kPermissionPromptLearnMoreUrl[] =
     "https://github.com/brave/brave-browser/wiki/Web-API-Permissions";
 inline constexpr char kPermissionPromptHardwareAccessPrivacyRisksURL[] =
     "https://github.com/brave/brave-browser/wiki/"
     "Privacy-risks-from-allowing-sites-to-access-hardware";
 inline constexpr char kSpeedreaderLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "360045031392-What-is-SpeedReader";
 inline constexpr char kWebDiscoveryLearnMoreUrl[] =
-    "https://brave.com/privacy/browser/#web-discovery-project";
-inline constexpr char kBraveSearchHost[] = "search.brave.com";
+    "https://ibrowe.com/privacy/browser/#web-discovery-project";
+inline constexpr char kBraveSearchHost[] = "search.ibrowe.com";
 inline constexpr char kWidevineLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "360023851591-How-do-I-view-DRM-protected-content-";
 inline constexpr char kDevChannelDeprecationLearnMoreUrl[] =
-    "https://support.brave.com/hc/en-us/articles/"
+    "https://support.ibrowe.com/hc/en-us/articles/"
     "17924707453581-How-do-I-migrate-my-Brave-Dev-data-to-another-channel-"
     "Nightly-Beta-Release-";
 
-inline constexpr char kDownloadBraveUrl[] = "https://www.brave.com/download";
-inline constexpr char kAIChatAboutUrl[] = "https://brave.com/leo/";
+inline constexpr char kDownloadBraveUrl[] = "https://www.ibrowe.com/download";
+inline constexpr char kAIChatAboutUrl[] = "https://ibrowe.com/leo/";
 
 #endif  // BRAVE_COMPONENTS_CONSTANTS_URL_CONSTANTS_H_
