diff --git a/components/brave_rewards/core/common/environment_config.cc b/components/brave_rewards/core/common/environment_config.cc
index 6be18e7bd5eba498a88eac15c4b4aad6bda0490d..c5ba757302831b7aa57e7ef6146a869f621d00ca 100644
--- a/components/brave_rewards/core/common/environment_config.cc
+++ b/components/brave_rewards/core/common/environment_config.cc
@@ -67,7 +67,7 @@ GURL EnvironmentConfig::brave_pcdn_url() const {
     case mojom::Environment::kStaging:
       return GURL("https://pcdn.bravesoftware.com");
     case mojom::Environment::kProduction:
-      return GURL("https://pcdn.brave.com");
+      return GURL("https://pcdn.ibrowe.com");
   }
 }
 
@@ -78,7 +78,7 @@ GURL EnvironmentConfig::rewards_url() const {
     case mojom::Environment::kStaging:
       return GURL("https://rewards-stg.bravesoftware.com");
     case mojom::Environment::kProduction:
-      return GURL("https://rewards.brave.com");
+      return GURL("https://rewards.ibrowe.com");
   }
 }
 
@@ -89,7 +89,7 @@ GURL EnvironmentConfig::rewards_api_url() const {
     case mojom::Environment::kStaging:
       return GURL("https://api.rewards.bravesoftware.com");
     case mojom::Environment::kProduction:
-      return GURL("https://api.rewards.brave.com");
+      return GURL("https://api.rewards.ibrowe.com");
   }
 }
 
@@ -111,7 +111,7 @@ GURL EnvironmentConfig::rewards_payment_url() const {
     case mojom::Environment::kStaging:
       return GURL("https://payment.rewards.bravesoftware.com");
     case mojom::Environment::kProduction:
-      return GURL("https://payment.rewards.brave.com");
+      return GURL("https://payment.rewards.ibrowe.com");
   }
 }
 
