diff --git a/components/brave_ads/core/internal/catalog/catalog_url_request_builder_unittest.cc b/components/brave_ads/core/internal/catalog/catalog_url_request_builder_unittest.cc
index 9ec5868b40d5247cf919dca065e92642863b95d2..f7dfb924b66954bc5df6b110753b900c9d2cea2e 100644
--- a/components/brave_ads/core/internal/catalog/catalog_url_request_builder_unittest.cc
+++ b/components/brave_ads/core/internal/catalog/catalog_url_request_builder_unittest.cc
@@ -26,7 +26,7 @@ TEST_F(BraveAdsCatalogUrlRequestBuilderTest, BuildUrl) {
   const mojom::UrlRequestInfoPtr expected_mojom_url_request =
       mojom::UrlRequestInfo::New();
   expected_mojom_url_request->url =
-      GURL("https://static.ads.bravesoftware.com/v9/catalog");
+      GURL("https://static.ads.ibrowesoftware.com/v9/catalog");
   expected_mojom_url_request->method = mojom::UrlRequestMethodType::kGet;
   EXPECT_EQ(expected_mojom_url_request, mojom_url_request);
 }
