diff --git a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/static_url_host.cc b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/static_url_host.cc
index d7ae3324c9378c8c094b2af3f870605b9f73731a..9493ba6af00dff4c082f743a7a64e149c5437274 100644
--- a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/static_url_host.cc
+++ b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/static_url_host.cc
@@ -13,7 +13,7 @@ namespace brave_ads {
 namespace {
 
 constexpr char kProductionHost[] = "https://static.ads.brave.com";
-constexpr char kStagingHost[] = "https://static.ads.bravesoftware.com";
+constexpr char kStagingHost[] = "https://static.ads.ibrowesoftware.com";
 
 }  // namespace
 
