diff --git a/components/brave_rewards/core/engine/test/data/publisher_info_db_v13.sql b/components/brave_rewards/core/engine/test/data/publisher_info_db_v13.sql
index f43f1cd496e28f6c7d1a47a3c390ed69cc885afb..0707785c1d5db558d0624151cb3faaa4d8c9edf9 100644
--- a/components/brave_rewards/core/engine/test/data/publisher_info_db_v13.sql
+++ b/components/brave_rewards/core/engine/test/data/publisher_info_db_v13.sql
@@ -151,7 +151,7 @@ INSERT INTO "activity_info" VALUES ('3zsistemi.si',9,1,1.00662511542715,100,100.
 INSERT INTO "activity_info" VALUES ('3zsistemi.si',15,1,1.04476927411666,100,100.0,1574671372);
 INSERT INTO "server_publisher_info" VALUES ('3zsistemi.si',2,0,'0fd15856-ada1-45a3-9ec7-74ac4db76eec');
 INSERT INTO "server_publisher_info" VALUES ('laurenwags.github.io',2,0,'abf1ff79-a239-42af-abff-20eb121edd1c');
-INSERT INTO "server_publisher_banner" VALUES ('laurenwags.github.io','Staging Banner Test','Lorem ipsum dolor sit amet, sale homero neglegentur ei vix, quo no tacimates vituperatoribus. Per elit luptatum temporibus ad, cibo minimum quaerendum no nec, atqui corpora complectitur te sed. Per ne vulputate neglegentur, id nec alia affert aperiri. Ea melius deserunt pro. Officiis sadipscing at nam, adhuc populo atomorum est.','chrome://rewards-image/https://rewards-stg.bravesoftware.com/xrEJASVGN9nQ5zJUnmoCxjEE','chrome://rewards-image/https://rewards-stg.bravesoftware.com/8eT9LXcpK3D795YHxvDdhrmg');
+INSERT INTO "server_publisher_banner" VALUES ('laurenwags.github.io','Staging Banner Test','Lorem ipsum dolor sit amet, sale homero neglegentur ei vix, quo no tacimates vituperatoribus. Per elit luptatum temporibus ad, cibo minimum quaerendum no nec, atqui corpora complectitur te sed. Per ne vulputate neglegentur, id nec alia affert aperiri. Ea melius deserunt pro. Officiis sadipscing at nam, adhuc populo atomorum est.','chrome://rewards-image/https://rewards-stg.ibrowesoftware.com/xrEJASVGN9nQ5zJUnmoCxjEE','chrome://rewards-image/https://rewards-stg.ibrowesoftware.com/8eT9LXcpK3D795YHxvDdhrmg');
 INSERT INTO "server_publisher_links" VALUES ('laurenwags.github.io','twitch','https://www.twitch.tv/laurenwags');
 INSERT INTO "server_publisher_links" VALUES ('laurenwags.github.io','twitter','https://twitter.com/bravelaurenwags');
 INSERT INTO "server_publisher_links" VALUES ('laurenwags.github.io','youtube','https://www.youtube.com/channel/UCCs7AQEDwrHEc86r0NNXE_A/videos');
