diff --git a/components/web_discovery/browser/server_config_loader.cc b/components/web_discovery/browser/server_config_loader.cc
index d2cb606ddf6533fe7e9ca7f97f02831eda524386..94f5f8bfcac376dd88fedda4ec9c6cc23d8adbf9 100644
--- a/components/web_discovery/browser/server_config_loader.cc
+++ b/components/web_discovery/browser/server_config_loader.cc
@@ -60,7 +60,7 @@ constexpr net::NetworkTrafficAnnotationTag kNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 constexpr char kGroupPubKeysFieldName[] = "groupPubKeys";
