diff --git a/components/brave_vpn/browser/brave_vpn_service.cc b/components/brave_vpn/browser/brave_vpn_service.cc
index 5276b18f241b459097f0e640051bdc358099b237..37234baa0b14707fc78b8fc816e29ca8d953a785 100644
--- a/components/brave_vpn/browser/brave_vpn_service.cc
+++ b/components/brave_vpn/browser/brave_vpn_service.cc
@@ -507,7 +507,7 @@ void BraveVpnService::OnFetchRegionList(GetAllRegionsCallback callback,
 
 void BraveVpnService::GetPurchaseToken(GetPurchaseTokenCallback callback) {
   std::string purchase_token_string = "";
-  std::string package_string = "com.brave.browser";
+  std::string package_string = "com.ibrowe.browser";
   std::string product_id_string = "brave-firewall-vpn-premium";
 
   // Get the Android purchase token (for Google Play Store).
