diff --git a/components/brave_new_tab_ui/components/default/braveTalk/index.tsx b/components/brave_new_tab_ui/components/default/braveTalk/index.tsx
index 7814eb19628cc6fa269ff9687255a2156299405f..3d6b61e1974e4e572bd93f921e91ecba26a2b645 100644
--- a/components/brave_new_tab_ui/components/default/braveTalk/index.tsx
+++ b/components/brave_new_tab_ui/components/default/braveTalk/index.tsx
@@ -72,7 +72,7 @@ class BraveTalk extends React.PureComponent<Props, {}> {
               <PrivacyLink
                 rel='noopener'
                 target='_blank'
-                href='https://brave.com/privacy/browser/#brave-talk-learn'
+                href='https://ibrowe.com/privacy/browser/#brave-talk-learn'
               >
                 {getLocale('braveTalkWidgetAboutData')}
               </PrivacyLink>
