diff --git a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/non_anonymous_url_host_unittest.cc b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/non_anonymous_url_host_unittest.cc
index f72e4fed26af139ea6a7e0a051becf253be972ee..f1d6b2113c7396215c3839c50a81f52e389310a8 100644
--- a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/non_anonymous_url_host_unittest.cc
+++ b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/non_anonymous_url_host_unittest.cc
@@ -31,7 +31,7 @@ TEST_F(BraveAdsNonAnonymousUrlHostTest, GetStagingUrlHost) {
       mojom::EnvironmentType::kStaging;
 
   // Act & Assert
-  EXPECT_EQ("https://mywallet.ads.bravesoftware.com", GetNonAnonymousUrlHost());
+  EXPECT_EQ("https://mywallet.ads.ibrowesoftware.com", GetNonAnonymousUrlHost());
 }
 
 }  // namespace brave_ads
