diff --git a/components/brave_vpn/common/brave_vpn_utils.cc b/components/brave_vpn/common/brave_vpn_utils.cc
index f8203e3cfb82f8c5104002cee8df2866fe74908b..7d67f8a1b893d9b9433fc7e2f760378cb892f0d6 100644
--- a/components/brave_vpn/common/brave_vpn_utils.cc
+++ b/components/brave_vpn/common/brave_vpn_utils.cc
@@ -154,7 +154,7 @@ GURL GetManageURLForUIType(mojom::ManageURLType type, const GURL& manage_url) {
       return manage_url.ReplaceComponents(replacements);
     }
     case mojom::ManageURLType::PRIVACY:
-      return GURL("https://brave.com/privacy/browser/#vpn");
+      return GURL("https://ibrowe.com/privacy/browser/#vpn");
     case mojom::ManageURLType::ABOUT:
       return GURL(brave_vpn::kAboutUrl);
     case mojom::ManageURLType::MANAGE:
@@ -262,7 +262,7 @@ std::string GetManageUrl(const std::string& env) {
 }
 
 // On desktop, the environment is tied to SKUs because you would purchase it
-// from `account.brave.com` (or similar, based on env). The credentials for VPN
+// from `account.ibrowe.com` (or similar, based on env). The credentials for VPN
 // will always be in the same environment as the SKU environment.
 //
 // When the vendor receives a credential from us during auth, it also includes
