diff --git a/components/sidebar/browser/sidebar_service_unittest.cc b/components/sidebar/browser/sidebar_service_unittest.cc
index 43573b69324431d116d5b64c2d8e998e817cf74a..be6b468fbfaaf9da7a678d8a0244f0603ae0b273 100644
--- a/components/sidebar/browser/sidebar_service_unittest.cc
+++ b/components/sidebar/browser/sidebar_service_unittest.cc
@@ -801,8 +801,8 @@ TEST_F(SidebarServiceTest, BuiltInItemUpdateTestWithBuiltInItemTypeKey) {
     dict.Set(sidebar::kSidebarItemTitleKey, "Brave together");
     dict.Set(sidebar::kSidebarItemTypeKey,
              static_cast<int>(SidebarItem::Type::kTypeBuiltIn));
-    dict.Set(sidebar::kSidebarItemBuiltInItemTypeKey,
-             static_cast<int>(SidebarItem::BuiltInItemType::kBraveTalk));
+    // dict.Set(sidebar::kSidebarItemBuiltInItemTypeKey,
+    //          static_cast<int>(SidebarItem::BuiltInItemType::kBraveTalk));
     dict.Set(sidebar::kSidebarItemOpenInPanelKey, true);
     base::Value::List list;
     list.Append(std::move(dict));
@@ -1014,7 +1014,8 @@ TEST_F(SidebarServiceOrderingTest, BuiltInItemsDefaultOrder) {
   EXPECT_EQ(0UL, service_->GetHiddenDefaultSidebarItems().size());
 
   EXPECT_TRUE(
-      ValidateBuiltInTypesOrdering({SidebarItem::BuiltInItemType::kBraveTalk,
+      ValidateBuiltInTypesOrdering({
+                                    // SidebarItem::BuiltInItemType::kBraveTalk,
                                     SidebarItem::BuiltInItemType::kWallet,
                                     SidebarItem::BuiltInItemType::kChatUI,
                                     SidebarItem::BuiltInItemType::kBookmarks,
@@ -1035,7 +1036,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsAllBuiltInVisible) {
       SidebarItem::BuiltInItemType::kWallet,
       SidebarItem::BuiltInItemType::kReadingList,
       SidebarItem::BuiltInItemType::kBookmarks,
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
   };
 
   auto expected_count = sidebar_items->size();
@@ -1055,7 +1056,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsWalletBuiltInHidden) {
   CHECK(sidebar_items);
 
   std::vector items = {
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
       SidebarItem::BuiltInItemType::kBookmarks,
       SidebarItem::BuiltInItemType::kReadingList,
       SidebarItem::BuiltInItemType::kChatUI,
@@ -1082,7 +1083,7 @@ TEST_F(SidebarServiceOrderingTest, LoadFromPrefsAIChatBuiltInNotListed) {
   CHECK(sidebar_items);
 
   std::vector items = {
-      SidebarItem::BuiltInItemType::kBraveTalk,
+      // SidebarItem::BuiltInItemType::kBraveTalk,
       SidebarItem::BuiltInItemType::kBookmarks,
       SidebarItem::BuiltInItemType::kChatUI,
       SidebarItem::BuiltInItemType::kReadingList,
