diff --git a/components/brave_ads/browser/reminder/reminder_util.cc b/components/brave_ads/browser/reminder/reminder_util.cc
index 374e124aa52ccde21da298fb4d90b9814155c150..43dd52388e1e9ea6bb838a7138a266c8a46b8c06 100644
--- a/components/brave_ads/browser/reminder/reminder_util.cc
+++ b/components/brave_ads/browser/reminder/reminder_util.cc
@@ -19,7 +19,7 @@ namespace {
 constexpr char kReminderNotificationAdPlacementId[] =
     "e64373ac-2ca5-4f6b-b497-1f1d7ccd40c8";
 constexpr char kReminderNotificationAdTargetUrl[] =
-    "https://support.brave.com/hc/en-us/articles/14648356808845";
+    "https://support.ibrowe.com/hc/en-us/articles/14648356808845";
 
 }  // namespace
 
