diff --git a/components/brave_ads/browser/ads_service_impl.cc b/components/brave_ads/browser/ads_service_impl.cc
index 2d9c6e6f55d2d73d044d0a33411246a66e1d3a09..2e069eb6d99fdcee8754ed7c8f833e856cabe124 100644
--- a/components/brave_ads/browser/ads_service_impl.cc
+++ b/components/brave_ads/browser/ads_service_impl.cc
@@ -165,7 +165,7 @@ net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTag() {
       policy {
         cookies_allowed: NO
         setting:
-          "You can enable or disable this feature by visiting brave://rewards."
+          "You can enable or disable this feature by visiting ibrowe://rewards."
         policy_exception_justification:
           "Not implemented."
       }
