diff --git a/components/brave_welcome_ui/components/help-improve/index.tsx b/components/brave_welcome_ui/components/help-improve/index.tsx
index f07cf35970f9f11ca3a77b025a4fa016ef64a7aa..10ee212dd93b457522463475df0e71bd239dc382 100644
--- a/components/brave_welcome_ui/components/help-improve/index.tsx
+++ b/components/brave_welcome_ui/components/help-improve/index.tsx
@@ -15,18 +15,25 @@ interface InputCheckboxProps {
   children: React.ReactNode
   id?: string
   isChecked: boolean
-}
+i}
 
 const changeSettingsNote = formatLocale('braveWelcomeChangeSettingsNote', {
-  $1: content => <a href="brave://settings/privacy" onClick={() => {
-    WelcomeBrowserProxyImpl.getInstance().openSettingsPage()
-  }}>
-    {content}
-  </a>
+    $1: content => (
+        <a
+            href="ibrowe://settings/privacy"
+            onClick={e => {
+                e.preventDefault()
+                chrome.send('openSettingsPage', ['privacy'])
+            }}
+        >
+            {content}
+        </a>
+    )
 })
 
+
 const readPrivacyPolicy = formatLocale('braveWelcomePrivacyPolicyNote', {
-  $1: content => <a href='https://brave.com/privacy/browser' target='_blank'>
+  $1: content => <a href='https://community.ibrowe.com/c/privacyupdates' target='_blank'>
     {content}
   </a>
 })
