diff --git a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
index d5b78ebad8bec1a905daeca7e5d9713393e1f638..6e7ecd88135f77aa671b6d0b512c1d90f89814f7 100644
--- a/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
+++ b/components/ai_chat/core/browser/ai_chat_credential_manager_unittest.cc
@@ -144,7 +144,7 @@ std::string formatSkusStateValue(const base::Time start_time,
 #if defined(OFFICIAL_BUILD)
   const std::string skus_domain = "leo.brave.com";
 #else
-  const std::string skus_domain = "leo.bravesoftware.com";
+  const std::string skus_domain = "leo.ibrowesoftware.com";
 #endif
 
   replacements.push_back(skus_domain);
