diff --git a/components/brave_shields/resources/panel/components/tree-list/index.tsx b/components/brave_shields/resources/panel/components/tree-list/index.tsx
index 6347d75edf89dce455c5e3afef40dd74a107f613..cca680b4d31882db47eeae70c27ffa3946f05748 100644
--- a/components/brave_shields/resources/panel/components/tree-list/index.tsx
+++ b/components/brave_shields/resources/panel/components/tree-list/index.tsx
@@ -21,7 +21,7 @@ import Toggle from '../../../../../web-components/toggle'
 
 interface WebcompatSettingsMap { [index: ContentSettingsType]: boolean }
 
-const kLearnMoreLink = 'https://support.brave.com/hc/en-us/articles/360022806212-How-do-I-use-Shields-while-browsing#h_01HXSZ8JPHR8YMBEZCT5M0VZTR'
+const kLearnMoreLink = 'https://support.ibrowe.com/hc/en-us/articles/360022806212-How-do-I-use-Shields-while-browsing#h_01HXSZ8JPHR8YMBEZCT5M0VZTR'
 
 interface Props {
   blockedList: Url[]
