diff --git a/components/ai_chat/resources/page/stories/story_utils/locale.ts b/components/ai_chat/resources/page/stories/story_utils/locale.ts
index c25940bb1bc37f518078680f9861a93e78de73a8..8cbe52a3761a2c6daa31d1c2e1dfe3b1dc82fe70 100644
--- a/components/ai_chat/resources/page/stories/story_utils/locale.ts
+++ b/components/ai_chat/resources/page/stories/story_utils/locale.ts
@@ -22,7 +22,7 @@ provideStrings({
   errorRateLimit: 'You\'ve reached the premium rate limit. Please try again in a few hours.',
   errorServiceOverloaded: 'The endpoint is currently overloaded. Please try again later.',
   braveLeoAssistantEndpointInvalidError: 'The endpoint URL is invalid. Please check the URL and try again.',
-  braveLeoAssistantEndpointValidAsPrivateIp: 'If you would like to use a private IP address, you must first enable "Private IP Addresses for Custom Model Enpoints" via brave://flags/#brave-ai-chat-allow-private-ips',
+  braveLeoAssistantEndpointValidAsPrivateIp: 'If you would like to use a private IP address, you must first enable "Private IP Addresses for Custom Model Enpoints" via ibrowe://flags/#brave-ai-chat-allow-private-ips',
   retryButtonLabel: 'Retry',
   learnMore: 'Learn more',
   noticeConversationHistoryBody: 'Leo will now remember your previous conversations so you can go back to them. They are stored privately on your device, and you can delete them any time.',
