diff --git a/components/web_discovery/browser/reporter.cc b/components/web_discovery/browser/reporter.cc
index a428682027962fce5d00a2fe9985714de27269ff..0c2ac006a524fc0fbae1f6c90cbc62ff68ea4c8b 100644
--- a/components/web_discovery/browser/reporter.cc
+++ b/components/web_discovery/browser/reporter.cc
@@ -48,7 +48,7 @@ constexpr net::NetworkTrafficAnnotationTag kSubmitNetworkTrafficAnnotation =
     policy {
       cookies_allowed: NO
       setting:
-        "Users can opt-in or out via brave://settings/search"
+        "Users can opt-in or out via ibrowe://settings/search"
     })");
 
 constexpr base::TimeDelta kRequestMaxAge = base::Hours(36);
