diff --git a/components/sidebar/browser/sidebar_service.cc b/components/sidebar/browser/sidebar_service.cc
index a51a30b94e453a8b4539cdd956a17d01f5c0697b..dd36db12cbc7a1426a2c474a3b31728dde2f6c02 100644
--- a/components/sidebar/browser/sidebar_service.cc
+++ b/components/sidebar/browser/sidebar_service.cc
@@ -49,10 +49,10 @@ SidebarItem::BuiltInItemType GetBuiltInItemTypeForLegacyURL(
   // A previous version of prefs used the URL even for built-in items, and not
   // the |SidebarItem::BuiltInItemType|. Therefore, this list should not
   // need to be updated.
-  if (url == "https://together.brave.com/" ||
-      url == "https://talk.brave.com/") {
-    return SidebarItem::BuiltInItemType::kBraveTalk;
-  }
+//  if (url == "https://together.brave.com/" ||
+//      url == "https://talk.brave.com/") {
+//    return SidebarItem::BuiltInItemType::kBraveTalk;
+//  }
 
   if (url == "chrome://wallet/") {
     return SidebarItem::BuiltInItemType::kWallet;
@@ -152,8 +152,8 @@ void SidebarService::MigratePrefSidebarBuiltInItemsToHidden() {
   }
   // Only include items that were known prior to this migration
   std::vector<SidebarItem> built_in_items_to_hide;
-  built_in_items_to_hide.push_back(
-      GetBuiltInItemForType(SidebarItem::BuiltInItemType::kBraveTalk));
+//  built_in_items_to_hide.push_back(
+//      GetBuiltInItemForType(SidebarItem::BuiltInItemType::kBraveTalk));
   built_in_items_to_hide.push_back(
       GetBuiltInItemForType(SidebarItem::BuiltInItemType::kWallet));
   built_in_items_to_hide.push_back(
@@ -591,13 +591,13 @@ std::vector<SidebarItem> SidebarService::GetDefaultSidebarItems() const {
 SidebarItem SidebarService::GetBuiltInItemForType(
     SidebarItem::BuiltInItemType type) const {
   switch (type) {
-    case SidebarItem::BuiltInItemType::kBraveTalk:
-      return SidebarItem::Create(GURL(kBraveTalkURL),
-                                 brave_l10n::GetLocalizedResourceUTF16String(
-                                     IDS_SIDEBAR_BRAVE_TALK_ITEM_TITLE),
-                                 SidebarItem::Type::kTypeBuiltIn,
-                                 SidebarItem::BuiltInItemType::kBraveTalk,
-                                 /* open_in_panel = */ false);
+//    case SidebarItem::BuiltInItemType::kBraveTalk:
+//      return SidebarItem::Create(GURL(kBraveTalkURL),
+//                                 brave_l10n::GetLocalizedResourceUTF16String(
+//                                     IDS_SIDEBAR_BRAVE_TALK_ITEM_TITLE),
+//                                 SidebarItem::Type::kTypeBuiltIn,
+//                                 SidebarItem::BuiltInItemType::kBraveTalk,
+//                                 /* open_in_panel = */ false);
     case SidebarItem::BuiltInItemType::kWallet: {
       if (brave_wallet::IsAllowed(prefs_)) {
         return SidebarItem::Create(GURL("chrome://wallet/"),
