diff --git a/components/brave_wallet_ui/common/constants/support_links.ts b/components/brave_wallet_ui/common/constants/support_links.ts
index 7edf99b894b308f4cb5b3f7d854e68d7b09a3a63..3aa5bcd5c61ae277fe956407147d16579bf0c406 100644
--- a/components/brave_wallet_ui/common/constants/support_links.ts
+++ b/components/brave_wallet_ui/common/constants/support_links.ts
@@ -5,5 +5,5 @@
 
 export const SUPPORT_LINKS = {
   WhatIsTheAssociatedTokenAccountProgram:
-    'https://support.brave.com/hc/en-us/articles/27490686330125-What-is-the-Associated-Token-Account-Program'
+    'https://support.ibrowe.com/hc/en-us/articles/27490686330125-What-is-the-Associated-Token-Account-Program'
 } as const
