diff --git a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/anonymous_search_url_host.cc b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/anonymous_search_url_host.cc
index 3f13e9fd4ccbcad7fbd54438c41b1b2e21b64779..204b6dede79f264a0cfb7610cd81507de3aa45bc 100644
--- a/components/brave_ads/core/internal/common/url/request_builder/host/hosts/anonymous_search_url_host.cc
+++ b/components/brave_ads/core/internal/common/url/request_builder/host/hosts/anonymous_search_url_host.cc
@@ -14,7 +14,7 @@ namespace {
 
 constexpr char kProductionHost[] = "https://search.anonymous.ads.brave.com";
 constexpr char kStagingHost[] =
-    "https://search.anonymous.ads.bravesoftware.com";
+    "https://search.anonymous.ads.ibrowesoftware.com";
 
 }  // namespace
 
