diff --git a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
index 4d70217a3b2659150fef6d7dd953341ef16a3f85..be4bd4b67946ddb33568e83263a04aa74e800b59 100644
--- a/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
+++ b/components/brave_wallet_ui/page/screens/onboarding/disclosures/disclosures.tsx
@@ -33,7 +33,7 @@ const TermsOfUseText: React.FC<{}> = () => {
   const text = formatLocale('braveWalletTermsOfServiceCheckboxText', {
     $1: (content) => (
       <TermsLink
-        href='https://brave.com/terms-of-use/'
+        href='https://ibrowe.com/terms-of-use/'
         target='_blank'
         rel='noopener noreferrer'
         onClick={
