diff --git a/components/brave_vpn/common/brave_vpn_utils.cc b/components/brave_vpn/common/brave_vpn_utils.cc
index 5126e09e8401c0a4e1cf5d3a451e3e5c6f1dd60a..4466bcdb5d6d1c8332ced9949028d2dcaf208e74 100644
--- a/components/brave_vpn/common/brave_vpn_utils.cc
+++ b/components/brave_vpn/common/brave_vpn_utils.cc
@@ -212,7 +212,7 @@ bool IsBraveVPNDisabledByPolicy(PrefService* prefs) {
   return prefs->FindPreference(prefs::kManagedBraveVPNDisabled) &&
   // Need to investigate more about this.
   // IsManagedPreference() gives false on macOS when it's configured by
-  // "defaults write com.brave.Browser.beta BraveVPNDisabled -bool true".
+  // "defaults write com.ibrowe.Browser.beta BraveVPNDisabled -bool true".
   // As kManagedBraveVPNDisabled is false by default and only can be set
   // by policy, I think skipping this condition checking will be fine.
 #if !BUILDFLAG(IS_MAC)
