diff --git a/components/brave_ads/core/internal/ads_impl.h b/components/brave_ads/core/internal/ads_impl.h
index 38857ef9d669588614458ef2f2a2cef2d67290b4..adda0ad5f2e83a26ca70d7f043a7e3f6db606922 100644
--- a/components/brave_ads/core/internal/ads_impl.h
+++ b/components/brave_ads/core/internal/ads_impl.h
@@ -55,7 +55,7 @@ class AdsImpl final : public Ads {
   void GetInternals(GetInternalsCallback callback) override;
 
   // TODO(https://github.com/brave/brave-browser/issues/42034): Transition
-  // diagnostics from brave://rewards-internals to brave://ads-internals.
+  // diagnostics from ibrowe://rewards-internals to ibrowe://ads-internals.
   void GetDiagnostics(GetDiagnosticsCallback callback) override;
 
   void GetStatementOfAccounts(GetStatementOfAccountsCallback callback) override;
