diff --git a/components/brave_new_tab_ui/containers/newTab/index.tsx b/components/brave_new_tab_ui/containers/newTab/index.tsx
index 9fc43cf8f5a69a84c244b2b6c18bfc97877fc46f..880b06820b8776e39828372047bfec77461f53bd 100644
--- a/components/brave_new_tab_ui/containers/newTab/index.tsx
+++ b/components/brave_new_tab_ui/containers/newTab/index.tsx
@@ -392,7 +392,7 @@ class NewTabPage extends React.Component<Props, State> {
   }
 
   learnMoreRewards = () => {
-    window.open('https://brave.com/brave-rewards/', '_blank', 'noopener')
+    window.open('https://ibrowe.com/ibrowe-rewards/', '_blank', 'noopener')
   }
 
   braveVPNSupported = loadTimeData.getBoolean('vpnWidgetSupported')
