diff --git a/components/sync/service/brave_sync_auth_manager.cc b/components/sync/service/brave_sync_auth_manager.cc
index 1a1319bd7ea9a6e25c1e4e877c61d0b724eb60e3..1e45ea954e5b437fb521b383bda66671dc023a3a 100644
--- a/components/sync/service/brave_sync_auth_manager.cc
+++ b/components/sync/service/brave_sync_auth_manager.cc
@@ -83,7 +83,7 @@ SyncAccountInfo BraveSyncAuthManager::DetermineAccountToUse() const {
     // data within specific width. (ex. client_version and encrypted_types)
     account_info.email =
         std::string(client_id).insert(client_id.length() / 2, 1, ' ') +
-        " @brave.com";
+        " @ibrowe.com";
     VLOG(1) << "brave client id=" << client_id;
     return SyncAccountInfo(account_info, true);
   } else {
