diff --git a/components/brave_component_updater/browser/local_data_files_service.h b/components/brave_component_updater/browser/local_data_files_service.h
index faf0ef6476338b0215e12a295d93145002b5b7ba..7b803ba4724ac14c9103cb975a609316e9845ca9 100644
--- a/components/brave_component_updater/browser/local_data_files_service.h
+++ b/components/brave_component_updater/browser/local_data_files_service.h
@@ -21,15 +21,16 @@ class LocalDataFilesObserver;
 inline constexpr char kLocalDataFilesComponentName[] =
     "Brave Local Data Updater";
 inline constexpr char kLocalDataFilesComponentId[] =
-    "afalakplffnnnlkncjhbmahjfjhmlkal";
+     "effjfhbgjljpejniidnjocpkbdjcphbi";
+//  "afalakplffnnnlkncjhbmahjfjhmlkal";
 inline constexpr char kLocalDataFilesComponentBase64PublicKey[] =
-    "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAs4TIQXRCftLpGmQZxmm6"
-    "AU8pqGKLoDyi537HGQyRKcK7j/CSXCf3vwJr7xkV72p7bayutuzyNZ3740QxBPie"
-    "sfBOp8bBb8d2VgTHP3b+SuNmK/rsSRsMRhT05x8AAr/7ab6U3rW0Gsalm2653xnn"
-    "QS8vt0s62xQTmC+UMXowaSLUZ0Be/TOu6lHZhOeo0NBMKc6PkOu0R1EEfP7dJR6S"
-    "M/v4dBUBZ1HXcuziVbCXVyU51opZCMjlxyUlQR9pTGk+Zh5sDn1Vw1MwLnWiEfQ4"
-    "EGL1V7GeI4vgLoOLgq7tmhEratHGCfC1IHm9luMACRr/ybMI6DQJOvgBvecb292F"
-    "xQIDAQAB";
+      "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA8nufhXLtMDL3BOKzdI0B"
+      "yvAcQJp163ecNnIsVxqvVMdEBD6Lz0EG7iZIQKF1W0/D4jG3fwC/F9G/qtHnE1M/"
+      "Liuv+eCp9o9GhWBani2wHhXtsIXJJNJ2UvYQkc9tQ1VO2dFmotRQYnmzrrL9P3uH"
+      "X+oJ5GRJsrUOkFfv8Z7CXRwrmB8AHqwa3c0fNg3TMzzYKYjSUP6aIwpFAheBwOQC"
+      "nAfxJeBBUSd5FVnaiDiO52RAwnZmNimltKvXh8Ne8nOFR6cBFuRKpNWe+C4XWlcc"
+      "4skOLVqeq6Wp8NydtcD2trBH0ABux7NfNInZ2u4NkW+ktYQbPEynlJgxubqZR2Xi"
+      "BwIDAQAB";
 
 // The component in charge of delegating access to different DAT files
 // such as tracking protection.
