diff --git a/components/brave_new_tab_ui/data/defaultTopSites.ts b/components/brave_new_tab_ui/data/defaultTopSites.ts
index de60029ecdc39975a7863cf93204d10a30eea2f1..04ada80be41186a6de7759ce074eac370cd2d5c3 100644
--- a/components/brave_new_tab_ui/data/defaultTopSites.ts
+++ b/components/brave_new_tab_ui/data/defaultTopSites.ts
@@ -12,7 +12,7 @@ import YouTubeFavicon from '../../img/newtab/defaultTopSitesIcon/youtube.png'
 export const defaultTopSitesData = [
   {
     name: 'App Store',
-    url: 'https://itunes.apple.com/app/brave-browser-fast-adblocker/id1052879175?mt=8',
+    url: 'https://itunes.apple.com/app/brave-browser-fast-adblocker/id6741485553?mt=8',
     favicon: AppStoreFavicon,
     background: 'rgba(255,255,255,0.8)'
   },
