diff --git a/ios/brave-ios/scripts/scheme_preaction.py b/ios/brave-ios/scripts/scheme_preaction.py
index 237871fbf329971d5a3b9a6ece7c5ff25477b3ae..575db8b01cfe2f56e42c7aaaa0be2f00216453bc 100755
--- a/ios/brave-ios/scripts/scheme_preaction.py
+++ b/ios/brave-ios/scripts/scheme_preaction.py
@@ -54,9 +54,10 @@ def main():
             err = inspect.cleandoc(f'''
             Expected out directory for chosen build doesn't exist:
 
-            {output_dir}
+              {output_dir}
 
-            Ensure you run the correct `npm run build` command prior to building
+            You must run a Release build first before attempting to Archive / App Store build.
+            Run the correct `npm run build` command prior to building in Xcode.
             ''')
             raise Exception(err)
     else:
