diff --git a/ios/brave-ios/Sources/AIChat/Constants/AIChatConstants.swift b/ios/brave-ios/Sources/AIChat/Constants/AIChatConstants.swift
index 3e704c9e4e3a9a6a963281bac3f595717c3d78bb..60a22b911034c05ed17f23b0d212e504be624f67 100644
--- a/ios/brave-ios/Sources/AIChat/Constants/AIChatConstants.swift
+++ b/ios/brave-ios/Sources/AIChat/Constants/AIChatConstants.swift
@@ -8,11 +8,11 @@ import Foundation
 public struct AIChatConstants {
   static let braveLeoPrivacyPolicyURL = URL(string: "https://brave.com/privacy/browser/#brave-leo")!
   static let braveLeoLearnMore = URL(
-    string: "https://support.brave.app/hc/en-us/categories/20990938292237-Brave-Leo"
+    string: "https://support.ibrowe.com/hc/en-us/categories/20990938292237-Brave-Leo"
   )!
   static let braveLeoSearchLearnMore = URL(
     string:
-      "https://support.brave.app/hc/en-us/articles/27586048343309-How-does-Leo-get-current-information"
+      "https://support.ibrowe.com/hc/en-us/articles/27586048343309-How-does-Leo-get-current-information"
   )!
   static let braveSearchURL = URL(string: "https://search.brave.com/search")!
 }
