diff --git a/ios/brave-ios/fastlane/Fastfile b/ios/brave-ios/fastlane/Fastfile
index 66fe06ab58409428df1affbcd90fea9aae076db8..c9bd5b6d688af794f699288d448de23a8e704d50 100644
--- a/ios/brave-ios/fastlane/Fastfile
+++ b/ios/brave-ios/fastlane/Fastfile
@@ -24,7 +24,7 @@ platform :ios do
     }))
   end
 
-  desc "Creates a Brave Beta build for TestFlight."
+  desc "Creates a iBrowe Beta build for TestFlight."
   lane :beta do |options|
     gymOverrides = {
       scheme: "Beta",
@@ -32,20 +32,20 @@ platform :ios do
       export_options: {
         manageAppVersionAndBuildNumber: false,
         provisioningProfiles: {
-          "com.brave.ios.browser.beta" => "Brave iOS Beta",
-          "com.brave.ios.browser.beta.ActionExtension" => "Brave iOS Beta Action Extension",
-          "com.brave.ios.browser.beta.ShareExtension" => "Brave iOS Beta Share Extension",
-          "com.brave.ios.browser.beta.BrowserIntents" => "Brave iOS Beta Intents Extension",
-          "com.brave.ios.browser.beta.BraveWidgetsExtension" => "Brave iOS Beta Widgets Extension",
-          "com.brave.ios.browser.beta.BraveWireGuard" => "Brave iOS Beta Wireguard Extension",
-          "com.brave.ios.browser.beta.AutoFillProvider" => "Brave iOS Beta AutoFill Provider",
+          "com.ibrowe.ios.browser.beta" => "iBrowe iOS Beta",
+          "com.ibrowe.ios.browser.beta.ActionExtension" => "iBrowe iOS Beta Action Extension",
+          "com.ibrowe.ios.browser.beta.ShareExtension" => "iBrowe iOS Beta Share Extension",
+          "com.ibrowe.ios.browser.beta.BrowserIntents" => "iBrowe iOS Beta Intents Extension",
+          "com.ibrowe.ios.browser.beta.BraveWidgetsExtension" => "iBrowe iOS Beta Widgets Extension",
+          "com.ibrowe.ios.browser.beta.BraveWireGuard" => "iBrowe iOS Beta Wireguard Extension",
+          "com.ibrowe.ios.browser.beta.AutoFillProvider" => "iBrowe iOS Beta AutoFill Provider",
         },
       }
     }
     testflight_build({gymOverrides: gymOverrides})
   end
 
-  desc "Creates a Brave Nightly build for TestFlight."
+  desc "Creates a iBrowe Nightly build for TestFlight."
   lane :nightly do |options|
     gymOverrides = {
       scheme: "Nightly",
@@ -53,13 +53,13 @@ platform :ios do
       export_options: {
         manageAppVersionAndBuildNumber: false,
         provisioningProfiles: {
-          "com.brave.ios.BrowserBeta" => "Brave iOS Nightly",
-          "com.brave.ios.BrowserBeta.ActionExtension" => "Brave iOS Nightly Action Extension",
-          "com.brave.ios.BrowserBeta.ShareExtension" => "Brave iOS Nightly Share Extension",
-          "com.brave.ios.BrowserBeta.BrowserIntents" => "Brave iOS Nightly Intents Extension",
-          "com.brave.ios.BrowserBeta.BraveWidgetsExtension" => "Brave iOS Nightly Widgets Extension",
-          "com.brave.ios.BrowserBeta.BraveWireGuard" => "Brave iOS Nightly Wireguard Extension",
-          "com.brave.ios.BrowserBeta.AutoFillProvider" => "Brave iOS Nightly AutoFill Provider",
+          "com.ibrowe.ios.BrowserBeta" => "iBrowe iOS Nightly",
+          "com.ibrowe.ios.BrowserBeta.ActionExtension" => "iBrowe iOS Nightly Action Extension",
+          "com.ibrowe.ios.BrowserBeta.ShareExtension" => "iBrowe iOS Nightly Share Extension",
+          "com.ibrowe.ios.BrowserBeta.BrowserIntents" => "iBrowe iOS Nightly Intents Extension",
+          "com.ibrowe.ios.BrowserBeta.BraveWidgetsExtension" => "iBrowe iOS Nightly Widgets Extension",
+          "com.ibrowe.ios.BrowserBeta.BraveWireGuard" => "iBrowe iOS Nightly Wireguard Extension",
+          "com.ibrowe.ios.BrowserBeta.AutoFillProvider" => "iBrowe iOS Nightly AutoFill Provider",
         },
       }
     }
@@ -74,13 +74,13 @@ platform :ios do
       export_options: {
         method: "app-store",
         provisioningProfiles: {
-          "com.brave.ios.browser" => "Brave iOS Release",
-          "com.brave.ios.browser.ActionExtension" => "Brave iOS Release Action Extension",
-          "com.brave.ios.browser.ShareExtension" => "Brave iOS Release Share Extension",
-          "com.brave.ios.browser.BrowserIntents" => "Brave iOS Release Intents Extension",
-          "com.brave.ios.browser.BraveWidgetsExtension" => "Brave iOS Release Widgets Extension",
-          "com.brave.ios.browser.BraveWireGuard" => "Brave iOS Release WireGuard Extension",
-          "com.brave.ios.browser.AutoFillProvider" => "Brave iOS Release AutoFill Provider",
+          "com.ibrowe.ios.browser" => "iBrowe iOS Release",
+          "com.ibrowe.ios.browser.ActionExtension" => "iBrowe iOS Release Action Extension",
+          "com.ibrowe.ios.browser.ShareExtension" => "iBrowe iOS Release Share Extension",
+          "com.ibrowe.ios.browser.BrowserIntents" => "iBrowe iOS Release Intents Extension",
+          "com.ibrowe.ios.browser.BraveWidgetsExtension" => "iBrowe iOS Release Widgets Extension",
+          "com.ibrowe.ios.browser.BraveWireGuard" => "iBrowe iOS Release WireGuard Extension",
+          "com.ibrowe.ios.browser.AutoFillProvider" => "iBrowe iOS Release AutoFill Provider",
         },
         manageAppVersionAndBuildNumber: false
       },
@@ -98,7 +98,7 @@ platform :ios do
       skip_waiting_for_build_processing: false,
       reject_build_waiting_for_review: true,
       ipa: "build/Client.ipa",
-      groups: ["Brave Internal"]
+      groups: ["iBrowe Internal"]
     }
     if options[:public] && options[:channel].downcase == "beta"
       pilotParams[:groups] << "Public Beta"
@@ -131,7 +131,7 @@ platform :ios do
       changelog: "Bug fixes & improvements",
       distribute_external: true,
       reject_build_waiting_for_review: true,
-      groups: ["Brave Internal"]
+      groups: ["iBrowe Internal"]
     }
     if options[:public] && options[:channel].downcase == "beta"
       pilotParams[:groups] << "Public Beta"
