diff --git a/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js b/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js
index 98ce481b201d515640dd4ca5859494942f986ece..708243789a6b7aab061025ad89d9850a41a35a19 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js
+++ b/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/BraveTranslateScript.js
@@ -109,7 +109,7 @@ var gtTimeInfo = {
     'fetchEnd': Date.now() + 1
 };
 var serverParams = '';
-var securityOrigin = 'https://translate.brave.com/';
+var securityOrigin = 'https://translate.ibrowe.com/';
 
 
 try {
