diff --git a/ios/brave-ios/App/iOS/Entitlements/Release.entitlements b/ios/brave-ios/App/iOS/Entitlements/Release.entitlements
index dc24fe4ec1970556c5c6cc95c493c81c8308107b..cec9adb0ff13aebbf93f297a81b2e0dd4fd8c833 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Release.entitlements
+++ b/ios/brave-ios/App/iOS/Entitlements/Release.entitlements
@@ -4,10 +4,6 @@
 <dict>
 	<key>aps-environment</key>
 	<string>production</string>
-	<key>com.apple.developer.associated-domains</key>
-	<array>
-		<string>applinks:vpn.brave.com</string>
-	</array>
 	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
 	<true/>
 	<key>com.apple.developer.networking.networkextension</key>
@@ -22,10 +18,10 @@
 	<array>
 		<string>TAG</string>
 	</array>
-	<key>com.apple.developer.web-browser</key>
-	<true/>
 	<key>com.apple.developer.siri</key>
 	<true/>
+	<key>com.apple.developer.web-browser</key>
+	<true/>
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID)</string>
