diff --git a/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift b/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
index 1ca0cdcb45e33fd7e46ff34ad42dafc88aff995b..68c564cabe905b8e95d34b62e7aed7f3dc260403 100644
--- a/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
+++ b/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
@@ -13,21 +13,21 @@ extension Strings {
       bundle: .module,
       value: "AutoFill Is On",
       comment:
-        "The title on a screen shown when the user turns on Brave as a password autofill provider"
+        "The title on a screen shown when the user turns on iBrowe as a password autofill provider"
     )
     public static let onboardingViewSubtitle = NSLocalizedString(
       "CredentialProvider.onboardingViewSubtitle",
       bundle: .module,
       value: "You can use saved passwords in other apps on your device",
       comment:
-        "The subtitle on a screen shown when the user turns on Brave as a password autofill provider"
+        "The subtitle on a screen shown when the user turns on iBrowe as a password autofill provider"
     )
     public static let onboardingViewFootnote = NSLocalizedString(
       "CredentialProvider.onboardingViewFootnote",
       bundle: .module,
       value: "To get to your passwords faster, deselect iCloud Keychain",
       comment:
-        "A footnote shown on a screen shown when the user turns on Brave as a password autofill provider. 'iCloud Keychain' is an Apple product."
+        "A footnote shown on a screen shown when the user turns on iBrowe as a password autofill provider. 'iCloud Keychain' is an Apple product."
     )
     public static let onboardingViewContinueCTA = NSLocalizedString(
       "CredentialProvider.onboardingViewContinueCTA",
@@ -38,9 +38,9 @@ extension Strings {
     public static let credentialListTitle = NSLocalizedString(
       "CredentialProvider.credentialListTitle",
       bundle: .module,
-      value: "Brave Passwords",
+      value: "iBrowe Passwords",
       comment:
-        "The title shown at the top of a list of credentials. 'Brave' in this case is the company name"
+        "The title shown at the top of a list of credentials. 'iBrowe' in this case is the company name"
     )
     public static let loginsForWebsite = NSLocalizedString(
       "CredentialProvider.loginsForWebsite",
