diff --git a/ios/brave-ios/Sources/BraveVPN/BraveVPN.swift b/ios/brave-ios/Sources/BraveVPN/BraveVPN.swift
index 8ac5eed34cd927a158c8b79a1472e6a5a9e54f64..cee072beada51c4385123f2a56d0a2d40f07028a 100644
--- a/ios/brave-ios/Sources/BraveVPN/BraveVPN.swift
+++ b/ios/brave-ios/Sources/BraveVPN/BraveVPN.swift
@@ -492,7 +492,7 @@ public class BraveVPN {
 
   /// Stores a in-memory list of vpn errors encountered during current browsing session.
   public private(set) static var errorLog = [(date: Date, message: String)]()
-  private static let errorLogQueue = DispatchQueue(label: "com.brave.errorLogQueue")
+  private static let errorLogQueue = DispatchQueue(label: "com.ibrowe.errorLogQueue")
 
   /// Prints out the error to the logger and stores it in a in memory array.
   /// This can be further used for a customer support form.
