diff --git a/ios/brave-ios/App/Configuration/Beta.xcconfig b/ios/brave-ios/App/Configuration/Beta.xcconfig
index 9c23358f834966e088cae90d32130786e576b922..8ce0435c8345e245690dab4e0eff83e9e0ef2223 100644
--- a/ios/brave-ios/App/Configuration/Beta.xcconfig
+++ b/ios/brave-ios/App/Configuration/Beta.xcconfig
@@ -6,9 +6,9 @@
 
 #include "Base.xcconfig"
 
-MOZ_BUNDLE_DISPLAY_NAME = BraveBeta
+MOZ_BUNDLE_DISPLAY_NAME = iBroweBeta
 
-BRAVE_URL_SCHEME = brave-beta
+BRAVE_URL_SCHEME = ibrowe-beta
 
 // Bundle Identifier
 MOZ_BUNDLE_ID = $(BASE_BUNDLE_ID).browser.beta
@@ -20,5 +20,5 @@ INCLUDE_SETTINGS_BUNDLE = NO
 SWIFT_ACTIVE_COMPILATION_CONDITIONS = BRAVE_CHANNEL_BETA $(brave_swift_active_compliation_conditions)
 
 // Manual Code-Signing
-CODE_SIGN_IDENTITY = Apple Distribution: Brave Software, Inc.
+CODE_SIGN_IDENTITY = Apple Development: Yongyut Jantabot (VJ4PQMKSUA)
 CODE_SIGN_STYLE = Manual
