diff --git a/ios/brave-ios/App/iOS/Entitlements/Enterprise.entitlements b/ios/brave-ios/App/iOS/Entitlements/Enterprise.entitlements
index 4f6ad4ad059d48c2c48fb7b915ed86fd18f9755a..57dbb7c984a60fee927374ad0f07e9a40cbaee18 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Enterprise.entitlements
+++ b/ios/brave-ios/App/iOS/Entitlements/Enterprise.entitlements
@@ -4,10 +4,6 @@
 <dict>
 	<key>aps-environment</key>
 	<string>production</string>
-	<key>com.apple.developer.associated-domains</key>
-	<array>
-		<string>applinks:vpn.brave.com</string>
-	</array>
 	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
 	<true/>
 	<key>com.apple.developer.networking.networkextension</key>
