diff --git a/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/ShredSettingsView.swift b/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/ShredSettingsView.swift
index fedbe21d81833fd177c6f75d88b3e281b3de6042..fc01ef11b4015a6bf23d847a77a6b6ebc6d01a58 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/ShredSettingsView.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/ShredSettingsView.swift
@@ -82,7 +82,7 @@ struct ShredSettingsView: View {
 
 #Preview {
   ShredSettingsView(
-    url: URL(string: "https://brave.com")!,
+    url: URL(string: "https://ibrowe.com")!,
     isPersistent: false
   ) {
     // Do nothing
