diff --git a/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift b/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
index 17aa2c3937ee6578897c568f8f54c8d1b1828c80..bd5e172cb20d126898dca2f1f120df12217d72e7 100644
--- a/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
+++ b/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
@@ -157,7 +157,7 @@ public class FeedDataSource: ObservableObject {
   public enum Environment: String, CaseIterable {
     case dev = "brave.software"
     case staging = "bravesoftware.com"
-    case production = "brave.com"
+    case production = "ibrowe.com"
   }
 
   /// The current Brave News environment.
@@ -1038,7 +1038,7 @@ public class FeedDataSource: ObservableObject {
       // Move to OSSignposter when we're 15+
       let log = OSLog(
         subsystem: Bundle.main.bundleIdentifier ?? "com.brave.ios",
-        category: "Brave News"
+        category: "iBrowe News"
       )
       let signpostID = OSSignpostID(log: log)
       os_signpost(.begin, log: log, name: "Card Generation", signpostID: signpostID)
@@ -1072,7 +1072,7 @@ public class FeedDataSource: ObservableObject {
       os_signpost(
         .end,
         log: log,
-        name: "Brave News Card Generation",
+        name: "iBrowe News Card Generation",
         signpostID: signpostID,
         "%d cards",
         generatedCards.count
