diff --git a/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusManager.swift b/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusManager.swift
index 84e1e161494a696b7447e288768bfa4261c9b313..ca30d4ba5dedb4a03369a38dee2c070e328958b7 100644
--- a/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusManager.swift
+++ b/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusManager.swift
@@ -184,8 +184,8 @@ private enum CredentialType {
 
   static func from(domain: String) -> CredentialType {
     switch domain {
-    case "vpn.brave.software", "vpn.bravesoftware.com", "vpn.brave.com": return .vpn
-    case "leo.brave.software", "leo.bravesoftware.com", "leo.brave.com": return .leo
+    case "vpn.ibrowe.software", "vpn.ibrowesoftware.com", "vpn.ibrowe.com": return .vpn
+    case "leo.ibrowe.software", "leo.ibrowesoftware.com", "leo.ibrowe.com": return .leo
     default:
       return .unknown
     }
