diff --git a/ios/brave-ios/Sources/Brave/Assets/Interstitial Pages/Pages/HTTPBlocked.html b/ios/brave-ios/Sources/Brave/Assets/Interstitial Pages/Pages/HTTPBlocked.html
index c007dacedaf84a1a6be72f1b4aa88329b045af8a..dc5c32bd86754f880f47d9350eb9aba99d7af560 100644
--- a/ios/brave-ios/Sources/Brave/Assets/Interstitial Pages/Pages/HTTPBlocked.html	
+++ b/ios/brave-ios/Sources/Brave/Assets/Interstitial Pages/Pages/HTTPBlocked.html	
@@ -20,7 +20,7 @@
       <h1>%blocked_title%</h1>
       <p class="description">
         %blocked_description%
-        <a href="https://support.brave.com/hc/en-us/articles/15513090104717-Strict-HTTPS-Upgrade-Mode-in-Brave-Browser" target="_blank" rel="noopener">
+        <a href="https://support.ibrowe.com/hc/en-us/articles/15513090104717-Strict-HTTPS-Upgrade-Mode-in-Brave-Browser" target="_blank" rel="noopener">
           %learn_more%
         </a>
       </p>
