diff --git a/ios/brave-ios/Sources/Growth/DAU.swift b/ios/brave-ios/Sources/Growth/DAU.swift
index 246527d7e52950af0768fd31ba98d1df5aab5b48..723b75131d7bd206259cf2a54080dc2ea32a4a11 100644
--- a/ios/brave-ios/Sources/Growth/DAU.swift
+++ b/ios/brave-ios/Sources/Growth/DAU.swift
@@ -20,8 +20,8 @@ public class DAU {
     // TODO: Handle via brave-stats-updater-server switch and get URL from brave_stats_updater_url
     let domain =
       AppConstants.isOfficialBuild
-      ? "https://usage-ping.brave.com/"
-      : "https://usage-ping.bravesoftware.com/"
+      ? "https://usage-ping.ibrowe.com/"
+      : "https://usage-ping.ibrowesoftware.com/"
 
     return "\(domain)\(apiVersion)/usage/ios?platform=ios"
   }
