diff --git a/ios/brave-ios/App/WebShell/ContentView.swift b/ios/brave-ios/App/WebShell/ContentView.swift
index 632860d5572c65c658377acef052f8e00da2b91f..a1f9f9c322972137547a6793e06c27af5f859424 100644
--- a/ios/brave-ios/App/WebShell/ContentView.swift
+++ b/ios/brave-ios/App/WebShell/ContentView.swift
@@ -182,7 +182,7 @@ struct ContentView: View {
     )
     .onAppear {
       DispatchQueue.main.async {
-        model.webView.load(URLRequest(url: URL(string: "https://search.brave.com")!))
+        model.webView.load(URLRequest(url: URL(string: "https://search.ibrowe.com")!))
       }
     }
   }
