diff --git a/ios/brave-ios/App/BraveWireGuard/Entitlements/BraveWireGuardNightly.entitlements b/ios/brave-ios/App/BraveWireGuard/Entitlements/BraveWireGuardNightly.entitlements
index 35b43d10a1c90fef5037d370f542d74dbf0dd197..12115c0d41acaae0878a5228945661e762fb7fe6 100644
--- a/ios/brave-ios/App/BraveWireGuard/Entitlements/BraveWireGuardNightly.entitlements
+++ b/ios/brave-ios/App/BraveWireGuard/Entitlements/BraveWireGuardNightly.entitlements
@@ -8,7 +8,9 @@
 	</array>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.$(MOZ_BUNDLE_ID).unique</string>
+		<string>group.com.ibrowe.ios.browser</string>
+		<string>group.com.ibrowe.ios.BrowserBeta.unique</string>
+		<string>group.com.ibrowe.ios.browser.beta</string>
 	</array>
 </dict>
 </plist>
