diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/UniversalLinkManager.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/UniversalLinkManager.swift
index d05a8553eb73d4b6f5c09825735e2347417be8b6..52604ec59cff63e16d21338dcc9b19f569222cfb 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/UniversalLinkManager.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/UniversalLinkManager.swift
@@ -13,7 +13,7 @@ public struct UniversalLinkManager {
 
     public var associatedDomain: String {
       switch self {
-      case .buyVPN: return "vpn.brave.com"
+      case .buyVPN: return "vpn.ibrowe.com"
       }
     }
 
