diff --git a/ios/brave-ios/App/iOS/Entitlements/Nightly.entitlements b/ios/brave-ios/App/iOS/Entitlements/Nightly.entitlements
index ab4a17804994bfea3931f0a9fc0a8f07bcd3e9b9..fc612b3b012bd4fbd5bf55fc396c11dde84c124c 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Nightly.entitlements
+++ b/ios/brave-ios/App/iOS/Entitlements/Nightly.entitlements
@@ -28,11 +28,5 @@
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID).unique</string>
 	</array>
-	<key>com.apple.developer.web-browser</key>
-	<true/>
-	<key>com.apple.developer.carplay-audio</key>
-	<true/>
-	<key>com.apple.developer.playable-content</key>
-	<true/>
 </dict>
 </plist>
