diff --git a/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
index 208d179003da74fe37acb4a7635a6a7c1cbf8f0f..868c71421be0216ecaf72bb09334499d90ab54bc 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
@@ -401,11 +401,11 @@ class SettingsViewController: TableViewController {
       )
     )
 
-    if !tabManager.privateBrowsingManager.isPrivateBrowsing && FeatureList.kAIChat.enabled {
-      section.rows.append(leoSettingsRow)
-    }
+//    if !tabManager.privateBrowsingManager.isPrivateBrowsing && FeatureList.kAIChat.enabled {
+//      section.rows.append(leoSettingsRow)
+//    }
 
-    section.rows.append(vpnSettingsRow)
+//    section.rows.append(vpnSettingsRow)
 
     section.rows.append(
       Row(
@@ -953,7 +953,7 @@ class SettingsViewController: TableViewController {
             // Rate Brave
             guard
               let writeReviewURL = URL(
-                string: "https://itunes.apple.com/app/id1052879175?action=write-review"
+                string: "https://apps.apple.com/us/app/ibrowe/id6741485553?action=write-review"
               )
             else { return }
             UIApplication.shared.open(writeReviewURL)
