diff --git a/ios/brave-ios/App/BraveWidgets/LockScreenFavoriteWidget.swift b/ios/brave-ios/App/BraveWidgets/LockScreenFavoriteWidget.swift
index 6dc484c8fef203a539b0a0bc5640f9d98adb4768..5bdc1ec1fe3abf126c37a7b6be259153e89f9db9 100644
--- a/ios/brave-ios/App/BraveWidgets/LockScreenFavoriteWidget.swift
+++ b/ios/brave-ios/App/BraveWidgets/LockScreenFavoriteWidget.swift
@@ -144,7 +144,7 @@ struct LockScreenFavoriteViewWidget_Previews: PreviewProvider {
         entry: .init(
           date: .now,
           favorite: .init(
-            url: URL(string: "https://brave.com")!,
+            url: URL(string: "https://ibrowe.com")!,
             title: "Brave",
             favicon: .init(
               image: nil,
