diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift
index bac25306a696d1a3223912633b1e9ebbdc85f3d7..e99e456479f93a2726234e58a534307b0c7c70df 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift
@@ -292,7 +292,7 @@ extension NTPBackgroundImage {
   static let fallback: NTPBackgroundImage = .init(
     imagePath: Bundle.module.url(forResource: "corwin-prescott-3", withExtension: "jpg")!,
     author: "Corwin Prescott",
-    link: URL(string: "https://www.brave.com")!
+    link: URL(string: "https://www.ibrowe.com")!
   )
 }
 
