diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Notifications/NTPLearnMoreViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Notifications/NTPLearnMoreViewController.swift
index e131d0773a85f8294336e09d1aa9a7e47efcfe1a..412d00baf907f102e4e02d65aa00b35b71bd06a7 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Notifications/NTPLearnMoreViewController.swift	
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Notifications/NTPLearnMoreViewController.swift	
@@ -22,8 +22,8 @@ class NTPLearnMoreViewController: BottomSheetViewController {
   private let state: BrandedImageCalloutState
   private let rewards: BraveRewards
 
-  private let termsOfServiceUrl = "https://www.brave.com/terms_of_use"
-  private let learnMoreAboutBraveRewardsUrl = "https://brave.com/brave-rewards/"
+  private let termsOfServiceUrl = "https://www.ibrowe.com/terms_of_use"
+  private let learnMoreAboutBraveRewardsUrl = "https://ibrowe.com/ibrowe-rewards/"
 
   init(state: BrandedImageCalloutState, rewards: BraveRewards) {
     self.state = state
