diff --git a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsBeta.entitlements b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsBeta.entitlements
index ee98f9a7c4b1b0f6ef5d7f6db75dda6c96a88c38..b0f5c77c0f86af9b4cfca217231fe19b473deda7 100644
--- a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsBeta.entitlements
+++ b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsBeta.entitlements
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.com.brave.ios.browser.beta</string>
+		<string>group.com.ibrowe.ios.BrowserBeta.unique</string>
 	</array>
 </dict>
 </plist>
