diff --git a/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift b/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
index 17aa2c3937ee6578897c568f8f54c8d1b1828c80..d62befc049bf91d036c474b8d8eb943cb38c5a28 100644
--- a/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
+++ b/ios/brave-ios/Sources/BraveNews/Composer/FeedDataSource.swift
@@ -111,8 +111,8 @@ public class FeedDataSource: ObservableObject {
   public var getAdsAPI: (() -> BraveAds)?
   public var historyAPI: BraveHistoryAPI?
 
-  private let todayQueue = DispatchQueue(label: "com.brave.today")
-  private let reloadQueue = DispatchQueue(label: "com.brave.today.reload")
+  private let todayQueue = DispatchQueue(label: "com.ibrowe.today")
+  private let reloadQueue = DispatchQueue(label: "com.ibrowe.today.reload")
 
   struct FeedDecodingError: Identifiable {
     var id: UUID = .init()
@@ -155,8 +155,8 @@ public class FeedDataSource: ObservableObject {
 
   /// A Brave News environment
   public enum Environment: String, CaseIterable {
-    case dev = "brave.software"
-    case staging = "bravesoftware.com"
+    case dev = "ibrowe.software"
+    case staging = "ibrowesoftware.com"
     case production = "brave.com"
   }
 
@@ -1037,7 +1037,7 @@ public class FeedDataSource: ObservableObject {
       )
       // Move to OSSignposter when we're 15+
       let log = OSLog(
-        subsystem: Bundle.main.bundleIdentifier ?? "com.brave.ios",
+        subsystem: Bundle.main.bundleIdentifier ?? "com.ibrowe.ios",
         category: "Brave News"
       )
       let signpostID = OSSignpostID(log: log)
