diff --git a/ios/brave-ios/App/BraveWidgetsExtensionRelease (AppStore).entitlements b/ios/brave-ios/App/BraveWidgetsExtensionRelease (AppStore).entitlements
index d2c5d08d55f5623e31d259668adc523c3f6ea3c5..6d8181045e4116e2e87100baac02b4aa5c385cf1 100644
--- a/ios/brave-ios/App/BraveWidgetsExtensionRelease (AppStore).entitlements	
+++ b/ios/brave-ios/App/BraveWidgetsExtensionRelease (AppStore).entitlements	
@@ -5,7 +5,6 @@
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID)</string>
-		<string>group.com.ibrowe.ios.BrowserBeta.unique</string>
 	</array>
 </dict>
 </plist>
