diff --git a/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
index 0adc27dde274b3948e6d475e24d5400e9316f597..3a22f145b1b704f43cbb1f2cba0cda14b6b30a16 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Settings/SettingsViewController.swift
@@ -335,7 +335,7 @@ class SettingsViewController: TableViewController {
             let controller = ChromeWebUIController(braveCore: braveCore, isPrivateBrowsing: false)
             let container = UINavigationController(rootViewController: controller)
             controller.title = Strings.braveAccount
-            controller.webView.load(URLRequest(url: URL(string: "brave://account")!))
+            controller.webView.load(URLRequest(url: URL(string: "ibrowe://account")!))
             controller.navigationItem.rightBarButtonItem = .init(
               systemItem: .done,
               primaryAction: .init { [unowned container] _ in
@@ -1142,13 +1142,13 @@ class SettingsViewController: TableViewController {
           text: Strings.settingsLicenses,
           selection: { [unowned self] in
             if FeatureList.kUseChromiumWebViews.enabled {
-              if let url = URL(string: "brave://credits") {
+              if let url = URL(string: "ibrowe://credits") {
                 settingsDelegate?.settingsOpenURLInNewTab(url)
               }
             } else {
               let controller = ChromeWebUIController(braveCore: braveCore, isPrivateBrowsing: false)
               let container = UINavigationController(rootViewController: controller)
-              controller.webView.load(URLRequest(url: URL(string: "brave://credits")!))
+              controller.webView.load(URLRequest(url: URL(string: "ibrowe://credits")!))
               controller.navigationItem.leftBarButtonItem = .init(
                 systemItem: .done,
                 primaryAction: .init { [unowned container] _ in
