diff --git a/ios/brave-ios/Sources/BraveStrings/BraveStrings+Sync.swift b/ios/brave-ios/Sources/BraveStrings/BraveStrings+Sync.swift
index e7e4ed74767af892c48e141bf028de9d3bfa726e..2eeda511efe3bae3c29bc29d75e7a01110b725f3 100644
--- a/ios/brave-ios/Sources/BraveStrings/BraveStrings+Sync.swift
+++ b/ios/brave-ios/Sources/BraveStrings/BraveStrings+Sync.swift
@@ -58,7 +58,7 @@ extension Strings {
       tableName: "BraveShared",
       bundle: .module,
       value:
-        "To start, you will need Brave installed on all the devices you plan to sync. To chain them together, start a sync chain that you will use to securely link all of your devices together.",
+        "To start, you will need iBrowe installed on all the devices you plan to sync. To chain them together, start a sync chain that you will use to securely link all of your devices together.",
       comment: "Sync settings welcome"
     )
     public static let newSyncCode = NSLocalizedString(
@@ -122,7 +122,7 @@ extension Strings {
       tableName: "BraveShared",
       bundle: .module,
       value:
-        "Using existing synced device open Brave Settings and navigate to Settings -> Sync. Choose \"Add Device\" and scan the code displayed on the screen.",
+        "Using existing synced device open iBrowe Settings and navigate to Settings -> Sync. Choose \"Add Device\" and scan the code displayed on the screen.",
       comment: "Sync to existing device description"
     )
     public static let addDeviceScanDescription = NSLocalizedString(
@@ -386,7 +386,7 @@ extension Strings {
       tableName: "BraveShared",
       bundle: .module,
       value:
-        "Data currently synced will be retained but all data in Brave’s Sync cache will be deleted. You will need to start a new sync chain to sync device data again.",
+        "Data currently synced will be retained but all data in iBrowe’s Sync cache will be deleted. You will need to start a new sync chain to sync device data again.",
       comment: "Message for removing last device from Sync"
     )
     public static let removeLastDeviceRemoveButtonName = NSLocalizedString(
@@ -446,7 +446,7 @@ extension Strings {
       tableName: "BraveShared",
       bundle: .module,
       value:
-        "It looks like this code was created on a device running an older version of Brave. Please update to the latest version of Brave on your other device, and then create a new sync code.",
+        "It looks like this code was created on a device running an older version of iBrowe. Please update to the latest version of iBrowe on your other device, and then create a new sync code.",
       comment: "Sync Error Description"
     )
     public static let expiredError = NSLocalizedString(
@@ -623,7 +623,7 @@ extension Strings {
         tableName: "BraveShared",
         bundle: .module,
         value:
-          "Deleting your account will remove your encrypted data from Brave servers and disable Sync on all of your connected devices.",
+          "Deleting your account will remove your encrypted data from iBrowe servers and disable Sync on all of your connected devices.",
         comment: "Part 1 Description for Alert used action Delete Sync Account."
       )
     public static let deleteAccountAlertDescriptionPart2 =
