diff --git a/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift b/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
index 1ca0cdcb45e33fd7e46ff34ad42dafc88aff995b..dc3b503cb311dc9e7a520ce35595f9375d5ab5fb 100644
--- a/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
+++ b/ios/brave-ios/Sources/CredentialProviderUI/CredentialProviderStrings.swift
@@ -38,7 +38,7 @@ extension Strings {
     public static let credentialListTitle = NSLocalizedString(
       "CredentialProvider.credentialListTitle",
       bundle: .module,
-      value: "Brave Passwords",
+      value: "iBrowe Passwords",
       comment:
         "The title shown at the top of a list of credentials. 'Brave' in this case is the company name"
     )
