diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchSuggestClient.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchSuggestClient.swift
index a80bff712e6edfbbde343022379226cae274550c..01b9c9b40e02f571c1a09d9001cc2a7cf0e9f46b 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchSuggestClient.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/Search/SearchSuggestClient.swift
@@ -10,7 +10,7 @@ import Shared
 ///
 /// Query callbacks that must run even if they are cancelled should wrap their contents in `withExtendendLifetime`.
 class SearchSuggestClient {
-  static let errorDomain = "com.brave.ios.SearchSuggestClient"
+  static let errorDomain = "com.ibrowe.ios.SearchSuggestClient"
   static let invalidEngineErrorCode = 0
   static let invalidResponseErrorCode = 1
 
