diff --git a/ios/brave-ios/App/iOS/Entitlements/Debug (AppStore).entitlements b/ios/brave-ios/App/iOS/Entitlements/Debug (AppStore).entitlements
index cb83da7993b1d6e5cbdaaeac2fc09fbdf093cb14..796847ea12efe5c9c5735d25eb20fa9a37ce8780 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Debug (AppStore).entitlements	
+++ b/ios/brave-ios/App/iOS/Entitlements/Debug (AppStore).entitlements	
@@ -5,15 +5,9 @@
 	<key>aps-environment</key>
 	<string>development</string>
 	<key>com.apple.developer.associated-domains</key>
-	<array>
-		<string>applinks:vpn.brave.com</string>
-	</array>
+	<array/>
 	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
 	<true/>
-	<key>com.apple.developer.browser.app-installation</key>
-	<true/>
-	<key>com.apple.developer.carplay-audio</key>
-	<true/>
 	<key>com.apple.developer.networking.networkextension</key>
 	<array>
 		<string>packet-tunnel-provider</string>
@@ -26,12 +20,8 @@
 	<array>
 		<string>TAG</string>
 	</array>
-	<key>com.apple.developer.playable-content</key>
-	<true/>
 	<key>com.apple.developer.siri</key>
 	<true/>
-	<key>com.apple.developer.web-browser</key>
-	<true/>
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID)</string>
