diff --git a/ios/brave-ios/App/iOS/Supporting Files/Info.plist b/ios/brave-ios/App/iOS/Supporting Files/Info.plist
index f67f8fd73d8dd45a920f27ca0eb99a4757ac5b86..24593c6117ad1fc919b855b08a1fd83307131d65 100644
--- a/ios/brave-ios/App/iOS/Supporting Files/Info.plist	
+++ b/ios/brave-ios/App/iOS/Supporting Files/Info.plist	
@@ -32,7 +32,7 @@
 			<key>CFBundleTypeRole</key>
 			<string>Editor</string>
 			<key>CFBundleURLName</key>
-			<string>com.brave.ios</string>
+			<string>com.ibrowe.ios</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>http</string>
@@ -68,15 +68,15 @@
 	<key>NSCameraUsageDescription</key>
 	<string>This lets you take and upload photos.</string>
 	<key>NSFaceIDUsageDescription</key>
-	<string>This authenticates your access to Brave</string>
+	<string>This authenticates your access to iBrowe</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string>Websites you visit may request your location.</string>
 	<key>NSMicrophoneUsageDescription</key>
-	<string>This allows Brave to submit search queries using your voice.</string>
+	<string>This allows iBrowe to submit search queries using your voice.</string>
 	<key>NSPhotoLibraryAddUsageDescription</key>
 	<string>This lets you save photos.</string>
 	<key>NSSpeechRecognitionUsageDescription</key>
-	<string>This permission is used to enable Apple Advanced Speech Recognition. If not allowed `on device` recognition will be used and searches will be totally private. Brave does not store or share your voice searches.</string>
+	<string>This permission is used to enable Apple Advanced Speech Recognition. If not allowed `on device` recognition will be used and searches will be totally private. iBrowe does not store or share your voice searches.</string>
 	<key>NSUserActivityTypes</key>
 	<array>
 		<string>LockScreenFavoriteConfigurationIntent</string>
@@ -86,8 +86,8 @@
 		<string>OpenWebsiteIntent</string>
 		<string>ShortcutsConfigurationIntent</string>
 		<string>StatsConfigurationIntent</string>
-		<string>com.brave.ios.browser-scene</string>
-		<string>com.brave.ios.browsing</string>
+		<string>com.ibrowe.ios.browser-scene</string>
+		<string>com.ibrowe.ios.browsing</string>
 	</array>
 	<key>UIApplicationSceneManifest</key>
 	<dict>
@@ -134,9 +134,7 @@
 		<string>armv7</string>
 	</array>
 	<key>UISupportedExternalAccessoryProtocols</key>
-	<array>
-		<string>com.yubico.ylp</string>
-	</array>
+	<array/>
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
