diff --git a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
index 7dbc2a561af36f4c4166e29fb0c44ea738dab3da..64e946175465dea9f702232aabc3de042d07b3e3 100644
--- a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
+++ b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
@@ -78,13 +78,13 @@ extension Action.Identifier {
     defaultVisibility: .hidden
   )
 
-  public static let braveTalk: Self = .init(
-    id: "BraveTalk",
-    title: Strings.ActionTitles.braveTalk,
-    braveSystemImage: "leo.product.brave-talk",
-    defaultRank: 2400,
-    defaultVisibility: .hidden
-  )
+  // public static let braveTalk: Self = .init(
+  //   id: "BraveTalk",
+  //   title: Strings.ActionTitles.braveTalk,
+  //   braveSystemImage: "leo.product.brave-talk",
+  //   defaultRank: 2400,
+  //   defaultVisibility: .hidden
+  // )
 
   public static let braveWallet: Self = .init(
     id: "BraveWallet",
