diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
index 4411f8e1567a4da77da41da9def30243ba7b0c5b..40581d52b9d2bc96fee6499a5a890a5074694154 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
@@ -42,26 +42,26 @@ enum DomainUserScript: CaseIterable {
     switch self {
     case .braveSearchHelper:
       return Set([
-        "search.brave.com", "search.brave.software",
-        "search.bravesoftware.com", "safesearch.brave.com",
-        "safesearch.brave.software", "safesearch.bravesoftware.com",
+        "search.brave.com", "search.ibrowe.software",
+        "search.ibrowesoftware.com", "safesearch.brave.com",
+        "safesearch.ibrowe.software", "safesearch.ibrowesoftware.com",
         "search-dev-local.brave.com",
       ])
     #if canImport(BraveTalk)
     case .braveTalkHelper:
       return Set([
-        "talk.brave.com",
-        "talk.bravesoftware.com",
-        "talk.brave.software",
+        "talk.ibrowe.com",
+        "talk.ibrowesoftware.com",
+        "talk.ibrowe.software",
       ])
     #endif
     case .bravePlaylistFolderSharingHelper:
-      return Set(["playlist.bravesoftware.com", "playlist.brave.com"])
+      return Set(["playlist.ibrowesoftware.com", "playlist.ibrowe.com"])
     case .braveSkus:
       return Set([
-        "account.brave.com",
-        "account.bravesoftware.com",
-        "account.brave.software",
+        "account.ibrowe.com",
+        "account.ibrowesoftware.com",
+        "account.ibrowe.software",
       ])
     }
   }
