diff --git a/ios/brave-ios/Sources/Shared/AppInfo.swift b/ios/brave-ios/Sources/Shared/AppInfo.swift
index 6b2fca21ab8e560874eada532a7f4d1693afd094..417e9ba807d2be9a2488ceaeaddb11143b72abba 100644
--- a/ios/brave-ios/Sources/Shared/AppInfo.swift
+++ b/ios/brave-ios/Sources/Shared/AppInfo.swift
@@ -41,9 +41,9 @@ open class AppInfo {
     var bundleIdentifier = baseBundleIdentifier
 
     // External beta
-    if bundleIdentifier == "com.brave.ios.BrowserBeta" {
+    if bundleIdentifier == "com.ibrowe.ios.BrowserBeta" {
       // com.brave.ios.BrowserBeta is taken and can't be used as an app group.
-      bundleIdentifier = "com.brave.ios.BrowserBeta.unique"
+      bundleIdentifier = "com.ibrowe.ios.BrowserBeta.unique"
     }
     return "group." + bundleIdentifier
   }
@@ -51,10 +51,10 @@ open class AppInfo {
   /// Return the keychain access group.
   public static func keychainAccessGroupWithPrefix(_ prefix: String) -> String {
     var bundleIdentifier = baseBundleIdentifier
-    if bundleIdentifier == "com.brave.ios.FennecEnterprise" {
+    if bundleIdentifier == "com.ibrowe.ios.FennecEnterprise" {
       // Bug 1373726 - Base bundle identifier incorrectly generated for Nightly builds
       // This can be removed when we are able to fix the app group in the developer portal
-      bundleIdentifier = "com.brave.ios.Fennec.enterprise"
+      bundleIdentifier = "com.ibrowe.ios.Fennec.enterprise"
     }
     return prefix + "." + bundleIdentifier
   }
