diff --git a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
index e1056153473474d81ece30e7379062fb0f09c1c4..6c902f1eaec24aa9d57a769a6728ff249f8b371a 100644
--- a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
+++ b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
@@ -77,13 +77,13 @@ extension Action.Identifier {
     defaultVisibility: .hidden
   )
 
-  public static let braveTalk: Self = .init(
-    id: "BraveTalk",
-    title: Strings.ActionTitles.braveTalk,
-    braveSystemImage: "leo.product.brave-talk",
-    defaultRank: 2400,
-    defaultVisibility: .hidden
-  )
+  // public static let braveTalk: Self = .init(
+  //   id: "BraveTalk",
+  //   title: Strings.ActionTitles.braveTalk,
+  //   braveSystemImage: "leo.product.brave-talk",
+  //   defaultRank: 2400,
+  //   defaultVisibility: .hidden
+  // )
 
   public static let braveWallet: Self = .init(
     id: "BraveWallet",
