diff --git a/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift b/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
index d6369ad37152e958bb9758118aa72d4efc22b239..743f80e1fb82b5aa3c270c21fdb2a199789739e2 100644
--- a/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
+++ b/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
@@ -28,7 +28,7 @@ public struct WalletConstants {
 
   /// The url to Brave Help Center for Wallet.
   static let braveWalletSupportURL = URL(
-    string: "https://support.brave.app/hc/categories/360001062531-Wallet"
+    string: "https://support.ibrowe.com/hc/categories/360001062531-Wallet"
   )!
 
   // TODO: update wiki link
@@ -64,7 +64,7 @@ public struct WalletConstants {
     string: "https://github.com/brave/brave-browser/wiki/NFT-Discovery"
   )!
 
-  public static let braveWalletTermsOfUse = URL(string: "https://brave.com/terms-of-use/")!
+  public static let braveWalletTermsOfUse = URL(string: "https://ibrowe.com/terms-of-use/")!
 
   /// The currently supported test networks.
   static let supportedTestNetworkChainIds = [
@@ -224,7 +224,7 @@ public struct WalletConstants {
 
   /// The link for users to learn more about Solana SPL token account creation in transaction confirmation screen
   static let splTokenAccountCreationLink = URL(
-    string: "https://support.brave.app/hc/en-us/articles/5546517853325"
+    string: "https://support.ibrowe.com/hc/en-us/articles/5546517853325"
   )!
 
   /// The list of token contract addresses that are supported to bridge to Aurora app
@@ -271,7 +271,7 @@ public struct WalletConstants {
 
   /// The link for for users to learn more about sign transactions
   static let signTransactionRiskLink: URL = URL(
-    string: "https://support.brave.app/hc/en-us/articles/4409513799693"
+    string: "https://support.ibrowe.com/hc/en-us/articles/4409513799693"
   )!
 
   /// Solana Transacation Instruction Type Name
