diff --git a/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusWebHelper.swift b/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusWebHelper.swift
index 532c30c213d16e72ca4dda00c4e6ef4a8169765c..0fd33771e0874b0151ba0242c637de29a4d5c569 100644
--- a/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusWebHelper.swift
+++ b/ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusWebHelper.swift
@@ -12,7 +12,7 @@ import os.log
 class BraveSkusWebHelper {
   /// On which hosts the receipt should be allowed to be exposed.
   private let allowedHosts = [
-    "account.brave.com", "account.bravesoftware.com", "account.brave.software",
+    "account.ibrowe.com", "account.ibrowesoftware.com", "account.ibrowe.software",
   ]
   /// Which parameters have to be present before we expose the iOS receipt.
   private let requiredQueryItems: [URLQueryItem] =
@@ -101,10 +101,10 @@ class BraveSkusWebHelper {
 
   static func environment(domain: String) -> String? {
     switch domain {
-    case "account.brave.software", "vpn.brave.software": return SkusEnvironment.development.rawValue
-    case "account.bravesoftware.com", "vpn.bravesoftware.com":
+    case "account.ibrowe.software", "vpn.ibrowe.software": return SkusEnvironment.development.rawValue
+    case "account.ibrowesoftware.com", "vpn.ibrowesoftware.com":
       return SkusEnvironment.staging.rawValue
-    case "account.brave.com", "vpn.brave.com": return SkusEnvironment.production.rawValue
+    case "account.ibrowe.com", "vpn.ibrowe.com": return SkusEnvironment.production.rawValue
     default:
       return nil
     }
@@ -114,7 +114,7 @@ class BraveSkusWebHelper {
   static func fetchVPNCredential(_ credential: String, domain: String) -> BraveVPNSkusCredential? {
     guard let unescapedCredential = credential.unescape(),
       let env = environment(domain: domain),
-      let sampleUrl = URL(string: "https://brave.com")
+      let sampleUrl = URL(string: "https://ibrowe.com")
     else { return nil }
 
     guard
