diff --git a/ios/brave-ios/App/WebShell/ContentView.swift b/ios/brave-ios/App/WebShell/ContentView.swift
index 6d87d6c5fcb76fd6bec26a5ac38d2575558714a8..ec3fdb8517d45bee0c507f7fdd943e3617edf3bf 100644
--- a/ios/brave-ios/App/WebShell/ContentView.swift
+++ b/ios/brave-ios/App/WebShell/ContentView.swift
@@ -180,7 +180,7 @@ struct ContentView: View {
     )
     .onAppear {
       DispatchQueue.main.async {
-        model.webView.load(URLRequest(url: URL(string: "https://search.brave.com")!))
+        model.webView.load(URLRequest(url: URL(string: "https://search.ibrowe.com")!))
       }
     }
   }
