diff --git a/ios/brave-ios/App/Configuration/Release.xcconfig b/ios/brave-ios/App/Configuration/Release.xcconfig
index 535ac4a6cff2be28f8635e41a2c69b803d548a3b..9861f1cef5175c16a5eaae0aaf819ce43a997a89 100644
--- a/ios/brave-ios/App/Configuration/Release.xcconfig
+++ b/ios/brave-ios/App/Configuration/Release.xcconfig
@@ -4,7 +4,7 @@
 
 #include "Base.xcconfig"
 
-MOZ_BUNDLE_DISPLAY_NAME = Brave
+MOZ_BUNDLE_DISPLAY_NAME = iBrowe
 
 // Bundle Identifier
 MOZ_BUNDLE_ID = $(BASE_BUNDLE_ID).browser
