diff --git a/ios/brave-ios/Sources/BrowserMenu/BrowserMenuModel.swift b/ios/brave-ios/Sources/BrowserMenu/BrowserMenuModel.swift
index e07c3866c758ba77fdcea986a1c2729a1a1b5eb5..6180187fcef36f8c564322b66705643c78e95c36 100644
--- a/ios/brave-ios/Sources/BrowserMenu/BrowserMenuModel.swift
+++ b/ios/brave-ios/Sources/BrowserMenu/BrowserMenuModel.swift
@@ -145,21 +145,21 @@ extension BrowserMenuModel {
     let vpnStatusPublisher = CurrentValueSubject<VPNStatus, Never>(mockStatus)
     let model = BrowserMenuModel(
       actions: [
-        .init(
-          id: .vpn,
-          title: "VPN On",
-          traits: .init(badgeColor: UIColor(braveSystemName: .primary50)),
-          state: true
-        ) { action in
-          var actionCopy = action
-          actionCopy.state?.toggle()
-          actionCopy.title = "VPN \(actionCopy.state! ? "On" : "Off")"
-          vpnStatusPublisher.send(actionCopy.state! ? mockStatus : .disconnected)
-          return .updateAction(actionCopy)
-        },
+//        .init(
+//          id: .vpn,
+//          title: "VPN On",
+//          traits: .init(badgeColor: UIColor(braveSystemName: .primary50)),
+//          state: true
+//        ) { action in
+//          var actionCopy = action
+//          actionCopy.state?.toggle()
+//          actionCopy.title = "VPN \(actionCopy.state! ? "On" : "Off")"
+//          vpnStatusPublisher.send(actionCopy.state! ? mockStatus : .disconnected)
+//          return .updateAction(actionCopy)
+//        },
         .init(id: .addBookmark, attributes: .disabled),
         .init(id: .history),
-        .init(id: .braveLeo),
+//        .init(id: .braveLeo),
         .init(id: .playlist),
         .init(id: .addFavourites),
         .init(
@@ -176,7 +176,7 @@ extension BrowserMenuModel {
         .init(id: .createPDF),
         .init(id: .pageZoom),
       ],
-      vpnStatusPublisher: vpnStatusPublisher.eraseToAnyPublisher(),
+//      vpnStatusPublisher: vpnStatusPublisher.eraseToAnyPublisher(),
       actionVisibility: {
         let pref = Preferences.Option<[String: Bool]>(
           key: "browser-menu-mock-action-visibility",
