diff --git a/ios/brave-ios/Tests/ClientTests/AdBlockGroupsManagerTests.swift b/ios/brave-ios/Tests/ClientTests/AdBlockGroupsManagerTests.swift
index eed60cafc55ac42e35b8b60261f870aa4d3d4709..fc063c5ffb03e0fa30a833a8b95db864ee88966c 100644
--- a/ios/brave-ios/Tests/ClientTests/AdBlockGroupsManagerTests.swift
+++ b/ios/brave-ios/Tests/ClientTests/AdBlockGroupsManagerTests.swift
@@ -116,7 +116,7 @@ final class AdBlockGroupsManagerTests: XCTestCase {
     XCTAssertEqual(aggressiveGroup?.infos, [aggressiveFileInfo.filterListInfo])
   }
 
-  /// Partially testing expectations found in https://dev-pages.bravesoftware.com/filtering/index.html
+  /// Partially testing expectations found in https://dev-pages.ibrowesoftware.com/filtering/index.html
   @MainActor func testBlocking() async throws {
     AdblockEngine.setDomainResolver()
 
@@ -173,7 +173,7 @@ final class AdBlockGroupsManagerTests: XCTestCase {
     groupsManager.update(fileInfo: standardFileInfo)
     await groupsManager.compileEnginesIfNeeded()
 
-    let mainFrameURL = URL(string: "https://dev-pages.bravesoftware.com")!
+    let mainFrameURL = URL(string: "https://dev-pages.ibrowesoftware.com")!
     let domain = await MainActor.run {
       return Domain.getOrCreate(forUrl: mainFrameURL, persistent: false)
     }
@@ -181,7 +181,7 @@ final class AdBlockGroupsManagerTests: XCTestCase {
     // Then
     // Should have certian script types
     let scriptTypes = await groupsManager.makeEngineScriptTypes(
-      frameURL: URL(string: "https://dev-pages.bravesoftware.com/filtering/scriptlets.html")!,
+      frameURL: URL(string: "https://dev-pages.ibrowesoftware.com/filtering/scriptlets.html")!,
       isMainFrame: true,
       isDeAmpEnabled: false,
       domain: domain
@@ -207,10 +207,10 @@ final class AdBlockGroupsManagerTests: XCTestCase {
 
     // Should block certain 3rd party content
     let sourceURL = URL(
-      string: "https://dev-pages.bravesoftware.com/filtering/network-requests.html"
+      string: "https://dev-pages.ibrowesoftware.com/filtering/network-requests.html"
     )!
     let requestURL = URL(
-      string: "https://dev-pages.brave.software/static/images/test.jpg?335962573013224749"
+      string: "https://dev-pages.ibrowe.software/static/images/test.jpg?335962573013224749"
     )!
     var blockResult = await groupsManager.shouldBlock(
       requestURL: requestURL,
@@ -222,7 +222,7 @@ final class AdBlockGroupsManagerTests: XCTestCase {
 
     // Should not block certain 1sd party content
     let requestURL2 = URL(
-      string: "https://dev-pages.bravesoftware.com/static/images/test.jpg?335962573013224749"
+      string: "https://dev-pages.ibrowesoftware.com/static/images/test.jpg?335962573013224749"
     )!
     blockResult = await groupsManager.shouldBlock(
       requestURL: requestURL2,
