diff --git a/ios/brave-ios/Sources/BraveShared/BraveURLs.swift b/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
index 9fbe6f9432c7dcbeca84e5c3b1727b047b563541..f93183940136e7e7f1003ecbb5a2654c2b9f492f 100644
--- a/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
+++ b/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
@@ -35,7 +35,7 @@ extension URL {
       string: "https://account.brave.com/?intent=connect-receipt&product=vpn"
     )!
     public static let braveVPNLinkReceiptStaging = URL(
-      string: "https://account.bravesoftware.com/?intent=connect-receipt&product=vpn"
+      string: "https://account.ibrowesoftware.com/?intent=connect-receipt&product=vpn"
     )!
     public static let braveVPNLinkReceiptDev = URL(
       string: "https://account.brave.software/?intent=connect-receipt&product=vpn"
@@ -56,7 +56,7 @@ extension URL {
       string: "https://account.brave.com/plans"
     )!
     public static let braveLeoManageSubscriptionStaging = URL(
-      string: "https://account.bravesoftware.com/plans"
+      string: "https://account.ibrowesoftware.com/plans"
     )!
     public static let braveLeoManageSubscriptionDev = URL(
       string: "https://account.brave.software/plans"
@@ -68,7 +68,7 @@ extension URL {
       string: "https://account.brave.com/?intent=link-order&product=leo"
     )!
     public static let braveLeoLinkReceiptStaging = URL(
-      string: "https://account.bravesoftware.com/?intent=link-order&product=leo"
+      string: "https://account.ibrowesoftware.com/?intent=link-order&product=leo"
     )!
     public static let braveLeoLinkReceiptDev = URL(
       string: "https://account.brave.software/?intent=link-order&product=leo"
