diff --git a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
index 6f9c28123caeb76191e65efb11a13dc3e6c6acb6..3f894b34d3cb692be245f4d6a36360c0b77cbd14 100644
--- a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
+++ b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
@@ -794,8 +794,8 @@ struct AIChatView_Preview: PreviewProvider {
 
       AIChatPageContextView(
         isToggleOn: .constant(true),
-        url: URL(string: "https://brave.com"),
-        pageTitle: "Brave Private Browser"
+        url: URL(string: "https://ibrowe.com"),
+        pageTitle: "iBrowe Private Browser"
       )
       .padding()
 
