diff --git a/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements b/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements
index c102394cdfe2aa6fd389dd4c8aeb6e510b8b8769..cf1a85feb6e1565e66d0eedd8279002bcb95f28b 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements	
+++ b/ios/brave-ios/App/iOS/Entitlements/Release (AppStore).entitlements	
@@ -4,14 +4,10 @@
 <dict>
 	<key>aps-environment</key>
 	<string>production</string>
-	<key>com.apple.developer.associated-domains</key>
-	<array>
-		<string>applinks:vpn.brave.com</string>
-	</array>
 	<key>com.apple.developer.authentication-services.autofill-credential-provider</key>
 	<true/>
-	<key>com.apple.developer.browser.app-installation</key>
-	<true/>
+<!--	<key>com.apple.developer.browser.app-installation</key>-->
+<!--	<true/>-->
 	<key>com.apple.developer.carplay-audio</key>
 	<true/>
 	<key>com.apple.developer.networking.networkextension</key>
