diff --git a/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift b/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift
index 4abaa27f41f4887eac2f02d4786218f931cd77bd..edf61ee69a20520ea88524f65f1c68689cd3c11e 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift
@@ -203,7 +203,7 @@ class BraveTranslateScriptHandler: NSObject, TabContentScript {
 
   private func downloadElementScript() async throws -> String {
     var urlRequest = URLRequest(
-      url: URL(string: "https://translate.brave.com/static/v1/element.js")!
+      url: URL(string: "https://translate.ibrowe.com/static/v1/element.js")!
     )
     urlRequest.httpMethod = "GET"
 
