diff --git a/ios/brave-ios/Sources/BraveShared/Extensions/URLExtensions.swift b/ios/brave-ios/Sources/BraveShared/Extensions/URLExtensions.swift
index b637965d35b497b3b243573d0e72f34730ca7956..6267e578ee384ce0a7a89f98e090814cd05b5407 100644
--- a/ios/brave-ios/Sources/BraveShared/Extensions/URLExtensions.swift
+++ b/ios/brave-ios/Sources/BraveShared/Extensions/URLExtensions.swift
@@ -288,7 +288,7 @@ extension URL {
 
   public var isPlaylistBlockedSiteURL: Bool {
     let urlHost = self.host ?? self.hostSLD
-    return urlHost == "talk.brave.com"
+    return urlHost == "talk.ibrowe.com"
   }
 
   /// Returns true if we should show Shred option for the given URL.
