diff --git a/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift b/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
index f1ed5705f3e1e06a4d1ec6e2660c7116ed2e166d..1469a9416d33af5532952416f9f6a835e897cdff 100644
--- a/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
+++ b/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
@@ -25,7 +25,7 @@ import os.log
 extension Logger {
   fileprivate static var module: Logger {
     .init(
-      subsystem: "\(Bundle.main.bundleIdentifier ?? "com.brave.ios")",
+      subsystem: "\(Bundle.main.bundleIdentifier ?? "com.ibrowe.ios")",
       category: "SceneDelegate"
     )
   }
