diff --git a/ios/brave-ios/Sources/Brave/Frontend/Sync/SyncSettingsTableViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Sync/SyncSettingsTableViewController.swift
index dec178987e4e32f8305413d3840c6502bcf3997c..ccae6b328b307084cc3aa0f227e7d8e294ea780c 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Sync/SyncSettingsTableViewController.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Sync/SyncSettingsTableViewController.swift
@@ -360,7 +360,7 @@ class SyncSettingsTableViewController: SyncViewController, UITableViewDelegate,
 
       let syncInternalsController = ChromeWebViewController(privateBrowsing: false).then {
         $0.title = Strings.Sync.internalsTitle
-        $0.loadURL("brave://sync-internals")
+        $0.loadURL("ibrowe://sync-internals")
       }
 
       self.navigationController?.pushViewController(syncInternalsController, animated: true)
