diff --git a/ios/brave-ios/Sources/Shared/Extensions/BundleExtension.swift b/ios/brave-ios/Sources/Shared/Extensions/BundleExtension.swift
index f868fca12627858a52f8a178f1994f0b8ea508c2..54d1862568e0926c55bc11660a87e7eefbaf1b1c 100644
--- a/ios/brave-ios/Sources/Shared/Extensions/BundleExtension.swift
+++ b/ios/brave-ios/Sources/Shared/Extensions/BundleExtension.swift
@@ -16,7 +16,7 @@ extension Bundle {
     guard let value = object(forInfoDictionaryKey: key) as? String else {
       if AppConstants.isRunningTest {
         // Using dummy info dictionary while running tests, without host app set.
-        return "com.brave.tests"
+        return "com.ibrowe.tests"
       }
       fatalError("Could not find a string value in the Info dictionary with the key: \(key)")
     }
