diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift
index d0ccb2c60986284313bf18d90d3e15f5295b140b..c86b52c908b00dc89f3e869f3fcb1b7e8ac84b77 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift	
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift	
@@ -901,7 +901,7 @@ class NewTabPageViewController: UIViewController {
       Preferences.Review.newsCardShownDate.value = Date()
       guard
         let writeReviewURL = URL(
-          string: "https://itunes.apple.com/app/id1052879175?action=write-review"
+          string: "https://itunes.apple.com/app/id6741485553?action=write-review"
         )
       else {
         return
@@ -1125,8 +1125,8 @@ class NewTabPageViewController: UIViewController {
   }
 
   private func tappedQRCode(_ code: String) {
-    // Super referrer websites come in format https://brave.com/r/REF_CODE
-    let refUrl = URL(string: "https://brave.com/")?
+    // Super referrer websites come in format https://ibrowe.com/r/REF_CODE
+    let refUrl = URL(string: "https://ibrowe.com/")?
       .appendingPathComponent("r")
       .appendingPathComponent(code)
 
