diff --git a/ios/brave-ios/App/Configuration/Base.xcconfig b/ios/brave-ios/App/Configuration/Base.xcconfig
index dd5561b77e60ee4c60b77afccba221584429db8c..3656c78b44aca69fbcb7797fad4c46193bbbb7e3 100644
--- a/ios/brave-ios/App/Configuration/Base.xcconfig
+++ b/ios/brave-ios/App/Configuration/Base.xcconfig
@@ -12,9 +12,9 @@ brave_version_build = 0
 
 #include "../../../../../out/ios_current_link/args.xcconfig"
 
-FRAMEWORK_BASE_BUNDLE_ID = com.brave
+FRAMEWORK_BASE_BUNDLE_ID = com.ibrowe
 BASE_BUNDLE_ID = $(FRAMEWORK_BASE_BUNDLE_ID).ios
 
-DEVELOPMENT_TEAM = KL8N8XSYF4
+DEVELOPMENT_TEAM = VJ4PQMKSUA
 
-BRAVE_URL_SCHEME = brave
+BRAVE_URL_SCHEME = ibrowe
