diff --git a/ios/brave-ios/App/Configuration/Debug-AppStore.xcconfig b/ios/brave-ios/App/Configuration/Debug-AppStore.xcconfig
index aa18dc9da706bfc13df4491b85d43dcb01d8273b..753522aea968685190281c63d9336c8f0ab75d97 100644
--- a/ios/brave-ios/App/Configuration/Debug-AppStore.xcconfig
+++ b/ios/brave-ios/App/Configuration/Debug-AppStore.xcconfig
@@ -11,5 +11,5 @@ BRAVE_GROUP_ID = group.$(MOZ_BUNDLE_ID)
 SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG BRAVE_CHANNEL_RELEASE $(brave_swift_active_compliation_conditions)
 
 // Manual Code-Signing
-DEVELOPMENT_TEAM = KL8N8XSYF4
+DEVELOPMENT_TEAM = D8FP7GS222
 CODE_SIGN_STYLE = Manual
