diff --git a/ios/brave-ios/App/WebShell/WebShell.entitlements b/ios/brave-ios/App/WebShell/WebShell.entitlements
index ff9f66523a465756e66b9f3d306b72218ed8e8d9..7de2440d78b311fe55cfdf81888f64d6ca1df7f2 100644
--- a/ios/brave-ios/App/WebShell/WebShell.entitlements
+++ b/ios/brave-ios/App/WebShell/WebShell.entitlements
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.brave.ios.webshell</string>
+		<string>group.com.ibrowe.ios.browser</string>
 	</array>
 </dict>
 </plist>
