diff --git a/ios/brave-ios/Tests/BraveNewsTests/FeedFillStrategyTests.swift b/ios/brave-ios/Tests/BraveNewsTests/FeedFillStrategyTests.swift
index c9028731c84d0f313fa19b73c6b2eae9e3cefe4a..4a0af8ed37663fff3eb2cf950adf827b472c1f14 100644
--- a/ios/brave-ios/Tests/BraveNewsTests/FeedFillStrategyTests.swift
+++ b/ios/brave-ios/Tests/BraveNewsTests/FeedFillStrategyTests.swift
@@ -190,7 +190,7 @@ extension FeedItem.Content {
   fileprivate static var mockArticle: FeedItem.Content {
     .init(
       publishTime: Date(timeIntervalSinceNow: -60),
-      url: URL(string: "https://brave.com/"),
+      url: URL(string: "https://ibrowe.com/"),
       imageURL: nil,
       title: "Test Title",
       description: "Test Description",
