diff --git a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntents.entitlements b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntents.entitlements
index a9502b44bbf72448e27509740f1dfe5e73c75e9a..b0f5c77c0f86af9b4cfca217231fe19b473deda7 100644
--- a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntents.entitlements
+++ b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntents.entitlements
@@ -4,7 +4,7 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.com.brave.ios.BrowserBeta.unique</string>
+		<string>group.com.ibrowe.ios.BrowserBeta.unique</string>
 	</array>
 </dict>
 </plist>
