diff --git a/ios/brave-ios/App/iOS/Supporting Files/Info.plist b/ios/brave-ios/App/iOS/Supporting Files/Info.plist
index c7f5fa0db35554d8ddfbc58f04e760481a8565c2..aa9d4733a5724221c67e4dfe9ebad13a2b2f0d29 100644
--- a/ios/brave-ios/App/iOS/Supporting Files/Info.plist	
+++ b/ios/brave-ios/App/iOS/Supporting Files/Info.plist	
@@ -32,7 +32,7 @@
 			<key>CFBundleTypeRole</key>
 			<string>Editor</string>
 			<key>CFBundleURLName</key>
-			<string>com.brave.ios</string>
+			<string>com.ibrowe.ios</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>http</string>
@@ -86,8 +86,8 @@
 		<string>OpenWebsiteIntent</string>
 		<string>ShortcutsConfigurationIntent</string>
 		<string>StatsConfigurationIntent</string>
-		<string>com.brave.ios.browser-scene</string>
-		<string>com.brave.ios.browsing</string>
+		<string>com.ibrowe.ios.browser-scene</string>
+		<string>com.ibrowe.ios.browsing</string>
 	</array>
 	<key>UIApplicationSceneManifest</key>
 	<dict>
