diff --git a/ios/browser/api/brave_shields/adblock_engine.mm b/ios/browser/api/brave_shields/adblock_engine.mm
index 74ede413de7d197ecf202e1f51948bde522033af..81de5b9e338217e2a75a9ff5a940cb5d882ed1c4 100644
--- a/ios/browser/api/brave_shields/adblock_engine.mm
+++ b/ios/browser/api/brave_shields/adblock_engine.mm
@@ -121,7 +121,7 @@ class AdblockEngineBox final {
 + (NSError*)adblockErrorForKind:(adblock::ResultKind)kind
                         message:(rust::String)message {
   return [NSError
-      errorWithDomain:@"com.brave.adblock"
+      errorWithDomain:@"com.ibrowe.adblock"
                  code:static_cast<NSInteger>(kind)
              userInfo:@{
                NSLocalizedDescriptionKey :
