diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/Favorites/Data/FavoritesPreloadedData.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/Favorites/Data/FavoritesPreloadedData.swift
index 51514a183a9b5ec7bda745206372843255919384..d8e40b95783fd8bf5c6737403733a03a8561e2ec 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/Favorites/Data/FavoritesPreloadedData.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/Favorites/Data/FavoritesPreloadedData.swift
@@ -23,8 +23,8 @@ struct FavoritesPreloadedData {
     title: "Facebook"
   )
   static let brave = FavoriteSite(
-    url: URL(string: "https://brave.com/whats-new/")!,
-    title: "What's new in Brave"
+    url: URL(string: "https://ibrowe.com/whats-new/")!,
+    title: "What's new in iBrowe"
   )
   static let popularFavorites = [youtube, wikipedia, facebook, brave]
 
