diff --git a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
index c2ed215d9d3fb9f55842694530fbfd02034762a7..58f87bd7954e96c38d4a4f1ce9e36a473dc44999 100644
--- a/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
+++ b/ios/brave-ios/Sources/BrowserMenu/Identifiers.swift
@@ -77,13 +77,13 @@ extension Action.Identifier {
     defaultVisibility: .hidden
   )
 
-  public static let braveTalk: Self = .init(
-    id: "BraveTalk",
-    title: Strings.ActionTitles.braveTalk,
-    braveSystemImage: "leo.product.brave-talk",
-    defaultRank: 2400,
-    defaultVisibility: .hidden
-  )
+  // public static let braveTalk: Self = .init(
+  //   id: "BraveTalk",
+  //   title: Strings.ActionTitles.braveTalk,
+  //   braveSystemImage: "leo.product.brave-talk",
+  //   defaultRank: 2400,
+  //   defaultVisibility: .hidden
+  // )
 
   public static let braveWallet: Self = .init(
     id: "BraveWallet",
