diff --git a/ios/brave-ios/Sources/BraveTalk/BraveTalkConferencingOptions.swift b/ios/brave-ios/Sources/BraveTalk/BraveTalkConferencingOptions.swift
index e520431da3ede5c584f49b45c0601ef1fa4e8138..a1fb4705f30c03acd86666ee28f7a875b77048df 100644
--- a/ios/brave-ios/Sources/BraveTalk/BraveTalkConferencingOptions.swift
+++ b/ios/brave-ios/Sources/BraveTalk/BraveTalkConferencingOptions.swift
@@ -16,7 +16,7 @@ extension JitsiMeetConferenceOptions {
     Self.fromBuilder { builder in
       builder.room = room
       builder.token = token
-      if host == "talk.brave.software" {
+      if host == "talk.ibrowe.software" {
         builder.serverURL = URL(string: "https://stage.8x8.vc")
       } else {
         builder.serverURL = URL(string: "https://8x8.vc")
