diff --git a/ios/brave-ios/Sources/BraveShared/BraveURLs.swift b/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
index 319f2f4f8c38218d0203f9fa491f0830bf602601..a2517d1e3aa855e1b52e43297f8b1215d38764fc 100644
--- a/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
+++ b/ios/brave-ios/Sources/BraveShared/BraveURLs.swift
@@ -7,32 +7,32 @@ import UIKit
 
 extension URL {
   public enum Brave {
-    public static let community = URL(string: "https://community.brave.com/")!
-    public static let account = URL(string: "https://account.brave.com")!
-    public static let privacy = URL(string: "https://brave.com/privacy/")!
-    public static let braveNews = URL(string: "https://brave.com/brave-news/")!
-    public static let braveNewsPrivacy = URL(string: "https://brave.com/privacy/#brave-news")!
-    public static let braveOffers = URL(string: "https://offers.brave.com/")!
-    public static let playlist = URL(string: "https://brave.com/playlist")!
-    public static let rewardsOniOS = URL(string: "https://brave.com/rewards-ios/")!
+    public static let community = URL(string: "https://community.ibrowe.com/")!
+    public static let account = URL(string: "https://account.ibrowe.com")!
+    public static let privacy = URL(string: "https://ibrowe.com/privacy/")!
+    public static let braveNews = URL(string: "https://ibrowe.com/brave-news/")!
+    public static let braveNewsPrivacy = URL(string: "https://ibrowe.com/privacy/#brave-news")!
+    public static let braveOffers = URL(string: "https://offers.ibrowe.com/")!
+    public static let playlist = URL(string: "https://ibrowe.com/playlist")!
+    public static let rewardsOniOS = URL(string: "https://ibrowe.com/rewards-ios/")!
     public static let rewardsUnverifiedPublisherLearnMoreURL = URL(
-      string: "https://brave.com/faq-rewards/#unclaimed-funds"
+      string: "https://ibrowe.com/faq-rewards/#unclaimed-funds"
     )!
-    public static let termsOfUse = URL(string: "https://www.brave.com/terms_of_use")!
+    public static let termsOfUse = URL(string: "https://www.ibrowe.com/terms_of_use")!
     public static let batTermsOfUse = URL(
       string: "https://basicattentiontoken.org/user-terms-of-service/"
     )!
-    public static let ntpTutorialPage = URL(string: "https://brave.com/ja/ntp-tutorial")!
-    public static let privacyFeatures = URL(string: "https://brave.com/privacy-features/")!
-    public static let support = URL(string: "https://support.brave.com")!
+    public static let ntpTutorialPage = URL(string: "https://ibrowe.com/ja/ntp-tutorial")!
+    public static let privacyFeatures = URL(string: "https://ibrowe.com/privacy-features/")!
+    public static let support = URL(string: "https://support.ibrowe.com")!
     public static let p3aHelpArticle = URL(
-      string: "https://support.brave.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-"
+      string: "https://support.ibrowe.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-"
     )!
     public static let braveVPNFaq = URL(
-      string: "https://support.brave.com/hc/en-us/articles/360045045952"
+      string: "https://support.ibrowe.com/hc/en-us/articles/360045045952"
     )!
     public static let braveVPNLinkReceiptProd = URL(
-      string: "https://account.brave.com/?intent=connect-receipt&product=vpn"
+      string: "https://account.ibrowe.com/?intent=connect-receipt&product=vpn"
     )!
     public static let braveVPNLinkReceiptStaging = URL(
       string: "https://account.bravesoftware.com/?intent=connect-receipt&product=vpn"
@@ -41,16 +41,16 @@ extension URL {
       string: "https://account.brave.software/?intent=connect-receipt&product=vpn"
     )!
     public static let braveVPNRefreshCredentials = URL(
-      string: "https://account.brave.com/?intent=recover&product=vpn&ux=mobile"
+      string: "https://account.ibrowe.com/?intent=recover&product=vpn&ux=mobile"
     )!
     public static let safeBrowsingHelp = URL(
-      string: "https://support.brave.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave"
+      string: "https://support.ibrowe.com/hc/en-us/articles/15222663599629-Safe-Browsing-in-Brave"
     )!
     public static let screenTimeHelp = URL(
       string: "https://support.apple.com/guide/security/secd8831e732/web"
     )!
     public static let braveLeoManageSubscriptionProd = URL(
-      string: "https://account.brave.com/plans"
+      string: "https://account.ibrowe.com/plans"
     )!
     public static let braveLeoManageSubscriptionStaging = URL(
       string: "https://account.bravesoftware.com/plans"
@@ -59,7 +59,7 @@ extension URL {
       string: "https://account.brave.software/plans"
     )!
     public static let braveLeoLinkReceiptProd = URL(
-      string: "https://account.brave.com/?intent=link-order&product=leo"
+      string: "https://account.ibrowe.com/?intent=link-order&product=leo"
     )!
     public static let braveLeoLinkReceiptStaging = URL(
       string: "https://account.bravesoftware.com/?intent=link-order&product=leo"
@@ -68,19 +68,19 @@ extension URL {
       string: "https://account.brave.software/?intent=link-order&product=leo"
     )!
     public static let braveLeoRefreshCredentials = URL(
-      string: "https://account.brave.com/?intent=recover&product=leo&ux=mobile"
+      string: "https://account.ibrowe.com/?intent=recover&product=leo&ux=mobile"
     )!
     public static let braveLeoModelCategorySupport = URL(
       string:
-        "https://support.brave.com/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models"
+        "https://support.ibrowe.com/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models"
     )!
     public static let braveVPNSmartProxySupport = URL(
       string:
-        "https://support.brave.com/hc/en-us/articles/32105253704333-What-is-Smart-Proxy-Routing"
+        "https://support.ibrowe.com/hc/en-us/articles/32105253704333-What-is-Smart-Proxy-Routing"
     )!
     public static let braveVPNKillSwitchSupport = URL(
       string:
-        "https://support.brave.com/hc/en-us/articles/32389914657549-What-is-the-Brave-VPN-Kill-Switch"
+        "https://support.ibrowe.com/hc/en-us/articles/32389914657549-What-is-the-Brave-VPN-Kill-Switch"
     )!
   }
   public enum Apple {
