diff --git a/ios/brave-ios/Sources/Brave/Helpers/UserActivityTabHelper.swift b/ios/brave-ios/Sources/Brave/Helpers/UserActivityTabHelper.swift
index 29e4560dde819dfcf5ab68e56d8f81de1bb49c34..0ce2a11fa698a3c28613e9a14d91f71270fb708e 100644
--- a/ios/brave-ios/Sources/Brave/Helpers/UserActivityTabHelper.swift
+++ b/ios/brave-ios/Sources/Brave/Helpers/UserActivityTabHelper.swift
@@ -8,7 +8,7 @@ import MobileCoreServices
 import Shared
 import Web
 
-private let browsingActivityType: String = "com.brave.ios.browsing"
+private let browsingActivityType: String = "com.ibrowe.ios.browsing"
 
 private let searchableIndex = CSSearchableIndex(name: "firefox")
 
