diff --git a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
index 84cd7990c7fddfe827068aeb685f7e662f44a31b..54a4aed3879863246277c5cbc3548425fcabf2a7 100644
--- a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
+++ b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
@@ -916,8 +916,8 @@ struct AIChatView_Preview: PreviewProvider {
 
       AIChatPageContextView(
         isToggleOn: .constant(true),
-        url: URL(string: "https://brave.com"),
-        pageTitle: "Brave Private Browser"
+        url: URL(string: "https://ibrowe.com"),
+        pageTitle: "iBrowe Private Browser"
       )
       .padding()
 
