diff --git a/ios/browser/ui/webui/skus/skus_internals_ui.mm b/ios/browser/ui/webui/skus/skus_internals_ui.mm
index a8996e6dbf67a9092639599588381c5d20dae1c8..4eca8ad71ce27167b9e112c48d9edd3711f87e2d 100644
--- a/ios/browser/ui/webui/skus/skus_internals_ui.mm
+++ b/ios/browser/ui/webui/skus/skus_internals_ui.mm
@@ -78,7 +78,7 @@
 SkusInternalsUI::SkusInternalsUI(web::WebUIIOS* web_ui, const GURL& url)
     : web::WebUIIOSController(web_ui, url.host()),
       local_state_(GetApplicationContext()->GetLocalState()) {
-  // Set up the brave://skus-internals/ source.
+  // Set up the ibrowe://skus-internals/ source.
   CreateAndAddWebUIDataSource(web_ui, url.host(), kSkusInternalsGenerated,
                               IDR_SKUS_INTERNALS_HTML);
 
