diff --git a/ios/brave-ios/App/Configuration/Nightly.xcconfig b/ios/brave-ios/App/Configuration/Nightly.xcconfig
index 32d1dbf6d8e9c81fe181ca4ea3787a4afff30cb6..66be6ab5cf3894972fe3d9a4dd14e49b664dd50c 100644
--- a/ios/brave-ios/App/Configuration/Nightly.xcconfig
+++ b/ios/brave-ios/App/Configuration/Nightly.xcconfig
@@ -7,9 +7,9 @@
 
 #include "Base.xcconfig"
 
-MOZ_BUNDLE_DISPLAY_NAME = Brave Nightly
+MOZ_BUNDLE_DISPLAY_NAME = iBrowe Nightly
 
-BRAVE_URL_SCHEME = brave-nightly
+BRAVE_URL_SCHEME = ibrowe-nightly
 
 // Bundle Identifier
 MOZ_BUNDLE_ID = $(BASE_BUNDLE_ID).BrowserBeta
@@ -23,5 +23,5 @@ SWIFT_ACTIVE_COMPILATION_CONDITIONS = BRAVE_CHANNEL_NIGHTLY $(brave_swift_active
 GCC_PREPROCESSOR_DEFINITIONS= DEBUG=1
 
 // Manual Code-Signing
-CODE_SIGN_IDENTITY = Apple Distribution: Brave Software, Inc.
+CODE_SIGN_IDENTITY = Apple Distribution: Yongyut Jantabot (VJ4PQMKSUA)
 CODE_SIGN_STYLE = Manual
