diff --git a/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift b/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
index 783b98b122df025d8915793388fbd1765b486ef7..9fb70a62474246645e834e010b9c66f088f5587f 100644
--- a/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
+++ b/ios/brave-ios/Sources/Onboarding/OnboardingStrings.swift
@@ -11,7 +11,7 @@ extension Strings {
     public static let introducingBravePlaylist = NSLocalizedString(
       "introducingBravePlaylist",
       bundle: .module,
-      value: "Introducing Brave Playlist",
+      value: "Introducing iBrowe Playlist",
       comment:
         "A title shown on the first 2 steps of onboarding for Brave Playlist. Brave Playlist is a product name"
     )
@@ -50,7 +50,7 @@ extension Strings {
     public static let playlistInfoFeaturePointTitleOne = NSLocalizedString(
       "playlistInfoFeaturePointTitleOne",
       bundle: .module,
-      value: "Add videos to Brave Playlist. Playback anytime. Even offline",
+      value: "Add videos to iBrowe Playlist. Playback anytime. Even offline",
       comment: "A list of features in Playlist"
     )
     public static let playlistInfoFeaturePointSubtitleOne = NSLocalizedString(
@@ -146,7 +146,7 @@ extension Strings {
       "confirmationActionButtonTitle",
       bundle: .module,
       value: "Take me to my Playlist",
-      comment: "A button title that when tapped opens Brave Playlist"
+      comment: "A button title that when tapped opens iBrowe Playlist"
     )
     public static let dismissStep4OnboardingButtonTitle = NSLocalizedString(
       "dismissStep4OnboardingButtonTitle",
