diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
index 4411f8e1567a4da77da41da9def30243ba7b0c5b..d040bf0237fc451601922a4829d6ba04c9b2ea58 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/DomainUserScript.swift
@@ -50,18 +50,18 @@ enum DomainUserScript: CaseIterable {
     #if canImport(BraveTalk)
     case .braveTalkHelper:
       return Set([
-        "talk.brave.com",
-        "talk.bravesoftware.com",
-        "talk.brave.software",
+        "talk.ibrowe.com",
+        "talk.ibrowesoftware.com",
+        "talk.ibrowe.software",
       ])
     #endif
     case .bravePlaylistFolderSharingHelper:
       return Set(["playlist.bravesoftware.com", "playlist.brave.com"])
     case .braveSkus:
       return Set([
-        "account.brave.com",
-        "account.bravesoftware.com",
-        "account.brave.software",
+        "account.ibrowe.com",
+        "account.ibrowesoftware.com",
+        "account.ibrowe.software",
       ])
     }
   }
