diff --git a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
index 48530efe85ca328eefdce646a8c3659527bf8274..03ce4b161cb92d2d0a5ca7b15ba036a48f4a5168 100644
--- a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
+++ b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
@@ -914,8 +914,8 @@ struct AIChatView_Preview: PreviewProvider {
 
       AIChatPageContextView(
         isToggleOn: .constant(true),
-        url: URL(string: "https://brave.com"),
-        pageTitle: "Brave Private Browser"
+        url: URL(string: "https://ibrowe.com"),
+        pageTitle: "iBrowe Private Browser"
       )
       .padding()
 
