diff --git a/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift b/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
index 616af16b1a0b30fba3c0b0b3371a4ec8f0db36a3..37d30db4345ada715737332ad1a37419bdab3e89 100644
--- a/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
+++ b/ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift
@@ -25,7 +25,7 @@ import os.log
 extension Logger {
   fileprivate static var module: Logger {
     .init(
-      subsystem: "\(Bundle.main.bundleIdentifier ?? "com.brave.ios")",
+      subsystem: "\(Bundle.main.bundleIdentifier ?? "com.ibrowe.ios")",
       category: "SceneDelegate"
     )
   }
