diff --git a/ios/brave-ios/Sources/BrowserMenu/BrowserMenuStrings.swift b/ios/brave-ios/Sources/BrowserMenu/BrowserMenuStrings.swift
index 52525b40c6b57fe3e72816d5a646410985bc2d25..c623e04d0cae9ba889a823394ad0e57dc455d240 100644
--- a/ios/brave-ios/Sources/BrowserMenu/BrowserMenuStrings.swift
+++ b/ios/brave-ios/Sources/BrowserMenu/BrowserMenuStrings.swift
@@ -73,31 +73,31 @@ extension Strings {
       "ActionTitles.playlist",
       bundle: .module,
       value: "Playlist",
-      comment: "A button title shown on the menu that presents the Brave Playlist feature"
+      comment: "A button title shown on the menu that presents the iBrowe Playlist feature"
     )
     public static let rewards = NSLocalizedString(
       "ActionTitles.rewards",
       bundle: .module,
       value: "Rewards",
-      comment: "A button title shown on the menu that presents the Brave Rewards feature."
+      comment: "A button title shown on the menu that presents the iBrowe Rewards feature."
     )
     public static let leoAIChat = NSLocalizedString(
       "ActionTitles.leoAIChat",
       bundle: .module,
       value: "AI Chat",
-      comment: "A button title shown on the menu that presents the Brave Leo AI feature."
+      comment: "A button title shown on the menu that presents the iBrowe Leo AI feature."
     )
     public static let braveTalk = NSLocalizedString(
       "ActionTitles.braveTalk",
       bundle: .module,
-      value: "Brave Talk",
-      comment: "A button title shown on the menu that opens a link to the Brave Talk feature."
+      value: "iBrowe Talk",
+      comment: "A button title shown on the menu that opens a link to the iBrowe Talk feature."
     )
     public static let braveNews = NSLocalizedString(
       "ActionTitles.braveNews",
       bundle: .module,
-      value: "Brave News",
-      comment: "A button title shown on the menu that presents the Brave News feature."
+      value: "iBrowe News",
+      comment: "A button title shown on the menu that presents the iBrowe News feature."
     )
     public static let share = NSLocalizedString(
       "ActionTitles.share",
@@ -118,7 +118,7 @@ extension Strings {
       bundle: .module,
       value: "VPN",
       comment:
-        "A button title shown on the menu that when tapped presents a Brave VPN paywall"
+        "A button title shown on the menu that when tapped presents a iBrowe VPN paywall"
     )
   }
 }
