diff --git a/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements b/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
index 6fd26ea80ec06ea489a1706bd0000256beacb42a..b1b5630eba26412453604739081462e69e04022b 100644
--- a/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
+++ b/ios/brave-ios/App/iOS/Entitlements/Debug.entitlements
@@ -28,7 +28,5 @@
 	<array>
 		<string>group.$(MOZ_BUNDLE_ID).unique</string>
 	</array>
-	<key>com.apple.developer.carplay-audio</key>
-	<true/>
 </dict>
 </plist>
