diff --git a/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/CustomFilterListView.swift b/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/CustomFilterListView.swift
index 70d4b012f739dadd91294c3c8ee141518bc23d3a..61268ec7481cc7f9f0c3aa1bd1c0fc5874805e4f 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/CustomFilterListView.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/CustomFilterListView.swift
@@ -179,8 +179,8 @@ struct CustomFilterListView: View {
       ! Hide the header on example.com (CF Test)
       example.com,example.net##h1
 
-      ! Hide the brave logo on brave.com (Network test)
-      ||brave.com/static-assets/images/brave-logo-sans-text.svg
+      ! Hide the brave logo on ibrowe.com (Network test)
+      ||ibrowe.com/static-assets/images/brave-logo-sans-text.svg
       """
     )
   )
