diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift
index e80ec2cf01435de73a4fa718b241172c66517427..dc05d5c633bd9abcdb4378c883a2f3035451b972 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift	
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift	
@@ -897,7 +897,7 @@ class NewTabPageViewController: UIViewController {
       Preferences.Review.newsCardShownDate.value = Date()
       guard
         let writeReviewURL = URL(
-          string: "https://itunes.apple.com/app/id1052879175?action=write-review"
+          string: "https://itunes.apple.com/app/id6741485553?action=write-review"
         )
       else {
         return
