diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/Playlist/Utilities/PlaylistThumbnailUtility.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/Playlist/Utilities/PlaylistThumbnailUtility.swift
index 7439cf5e7af3e86435535ae17942e02d49522db8..18338edf06247431a0c756f7ef78bafd1e2161e2 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/Playlist/Utilities/PlaylistThumbnailUtility.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/Playlist/Utilities/PlaylistThumbnailUtility.swift
@@ -194,7 +194,7 @@ private class HLSThumbnailGenerator {
   private var observer: NSKeyValueObservation?
   private var state: State = .loading
   private let completion: (UIImage?, HLSThumbnailGeneratorError?) -> Void
-  private let queue = DispatchQueue(label: "com.brave.hls-thumbnail-generator")
+  private let queue = DispatchQueue(label: "com.ibrowe.hls-thumbnail-generator")
 
   init(
     url: URL,
