diff --git a/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift b/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
index 6f8e5eee3f831399087808aa564d009d5fd5ebe2..d08e2856381cdc397ccd63a40703e7e20d2a3423 100644
--- a/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
+++ b/ios/brave-ios/Sources/BraveWallet/WalletConstants.swift
@@ -28,13 +28,13 @@ public struct WalletConstants {
 
   /// The url to Brave Help Center for Wallet.
   static let braveWalletSupportURL = URL(
-    string: "https://support.brave.com/hc/categories/360001062531-Wallet"
+    string: "https://support.ibrowe.com/hc/categories/360001062531-Wallet"
   )!
 
   // TODO: update wiki link
   /// Brave Wiki page for Solana Name Service (SNS)
   public static let snsBraveWikiURL: URL = URL(
-    string: "https://github.com/brave/brave-browser/wiki/Resolve-Methods-for-Solana-Name-Service"
+    string: "https://github.com/yuthstyle88/ibrowe-browser/wiki/Resolve-Methods-for-Solana-Name-Service"
   )!
 
   /// Terms of Use for Ethereum Name Service (ENS)
@@ -50,7 +50,7 @@ public struct WalletConstants {
 
   /// The url to learn more about Unstoppable Domains resolve methods.
   public static let braveWalletUnstoppableDomainsURL = URL(
-    string: "https://github.com/brave/brave-browser/wiki/Resolve-Methods-for-Unstoppable-Domains"
+    string: "https://github.com/yuthstyle88/ibrowe-browser/wiki/Resolve-Methods-for-Unstoppable-Domains"
   )!
 
   /// The url to the privacy policy for 0x swaps
@@ -64,7 +64,7 @@ public struct WalletConstants {
     string: "https://github.com/brave/brave-browser/wiki/NFT-Discovery"
   )!
 
-  public static let braveWalletTermsOfUse = URL(string: "https://brave.com/terms-of-use/")!
+  public static let braveWalletTermsOfUse = URL(string: "https://ibrowe.com/terms-of-use/")!
 
   /// The currently supported test networks.
   static let supportedTestNetworkChainIds = [
