diff --git a/ios/brave-ios/Sources/CertificateUtilities/BraveCertificateUtils.swift b/ios/brave-ios/Sources/CertificateUtilities/BraveCertificateUtils.swift
index f4367760329530d8c6fd7f1d3c5abeb197c5a244..a80f21cc6523c4d27dd0c044dd64bbb944209dde 100644
--- a/ios/brave-ios/Sources/CertificateUtilities/BraveCertificateUtils.swift
+++ b/ios/brave-ios/Sources/CertificateUtilities/BraveCertificateUtils.swift
@@ -191,7 +191,7 @@ public enum BraveCertificateUtilError: LocalizedError {
 
 extension BraveCertificateUtils {
   private static let evaluationQueue = DispatchQueue(
-    label: "com.brave.cert-utils-evaluation-queue",
+    label: "com.ibrowe.cert-utils-evaluation-queue",
     qos: .userInitiated
   )
 
