diff --git a/ios/brave-ios/Sources/Brave/Frontend/Brave Notifications/Brave Rewards/RewardsNotification.swift b/ios/brave-ios/Sources/Brave/Frontend/Brave Notifications/Brave Rewards/RewardsNotification.swift
index 2e624a8a22581eaadbbe4016af7f43a8f448f181..c0b5b664309b299ed538b3be604ae4ce16fd20ec 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Brave Notifications/Brave Rewards/RewardsNotification.swift	
+++ b/ios/brave-ios/Sources/Brave/Frontend/Brave Notifications/Brave Rewards/RewardsNotification.swift	
@@ -78,7 +78,7 @@ extension RewardsNotification {
     let notification = NotificationAd.customAd(
       title: Strings.Ads.myFirstAdTitle,
       body: Strings.Ads.myFirstAdBody,
-      url: "https://brave.com/my-first-ad"
+      url: "https://ibrowe.com/my-first-ad"
     )
 
     guard let targetURL = URL(string: notification.targetURL) else {
