diff --git a/ios/brave-ios/Sources/Shared/SharedStrings.swift b/ios/brave-ios/Sources/Shared/SharedStrings.swift
index 796a92becaf467e9d27a6dc439d150be14861ef5..af08573892444826318bc00160ac46f9dd920ff5 100644
--- a/ios/brave-ios/Sources/Shared/SharedStrings.swift
+++ b/ios/brave-ios/Sources/Shared/SharedStrings.swift
@@ -156,7 +156,7 @@ extension Strings {
   public static let unableToOpenURLError = NSLocalizedString(
     "UnableToOpenURLError",
     bundle: .module,
-    value: "Brave cannot open the page because it has an invalid address.",
+    value: "iBrowe cannot open the page because it has an invalid address.",
     comment:
       "The message displayed to a user when they try to open a URL that cannot be handled by Brave, or any external app."
   )
@@ -343,7 +343,7 @@ extension Strings {
   public static let searchResultAdClickedInfoBarTitle = NSLocalizedString(
     "SearchResultAdClickedInfoBarTitle",
     bundle: .module,
-    value: "Thanks for supporting Brave Search by clicking a private ad. Unlike Big Tech, we measure ad performance anonymously to preserve your privacy.",
+    value: "Thanks for supporting iBrowe Search by clicking a private ad. Unlike Big Tech, we measure ad performance anonymously to preserve your privacy.",
     comment: "The text label of creative search result ad infobar message."
   )
 
