diff --git a/ios/brave-ios/Sources/BraveWallet/WalletStrings.swift b/ios/brave-ios/Sources/BraveWallet/WalletStrings.swift
index 759f0474c43a83ef3cc90361ab63b5c0cc3863d2..afe6b49ddfa0c23e157871675cb569499988d0c3 100644
--- a/ios/brave-ios/Sources/BraveWallet/WalletStrings.swift
+++ b/ios/brave-ios/Sources/BraveWallet/WalletStrings.swift
@@ -14,7 +14,7 @@ extension Strings {
       "wallet.module",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave Wallet",
+      value: "iBrowe Wallet",
       comment:
         "The title shown on the wallet settings page, and the value shown when selecting the default wallet as Brave Wallet in wallet settings."
     )
@@ -23,7 +23,7 @@ extension Strings {
       tableName: "BraveWallet",
       bundle: .module,
       value: "Wallet",
-      comment: "The title shown on the menu to access Brave Wallet"
+      comment: "The title shown on the menu to access iBrowe Wallet"
     )
     public static let web3 = NSLocalizedString(
       "wallet.web3",
@@ -488,7 +488,7 @@ extension Strings {
       "wallet.cryptoTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave Wallet",
+      value: "iBrowe Wallet",
       comment: "The title of the crypto tab"
     )
     public static let backupWalletWarningMessage = NSLocalizedString(
@@ -664,7 +664,7 @@ extension Strings {
       "wallet.setupCryptoCreateNewSubTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Get started with Brave Wallet within minutes.",
+      value: "Get started with iBrowe Wallet within minutes.",
       comment:
         "The subtitle displayed in the section for user that need to create a brand new wallet"
     )
@@ -819,7 +819,7 @@ extension Strings {
       "wallet.onbordingCompletedSubTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Your Brave Wallet is ready to use.",
+      value: "Your iBrowe Wallet is ready to use.",
       comment: "The subtitle of the last step of creating a new wallet."
     )
     public static let onboardingCompletedButtonTitle = NSLocalizedString(
@@ -918,8 +918,8 @@ extension Strings {
       "wallet.restoreLegacyBraveWalletToggleTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Import from legacy Brave crypto wallets?",
-      comment: "A label for toggle for user to use to indicate importing a legacy brave wallet or not"
+      value: "Import from legacy iBrowe crypto wallets?",
+      comment: "A label for toggle for user to use to indicate importing a legacy iBrowe wallet or not"
     )
     public static let createWalletBackButtonTitle = NSLocalizedString(
       "wallet.createWalletBackButtonTitle",
@@ -1002,7 +1002,7 @@ extension Strings {
       "wallet.settingsResetWalletAlertTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Are you sure you want to reset Brave Wallet?",
+      value: "Are you sure you want to reset iBrowe Wallet?",
       comment:
         "The title the confirmation dialog when resetting the wallet. As in to erase the users wallet from the device"
     )
@@ -1201,7 +1201,7 @@ extension Strings {
       "wallet.biometricsSetupTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Unlock Brave Wallet with %@",
+      value: "Unlock iBrowe Wallet with %@",
       comment:
         "The title shown when a user is asked if they would like to setup biometric unlock. `%@` will be replaced with the biometric type name of the current device."
     )
@@ -1209,7 +1209,7 @@ extension Strings {
       "wallet.biometricsSetupSubTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Would you like to use %@ to unlock Brave Wallet?",
+      value: "Would you like to use %@ to unlock iBrowe Wallet?",
       comment:
         "The sub-title shown when a user is asked if they would like to setup biometric unlock. `%@` will be replaced with the biometric type name of the current device."
     )
@@ -1255,7 +1255,7 @@ extension Strings {
       "wallet.autoLockFooter",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "The number of minutes to wait until the Brave Wallet is automatically locked",
+      value: "The number of minutes to wait until the iBrowe Wallet is automatically locked",
       comment: "The footer beneath the auto-lock title and interval duration"
     )
     public static let enterAmount = NSLocalizedString(
@@ -1348,7 +1348,7 @@ extension Strings {
       "wallet.swapCryptoUnsupportNetworkBody",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave doesn't currently support swap on the %@. Please switch to a supported chain.",
+      value: "iBrowe doesn't currently support swap on the %@. Please switch to a supported chain.",
       comment:
         "The description of where user will see once a test network has been picked in swap screen. '%@' will be replaced with a network such as 'Rinkeby Test Network' or 'Ropsten Test Network'"
     )
@@ -1396,7 +1396,7 @@ extension Strings {
       "wallet.braveFeeLabel",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave Fee: %@",
+      value: "iBrowe Fee: %@",
       comment:
         "The title for Brave Fee label in Swap. The fee percentage is displayed beside the label."
     )
@@ -2185,7 +2185,7 @@ extension Strings {
       "wallet.swapDexAggrigatorNote",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave uses %@ as a DEX aggregator.",
+      value: "iBrowe uses %@ as a DEX aggregator.",
       comment:
         "A disclaimer note shown on the Swap screen. '%@' will be replaced by a company name, ex. '0x' / 'Jupiter'. 'DEX aggregator' is a type of blockchain-based service (decentralized exchange)"
     )
@@ -2226,7 +2226,7 @@ extension Strings {
       "wallet.braveSwapFeeDisclaimer",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Quote includes a %@ Brave fee.",
+      value: "Quote includes a %@ iBrowe fee.",
       comment:
         "A disclaimer that appears at the bottom of an swap screen which discloses the fixed Brave fee included in the swap quotes. '%@' will be replaced by a percentage. For example: 'Quote includes a 0.875% Brave fee'"
     )
@@ -2808,7 +2808,7 @@ extension Strings {
       "wallet.dappsConnectionNotificationTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "This page wants to interact with Brave Wallet",
+      value: "This page wants to interact with iBrowe Wallet",
       comment:
         "The title of the notification which will prompt at the top of the browser when users are visiting web3 site that is not yet connected with Brave Wallet."
     )
@@ -2816,7 +2816,7 @@ extension Strings {
       "wallet.dappsConnectionNotificationOriginTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "%@ wants to interact with Brave Wallet",
+      value: "%@ wants to interact with iBrowe Wallet",
       comment:
         "The title of the notification which will prompt at the top of the browser when users are visiting web3 site that is not yet connected with Brave Wallet. The '%@' will be the site attempting to connect. For example: \"app.uniswap.org wants to interact with Brave Wallet\""
     )
@@ -2877,7 +2877,7 @@ extension Strings {
       "wallet.newSiteConnectMessage",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Connect to Brave Wallet",
+      value: "Connect to iBrowe Wallet",
       comment: "The message displayed below the dapp's origin url in new site connection screen."
     )
     public static let newSiteConnectFooter = NSLocalizedString(
@@ -2978,7 +2978,7 @@ extension Strings {
       "wallet.addNetworkDescription",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "This will allow this network to be used within Brave Wallet.",
+      value: "This will allow this network to be used within iBrowe Wallet.",
       comment:
         "The description of the view shown over a dapps website that describes what adding a new network will do."
     )
@@ -3621,7 +3621,7 @@ extension Strings {
       "wallet.dappsSettingsConnectedSitesSectionEmpty",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "DApps you connect to Brave Wallet will appear here",
+      value: "DApps you connect to iBrowe Wallet will appear here",
       comment:
         "A message that will be displayed under the section header when there is no dapps have been granted wallet connection."
     )
@@ -4600,7 +4600,7 @@ extension Strings {
       "wallet.ensOffchainGatewayTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave supports using offchain gateways to resolve .eth domains.",
+      value: "iBrowe supports using offchain gateways to resolve .eth domains.",
       comment:
         "Title shown send address / ENS domain when requesting to do an ENS off chain lookup."
     )
@@ -4649,7 +4649,7 @@ extension Strings {
       "wallet.snsDomainInterstitialPageTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Enable Support of Solana Name Service (SNS) in Brave?",
+      value: "Enable Support of Solana Name Service (SNS) in iBrowe?",
       comment:
         "Title displayed when users chose Brave to ask them if they want the SNS to be resolved every time they enter one."
     )
@@ -4675,7 +4675,7 @@ extension Strings {
       "wallet.ensDomainInterstitialPageTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Enable Support of Ethereum Name Service (ENS) in Brave?",
+      value: "Enable Support of Ethereum Name Service (ENS) in iBrowe?",
       comment:
         "Title displayed when users chose Brave to ask them if they want the ENS domain to be resolved every time they enter one."
     )
@@ -4701,7 +4701,7 @@ extension Strings {
       "wallet.ensOffchainDomainInterstitialPageTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Enable support of Ethereum Name Service (ENS) offchain lookup in Brave?",
+      value: "Enable support of Ethereum Name Service (ENS) offchain lookup in iBrowe?",
       comment:
         "Title displayed when users chose Brave to ask them if they want the ENS Offchain domain to be resolved every time they enter one."
     )
@@ -4727,7 +4727,7 @@ extension Strings {
       "wallet.udDomainInterstitialPageTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Enable support of Unstoppable Domains in Brave?",
+      value: "Enable support of Unstoppable Domains in iBrowe?",
       comment:
         "Title displayed when users chose Brave to ask them if they want the Unstoppable Domains domain to be resolved every time they enter one."
     )
@@ -4944,7 +4944,7 @@ extension Strings {
       "wallet.nftRemoveFromWalletAlertTitle",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Remove from Brave Wallet?",
+      value: "Remove from iBrowe Wallet?",
       comment: "The title of the alert when user attempts to remove an NFT from wallet."
     )
     public static let nftRemoveFromWalletAlertDescription = NSLocalizedString(
@@ -5224,7 +5224,7 @@ extension Strings {
       "wallet.signInWithBraveWallet",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Sign in with Brave Wallet",
+      value: "Sign in with iBrowe Wallet",
       comment: "The title of the view shown above a Sign In With Ethereum request."
     )
     public static let securityRiskDetectedTitle = NSLocalizedString(
@@ -5239,7 +5239,7 @@ extension Strings {
       "wallet.signInWithBraveWalletMessage",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "You are signing into %@. Brave Wallet will share your wallet address with %@.",
+      value: "You are signing into %@. iBrowe Wallet will share your wallet address with %@.",
       comment:
         "The title of the view shown when a security issue is detected with a Sign In With Ethereum request."
     )
@@ -5349,7 +5349,7 @@ extension Strings {
       tableName: "BraveWallet",
       bundle: .module,
       value: "Details",
-      comment: "The title of the details view for Sign In With Ethereum/Brave Wallet requests."
+      comment: "The title of the details view for Sign In With Ethereum/iBrowe Wallet requests."
     )
     public static let transactionSummaryIntentLabel = NSLocalizedString(
       "wallet.transactionSummaryIntentLabel",
@@ -5518,7 +5518,7 @@ extension Strings {
       "wallet.btcOrdinalsUnsupportedWarning",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Brave Wallet does not currently support Bitcoin NFTs (ordinals). Sending BTC from an address that has ordinals may result in its ordinals being transferred inadvertently.",
+      value: "iBrowe Wallet does not currently support Bitcoin NFTs (ordinals). Sending BTC from an address that has ordinals may result in its ordinals being transferred inadvertently.",
       comment: "A warning displayed in row before the Send button in Send Crypto view when a Bitcoin asset is selected."
     )
     public static let inputLabel = NSLocalizedString(
