diff --git a/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift b/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
index dd36662e131902517560554e7512d58c6e4f1972..5b94af51d16e52ee1a69bb0ff902ae71784c50e3 100644
--- a/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
+++ b/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
@@ -13,12 +13,12 @@ public class BasicAuthCredentialsManager: NSObject, URLSessionDataDelegate {
 
   public static let validDomains: Set<String> = [
     // Brave Search
-    "search.brave.software", "search.bravesoftware.com",
-    "safesearch.brave.software", "safesearch.bravesoftware.com",
+    "search.ibrowe.software", "search.ibrowesoftware.com",
+    "safesearch.ibrowe.software", "safesearch.ibrowesoftware.com",
     "search-dev-local.brave.com",
 
     // Playlist
-    "playlist.bravesoftware.com",
+    "playlist.ibrowesoftware.com",
   ]
 
   public static func setCredential(origin: String, credential: URLCredential?) {
