diff --git a/ios/brave-ios/App/Configuration/Base.xcconfig b/ios/brave-ios/App/Configuration/Base.xcconfig
index dd5561b77e60ee4c60b77afccba221584429db8c..97caf1f653b6369bd76e898899cfe439f1f0a749 100644
--- a/ios/brave-ios/App/Configuration/Base.xcconfig
+++ b/ios/brave-ios/App/Configuration/Base.xcconfig
@@ -11,10 +11,10 @@ brave_ios_marketing_version = 0.0
 brave_version_build = 0
 
 #include "../../../../../out/ios_current_link/args.xcconfig"
-
-FRAMEWORK_BASE_BUNDLE_ID = com.brave
+brave_version_build = 3
+FRAMEWORK_BASE_BUNDLE_ID = com.ibrowe
 BASE_BUNDLE_ID = $(FRAMEWORK_BASE_BUNDLE_ID).ios
 
-DEVELOPMENT_TEAM = KL8N8XSYF4
+DEVELOPMENT_TEAM = D8FP7GS222
 
-BRAVE_URL_SCHEME = brave
+BRAVE_URL_SCHEME = ibrowe
