diff --git a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
index a5cb7c988957a82fa338cbe9d72925de7e0c823d..ba1d64d828f453eda743d81cf32f67333f88c999 100644
--- a/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
+++ b/ios/brave-ios/Sources/AIChat/Components/AIChatView.swift
@@ -904,8 +904,8 @@ struct AIChatView_Preview: PreviewProvider {
 
       AIChatPageContextView(
         isToggleOn: .constant(true),
-        url: URL(string: "https://brave.com"),
-        pageTitle: "Brave Private Browser"
+        url: URL(string: "https://ibrowe.com"),
+        pageTitle: "iBrowe Private Browser"
       )
       .padding()
 
