diff --git a/ios/brave-ios/Sources/Brave/Networking/NetworkMonitor.swift b/ios/brave-ios/Sources/Brave/Networking/NetworkMonitor.swift
index 69228d1f299a2e33b33efc63aa5f05f0f7727e1c..9bf0a91365110879adf488ec66c03c71a63f4329 100644
--- a/ios/brave-ios/Sources/Brave/Networking/NetworkMonitor.swift
+++ b/ios/brave-ios/Sources/Brave/Networking/NetworkMonitor.swift
@@ -9,7 +9,7 @@ import Network
 
 public class NetworkMonitor: NSObject {
   private let monitor = NWPathMonitor()
-  private let queue = DispatchQueue(label: "com.brave.network-monitor")
+  private let queue = DispatchQueue(label: "com.ibrowe.network-monitor")
 
   public static let shared = NetworkMonitor()
 
