diff --git a/ios/brave-ios/Sources/Onboarding/WelcomeFocus/Resources/FocusOnbordingConstants.swift b/ios/brave-ios/Sources/Onboarding/WelcomeFocus/Resources/FocusOnbordingConstants.swift
index 2497d59e5042dc00d8b3443a20b2073adf4c8208..62e5c2ebe206be4316e51bfbf39e176942fa1fd9 100644
--- a/ios/brave-ios/Sources/Onboarding/WelcomeFocus/Resources/FocusOnbordingConstants.swift
+++ b/ios/brave-ios/Sources/Onboarding/WelcomeFocus/Resources/FocusOnbordingConstants.swift
@@ -8,6 +8,6 @@ import UIKit
 
 class FocusOnboardingConstants {
   static let p3aHelpArticle = URL(
-    string: "https://support.brave.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-"
+    string: "https://support.ibrowe.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-"
   )!
 }
