diff --git a/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift b/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
index dd36662e131902517560554e7512d58c6e4f1972..42cb51672140f1d29b84f14aefc31f17cbb6ef51 100644
--- a/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
+++ b/ios/brave-ios/Sources/BraveShared/BasicAuthCredentials.swift
@@ -18,7 +18,7 @@ public class BasicAuthCredentialsManager: NSObject, URLSessionDataDelegate {
     "search-dev-local.brave.com",
 
     // Playlist
-    "playlist.bravesoftware.com",
+    "playlist.ibrowesoftware.com",
   ]
 
   public static func setCredential(origin: String, credential: URLCredential?) {
