diff --git a/ios/brave-ios/Sources/Brave/Frontend/Browser/Tabs/Synced/SyncedTabsView.swift b/ios/brave-ios/Sources/Brave/Frontend/Browser/Tabs/Synced/SyncedTabsView.swift
index 37512463765a3782f089837360fc630ff6552483..d48c19470c42a6128c2cb98f8619f85e9ee00e13 100644
--- a/ios/brave-ios/Sources/Brave/Frontend/Browser/Tabs/Synced/SyncedTabsView.swift
+++ b/ios/brave-ios/Sources/Brave/Frontend/Browser/Tabs/Synced/SyncedTabsView.swift
@@ -439,7 +439,7 @@ extension OpenDistantSession {
           deviceFormFactor: .phone,
           tabs: [
             .init(
-              url: URL(string: "https://brave.com")!,
+              url: URL(string: "https://ibrowe.com")!,
               title: "Brave",
               tabId: 1,
               sessionTag: "brave"
@@ -451,7 +451,7 @@ extension OpenDistantSession {
           deviceFormFactor: .desktop,
           tabs: [
             .init(
-              url: URL(string: "https://brave.com")!,
+              url: URL(string: "https://ibrowe.com")!,
               title: "Brave",
               tabId: 1,
               sessionTag: "brave2"
