diff --git a/ios/brave-ios/Sources/Brave/DeviceCheck/DeviceCheck.swift b/ios/brave-ios/Sources/Brave/DeviceCheck/DeviceCheck.swift
index d597584841fc1da62002cdcf07d8b32acb14265c..c3ee5235f258166bc42f9bde93e7b7c602a16cc1 100644
--- a/ios/brave-ios/Sources/Brave/DeviceCheck/DeviceCheck.swift
+++ b/ios/brave-ios/Sources/Brave/DeviceCheck/DeviceCheck.swift
@@ -28,7 +28,7 @@ public struct DeviceCheckRegistration: Codable {
     guard let data = Data(base64Encoded: try container.decode(String.self, forKey: .enrollmentBlob))
     else {
       throw NSError(
-        domain: "com.brave.device.check.enrollment",
+        domain: "com.ibrowe.device.check.enrollment",
         code: -1,
         userInfo: [
           NSLocalizedDescriptionKey: "Cannot decode enrollmentBlob"
@@ -91,7 +91,7 @@ public struct AttestationVerifcation: Codable {
       let data = Data(base64Encoded: try container.decode(String.self, forKey: .attestationBlob))
     else {
       throw NSError(
-        domain: "com.brave.device.check.enrollment",
+        domain: "com.ibrowe.device.check.enrollment",
         code: -1,
         userInfo: [
           NSLocalizedDescriptionKey: "Cannot decode attestationBlob"
@@ -139,7 +139,7 @@ public struct Attestation: Codable {
 public class DeviceCheckClient {
 
   // The ID of the private-key stored in the secure-enclave chip
-  private static let privateKeyId = "com.brave.device.check.private.key"
+  private static let privateKeyId = "com.ibrowe.device.check.private.key"
 
   // The current build environment
   private let environment: BraveRewards.Configuration.Environment
@@ -290,11 +290,11 @@ extension DeviceCheckClient {
   private var baseURL: URL? {
     switch environment {
     case .development:
-      return URL(string: "https://grant.rewards.brave.software")
+      return URL(string: "https://grant.rewards.ibrowe.software")
     case .staging:
-      return URL(string: "https://grant.rewards.bravesoftware.com")
+      return URL(string: "https://grant.rewards.ibrowesoftware.com")
     case .production:
-      return URL(string: "https://grant.rewards.brave.com")
+      return URL(string: "https://grant.rewards.ibrowe.com")
     }
   }
 
