diff --git a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsRelease (AppStore).entitlements b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsRelease (AppStore).entitlements
index 640e674f36a8dbe22a2c5aa845659c00779bd74a..6610d0883a4b10490e623de078c14d9e99b8099b 100644
--- a/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsRelease (AppStore).entitlements	
+++ b/ios/brave-ios/App/BrowserIntents/Entitlements/BrowserIntentsRelease (AppStore).entitlements	
@@ -4,7 +4,9 @@
 <dict>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.com.brave.ios.browser</string>
+		<string>group.com.ibrowe.ios.BrowserBeta.unique</string>
+		<string>group.com.ibrowe.ios.browser</string>
+		<string>group.com.ibrowe.ios.browser.beta</string>
 	</array>
 </dict>
 </plist>
