diff --git a/script/signing_helper.py b/script/signing_helper.py
index 77f8cbb9ee00d4766243d6660f50f9d891ab932d..d61366322d6714447f7a3b255981282aab943dd0 100755
--- a/script/signing_helper.py
+++ b/script/signing_helper.py
@@ -92,8 +92,8 @@ def BraveModifyPartsForSigning(parts, config):
         # We should instead use upstream's channel_customize mechanism.
         # See https://github.com/brave/brave-browser/issues/39347.
         privileged_helper = parts['privileged-helper']
-        channel_re = 'com.brave.Browser(.*).UpdaterPrivilegedHelper'
-        replacement = 'com.brave.Browser.UpdaterPrivilegedHelper'
+        channel_re = 'com.ibrowe.Browser(.*).UpdaterPrivilegedHelper'
+        replacement = 'com.ibrowe.Browser.UpdaterPrivilegedHelper'
         privileged_helper.path = re.sub(channel_re, replacement,
                                         privileged_helper.path)
         privileged_helper.identifier = re.sub(channel_re, replacement,
