diff --git a/script/signing_helper.py b/script/signing_helper.py
index 77f8cbb9ee00d4766243d6660f50f9d891ab932d..6d676e973016bdd4747cb58e6a429719c5fee87c 100755
--- a/script/signing_helper.py
+++ b/script/signing_helper.py
@@ -85,15 +85,15 @@ def BraveModifyPartsForSigning(parts, config):
                                    | CodeSignOptions.HARDENED_RUNTIME)
 
     if config.enable_updater:
-        # The privileged helper is com.brave.Browser.UpdaterPrivilegedHelper.
+        # The privileged helper is com.ibrowe.Browser.UpdaterPrivilegedHelper.
         # But the value here is
-        # com.brave.Browser.<channel>.UpdaterPrivilegedHelper. This is because
+        # com.ibrowe.Browser.<channel>.UpdaterPrivilegedHelper. This is because
         # our current branding logic treats each channel as a separate product.
         # We should instead use upstream's channel_customize mechanism.
         # See https://github.com/brave/brave-browser/issues/39347.
         privileged_helper = parts['privileged-helper']
-        channel_re = 'com.brave.Browser(.*).UpdaterPrivilegedHelper'
-        replacement = 'com.brave.Browser.UpdaterPrivilegedHelper'
+        channel_re = 'com.ibrowe.Browser(.*).UpdaterPrivilegedHelper'
+        replacement = 'com.ibrowe.Browser.UpdaterPrivilegedHelper'
         privileged_helper.path = re.sub(channel_re, replacement,
                                         privileged_helper.path)
         privileged_helper.identifier = re.sub(channel_re, replacement,
