diff --git a/test/data/speedreader/rewriter/pages/tools/collect/issues_urls.list b/test/data/speedreader/rewriter/pages/tools/collect/issues_urls.list
index fba7b65886c6626d3b30f36e579bb050a6969c8f..7384819abc7826467c56dfe89fa8f9fa2acf2315 100644
--- a/test/data/speedreader/rewriter/pages/tools/collect/issues_urls.list
+++ b/test/data/speedreader/rewriter/pages/tools/collect/issues_urls.list
@@ -1,6 +1,6 @@
 24316 https://blog.evjang.com/2019/02/maml-jax.html
 23771 https://www.sixthtone.com/news/1010653/she-spent-a-decade-writing-fake-russian-history.-wikipedia-just-noticed.-?source=channel_rising
-19593 https://support.brave.com/hc/en-us/articles/360056508071
+19593 https://support.ibrowe.com/hc/en-us/articles/360056508071
 20227 https://ferrous-systems.com/blog/dlx-in-rust/
 18164 https://www.tmz.com/2021/09/15/spacex-inspiration4-first-all-civilian-crew-space-mission-tourists-no-astronauts/
 18231 https://www.eweek.com/news/
