diff --git a/test/filters/unit_tests.filter b/test/filters/unit_tests.filter
index 6e8952e53e73a737296037cda11ccae05f1bb13a..f46067d81ffece66e0333b165e7839477069030b 100644
--- a/test/filters/unit_tests.filter
+++ b/test/filters/unit_tests.filter
@@ -421,7 +421,7 @@
 -All/UnusedSitePermissionsServiceTest.UnusedSitePermissionsServiceTest/3
 -SiteSettingsHandlerTest.NotificationPermissionRevokeUkm
 
-# This test fails because we replace WebUI chrome:// scheme with brave://
+# This test fails because we replace WebUI chrome:// scheme with ibrowe://
 -ChromeURLsTest.ContainsConfigURLs
 
 # These tests fail because we changed idle detection content setting to BLOCK
@@ -498,7 +498,7 @@
 -PKIMetadataComponentInstallerTest.InstallComponentUpdatesPinningConfig
 
 # These tests are disabled because we hide the password card in
-# brave://settings/safetyCheck which also involves always returning a "safe"
+# ibrowe://settings/safetyCheck which also involves always returning a "safe"
 # password card state
 -All/PasswordStatusCheckServiceParameterizedCardTest.PasswordCardState/*
 -PasswordStatusCheckServiceBaseTest.PasswordCardCheckTime
