diff --git a/test/data/speedreader/rewriter/pages/issues_pages/support.brave.com/page.url b/test/data/speedreader/rewriter/pages/issues_pages/support.brave.com/page.url
index c20757abca1da7e21bba351dcbe33377e801eef4..3ee332f95167d9dfe259a9590266bba0171808ac 100644
--- a/test/data/speedreader/rewriter/pages/issues_pages/support.brave.com/page.url
+++ b/test/data/speedreader/rewriter/pages/issues_pages/support.brave.com/page.url
@@ -1 +1 @@
-https://support.brave.com/hc/en-us/articles/360034841711-What-is-a-verified-Brave-Rewards-profile-How-do-I-verify-my-Brave-Rewards-
\ No newline at end of file
+https://support.ibrowe.com/hc/en-us/articles/360034841711-What-is-a-verified-Brave-Rewards-profile-How-do-I-verify-my-Brave-Rewards-
\ No newline at end of file
