diff --git a/third_party/android_deps/libs/com_brave_playlist/README.chromium b/third_party/android_deps/libs/com_brave_playlist/README.chromium
index 66d4e657dac52338c132f8daf315745303a200b7..8ccab6fa486c0874275ed5e2fb3bc7da9624d013 100644
--- a/third_party/android_deps/libs/com_brave_playlist/README.chromium
+++ b/third_party/android_deps/libs/com_brave_playlist/README.chromium
@@ -1,4 +1,4 @@
 Name: Brave Playlist
-URL: https://github.com/brave-experiments/playlist-kotlin
+URL: https://github.com/yuthstyle88/ibrowe-playlist
 License: MPL-2.0
 License File: /brave/common/licenses/MPL-2.0
\ No newline at end of file
